# plasmid tools

**Table of Contents**
- [unclassified](#unclassified)
- [review](#review)
- [updates](#updates)
- [data](#data)
- [tools](#tools)
- [kazumaxneo](#kazumaxneo)

----------
## unclassified
## YYYY-MM-DD

https://twitter.com/search?q=plasmid%20database&src=typed_query&f=live

10:56 PM · Nov 2, 2022
https://twitter.com/pacyc1841/status/1587805748435640320
K TOMINAGA(tomiken) on Twitter: "ウイルス分野ツール多すぎる問題を少しでも解決したくて微力ながら集めてみました https://t.co/vQ9UNP9mNd" / Twitter

https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.html
Antibiotic resistance detection


----------
## review

## tools

https://omictools.com/plasmid-detection-category
Plasmid detection bioinformatics tools | Metagenomic sequencing analysis - omicX

----------

## PTUs
plasmid taxonomic units (PTUs)

### COPLA
https://castillo.dicom.unican.es/copla/

### 2021-07-31
https://pubmed.ncbi.nlm.nih.gov/34332528/
BMC Bioinformatics
. 2021 Jul 31;22(1):390. doi: 10.1186/s12859-021-04299-x.
COPLA, a taxonomic classifier of plasmids
Santiago Redondo-Salvo 1, Roger Bartomeus-Peñalver 1, Luis Vielva 2, Kaitlin A Tagg 3 4, Hattie E Webb 3 4, Raúl Fernández-López 1, Fernando de la Cruz 5
https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-04299-x
https://pmc.ncbi.nlm.nih.gov/articles/PMC8325299/


### 2020-07-17
https://pubmed.ncbi.nlm.nih.gov/32681114/
Nat Commun
. 2020 Jul 17;11(1):3602. doi: 10.1038/s41467-020-17278-2.
Pathways for horizontal gene transfer in bacteria revealed by a global map of their plasmids
Santiago Redondo-Salvo # 1, Raúl Fernández-López # 1, Raúl Ruiz 1, Luis Vielva 2, María de Toro 3, Eduardo P C Rocha 4, M Pilar Garcillán-Barcia 1, Fernando de la Cruz 5
https://www.nature.com/articles/s41467-020-17278-2
https://pmc.ncbi.nlm.nih.gov/articles/PMC7367871/

----------
## PlasAnn

査読なし？

https://github.com/ajlopatkin/PlasAnn
ajlopatkin/PlasAnn: Plasmid annotation pipeline

PlasAnn - Plasmid Annotation Tool
This tool is designed for annotating plasmid sequences from FASTA or GenBank files. It provides comprehensive annotation including coding sequences (CDS), origins of replication (oriC/oriV), origins of transfer (oriT), transposons, replicons, and non-coding RNAs.

If you use PlasAnn in your research, please cite:

PlasAnn: Comprehensive Plasmid Annotation Pipeline
Habibul Islam
University of Rochester

----------
## PlasmidScope
https://plasmid.deepomics.org/
https://pubmed.ncbi.nlm.nih.gov/39441081/
Nucleic Acids Res
. 2025 Jan 6;53(D1):D179-D188. doi: 10.1093/nar/gkae930.
PlasmidScope: a comprehensive plasmid database with rich annotations and online analytical tools
Yinhu Li 1 2 3, Xikang Feng 4 5, Xuhua Chen 2, Shuo Yang 6, Zicheng Zhao 7, Yu Chen 1 2 3, Shuai Cheng Li 6 8
https://academic.oup.com/nar/article/53/D1/D179/7832352?login=false
https://pmc.ncbi.nlm.nih.gov/articles/PMC11701673/

----------
## Dnaapler

https://github.com/gbouras13/dnaapler

3:00 PM · Jan 24, 2024
https://x.com/GB13Faithless/status/1750035858756178201
George Bouras on X: "Happy to share that Dnaapler is published https://t.co/lsKHGp6Msr - if you need to reorient complete, plasmid or phage genomes give it a go. I really liked the @JOSS_TheOJ process - perhaps open reviews are more constructive @susiegriggo_ @BhavyaPapudeshi @viji112 @BeardyMcFace" / X

https://joss.theoj.org/papers/10.21105/joss.05968
Journal of Open Source Software: Dnaapler: A tool to reorient circular microbial genomes
Submitted 09 August 2023 • Published 11 January 2024


----------
## PLSDB
https://ccb-microbe.cs.uni-saarland.de/plsdb/

https://github.com/VGalata/plsdb

https://github.com/VGalata/plsdb/issues/15
README.md for PLSDB v. 2021_06_23 · Issue #15 · VGalata/plsdb
haruosuz commented on Feb 3, 2022
https://github.com/VGalata/plsdb/issues/15#issuecomment-1089840333
pMCBF1 (AY950444) is not included in PLSDB v. 2021_06_23_v2.
@Xethic Xethic closed this as completed on Apr 20

https://github.com/VGalata/plsdb/issues/10
PLSDB (v. 2020_11_19) BioSample Attributes · Issue #10 · VGalata/plsdb
haruosuz commented on Mar 1, 2021 • 
@SmalJonni SmalJonni closed this as completed on Nov 26, 2021

https://github.com/VGalata/plsdb/issues/5
duplicate records · Issue #5 · VGalata/plsdb
haruosuz commented on Nov 3, 2020 • 
@Xethic Xethic closed this as completed on Dec 11, 2020

## 2022-01-07
https://pubmed.ncbi.nlm.nih.gov/34850116/
Nucleic Acids Res
. 2022 Jan 7;50(D1):D273-D278. doi: 10.1093/nar/gkab1111.
PLSDB: advancing a comprehensive database of bacterial plasmids
Georges P Schmartz 1, Anna Hartung 1, Pascal Hirsch 1 2, Fabian Kern 1, Tobias Fehlmann 1, Rolf Müller 2 3, Andreas Keller 1 2
https://academic.oup.com/nar/article/50/D1/D273/6439675
https://pmc.ncbi.nlm.nih.gov/articles/PMC8728149/
A new plasmid collection by Brooks et al., for example, tries to bundle NCBI plasmid information in a collection (19). Another recent database, mMGE (20) has the advantage of unifying phage and plasmid information in a single catalog. However, the database creation workflow is solely focused on the human microbiome. The COMPASS database (21), is of comparable scope to PLSDB and focuses extensively on replicon typing. Due to the scope, size, functionality, and quality of its content, PLSDB is widely used in the scientific community as a central resource for reference data on natural occurring plasmids. By focusing on this domain, the resource finds extensive usage in environmental studies (22). Further, antibiotic resistance analyses with a diverse scope profit frequently from the resistance annotations found in PLSDB (23,24).

## 2019-01-08
https://pubmed.ncbi.nlm.nih.gov/30380090/
Nucleic Acids Res
. 2019 Jan 8;47(D1):D195-D202. doi: 10.1093/nar/gky1050.
PLSDB: a resource of complete bacterial plasmids
Valentina Galata 1, Tobias Fehlmann 1, Christina Backes 1, Andreas Keller 1
https://academic.oup.com/nar/article/47/D1/D195/5149885
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323999/
Data retrieval and processing pipeline
Data collection
Plasmid records were searched in the NCBI nucleotide database by using the query from Orlek et al. (4) and filtering the results to have ‘plasmid’ as location tag, being assigned to a bacterial organism and being from the specified resource (INSDC or RefSeq). 

Overview of collected plasmids
In total, 13 789 plasmid records (2945 from INSDC and 10 844 from RefSeq) were retrieved from the NCBI nucleotide database.

https://integbio.jp/dbcatalog/record/nbdc02515
NCBI nucleotide database から取得したバクテリアプラスミドの配列とメタデータを整理したデータベースです。各レコードには配列長、トポロジー、分類学的情報のような種々のパラメータを付けており、これらでフィルタリングができます。 また、コンティグやロングリードのようなより長い配列についてはMashを使用することによって、遺伝子のようなより短い配列についてはBLASTnを使用することによって配列を検索することが可能です。

https://kazumaxneo.hatenablog.com/entry/2023/03/23/010432
細菌プラスミドのデータベース PLSDB - macでインフォマティクス

----------
## updates

----------
## 2024

## 2024-08
## geNomad

11:01 AM · Nov 1, 2023
https://twitter.com/jgi/status/1719535041955402030
Joint Genome Institute on X: "As mobile genetic elements like viruses spread their DNA, they can affect how microbes adapt to climate change. geNomad is a software that finds plasmids and viruses w/in microbial genomes — learn more: https://t.co/OW56AXL5RI @MicrobiomeData @kyrpides @apcamargo_ @simroux_virus https://t.co/WateZqbfb4" / X

https://twitter.com/NatureBiotech/status/1704927640619000245
https://twitter.com/jgi/status/1704877480728018964
https://twitter.com/jgi/status/1704873642394272096
https://twitter.com/MicrobiomeData/status/1704873416363294919

https://pubmed.ncbi.nlm.nih.gov/37735266/
Nat Biotechnol
. 2024 Aug;42(8):1303-1312. doi: 10.1038/s41587-023-01953-y. Epub 2023 Sep 21.
Identification of mobile genetic elements with geNomad
Antonio Pedro Camargo 1, Simon Roux 2, Frederik Schulz 2, Michal Babinski 3, Yan Xu 3, Bin Hu 3, Patrick S G Chain 3, Stephen Nayfach 2, Nikos C Kyrpides 4
https://www.nature.com/articles/s41587-023-01953-y
https://pmc.ncbi.nlm.nih.gov/articles/PMC11324519/

## 2024-01-05
https://pubmed.ncbi.nlm.nih.gov/37930866/
Nucleic Acids Res
. 2024 Jan 5;52(D1):D164-D173. doi: 10.1093/nar/gkad964.
IMG/PR: a database of plasmids from genomes and metagenomes with rich annotations and metadata
Antonio Pedro Camargo 1, Lee Call 1, Simon Roux 1, Stephen Nayfach 1, Marcel Huntemann 1, Krishnaveni Palaniappan 1, Anna Ratner 1, Ken Chu 1, Supratim Mukherjeep 1, T B K Reddy 1, I-Min A Chen 1, Natalia N Ivanova 1, Emiley A Eloe-Fadrosh 1, Tanja Woyke 1, David A Baltrus 2 3, Salvador Castañeda-Barba 4, Fernando de la Cruz 5, Barbara E Funnell 6, James P J Hall 7, Aindrila Mukhopadhyay 8 9, Eduardo P C Rocha 10, Thibault Stalder 4, Eva Top 4, Nikos C Kyrpides 1
https://academic.oup.com/nar/article/52/D1/D164/7335748
https://pmc.ncbi.nlm.nih.gov/articles/PMC10767988/
To tackle these issues, plasmid-specific databases such as pATLAS (11) PLSDB (12,13) and COMPASS (14) provide data such as replicon and MOB typing, antibiotic resistance genes and virulence genes. 

To address this issue, we have developed IMG/PR, a comprehensive database of plasmid sequences derived from the Integrated Microbial Genomes & Microbiomes (IMG/M) system (27). IMG/PR amasses the most extensive collection of publicly available plasmids, systematically identified across several thousand genomes, metagenomes and metatranscriptomes using the geNomad tool (28). 


https://genome.jgi.doe.gov/portal/IMG_PR/IMG_PR.home.html

The sequences and metadata associated with IMG/PR plasmids are organized in the following files available for download:

IMGPR_plasmid_data.tsv: Metadata related to plasmids in IMG/PR.
IMGPR_nucl.fna.gz: Nucleotide sequences of plasmids in IMG/PR.
IMGPR_prot.faa.gz: Sequences of proteins encoded by plasmids in IMG/PR.


----------
## 2023

## SHIP
https://github.com/AbeelLab/SHIP
Synteny-aware HGT Identification in Plasmids

## 2023-10-03
https://pubmed.ncbi.nlm.nih.gov/37796811/
Bioinformatics
. 2023 Oct 3;39(10):btad612. doi: 10.1093/bioinformatics/btad612.
SHIP: identifying antimicrobial resistance gene transfer between plasmids
Marco Teixeira 1 2 3, Stephanie Pillay 3, Aysun Urhan 3 4, Thomas Abeel 3 4
https://academic.oup.com/bioinformatics/article/39/10/btad612/7291867
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10598575/

## HOTSPOT
https://github.com/Orin-beep/HOTSPOT

## 2023-05-04
https://pubmed.ncbi.nlm.nih.gov/37086432/
Bioinformatics
. 2023 May 4;39(5):btad283. doi: 10.1093/bioinformatics/btad283.
HOTSPOT: hierarchical host prediction for assembled plasmid contigs with transformer
Yongxin Ji 1, Jiayu Shang 1, Xubo Tang 1, Yanni Sun 1
https://academic.oup.com/bioinformatics/article/39/5/btad283/7136643
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10159655/

----------
## 2022



----------

## 2022-09-22
https://pubmed.ncbi.nlm.nih.gov/36036594/
Appl Environ Microbiol
. 2022 Sep 22;88(18):e0099122. doi: 10.1128/aem.00991-22. Epub 2022 Aug 29.
mobileOG-db: a Manually Curated Database of Protein Families Mediating the Life Cycle of Bacterial Mobile Genetic Elements
Connor L Brown 1, James Mullet 2, Fadi Hindi 2, James E Stoll 3, Suraj Gupta 1, Minyoung Choi 2, Ishi Keenum 2, Peter Vikesland 2, Amy Pruden 2, Liqing Zhang 4
https://journals.asm.org/doi/full/10.1128/aem.00991-22
https://pmc.ncbi.nlm.nih.gov/articles/PMC9499024/
Furthermore, there is no centralized resource for MGE hallmark genes that could serve as the basis for annotating diverse classes of MGEs. Instead, decentralized databases exist for phages, including pVOG (17) and the GutPhage Database (GPD) (18); insertion sequences, ISfinder (19); integrative genomic elements (IGEs), ICEberg (11), immedb (20); or plasmids, COMPASS (21), NCBI Plasmid RefSeq (22). While ACLAME (23) combines multiple element types, there has not been a substantial update to this database since 2010 (23, 24).


## PlasmidHostFinder
https://cge.cbs.dtu.dk/services/PlasmidHostFinder/

## 2022-10-26
https://pubmed.ncbi.nlm.nih.gov/36069451/
mSystems
. 2022 Oct 26;7(5):e0019122. doi: 10.1128/msystems.00191-22. Epub 2022 Sep 7.
Global Distribution and Diversity of Prevalent Sewage Water Plasmidomes
Frederik Teudt 1, Saria Otani 1, Frank M Aarestrup 1
https://journals.asm.org/doi/10.1128/msystems.00191-22
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9600348/
Plasmid host prediction. The homology reduced circular elements were analyzed with a random forest model plasmid host predictor created by Aytan-Aktug et al. (51). The genus predicted with the highest probability was chosen as the host taxon.

## 2022-04-26
https://pubmed.ncbi.nlm.nih.gov/35382558/
mSystems
. 2022 Apr 26;7(2):e0118021. doi: 10.1128/msystems.01180-21. Epub 2022 Apr 6.
PlasmidHostFinder: Prediction of Plasmid Hosts Using Random Forest
Derya Aytan-Aktug 1, Philip T L C Clausen 1, Judit Szarvas 1, Patrick Munk 1, Saria Otani 1, Marcus Nguyen 2 3, James J Davis 2 3 4, Ole Lund 1, Frank M Aarestrup 1
https://journals.asm.org/doi/10.1128/msystems.01180-21
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9040769/


----------
## Deeplasmid

https://github.com/wandreopoulos/deeplasmid
August 22, 2024

2022-02-22
https://pubmed.ncbi.nlm.nih.gov/34871418/
Nucleic Acids Res
. 2022 Feb 22;50(3):e17. doi: 10.1093/nar/gkab1115.
Deeplasmid: deep learning accurately separates plasmids from bacterial chromosomes
William B Andreopoulos 1 2, Alexander M Geller 3, Miriam Lucke 3, Jan Balewski 4, Alicia Clum 1, Natalia N Ivanova 1, Asaf Levy 3
https://academic.oup.com/nar/article/50/3/e17/6454267?login=false
https://pmc.ncbi.nlm.nih.gov/articles/PMC8860608/

Existing tools have limitations due to their reliance on the circularity of the topology, bias towards certain taxonomies used in training (e.g. PlasmidFinder, PLACNET, Kraken) and coverage of a de Bruijn assembly graph constructed from k-mers found in reads (e.g. Recycler and plasmidSPAdes). The software packages cBAR (16), PlasFlow (17), mlplasmids (22), Platon (23) and PlasmidVerify (24) satisfy the above criteria, since they utilize only two types of data: assembled sequences themselves (PlasFlow) and features extracted from assembled sequences (cBar, mlplasmids, Platon, PlasmidVerify). 



----------
## 2021

----------

## bakta
## 2021-11
https://github.com/oschwengers/bakta

https://pubmed.ncbi.nlm.nih.gov/34739369/
Microb Genom
. 2021 Nov;7(11):000685. doi: 10.1099/mgen.0.000685.
Bakta: rapid and standardized annotation of bacterial genomes via alignment-free sequence identification
Oliver Schwengers 1, Lukas Jelonek 1, Marius Alfred Dieckmann 1, Sebastian Beyvers 1, Jochen Blom 1, Alexander Goesmann 1
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000685#tab2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8743544/

https://kazumaxneo.hatenablog.com/entry/2021/04/13/193935
バクテリアのゲノムやプラスミドのアノテーションを行う Bakta - macでインフォマティクス

----------
### pLannotate
### 2021-07-02
https://github.com/mmcguffi/pLannotate

https://github.com/mmcguffi/pLannotate/releases/tag/v1.2.4

https://pubmed.ncbi.nlm.nih.gov/34019636/
Nucleic Acids Res
. 2021 Jul 2;49(W1):W516-W522. doi: 10.1093/nar/gkab374.
pLannotate: engineered plasmid annotation
Matthew J McGuffie 1, Jeffrey E Barrick 1
https://academic.oup.com/nar/article/49/W1/W516/6279845
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC8262757/

----------

## SCAPP
## 2021-06-25
SCAPP (Sequence Contents Aware Plasmid Peeler) は、メタゲノム・アセンブリ・グラフからプラスミド配列を構築する。他のツール（RecyclerやmetaplasmidSPAdes）と同等以上のパフォーマンス。

https://github.com/Shamir-Lab/SCAPP

https://pubmed.ncbi.nlm.nih.gov/34172093/
Microbiome
. 2021 Jun 25;9(1):144. doi: 10.1186/s40168-021-01068-z.
SCAPP: an algorithm for improved plasmid assembly in metagenomes
David Pellow 1, Alvah Zorea 2, Maraike Probst 3, Ori Furman 2, Arik Segal 4 5, Itzhak Mizrahi 2, Ron Shamir 6
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01068-z
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8228940/

Posted August 12, 2020.
https://www.biorxiv.org/content/10.1101/2020.01.12.903252v3

https://kazumaxneo.hatenablog.com/entry/2020/02/24/073000
（メタゲノムの）アセンブリグラフから環状plasmid配列を出力する SCAPP - macでインフォマティクス

----------
## 2020
----------

----------
## 2020-11

情報理論の情報エントロピーを用いて、4,424個のプラスミドゲノムにおける同義コドン使用のランダム性を評価

https://pubmed.ncbi.nlm.nih.gov/33074084/
Microb Genom
. 2020 Nov;6(11):mgen000459. doi: 10.1099/mgen.0.000459. Epub 2020 Oct 19.
Identification of the conjugative and mobilizable plasmid fragments in the plasmidome using sequence signatures
Zhencheng Fang 1 2, Hongwei Zhou 1 3
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000459#tab2
https://pmc.ncbi.nlm.nih.gov/articles/PMC7725325/

Additionally, many computational tools for sequence classification and assembly that identify plasmid sequences from chromosome-derived sequences in metagenomic data have been developed, such as cBar [5], PlasFlow [6], PPR-Meta [7], PlasClass [8], Recycler [9] and metaplasmidSPAdes [10].

Randomness of synonymous codon usage
Because of genome amelioration, plasmids adjust their sequence signatures based on their host [15]. Codon usage is one of the most important features of sequence signatures. Since transmissible plasmids may exist in a wide range of hosts, their codon usage may exhibit mixed patterns in different hosts. Therefore, we surmise that the usage of synonymous codons of each amino acid on transmissible plasmids is more random than on non-transmissible plasmids.

It has been shown that bacteria of different species contain different sequence signatures [19], and during co-evolution, plasmids adjust their sequence signatures based on their hosts [15]. The similarity of sequence signatures between foreign DNA, such as plasmids and phages, and bacterial chromosomal DNA has also been widely used to predict the hosts of foreign DNA [15, 20, 21]. 

15.Suzuki H, Yano H, Brown CJ, Top EM. Predicting plasmid promiscuity based on genomic signature. J Bacteriol. 2010;192:6045–6055. doi: 10.1128/JB.00277-10. [DOI] [PMC free article] [PubMed] [Google Scholar]

21.Galiez C, Siebert M, Enault F, Vincent J, Söding J, et al. Wish: who is the host? predicting prokaryotic hosts from metagenomic phage contigs. Bioinformatics. 2017;33:3113–3114. doi: 10.1093/bioinformatics/btx383. [DOI] [PMC free article] [PubMed] [Google Scholar]


----------
### PlasmidFinder
https://cge.cbs.dtu.dk/services/PlasmidFinder/
PlasmidFinder 2.1
Software version: 2.0.1 (2020-02-07)
Database version: (2020-04-02)

https://bitbucket.org/genomicepidemiology/plasmidfinder_db/src/master/
クローンの作成

Oct 10, 2018
https://github.com/tseemann/abricate/issues/66
PlasmidFinder data should be downloaded from BitBucket · Issue #66 · tseemann/abricate

## 2020
https://pubmed.ncbi.nlm.nih.gov/31584170/
Methods Mol Biol
. 2020;2075:285-294. doi: 10.1007/978-1-4939-9877-7_20.
PlasmidFinder and In Silico pMLST: Identification and Typing of Plasmid Replicons in Whole-Genome Sequencing (WGS)
Alessandra Carattoli 1, Henrik Hasman 2
https://link.springer.com/protocol/10.1007/978-1-4939-9877-7_20

## 2014-07
https://pubmed.ncbi.nlm.nih.gov/24777092/
Antimicrob Agents Chemother
. 2014 Jul;58(7):3895-903. doi: 10.1128/AAC.02412-14. Epub 2014 Apr 28.
In silico detection and typing of plasmids using PlasmidFinder and plasmid multilocus sequence typing
Alessandra Carattoli 1, Ea Zankari 2, Aurora García-Fernández 1, Mette Voldby Larsen 3, Ole Lund 3, Laura Villa 1, Frank Møller Aarestrup 2, Henrik Hasman 4
https://aac.asm.org/content/58/7/3895.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4068535/

----------

## PlasmIdent
https://github.com/imgag/plasmIDent
## 2020-08-19
https://pubmed.ncbi.nlm.nih.gov/32817379/
mSphere
. 2020 Aug 19;5(4):e00525-20. doi: 10.1128/mSphere.00525-20.
Tracking of Antibiotic Resistance Transfer and Rapid Plasmid Evolution in a Hospital Setting by Nanopore Sequencing
Silke Peter 1 2, Mattia Bosio # 3 4, Caspar Gross # 5, Daniela Bezdan 3, Javier Gutierrez 6, Philipp Oberhettinger 7 2, Jan Liese 7 2, Wichard Vogel 8, Daniela Dörfel 8 9, Lennard Berger 5, Matthias Marschal 7 2, Matthias Willmann 7 2, Ivo Gut 6 10, Marta Gut 6 10, Ingo Autenrieth 7 2, Stephan Ossowski 11 5 10
https://journals.asm.org/doi/full/10.1128/mSphere.00525-20
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7440845/
We developed the computational platform plasmIDent for Nanopore-based characterization of clinical isolates and monitoring of ARG transfer, comprising de novo assembly of genomes and plasmids, plasmid circularization, ARG annotation, comparative genome analysis of multiple isolates, and visualization of results. Using plasmIDent, we identified a 40-kb plasmid carrying bla IMP-8 in P. aeruginosa and C. freundii, verifying the plasmid transfer. 

https://kazumaxneo.hatenablog.com/entry/2020/08/24/073000
ロングリードシークエンシングから環状プラスミドを同定しARGsなどのアノテーションをつける PlasmIdent - macでインフォマティクス

Aug 20, 2020
https://twitter.com/mSphereJ/status/1296110566424141825
mSphere EIC on Twitter: "Tracking of Antibiotic Resistance Transfer and Rapid Plasmid Evolution in a Hospital Setting by Nanopore Sequencing https://t.co/VLrQf6hJsc @ASMicrobiology #mSphere" / Twitter
https://msphere.asm.org/content/5/4/e00525-20
Tracking of Antibiotic Resistance Transfer and Rapid Plasmid Evolution in a Hospital Setting by Nanopore Sequencing | mSphere 
- We developed the computational platform plasmIDent for Nanopore-based characterization of clinical isolates and monitoring of ARG transfer, comprising de novo assembly of genomes and plasmids, plasmid circularization, ARG annotation, comparative genome analysis of multiple isolates, and visualization of results."
- pathoLogic, plasmIDent and a graphical user interface (GUI) are freely available on github (plasmIDent pipeline, https://github.com/imgag/plasmIDent; pathoLogic pipeline, https://github.com/imgag/pathoLogic).

----------

## MOB-suite
https://github.com/phac-nml/mob-suite

11:40 PM · Sep 25, 2020
https://twitter.com/JohnHENash/status/1309503134423748608
John Nash on Twitter: "Our new paper on plasmid typing is out. I have the most amazing co-authors... https://t.co/49iOzFSWvz" / Twitter

## 2020-10
https://pubmed.ncbi.nlm.nih.gov/32969786/
Microb Genom
. 2020 Oct;6(10):mgen000435. doi: 10.1099/mgen.0.000435.
Universal whole-sequence-based plasmid typing and its utility to prediction of host range and epidemiological surveillance
James Robertson 1, Kyrylo Bessonov 1, Justin Schonfeld 1, John H E Nash 1
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000435#tab2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7660255/

## 2020-08
https://pubmed.ncbi.nlm.nih.gov/30052170/
Microb Genom
. 2018 Aug;4(8):e000206. doi: 10.1099/mgen.0.000206. Epub 2018 Jul 27.
MOB-suite: software tools for clustering, reconstruction and typing of plasmids from draft assemblies
James Robertson 1, John H E Nash 2
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000206#tab2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6159552/
- Introduction
  - There is a strong need for automatic tools for the detection of plasmids from WGS data and numerous tools have been developed using a diverse array of approaches [1, 4, 16–18]. 
  - Additionally, the suite provides a scalable nomenclature for identifying plasmids over short evolutionary periods by estimating genomic distances based on mash min-hashing [22]. 
- MOB-cluster method
MOB-cluster uses mash dist [22] with default parameters to calculate all pairwise genomic distances for each plasmid contained in the closed plasmid reference database. 
We selected a permissive mash dist clustering threshold of 0.05, which resulted in clusters consisting largely of single replicon and MOB types. The distance metric used by mash dist allows for plasmids with considerable differences in size to be grouped together.

PlasFlowの論文（17.）は、引用されているものの、議論されていない。

----------
## ZhuLab
Zhencheng Fang

## PlasGUN
## 2020-05-01
http://cqb.pku.edu.cn/ZhuLab/PlasGUN/
HTTP 404 error

https://github.com/zhenchengfang/PlasGUN/

https://pubmed.ncbi.nlm.nih.gov/32091572/
Bioinformatics
. 2020 May 1;36(10):3239-3241. doi: 10.1093/bioinformatics/btaa103.
PlasGUN: gene prediction in plasmid metagenomic short reads using deep learning
Zhencheng Fang 1, Jie Tan 1, Shufang Wu 1, Mo Li 1, Chunhui Wang 1, Yongchu Liu 1, Huaiqiu Zhu 1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7214025/

## PPR-Meta
## 2019-06-01
http://cqb.pku.edu.cn/ZhuLab/PPR_Meta
HTTP 404 error

https://github.com/zhenchengfang/PPR-Meta

https://pubmed.ncbi.nlm.nih.gov/31220250/
Gigascience
. 2019 Jun 1;8(6):giz066. doi: 10.1093/gigascience/giz066.
PPR-Meta: a tool for identifying phages and plasmids from metagenomic fragments using deep learning
Zhencheng Fang 1 2, Jie Tan 1 2, Shufang Wu 1 2, Mo Li 1 2 3, Congmin Xu 1 2 4, Zhongjie Xie 1 2, Huaiqiu Zhu 1 2
https://academic.oup.com/gigascience/article/8/6/giz066/5521157?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6586199/
- Mathematical model of DNA sequences
Here, we use a more detailed approach to represent the DNA fragments. Specifically, each sequence is represented by “base one-hot matrix (BOH)” and “codon one-hot matrix (COH).”


----------
## COMPASS
## 2020-03-24
https://github.com/itsmeludo/COMPASS

https://colil.dbcls.jp/browse/papers/32265894/1/100/year
Analysis of COMPASS, a New Comprehensive Plasmid Database Revealed Prevalence of Multireplicon and Extensive Diversity of IncF Plasmids.
Douarre PE, Mallet L, Radomski N, Felten A, Mistou MY Front Microbiol. 2020;11:483. PMID:32265894
    is cited by the following papers.

← 前へ  合計: 19, 

https://pubmed.ncbi.nlm.nih.gov/32265894/
Front Microbiol
. 2020 Mar 24;11:483. doi: 10.3389/fmicb.2020.00483. eCollection 2020.
Analysis of COMPASS, a New Comprehensive Plasmid Database Revealed Prevalence of Multireplicon and Extensive Diversity of IncF Plasmids
Pierre-Emmanuel Douarre 1, Ludovic Mallet 1, Nicolas Radomski 1, Arnaud Felten 1, Michel-Yves Mistou 2
https://www.frontiersin.org/articles/10.3389/fmicb.2020.00483/full
https://pmc.ncbi.nlm.nih.gov/articles/PMC7105883/

https://www.ncbi.nlm.nih.gov/pubmed/31584171
Methods Mol Biol. 2020;2075:295-308. doi: 10.1007/978-1-4939-9877-7_21.
MOBscan: Automated Annotation of MOB Relaxases.
Garcillán-Barcia MP1, Redondo-Salvo S2, Vielva L3, de la Cruz F4.


### gplas
https://gitlab.com/sirarredondo/gplas

https://pubmed.ncbi.nlm.nih.gov/32271863/
Bioinformatics
. 2020 Jun 1;36(12):3874-3876. doi: 10.1093/bioinformatics/btaa233.
gplas: a comprehensive tool for plasmid analysis using short-read graphs
Sergio Arredondo-Alonso 1, Martin Bootsma 2 3, Yaïr Hein 3, Malbert R C Rogers 1, Jukka Corander 4 5 6, Rob J L Willems 1, Anita C Schürch 1



## 2019

## RefSoil
## 2019-02-26
RefSoil+：土壌プラスミドの遺伝子と形質のリファレンスデータベース

https://github.com/ShadeLab/RefSoil_plasmids

https://pubmed.ncbi.nlm.nih.gov/30834332/
mSystems
. 2019 Feb 26;4(1):e00349-18. doi: 10.1128/mSystems.00349-18. eCollection 2019 Jan-Feb.
RefSoil+: a Reference Database for Genes and Traits of Soil Plasmids
Taylor K Dunivin 1 2, Jinlyung Choi 3, Adina Howe 3, Ashley Shade 1 4 5 6
https://journals.asm.org/doi/full/10.1128/mSystems.00349-18
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6392096/
RefSoil+ is publicly available on GitHub (https://github.com/ShadeLab/RefSoil_plasmids). 


## WASPS
## 2020-03-01
https://archaea.i2bc.paris-saclay.fr/wasps/

https://pubmed.ncbi.nlm.nih.gov/31589313/
Bioinformatics
. 2020 Mar 1;36(5):1629-1631. doi: 10.1093/bioinformatics/btz745.
WASPS: web-assisted symbolic plasmid synteny server
Catherine Badel 1, Violette Da Cunha 1, Ryan Catchpole 1, Patrick Forterre 1 2, Jacques Oberto 1
https://academic.oup.com/bioinformatics/article/36/5/1629/5582648
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7703779/

Recently, plasmid assets have been developed proposing either a comprehensive manually curated bacterial plasmid list (Brooks et al., 2019) or a bacterial plasmid database which can be interrogated using sequence similarity programs (Galata et al., 2019). 

2:32 AM · Oct 15, 2019
https://twitter.com/Catherine_Badel/status/1183797777110843393
Catherine Badel on Twitter: "WASPS, a new plasmid web service for similarity analyses and synteny map construction. Also includes a comprehensive and updated natural plasmid database. https://t.co/ee2kqXEAoz" / Twitter

### HyAsP
https://github.com/cchauve/HyAsP

https://www.ncbi.nlm.nih.gov/pubmed/31116364
Bioinformatics. 2019 May 22. pii: btz413. doi: 10.1093/bioinformatics/btz413. [Epub ahead of print]
HyAsP, a greedy tool for plasmids identification.
Müller R1, Chauve C2.
https://academic.oup.com/bioinformatics/article-abstract/35/21/4436/5497247?redirectedFrom=fulltext
You do not currently have access to this article.

https://kazumaxneo.hatenablog.com/entry/2019/07/14/073000
アセンブリグラフからプラスミドを検出する HyAsP - macでインフォマティクス


## DoriC
## 2019-01-08
https://tubic.org/doric/
https://pubmed.ncbi.nlm.nih.gov/30364951/
Nucleic Acids Res
. 2019 Jan 8;47(D1):D74-D77. doi: 10.1093/nar/gky1014.
DoriC 10.0: an updated database of replication origins in prokaryotic genomes including chromosomes and plasmids
Hao Luo 1 2 3, Feng Gao 1 2 3
https://academic.oup.com/nar/article/47/D1/D74/5144951
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323995/
In a large number of plasmids, the origin of vegetative replication (oriV) often consists of direct repeats or iteron DNA sequences, which interact with Rep proteins to form the initial complex during the process of replication initiation (8). There is also an AT-rich region near the location of iterons in oriV, which serves as the DNA unwinding element (9).

Replication origins on plasmids

https://kazumaxneo.hatenablog.com/entry/2019/06/21/073000
バクテリア、アーキア、プラスミドの複製起点（ori）データベース DoriC - macでインフォマティクス
多数のプラスミドにおいて、origin of vegetative replication (oriV) はしばしばダイレクトリピートまたはiteron DNA配列（wiki）からなり、これらはRepタンパク質と相互作用して複製開始の過程で初期複合体を形成する（ref.8）。oriVのiteronの位置の近くにATに富む領域もあり、これはDNA巻き戻し要素として働く（ref.9）。

### tiptoft

https://twitter.com/andrewjpage/status/1101497825324609537
Andrew Page on Twitter: "Small plasmids can disappear at the read correction step of long read assembly.Tiptoft published today in #JOSS by @andrewjpage & @torstenseemann takes uncorrected long reads & identifies what plasmids are actually present. Install: pip3 install tiptoft https://t.co/xru3UyeIL0"
10:02 AM - 1 Mar 2019
http://joss.theoj.org/papers/10.21105/joss.01021
The Journal of Open Source Software: TipToft: detecting plasmids contained in uncorrected long read sequencing data

https://kazumaxneo.hatenablog.com/entry/2019/03/05/073000
ナノポアのrawロングリードからプラスミドを探索する tiptoft - macでインフォマティクス

### meta_arsenic
ヒ素耐性遺伝子を検出するためのツールキット

https://github.com/ShadeLab/PAPER_Dunivin_meta_arsenic
ShadeLab/PAPER_Dunivin_meta_arsenic: Toolkit for detecting arsenic resistance genes in nucleotide and amino acid sequences. A meta-analysis of arsenic resistance genes in soil genomes (RefSoil+) and metagenomes

To cite this work or code
Dunivin, T.K., Yeh, S.Y. & Shade, A. A global survey of arsenic-related genes in soil microbiomes. BMC Biol 17, 45 (2019). https://doi.org/10.1186/s12915-019-0661-5


https://www.biorxiv.org/content/10.1101/445502v1
Targeting microbial arsenic resistance genes: a new bioinformatic toolkit informs arsenic ecology and evolution in soil genomes and metagenomes | bioRxiv
https://www.biorxiv.org/content/biorxiv/early/2018/10/17/445502.full.pdf

 All arsenic resistance
gene databases are freely available on FunGene (http://fungene.cme.msu.edu/). 

 Reference and BLAST databases are publicly
available on GitHub (https://github.com/ShadeLab/meta_arsenic). 

## 2018

https://www.ncbi.nlm.nih.gov/pubmed/30619542
Comput Struct Biotechnol J. 2018 Dec 15;17:70-81. doi: 10.1016/j.csbj.2018.12.001. eCollection 2019.
Reconsidering plasmid maintenance factors for computational plasmid design.
Yano H1, Shintani M2,3, Tomita M4,5, Suzuki H4,5, Oshima T6.
https://doi.org/10.1016/j.csbj.2018.12.001
https://www.sciencedirect.com/science/article/pii/S2001037018301685

3. Publicly Available Resources
3.1. Plasmid Sequence Data
3.2. Bioinformatics Tools


----------
## mlplasmids

2018-11
https://pubmed.ncbi.nlm.nih.gov/30383524/
Microb Genom
. 2018 Nov;4(11):e000224. doi: 10.1099/mgen.0.000224. Epub 2018 Nov 1.
mlplasmids: a user-friendly tool to predict plasmid- and chromosome-derived sequences for single species
Sergio Arredondo-Alonso 1, Malbert R C Rogers 1, Johanna C Braat 1, Tess D Verschuuren 2, Janetta Top 1, Jukka Corander 3 4 5, Rob J L Willems 1, Anita C Schürch 1
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000224#tab2
https://pmc.ncbi.nlm.nih.gov/articles/PMC6321875/

----------
## 2017

----------
## 2017-12-20

2017年12月20日
https://www.yodosha.co.jp/yodobook/book/9784758125031/
実験医学：どこでも　誰でも　より長く　ナノポアシークエンサーが研究の常識を変える！ - 羊土社
■サマリーを表示する
ナノポアシークエンサーによる薬剤耐性菌ゲノムの解析【鈴木仁人，矢原耕史，平林亜希，柴山恵吾】
近年，ナノポアシークエンサーによる1 分子長鎖（ロングリード）型シークエンシング技術の革新により，超長鎖DNA配列（～1 Mbp）の解読，RNA配列の直接解読，メチル化などの修飾塩基の検出などが可能になってきた．これらの技術は，細菌の染色体やプラスミドを含む完全配列の解析に加え，

https://www.ncbi.nlm.nih.gov/pubmed/29220507
Brief Bioinform. 2017 Dec 5. doi: 10.1093/bib/bbx162. [Epub ahead of print]
Assessing the heterogeneity of in silico plasmid predictions based on whole-genome-sequenced clinical isolates.
Laczny CC, Galata V, Plum A, Posch AE, Keller A.
- tools (cBar, PlasmidFinder, plasmidSPAdes and Recycler)

https://gitlab.com/sirarredondo/Plasmid_Assembly/tree/master
Sergio_Arredondo / Plasmid_Assembly · GitLab

https://www.ncbi.nlm.nih.gov/pubmed/29177087
Microb Genom. 2017 Aug 18;3(10):e000128. doi: 10.1099/mgen.0.000128. eCollection 2017 Oct.
On the (im)possibility of reconstructing plasmids from whole-genome short-read sequencing data.
Arredondo-Alonso S1, Willems RJ1, van Schaik W1,2, Schürch AC1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5695206/
- four programs (PlasmidSPAdes, Recycler, cBar and PlasmidFinder) 
- Table 1.
Overview of the programs to predict plasmids from short-read sequencing data

----------
## 2017-10_WIsH

https://colil.dbcls.jp/browse/papers/28957499/1/100/year
WIsH: who is the host? Predicting prokaryotic hosts from metagenomic phage contigs.
Galiez C, Siebert M, Enault F, Vincent J, Soding J Bioinformatics. 2017 Oct 1;33(19):3113-3114. PMID:28957499
    is cited by the following papers.

← 前へ  合計: 112, 

https://pubmed.ncbi.nlm.nih.gov/28957499/
Bioinformatics
. 2017 Oct 1;33(19):3113-3114. doi: 10.1093/bioinformatics/btx383.
WIsH: who is the host? Predicting prokaryotic hosts from metagenomic phage contigs
Clovis Galiez 1, Matthias Siebert 1, François Enault 2, Jonathan Vincent 2, Johannes Söding 1
https://academic.oup.com/bioinformatics/article/33/19/3113/3964377?login=false
https://pmc.ncbi.nlm.nih.gov/articles/PMC5870724/
2 Materials and methods
The estimated k-mer frequencies classically used for host prediction using genomic composition become very noisy for short phage contigs. We therefore adopted a suited probabilistic approach. First, we train a homogeneous Markov model of order 8 (Supplementary Fig. S2) for each potential host genome (

----------
## Orlek

## 2017-05
https://pubmed.ncbi.nlm.nih.gov/28286183/
Plasmid
. 2017 May;91:42-52. doi: 10.1016/j.plasmid.2017.03.002. Epub 2017 Mar 9.
Ordering the mob: Insights into replicon and MOB typing schemes from analysis of a curated dataset of publicly available plasmids
Alex Orlek 1, Hang Phan 2, Anna E Sheppard 2, Michel Doumith 3, Matthew Ellington 4, Tim Peto 2, Derrick Crook 2, A Sarah Walker 2, Neil Woodford 4, Muna F Anjum 5, Nicole Stoesser 6
https://www.sciencedirect.com/science/article/pii/S0147619X16301032?via%3Dihub
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5466382/

Biopython scripts (Cock et al., 2009) were used to retrieve information and filter accessions, including filtering-out non-coding sequences, and eliminating incomplete plasmid sequences using a regular expression search of accession title descriptions (Supplementary methods S1). 

1-s2.0-S0147619X16301032-mmc2.docx

## 2017-04-23
https://pubmed.ncbi.nlm.nih.gov/28516137/
Data Brief
. 2017 Apr 23;12:423-426. doi: 10.1016/j.dib.2017.04.024. eCollection 2017 Jun.
A curated dataset of complete Enterobacteriaceae plasmids compiled from the NCBI nucleotide database
Alex Orlek 1 2, Hang Phan 1 2, Anna E Sheppard 1 2, Michel Doumith 3, Matthew Ellington 2 3, Tim Peto 1 2, Derrick Crook 1 2, A Sarah Walker 1 2, Neil Woodford 2 3, Muna F Anjum 2 4, Nicole Stoesser 1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5426034/
Data accessibility	Data is publicly available in the Figshare repository.
https://figshare.com/s/18de8bdcbba47dbaba41

A bioinformatic protocol and code for MOB typing using the protein dataset are provided on GitHub (https://github.com/AlexOrlek/MOBtyping).






## 2017-03-28
Posted March 28, 2017.
doi: https://doi.org/10.1101/121350
Plasmid Profiler: Comparative Analysis of Plasmid Content in WGS Data
Adrian Zetner, Jennifer Cabral, Laura Mataseje, Natalie C Knox, Philip Mabon, Michael Mulvey, Gary Van Domselaar
doi: https://doi.org/10.1101/121350

## 2016

https://www.cell.com/molecular-therapy-family/molecular-therapy/fulltext/S1525-0016(16)32965-3
156. A Plasmid Manager, a Comprehensive Database for Archiving Plasmid DNA Sequence: Molecular Therapy
OLIGONUCLEOTIDE THERAPEUTICS I| VOLUME 24, SUPPLEMENT 1, S61, MAY 01, 2016

## 2014

Apr 22, 2014
https://github.com/mscook/VFDB
Sequences of 19 plasmid replicon types.
https://github.com/mscook/VFDB/blob/master/plasmid_replicons.fa

----------

## data

https://twitter.com/BioMickWatson/status/1055412464039460865
Mick Watson on Twitter: "Best database of microbial plasmid sequences?"
6:55 AM - 25 Oct 2018
not sure if "best" but: pATLAS (http://www.patlas.site/ ) and our, recently created, database PLSDB (https://ccb-microbe.cs.uni-saarland.de/plsdb/ )

### Sistrom
Sistrom, Mark (2018), Plasmids database, Dryad, Dataset, https://doi.org/10.15146/R33X2J

https://twitter.com/Tyu_Shi/status/1083013705514680320
Tyu_Shi on Twitter: "論文読んだ限りこのmethodでcuratedでcomprehensiveなplasmid sequence databaseと言う気にはならないな。"A Curated, Comprehensive Database of Plasmid Sequences" https://t.co/hndJfjzrYd 出張先なので800MB超のファイルダウンロードするのは避けたいが、 帰ったら一応ダウンロードしてみよう。"
9:52 AM - 9 Jan 2019

## 2019-01-05
https://pubmed.ncbi.nlm.nih.gov/30625112/
Microb Genom
. 2019 Jan;5(1):e000244. doi: 10.1099/mgen.0.000244.
Where the plasmids roam: large-scale sequence analysis reveals plasmids with large host ranges
Lauren Elisabeth Brooks 1 2, Mo Kaze 2, Mark Sistrom 2
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000244#tab2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6412061/

## 2019-01-03
https://pubmed.ncbi.nlm.nih.gov/30637385/
Microbiol Resour Announc
. 2019 Jan 3;8(1):e01325-18. doi: 10.1128/MRA.01325-18. eCollection 2019 Jan.
A Curated, Comprehensive Database of Plasmid Sequences
Lauren Brooks 1, Mo Kaze 2, Mark Sistrom 2
https://journals.asm.org/doi/full/10.1128/MRA.01325-18
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6318356/
ftp://ftp.ncbi.nlm.nih.gov/refseq/release/plasmid/, resulting in 11,677 plasmid sequences. 
This resulted in 7,426 complete and assembled plasmid sequences following this initial screening.
(https://www.ncbi.nlm.nih.gov/genome/browse#!/prokaryotes/).
This allowed us to include an additional 3,466 complete, annotated plasmid sequences, resulting in our database of 10,892 complete and annotated plasmid sequences for subsequent analyses.

Data availability. The plasmid database is available in fasta format and associated metadata are available in csv format at https://doi.org/10.15146/R33X2J.


### pATLAS
http://www.patlas.site

## 2019-01-08
https://pubmed.ncbi.nlm.nih.gov/30395323/
Nucleic Acids Res
. 2019 Jan 8;47(D1):D188-D194. doi: 10.1093/nar/gky1073.
Plasmid ATLAS: plasmid visual analytics and identification in high-throughput sequencing data
Tiago F Jesus 1, Bruno Ribeiro-Gonçalves 1, Diogo N Silva 1, Valeria Bortolaia 2, Mário Ramirez 1, João A Carriço 1
https://academic.oup.com/nar/article/47/D1/D188/5160973?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323984/
Local installation pATLAS can also be locally installed to allow custom built plasmid databases (https://patlas.gitbook.io/docs/api/local_installation). This allows users to explore their own plasmid database in the context of the plasmid RefSeq database available in pATLAS or any subset of it.
To create the default pATLAS database from the current NCBI RefSeq release, the user can take advantage of the scripts available in pATLAS github (https://github.com/B-UMMI/pATLAS) or the nextflow pipeline (https://github.com/tiagofilipe12/pATLAS-db-creation). A docker compose for the latest stable release (currently 1.6.0) of pATLAS (https://github.com/bfrgoncalves/patlas-compose) is also available to easily launch the pATLAS service in a local machine.



https://integbio.jp/dbcatalog/record/nbdc02514
pATLAS:  Plasmid Atlas
説明: 	NCBI RefSeqに登録されているプラスミドとそのメタデータを検索できるデータベースです。プラスミド名、生物種、抗生物質耐性、病原性、プラスミドファミリー、配列長、類似性による検索が可能で、プラスミド間の関係を可視化することができます。また外部パイプラインによって同定されたプラスミドセットをプラスミドネットワークに表示することもできます。


https://kazumaxneo.hatenablog.com/entry/2019/11/08/073000
pATLASflow - macでインフォマティクス

https://kazumaxneo.hatenablog.com/entry/2019/11/06/073000
プラスミドの分析と視覚化のwebサービス Plasmid ATLAS - macでインフォマティクス

https://twitter.com/WvSchaik/status/1060486271557091328
Willem van Schaik on Twitter: "Very interesting tool by @jacarrico: 'Plasmid ATLAS: plasmid visual analytics and identification in high-throughput sequencing data' https://t.co/CWIAoV3449"
5:56 AM - 8 Nov 2018

## 2017


## 2017-02-09

MINI REVIEW article
https://pubmed.ncbi.nlm.nih.gov/28232822/
Front Microbiol
. 2017 Feb 9;8:182. doi: 10.3389/fmicb.2017.00182. eCollection 2017.
Plasmid Classification in an Era of Whole-Genome Sequencing: Application in Studies of Antibiotic Resistance Epidemiology
Alex Orlek 1, Nicole Stoesser 2, Muna F Anjum 3, Michel Doumith 4, Matthew J Ellington 5, Tim Peto 1, Derrick Crook 1, Neil Woodford 5, A Sarah Walker 1, Hang Phan 1, Anna E Sheppard 1
https://www.frontiersin.org/articles/10.3389/fmicb.2017.00182/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5299020/
Table 2
Summary of common in silico tools used for plasmid analysis.



### aclame
ACLAME は、version 0.4 (5 August 2009) 以降は更新されていない。

http://aclame.ulb.ac.be/

https://pubmed.ncbi.nlm.nih.gov/19933762/
Nucleic Acids Res
. 2010 Jan;38(Database issue):D57-61. doi: 10.1093/nar/gkp938. Epub 2009 Nov 23.
ACLAME: a CLAssification of Mobile genetic Elements, update 2010
Raphaël Leplae 1, Gipsi Lima-Mendez, Ariane Toussaint
https://academic.oup.com/nar/article/38/suppl_1/D57/3112288
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808911/

https://pubmed.ncbi.nlm.nih.gov/17064288/
Review FEMS Microbiol Rev
. 2006 Nov;30(6):980-94. doi: 10.1111/j.1574-6976.2006.00044.x.
A first global analysis of plasmid encoded proteins in the ACLAME database
Raphaël Leplae 1, Gipsi Lima-Mendez, Ariane Toussaint
https://academic.oup.com/femsre/article/30/6/980/495140


https://www.ncbi.nlm.nih.gov/pubmed/17064288
FEMS Microbiol Rev. 2006 Nov;30(6):980-94.
A first global analysis of plasmid encoded proteins in the ACLAME database.
Leplae R1, Lima-Mendez G, Toussaint A.



----------


### oriTfinder
https://bioinfo-mml.sjtu.edu.cn/oriTfinder/

## 2018-07-02
https://pubmed.ncbi.nlm.nih.gov/29733379/
Nucleic Acids Res
. 2018 Jul 2;46(W1):W229-W234. doi: 10.1093/nar/gky352.
oriTfinder: a web-based tool for the identification of origin of transfers in DNA sequences of bacterial mobile genetic elements
Xiaobin Li 1, Yingzhou Xie 1, Meng Liu 1, Cui Tai 1, Jingyong Sun 2, Zixin Deng 1, Hong-Yu Ou 1 3
https://academic.oup.com/nar/article/46/W1/W229/4992657
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6030822/


----------

## MacSyFinder
CONJscan

https://research.pasteur.fr/en/software/conjscan-t4ssscan/
Eduardo Rocha - CONJscan-T4SSscan - Research - Institut Pasteur
MacSyFinder. The T4SS for protein secretion can be identified using TXSScan. The CONJscan part is now a module in MacSyFinder.

## Jean Cury
## 2018-09-01
https://pubmed.ncbi.nlm.nih.gov/29905872/
Mol Biol Evol
. 2018 Sep 1;35(9):2230-2239. doi: 10.1093/molbev/msy123.
Host Range and Genetic Plasticity Explain the Coexistence of Integrative and Extrachromosomal Mobile Genetic Elements
Jean Cury 1 2, Pedro H Oliveira 1 2, Fernando de la Cruz 3, Eduardo P C Rocha 1 2
https://academic.oup.com/mbe/article/35/9/2230/5037826?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6107060/
Conjugative systems were detected using the CONJscan module of MacSyFinder (Abby et al. 2014), with protein profiles and definitions of the MPF type T, published previously (Guglielmini et al. 2014).

## 2017-09-06
https://pubmed.ncbi.nlm.nih.gov/28911112/
Nucleic Acids Res
. 2017 Sep 6;45(15):8943-8956. doi: 10.1093/nar/gkx607.
Integrative and conjugative elements and their hosts: composition, distribution and organization
Jean Cury 1 2, Marie Touchon 1 2, Eduardo P C Rocha 1 2
https://academic.oup.com/nar/article/45/15/8943/3958713?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5587801/
Detection of conjugative systems
Conjugative systems were found with the CONJscan module of MacSyFinder (29), using protein profiles and definitions following a previous work (16) 


## Sophie S Abby
## 2014-10-17
https://pubmed.ncbi.nlm.nih.gov/25330359/
PLoS One
. 2014 Oct 17;9(10):e110726. doi: 10.1371/journal.pone.0110726. eCollection 2014.
MacSyFinder: a program to mine genomes for molecular systems with an application to CRISPR-Cas systems
Sophie S Abby 1, Bertrand Néron 2, Hervé Ménager 2, Marie Touchon 1, Eduardo P C Rocha 1
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0110726
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4201578/
Macromolecular System Finder (MacSyFinder)
https://github.com/gem-pasteur/macsyfinder

sophie.abby@pasteur.fr		Sophie S Abby

## Julien Guglielmini
## 2014-05
https://pubmed.ncbi.nlm.nih.gov/24623814/
Nucleic Acids Res
. 2014 May;42(9):5715-27. doi: 10.1093/nar/gku194. Epub 2014 Mar 12.
Key components of the eight classes of type IV secretion systems involved in bacterial conjugation or protein secretion
Julien Guglielmini 1, Bertrand Néron 2, Sophie S Abby 3, María Pilar Garcillán-Barcia 4, Fernando de la Cruz 4, Eduardo P C Rocha 3
https://academic.oup.com/nar/article/42/9/5715/1253976?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4027160/
Webservers
We had previously made available a web site (now called CONJscan, http://mobyle.pasteur.fr/cgi-bin/portal.py#forms::CONJscan-T4SSscan or the short URL http://bit.ly/CONJscan) that allows searching for T4SS and conjugation-related protein profiles.

jgugliel@pasteur.fr		Julien Guglielmini

----------

## 2019-09

https://pubmed.ncbi.nlm.nih.gov/31187446/
Comparative Study Genes Genomics
. 2019 Sep;41(9):1077-1083. doi: 10.1007/s13258-019-00839-1. Epub 2019 Jun 11.
Comparison of de-novo assembly tools for plasmid metagenome analysis
Sachin Kumar Gupta 1, Shahbaz Raza 1, Tatsuya Unno 2 3
https://link.springer.com/article/10.1007/s13258-019-00839-1
we suggest that the use of MEGAHIT for plasmid metagenome analysis may offer more diverse plasmids with less computer resource required.


### metaplasmidSPAdes

https://www.ncbi.nlm.nih.gov/pubmed/31048319
Genome Res. 2019 Jun;29(6):961-968. doi: 10.1101/gr.241299.118. Epub 2019 May 2.
Plasmid detection and assembly in genomic and metagenomic data sets.
Antipov D#1, Raiko M#1, Lapidus A1, Pevzner PA1,2.


### plasmidSPAdes
- https://cab.spbu.ru/software/plasmid-spades/
- https://cab.spbu.ru/software/spades/
- https://cab.spbu.ru/files/release3.15.2/manual.html

https://pubmed.ncbi.nlm.nih.gov/27466620/
Bioinformatics
. 2016 Nov 15;32(22):3380-3387. doi: 10.1093/bioinformatics/btw493. Epub 2016 Jul 27.
plasmidSPAdes: assembling plasmids from whole genome sequencing data
Dmitry Antipov 1, Nolan Hartwick 2, Max Shen 3, Mikhail Raiko 2, Alla Lapidus 1, Pavel A Pevzner 1 2
https://academic.oup.com/bioinformatics/article/32/22/3380/2525610





### mlplasmids
https://gitlab.com/sirarredondo/mlplasmids

https://www.ncbi.nlm.nih.gov/pubmed/30383524
Microb Genom. 2018 Nov;4(11). doi: 10.1099/mgen.0.000224. Epub 2018 Nov 1.
mlplasmids: a user-friendly tool to predict plasmid- and chromosome-derived sequences for single species.
Arredondo-Alonso S1, Rogers MRC1, Braat JC1, Verschuuren TD2, Top J1, Corander J3,4,5, Willems RJL1, Schürch AC1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6321875/
We evaluated the performance of mlplasmids against PlasFlow (version 1.0), PlasmidSPAdes (version 3.8.2) and cBar (version 1.2). 


### PlaScope
https://github.com/GuilhemRoyer/PlaScope

https://www.ncbi.nlm.nih.gov/pubmed/30265232
Microb Genom. 2018 Sep;4(9). doi: 10.1099/mgen.0.000211.
PlaScope: a targeted approach to assess the plasmidome from genome assemblies at the species level.
Royer G1,2,3, Decousser JW1,3, Branger C1, Dubois M2, Médigue C2, Denamur E4,1, Vallenet D2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6202455/
Compared to other plasmid classifiers, PlasFlow and cBar, it achieves better recall (0.87), specificity (0.99), precision (0.96) and accuracy (0.98) on a dataset of 70 genomes of Escherichia coli containing plasmids. 


### PlasmidSeeker
https://github.com/bioinfo-ut/PlasmidSeeker
A k-mer based program for the identification of known plasmids from bacterial whole genome sequencing reads

https://www.ncbi.nlm.nih.gov/pubmed/29629246
PeerJ. 2018 Apr 2;6:e4588. doi: 10.7717/peerj.4588. eCollection 2018.
PlasmidSeeker: identification of known plasmids from bacterial whole genome sequencing reads.
Roosaare M1, Puustusmaa M1, Möls M1,2, Vaher M1, Remm M1.
The PlasmidSeeker algorithm is based on k-mers and uses k-mer abundance to distinguish between plasmid and bacterial sequences. 

### PlasmidTron
https://github.com/sanger-pathogens/plasmidtron

https://www.ncbi.nlm.nih.gov/pubmed/29533742
Microb Genom. 2018 Mar 12. doi: 10.1099/mgen.0.000164. [Epub ahead of print]
PlasmidTron: assembling the cause of phenotypes and genotypes from NGS data.
Page AJ1,2, Wailan A1, Shao Y1, Judge K1, Dougan G3,1, Klemm EJ1, Thomson NR4,1, Keane JA1.
PlasmidTron uses a k-mer-based approach to identify reads associated with a phylogenetically unlinked phenotype. 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5885016/
Recently a number of methods have been developed to address the problem of assembling mobile genetic elements from next generation sequencing (NGS) data [1].
plasmidSPAdes[2] 
Unicycler [3] 
Recycler [5] 

https://twitter.com/pathogenomenick/status/978273798687477763
Nick Loman on Twitter: ".@andrewjpage - @TheQuadram's new Head of Informatics presenting his open wares to the @MRCClimb quarterly strategy meeting: 1. Docker containers for MLST software evaluation: https://t.co/1Qy0qCIVdd 2. PlasmidTron - k-mer approaches to detect plasmids: https://t.co/ZoFHCAhT5N… https://t.co/kBZ066JeyH"
10:13 AM - 26 Mar 2018

https://twitter.com/andrewjpage/status/908946651548717056
Andrew Page on Twitter: "Check out our latest paper PlasmidTron, it's like GWAS but focused on assembling MGEs https://t.co/B2T3d0ZU0k"
2:52 AM - 16 Sep 2017

----------
### PlasFlow

https://colil.dbcls.jp/browse/papers/29346586/1/100/year
PlasFlow: predicting plasmid sequences in metagenomic data using genome signatures.
Krawczyk PS, Lipinski L, Dziembowski A Nucleic Acids Res. 2018 Apr 6;46(6):e35. PMID:29346586
    is cited by the following papers.
合計: 102,

Deeplasmid: deep learning accurately separates plasmids from bacterial chromosomes.
Andreopoulos WB, Geller AM, Lucke M, Balewski J, Clum A, Ivanova NN, Levy A     Nucleic Acids Res. 2022 Feb 22;50(3):e17.

PathoFact: a pipeline for the prediction of virulence factors and antimicrobial resistance genes in metagenomic data.
de Nies L, Lopes S, Busi SB, Galata V, Heintz-Buschart A, Laczny CC, May P, Wilmes P     Microbiome. 2021 Feb 17;9(1):49.

PlasGUN: gene prediction in plasmid metagenomic short reads using deep learning.
Fang Z, Tan J, Wu S, Li M, Wang C, Liu Y, Zhu H     Bioinformatics. 2020 May 1;36(10):3239-3241.

PlasClass improves plasmid sequence classification.
Pellow D, Mizrahi I, Shamir R     PLoS Comput Biol. 2020 Apr;16(4):e1007781.

PPR-Meta: a tool for identifying phages and plasmids from metagenomic fragments using deep learning.
Fang Z, Tan J, Wu S, Li M, Xu C, Xie Z, Zhu H     Gigascience. 2019 Jun 1;8(6):.

https://github.com/smaegol/PlasFlow
NOT MAINTAINED
Use at your own risk. I am very grateful that it is being widely used but, as I completely changed my research area I cannot give my time to maintain this project. There are other, newer packages developed, which can be used instead.
メンテナンスされていません
ご利用は自己責任でお願いします。広く使われていることに感謝していますが、研究分野を完全に変更したため、このプロジェクトを維持する時間が取れません。代わりに使用できる、より新しいパッケージが他にも開発されています。

https://www.ncbi.nlm.nih.gov/pubmed/29346586
Nucleic Acids Res. 2018 Jan 13. doi: 10.1093/nar/gkx1321. [Epub ahead of print]
PlasFlow: predicting plasmid sequences in metagenomic data using genome signatures.
Krawczyk PS1,2, Lipinski L1, Dziembowski A1,2.
https://pmc.ncbi.nlm.nih.gov/articles/PMC5887522/

https://galaxyproject.github.io/training-material/topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.html
Species and plasmids
Prediction of plasmid sequences and classes using PlasFlow

----------

### Recycler
https://github.com/Shamir-Lab/Recycler

https://www.ncbi.nlm.nih.gov/pubmed/28003256
Bioinformatics. 2017 Feb 15;33(4):475-482. doi: 10.1093/bioinformatics/btw651.
Recycler: an algorithm for detecting plasmids from de novo assembly graphs.
Rozov R1, Brown Kav A2, Bogumil D2, Shterzer N2, Halperin E1,3,4, Mizrahi I2, Shamir R1.

- [Rozov R et al. (2017) Bioinformatics. 33(4):475-482. "Recycler: an algorithm for detecting plasmids from de novo assembly graphs."](https://www.ncbi.nlm.nih.gov/pubmed/28003256)



## PLACNETw
https://castillo.dicom.unican.es/upload/
## 2017-12-01
https://pubmed.ncbi.nlm.nih.gov/29036591/
Bioinformatics
. 2017 Dec 1;33(23):3796-3798. doi: 10.1093/bioinformatics/btx462.
PLACNETw: a web-based tool for plasmid reconstruction from bacterial genomes
Luis Vielva 1, María de Toro 2, Val F Lanza 3, Fernando de la Cruz 4
https://academic.oup.com/bioinformatics/article/33/23/3796/3983262?login=false
https://digital.csic.es/handle/10261/189719

## PLACNET
PLAsmid Constellation NET
https://sourceforge.net/projects/placnet/
## 2014-12-18
https://pubmed.ncbi.nlm.nih.gov/25522143/
PLoS Genet
. 2014 Dec 18;10(12):e1004766. doi: 10.1371/journal.pgen.1004766. eCollection 2014 Dec.
Plasmid flux in Escherichia coli ST131 sublineages, analyzed by plasmid constellation network (PLACNET), a new method for plasmid reconstruction from whole genome sequences
Val F Lanza 1, María de Toro 1, M Pilar Garcillán-Barcia 1, Azucena Mora 2, Jorge Blanco 2, Teresa M Coque 3, Fernando de la Cruz 1
https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004766
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4270462/


### AcCNET
http://sourceforge.net/projects/accnet

https://www.ncbi.nlm.nih.gov/pubmed/27663497
Bioinformatics. 2017 Jan 15;33(2):283-285. doi: 10.1093/bioinformatics/btw601. Epub 2016 Sep 23.
AcCNET (Accessory Genome Constellation Network): comparative genomics software for accessory genome analysis using bipartite networks.
Lanza VF1,2,3, Baquero F1,2,3, de la Cruz F4, Coque TM1,2,3.
https://academic.oup.com/bioinformatics/article/33/2/283/2525709


### cBar

http://csbl.bmb.uga.edu/~ffzhou/cBar/
リンク切れ

https://pubmed.ncbi.nlm.nih.gov/20538725/
Bioinformatics
. 2010 Aug 15;26(16):2051-2. doi: 10.1093/bioinformatics/btq299. Epub 2010 Jun 10.
cBar: a computer program to distinguish plasmid-derived from chromosome-derived sequence fragments in metagenomics data
Fengfeng Zhou 1, Ying Xu
https://academic.oup.com/bioinformatics/article/26/16/2051/215085?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2916713/

----------
### PlasMapper

リンク切れ
http://wishart.biology.ualberta.ca/PlasMapper/
Service Temporarily Unavailable

https://www.ncbi.nlm.nih.gov/pubmed/15215471
Nucleic Acids Res. 2004 Jul 1;32(Web Server issue):W660-4.
PlasMapper: a web server for drawing and auto-annotating plasmid maps.
Dong X1, Stothard P, Forsythe IJ, Wishart DS.

https://en.wikipedia.org/wiki/Plasmapper
PlasMapper (Plasmid Mapper) [1] is a freely available web server that automatically generates and annotates high-quality circular plasmid maps.

http://togodb.biosciencedbc.jp/entry/stga_tool/410
プラスミドなどの塩基配列に対して、制限酵素切断部位を始め、複製開始点、プロモーターなどの情報、さらに自分の定義した領域のマッピングを同時に行い、ベクトル形式(SVG)で描画するツール。

----------

### SnapGene
http://www.snapgene.com
SnapGene | Software for everyday molecular biology

### ApE
ApE(A plasmid Editor)

- 2013/12/04 [ApEでシーケンスアライメント - ニュートラルな研究者のメモ](http://cobucchii.hatenablog.com/entry/2013/12/04/214725)
- [ApE(A plasmid Editor)を利用してプラスミドを設計する - 統合TV (togotv)(2013-05-27)](http://togotv.dbcls.jp/20130527.html)
- 2012/7/7 [ApEでできること一覧 | LifeScienceProject](http://life-science-project.com/540/)
- 2009.08.03 [DNA情報の取得とApEでの操作 - 花井＠産総研](https://staff.aist.go.jp/s-hanai/gene_download.html)
- [堀田研究室｜リンク](http://www.cira.kyoto-u.ac.jp/hotta/link.html) ApE フリーであるにも関わらず、非常に優秀な塩基配列編集ソフトウェア。GeneBank形式等の各種塩基配列ファイルを読み込んで、アノテーション付きで表示や編集が可能。制限酵素の切断部位やサイズを様々に表示したり、さらにはDam/Dcm依存性の制限酵素の切り替えもボタン一つ。プラスミドマップの描画にも対応しており、シンプルだが実用上は十分。Mac, Win, Unix/Linux対応。

### [Addgene](https://www.addgene.org) non-profit plasmid repository
- [Addgene (@Addgene) | Twitter](https://twitter.com/addgene)

----------
### cira.kyoto-u

https://www.cira.kyoto-u.ac.jp/hotta/link.html
京大CiRA堀田研｜リンク集

分子生物学実験関連
- PlasMapperr
未知のプラスミドDNAの配列を入力すると、自動的にアノテーションをして美しいマップを表示してくれる。アノテーション結果をGeneBank形式で書き出しApEで開く事も可能。

研究用ソフトウェア
- ApE
無償であるにも関わらず、非常に優秀な塩基配列編集ソフトウェア。GeneBank形式等の各種塩基配列ファイルを読み込んで、アノテーション付きで表示や編集が可能。制限酵素の切断部位やサイズを様々に表示したり、さらにはDam/Dcm依存性の制限酵素の切り替えもボタン一つ。プラスミドマップの描画にも対応しており、シンプルだが実用上は十分。Mac, Win, Unix/Linux対応。
- SnapGene Viewer
有償ソフトSnapGeneのトライアル版的な位置付けで、プラスミドDNAなどの塩基配列管理ソフトウェア。GeneBank形式等の各種塩基配列ファイルを読み込んで、アノテーション付きで表示が可能。編集機能を利用するには有償版を購入する必要があるが、グラフィックスの美しさはさすが商用ソフト。Mac, Win, Linux対応。

研究用データベース
- Addgenes
研究用プラスミドDNAの非営利分与団体。MTA(研究資料分与契約書)を締結すれば、登録されているプラスミドDNA一つをUS$65程度(+送付料)で分与してもらえる。また、自身の構築したプラスミドDNAを委託して、世界中に分与してもらうことも可能。

----------

### kazumaxneo

https://kazumaxneo.hatenablog.com/archive/category/plasmid
plasmid カテゴリーの記事一覧 - macでインフォマティクス


https://kazumaxneo.hatenablog.com/entry/2021/04/13/193935
バクテリアのゲノムやプラスミドのアノテーションを行う Bakta - macでインフォマティクス


https://kazumaxneo.hatenablog.com/entry/2020/07/01/140556
ターゲットアセンブリにより保存されたプラスミド配列を再構成してアノテーションをつける PlasmidID - macでインフォマティクス


https://kazumaxneo.hatenablog.com/entry/2019/10/09/234533
メタゲノムのアセンブリ配列からファージやプラスミドを予測する PPR-Meta - macでインフォマティクス

https://kazumaxneo.hatenablog.com/entry/2019/06/05/073000
metaplasmidSPAdes - macでインフォマティクス

https://kazumaxneo.hatenablog.com/entry/2019/05/11/073000
メタゲノムのアセンブリ配列からプラスミド配列を予測する PlasFlow - macでインフォマティクス

https://kazumaxneo.hatenablog.com/entry/2018/07/03/113439
抗生物質耐性遺伝子や病原性遺伝子を素早く検索できる ABRicate - macでインフォマティクス

https://kazumaxneo.hatenablog.com/entry/2018/07/02/142609
腸内細菌科（エンテロバクター科）のプラスミド同定ツール PlasmidFinder - macでインフォマティクス

https://kazumaxneo.hatenablog.com/entry/2018/05/28/203726
NGSデータからプラスミドを検出するplasmidseeker - macでインフォマティクス

https://kazumaxneo.hatenablog.com/entry/2018/04/02/123713
ロングリードを使い環状DNAかどうか調べる Circlator - macでインフォマティクス
多くの重要な抗菌剤耐性および病原性決定因子がプラスミド上に担持され、これらの環状配列について完全で正確な情報を有することの重要性が示される。

https://kazumaxneo.hatenablog.com/entry/2017/12/17/142135
cBarでプラスミド配列を区別する - macでインフォマティクス

https://kazumaxneo.hatenablog.com/entry/2017/10/01/182022
環状ゲノムを探す ccontigs - macでインフォマティクス
ccontigsはコンティグ末端の配列がオーバーラップしているかどうか調べることで、コンティグからclosed circularなゲノムを探すツール。環状のプラスミドやウィルスを検出できるとされる。論文にはなっていない。

https://kazumaxneo.hatenablog.com/entry/2017/08/27/233654
アセンブルのgraphからプラスミドデータを検出するツール Recycler - macでインフォマティクス

https://kazumaxneo.hatenablog.com/entry/2017/06/27/133058
DNA解析ソフト1 Snapgene - macでインフォマティクス
SnapGeneはプラスミドやゲノムの解析ソフトである。ビジュアルで見やすく表示する機能に特化しており、制限酵素処理結果を画面で見やすく表示したり、TA cloningやgibson、in fuisionなどの反応結果を表示し設計を支援する機能などを持つ。

----------

