# plasmid metagenome

**Table of Contents**
- [unclassified](#unclassified)
- [updates](#updates)
  - [2020](#2020)
  - [2019](#2019)
  - [2018](#2018)
  - [2017](#2017)
- [kazumaxneo](#kazumaxneo)

----------
## unclassified


----------

### Orlek_2017_Review

review
https://pubmed.ncbi.nlm.nih.gov/28232822/
Front Microbiol
. 2017 Feb 9;8:182. doi: 10.3389/fmicb.2017.00182. eCollection 2017.
Plasmid Classification in an Era of Whole-Genome Sequencing: Application in Studies of Antibiotic Resistance Epidemiology
Alex Orlek 1, Nicole Stoesser 2, Muna F Anjum 3, Michel Doumith 4, Matthew J Ellington 5, Tim Peto 1, Derrick Crook 1, Neil Woodford 5, A Sarah Walker 1, Hang Phan 1, Anna E Sheppard 1
https://www.frontiersin.org/articles/10.3389/fmicb.2017.00182/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5299020/
This review focuses on WGS datasets from cultured rather than metagenomic samples; for the latter, see recent reviews (Jørgensen et al., 2014; Martínez et al., 2016).

### Martinez_2017
### 2017-01

https://pubmed.ncbi.nlm.nih.gov/27861983/
Review Ann N Y Acad Sci
. 2017 Jan;1388(1):26-41. doi: 10.1111/nyas.13282. Epub 2016 Nov 10.
Genomic and metagenomic technologies to explore the antibiotic resistance mobilome
José L Martínez 1, Teresa M Coque 2 3 4, Val F Lanza 2 3 4, Fernando de la Cruz 5, Fernando Baquero 2 3 4
https://nyaspubs.onlinelibrary.wiley.com/doi/abs/10.1111/nyas.13282
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_proquest_miscellaneous_1868320943
Wiley Online Library Database Model 2024
https://nyaspubs-onlinelibrary-wiley-com.kras.lib.keio.ac.jp/doi/full/10.1111/nyas.13282
Methods for analyzing the mobilome and for extracting mobilome information from genomes and metagenomes

Methods, such as PlasmidFinder, pMLST,133 or in silico degenerate plasmid MOB typing,122 identify and classify plasmids by the presence of plasmid-specific characteristics, such as the replication system, a multilocus sequence type, or its mobilization system. 

### Jorgensen_2015

https://pubmed.ncbi.nlm.nih.gov/25657641/
Front Microbiol
. 2015 Jan 22;5:750. doi: 10.3389/fmicb.2014.00750. eCollection 2014.
Current strategies for mobilome research
Tue S Jørgensen 1, Anne S Kiil 1, Martin A Hansen 1, Søren J Sørensen 1, Lars H Hansen 2
https://www.frontiersin.org/articles/10.3389/fmicb.2014.00750/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302988/

----------
## updates

## 2024

## 2024-08-28
https://pubmed.ncbi.nlm.nih.gov/39198442/
Nat Commun
. 2024 Aug 28;15(1):7450. doi: 10.1038/s41467-024-51929-y.
Nanopore R10.4 metagenomic detection of blaCTX-M/blaDHA antimicrobial resistance genes and their genetic environments in stool
Edgar I Campos-Madueno 1 2, Claudia Aldeia 1, Andrea Endimiani 3
https://www.nature.com/articles/s41467-024-51929-y
https://pmc.ncbi.nlm.nih.gov/articles/PMC11358271/
In 2 native Nanopore-SMS-positive samples, the location of a single blaCTX-M-117-like read in sample S1-NGR-02 had a 97.7% identity and 5% coverage to IncFIB(H89-PhagePlasmid)-type plasmid p2-S1-NGR-02-A, while the S1-ESP-03 MAG-positive for blaCTX-M-15 mapped with 99.9% identity and 100% coverage to IncFII-type plasmid p1-S1-ESP-03-A (Supplementary Data 1 and Fig. 2).
|
Furthermore, the extracted reads, and MAG contigs were screened with PlasmidFinder v2.0 (parameters: 50% minimum template identity and 60% template length coverage) and MLST v2.0 (parameter: E. coli number 1 configuration)57,58.

## 2022

## 2022-09-15
https://pubmed.ncbi.nlm.nih.gov/35654190/
Sci Total Environ
. 2022 Sep 15:839:156313. doi: 10.1016/j.scitotenv.2022.156313. Epub 2022 May 30.
Metagenomic insights into the influence of mobile genetic elements on ARGs along typical wastewater treatment system on pig farms in China
Rong-Min Zhang 1, Mei-Na Liao 2, Jia-En Wu 2, Xiao-Qing Lu 2, Hui-Zhen Tan 2, Jian Sun 1, Xiao-Ping Liao 3, Ya-Hong Liu 4
https://www.sciencedirect.com/science/article/pii/S0048969722034106?via=ihub
The metagenomic sequence data from our 27 samples were mapped to the PlasmidFinder database (Carattoli et al., 2014) including plasmid replicon types that originated from both Gram-positive bacteria and the Enterobacteriaceae. 

## 2020

## 2020-07
## Walter_2020_Plasmid
https://pubmed.ncbi.nlm.nih.gov/32380021/
Plasmid
. 2020 Jul;110:102505. doi: 10.1016/j.plasmid.2020.102505. Epub 2020 May 4.
Plasmidome of an environmental Acinetobacter lwoffii strain originating from a former gold and arsenic mine
Tomasz Walter 1, Joanna Klim 1, Marcin Jurkowski 1, Jan Gawor 1, Iwona Köhling 1, Małgorzata Słodownik 1, Urszula Zielenkiewicz 2
https://www.sciencedirect.com/science/article/pii/S0147619X20300172

9:33 AM · Aug 12, 2020
https://twitter.com/EicJrood/status/1293344783956447232
JRood_Plasmid_EIC on Twitter: "Congratulations to Tomasz Walter, Urszula Zielenkiewicz and colleagues whose interesting paper on the plasmidome of an environmental Acinetobacter lwoffii strain originating from a former gold and arsenic mine has now been published in Plasmid (https://t.co/h675rmQeDO). #Plasmid" / Twitter

4:44 PM · May 12, 2020
https://twitter.com/gaworj/status/1260113697046921221
Jan Gawor on Twitter: "Our recent paper is out. Genome sequence obtained with my little help and @nanopore MinION: Plasmidome of an environmental Acinetobacter lwoffii strain originating from a former gold and arsenic mine. https://t.co/4cv1RsOX6p" / Twitter

##

https://pubmed.ncbi.nlm.nih.gov/32655530/
Front Microbiol
. 2020 Jun 23;11:1343. doi: 10.3389/fmicb.2020.01343. eCollection 2020.
First Report on the Plasmidome From a High-Altitude Lake of the Andean Puna
María Florencia Perez 1, Daniel Kurth 1, María Eugenia Farías 1, Mariana Noelia Soria 1, Genis Andrés Castillo Villamizar 2 3, Anja Poehlein 2, Rolf Daniel 2, Julián Rafael Dib 1 4
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7324554/

## 2020-05-18
## Lira_2020
https://pubmed.ncbi.nlm.nih.gov/32424207/
Sci Rep
. 2020 May 18;10(1):8174. doi: 10.1038/s41598-020-65031-y.
Metagenomic analysis of an urban resistome before and after wastewater treatment
Felipe Lira 1, Ivone Vaz-Moreira 2, Javier Tamames 1, Célia M Manaia 2, José Luis Martínez 3
https://www.nature.com/articles/s41598-020-65031-y
https://pmc.ncbi.nlm.nih.gov/articles/PMC7235214/
ESKAPE

Plasmid replicons

The most abundant replicons were repUS2, IncP(6), and IncQ2, in this order. RepUS2 plasmids are narrow-range plasmids, present in Gram-positive bacteria, and have been recently found to be prevalent in gut microbiome40, while IncP(6) and IncQ2 are broad-range conjugative plasmids that contribute to the acquisition of multidrug resistance in a variety of Gram-negative bacteria41–45. (Fig. 1, Table S2). In agreement with the results obtained with ARGs data, PCA showed that influent samples grouped together in comparison with treated wastewater samples with a spreader distribution (Fig. 3). In this case, just one plasmid replicon was detected in the three sampling dates in the final UV, the IncP(Beta) replicon type (Table S2).

Figure 5

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7235214/#Sec7title

Discussion


However the higher number of reads belonged to macrolide resistance genes, mainly msrE and mphE. Plasmids as those belonging to the repUS2 group, which have been recently found to be prevalent in gut microbiome40, as well as the IncP(6) and IncQ groups that include several multidrug resistance plasmids41–45 were highly abundant in RAW water and their presence was much lower after UV treatment. 


40. Munck, C., Sheth, R. U., Freedberg, D. E. & Wang, H. H. Recording mobile DNA in the gut microbiota using an Escherichia coli CRISPR-Cas spacer acquisition platform. Nat Commun 11, 10.1038/s41467-019-14012-5 (2020). [PMC free article] [PubMed] [Ref list]


To note here that IncQ2 plasmids have been recently found in urban agriculture fields irrigated with wastewater in Africa cities65 and that IncP-6 plasmids have been involved in carbapenem-resistance circulation among several Enterobacteriaceae species from wastewater and a hospital43.


https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7235214/#Sec8title

Materials and Methods

Analysis of shotgun metagenomic data

Plasmid replicon types present in the samples were identified by performing a local alignment of all reads from each sample against the sequences deposited at the PlasmidFinder v1.3 database67, once downloaded and formatted as database in a local server. For comparative purposes, the relative abundance of replicons and of ARGs was normalized by the number of reads present in 10 million of total reads.

https://en.wikipedia.org/wiki/ESKAPE
```
Enterococcus faecium

Staphylococcus aureus

Klebsiella pneumoniae

Acinetobacter baumannii

Pseudomonas aeruginosa

Enterobacter spp.
```

IncP-6


### 2020-01-07

Plasmid Finder
-> should be
PlasmidFinder

https://colil.dbcls.jp/browse/papers/31911609/1/100/year
https://pubmed.ncbi.nlm.nih.gov/31911609/
Nat Commun
. 2020 Jan 7;11(1):95. doi: 10.1038/s41467-019-14012-5.
Recording mobile DNA in the gut microbiota using an Escherichia coli CRISPR-Cas spacer acquisition platform
Christian Munck 1, Ravi U Sheth 1 2, Daniel E Freedberg 3, Harris H Wang 4 5
https://www.nature.com/articles/s41467-019-14012-5
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6946703/

Fig. 5
Analysis of human-associated mobilome from HGT recordings.
a Exogenous spacers mapped to the custom plasmid database, each row represents a plasmid (denoted by accession number). The mappings are filtered to include the fewest number of plasmids covering all spacers. Rows are sorted in order of the number of spacers that map to the plasmid. The sorting enables easy identification of discrete transferred elements. Each spacer cluster is annotated with the predicted plasmid group based on Plasmid Finder36. 


Fig. 6
Metagenomic verification of predicted transfer events.

Using the Plasmid Finder database36, we identified putative plasmid contigs across the metagenomes.

36. Carattoli A, et al. In silico detection and typing of plasmids using plasmidfinder and plasmid multilocus sequence typing. Antimicrob. Agents Ch. 2014;58:3895–3903. doi: 10.1128/AAC.02412-14. [PMC free article] [PubMed] [CrossRef] [Google Scholar]

### Brown_Kav_2020
https://pubmed.ncbi.nlm.nih.gov/31602783/
Environ Microbiol
. 2020 Jan;22(1):32-44. doi: 10.1111/1462-2920.14813. Epub 2019 Dec 15.
Unravelling plasmidome distribution and interaction with its hosting microbiome
Aya Brown Kav 1 2, Roye Rozov 3, David Bogumil 1, Søren Johannes Sørensen 4, Lars Hestbjerg Hansen 5, Itai Benhar 2, Eran Halperin 2 3 6, Ron Shamir 3, Itzhak Mizrahi 1
https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/1462-2920.14813


## 2019

### Stalder_EvaTop_2019
## 2019-10
https://pubmed.ncbi.nlm.nih.gov/31147603/
ISME J
. 2019 Oct;13(10):2437-2446. doi: 10.1038/s41396-019-0446-4. Epub 2019 May 30.
Linking the resistome and plasmidome to the microbiome
Thibault Stalder 1 2, Maximilian O Press 3, Shawn Sullivan 3, Ivan Liachko 3, Eva M Top 4 5
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6776055/
Here we demonstrate that the in vivo proximity-ligation method Hi-C can reconstruct a known plasmid-host association from a wastewater community, and identify the in situ host range of ARGs, plasmids, and integrons by physically linking them to their host chromosomes. Hi-C detected both previously known and novel associations between ARGs, mobile genetic elements and host genomes, thus validating this method. 



https://pubmed.ncbi.nlm.nih.gov/31187446/
Comparative Study Genes Genomics
. 2019 Sep;41(9):1077-1083. doi: 10.1007/s13258-019-00839-1. Epub 2019 Jun 11.
Comparison of de-novo assembly tools for plasmid metagenome analysis
Sachin Kumar Gupta 1, Shahbaz Raza 1, Tatsuya Unno 2 3
https://link.springer.com/article/10.1007/s13258-019-00839-1
MEGAHIT


https://pubmed.ncbi.nlm.nih.gov/31455406/
Microbiome
. 2019 Aug 27;7(1):119. doi: 10.1186/s40168-019-0737-z.
Long-read metagenomic exploration of extrachromosomal mobile genetic elements in the human gut
Yoshihiko Suzuki 1, Suguru Nishijima 1 2 3, Yoshikazu Furuta 4, Jun Yoshimura 1, Wataru Suda 1 5, Kenshiro Oshima 1, Masahira Hattori 6 7 8, Shinichi Morishita 9
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6712665/
plasmids and bacteriophages


### Che_2019_Microbiome

https://pubmed.ncbi.nlm.nih.gov/30898140/
Microbiome
. 2019 Mar 21;7(1):44. doi: 10.1186/s40168-019-0663-0.
Mobile antibiotic resistome in wastewater treatment plants revealed by Nanopore metagenomic sequencing
You Che 1, Yu Xia 1, Lei Liu 1, An-Dong Li 1, Yu Yang 1, Tong Zhang 2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6429696/

plasmids. Transposons and integrons 
A broad spectrum of ARGs carried by plasmids (29 subtypes) and ICEs (4 subtypes) was persistent across the WWTPs. 

ESKAPE

Then, PlasFlow [24] was used to identify the ARGs carried by plasmids from all the detected ARG reads. Taxonomic identification of the ARG-carrying reads was conducted using Centrifuge (v1.0.3) [25] with the NCBI nonredundant nucleotide sequence database; the classification results were visualized with Pavian (https://github.com/fbreitwieser/pavian). 

### Kothari_2019_mBio

https://pubmed.ncbi.nlm.nih.gov/30808697/
mBio
. 2019 Feb 26;10(1):e02899-18. doi: 10.1128/mBio.02899-18.
Large Circular Plasmids from Groundwater Plasmidomes Span Multiple Incompatibility Groups and Are Enriched in Multimetal Resistance Genes
Ankita Kothari 1, Yu-Wei Wu 1 2, John-Marc Chandonia 3 4, Marimikel Charrier 1, Lara Rajeev 1, Andrea M Rocha 5, Dominique C Joyner 6, Terry C Hazen 5 6, Steven W Singer 1, Aindrila Mukhopadhyay 7 3
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6391923/

The circular plasmids from groundwater were diverse in encoding five out of six different relaxase groups and seven incompatibility groups into which plasmids are classified. Based on the relaxase classification (37), MOBQ and MOBP were the most abundant (Fig. 6a).


## 2018

https://link.springer.com/referenceworkentry/10.1007%2F978-1-4614-1531-2_565
Cite this entry as:
Thomas C.M. (2018) Plasmid Incompatibility. In: Wells R.D., Bond J.S., Klinman J., Masters B.S.S. (eds) Molecular Life Sciences. Springer, New York, NY. https://doi.org/10.1007/978-1-4614-1531-2_565
During the analysis and assembly of genome or metagenome ( Metamobilomics: The Plasmid Metagenome of Natural Environments) sequence data, one feature that will mark out a contig (an assembly of overlapping sequence reads into a continuous block of reliable DNA sequence data) as coming from a plasmid is the presence of a replicon (although some “plasmids” have become “chromosomes” by virtue of acquiring genes essential to their hosts – see  “Plasmids as Secondary Chromosomes”).


https://link.springer.com/referenceworkentry/10.1007%2F978-1-4614-1531-2_576
Cite this entry as:
Li L., Luo W., Hansen L.H., Sørensen S.J. (2018) Metamobilomics – The Plasmid Metagenome of Natural Environments. In: Wells R.D., Bond J.S., Klinman J., Masters B.S.S. (eds) Molecular Life Sciences. Springer, New York, NY. https://doi.org/10.1007/978-1-4614-1531-2_576


## 2016

https://pubmed.ncbi.nlm.nih.gov/27530840/
J Glob Antimicrob Resist
. 2016 Sep;6:57-66. doi: 10.1016/j.jgar.2016.03.001. Epub 2016 Apr 19.
Plasmid metagenomics reveals multiple antibiotic resistance gene classes among the gut microbiomes of hospitalised patients
Tossawan Jitwasinkul 1, Prapat Suriyaphol 2, Sithichoke Tangphatsornruang 3, Martin Asser Hansen 4, Lars Hestbjerg Hansen 5, Søren Johannes Sørensen 4, Chairat Permpikul 6, Yong Rongrungruang 6, Chanwit Tribuddharat 7
https://www.sciencedirect.com/science/article/abs/pii/S2213716516300261?via%3Dihub

## 2015

### Li_2015

https://pubmed.ncbi.nlm.nih.gov/26441947/
Front Microbiol
. 2015 Sep 24;6:1025. doi: 10.3389/fmicb.2015.01025. eCollection 2015.
Exploring antibiotic resistance genes and metal resistance genes in plasmid metagenomes from wastewater treatment plants
An-Dong Li 1, Li-Guan Li 1, Tong Zhang 1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4585309/

### Dib_2015

https://pubmed.ncbi.nlm.nih.gov/26074886/
Review Front Microbiol
. 2015 May 26;6:463. doi: 10.3389/fmicb.2015.00463. eCollection 2015.
Strategies and approaches in plasmidome studies-uncovering plasmid diversity disregarding of linear elements?
Julián R Dib 1, Martin Wagenknecht 2, María E Farías 3, Friedhelm Meinhardt 4
https://www.frontiersin.org/articles/10.3389/fmicb.2015.00463/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4443254/

Global Plasmid Studies in Cultured Strains

Culture-Independent Plasmidome

Metagenomic Plasmidome


## 2014

https://ci.nii.ac.jp/naid/110009972449
CiNii 論文 -  S32-03 活性汚泥細菌叢におけるプラスミドメタゲノム解析(S32 微生物社会学を基盤とした革新的水利用技術への挑戦,シンポジウム)
日本微生物生態学会講演要旨集 2014, 69, 2014
宮崎 亮
https://dl.ndl.go.jp/info:ndljp/pid/10882613


## 2012




https://pubmed.ncbi.nlm.nih.gov/22580363/
Nat Rev Microbiol
. 2012 May 14;10(6):379. doi: 10.1038/nrmicro2804.
Welcome to the plasmidome
Alan Walker
PMID: 22580363 DOI: 10.1038/nrmicro2804
Abstract
This month's Genome Watch reviews a recent paper that used a metagenomics approach to characterize the plasmid content of bovine rumen samples.
https://www.nature.com/articles/nrmicro2804



### Brown_Kav_2012

https://pubmed.ncbi.nlm.nih.gov/22431592/
Proc Natl Acad Sci U S A
. 2012 Apr 3;109(14):5452-7. doi: 10.1073/pnas.1116410109. Epub 2012 Mar 19.
Insights into the bovine rumen plasmidome
Aya Brown Kav 1, Goor Sasson, Elie Jami, Adi Doron-Faigenboim, Itai Benhar, Itzhak Mizrahi
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3325734/


https://jp.illumina.com/content/dam/illumina-marketing/apac/japan/documents/pdf/publication_metagenome-j.pdf
総説：メタゲノム研究 - イルミナ株式会社

プラスミド（プラスミド解析）

プラスミドは、微生物環境における強力で重要な進化因子である、平行遺伝子伝播を媒介する場合が多くあります24。

参考文献：
Kav, A. B., Sasson G., Jami E., Doron-Faigenboim A., Benhar I., et al. (2012) Insights into the bovine rumen
plasmidome. Proc Natl Acad Sci U S A 109: 5452–5457
著者らは、ルーメンのプラスミドームが門を超え得るほどの高度なモザイク性を有していると示しています。ルーメン
のプラスミドームの機能的プロファイルは、ルーメンの生態学的ニッチに多く存在する機能をコードしています。
これにより宿主に優位性が付与され、可動性の遺伝因子が環境への適応に重要な役割を果たしていることが示唆され
ています。
イルミナ技術：Genome AnalyzerIIx およびペアエンドリード

### Ma_2012

https://pubmed.ncbi.nlm.nih.gov/22059529/
Environ Microbiol
. 2012 Feb;14(2):453-66. doi: 10.1111/j.1462-2920.2011.02633.x. Epub 2011 Nov 8.
Analysis of two marine metagenomes reveals the diversity of plasmids in oceanic environments
Yingfei Ma 1, Ian T Paulsen, Brian Palenik
https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/j.1462-2920.2011.02633.x



## 2011

https://pubmed.ncbi.nlm.nih.gov/22016806/
PLoS One
. 2011;6(10):e26041. doi: 10.1371/journal.pone.0026041. Epub 2011 Oct 10.
Plasmid metagenome reveals high levels of antibiotic resistance genes and mobile genetic elements in activated sludge
Tong Zhang 1, Xu-Xiang Zhang, Lin Ye
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189950/

## 2009

https://pubmed.ncbi.nlm.nih.gov/19389756/
Microbiology (Reading)
. 2009 Jul;155(Pt 7):2306-2319. doi: 10.1099/mic.0.028233-0. Epub 2009 Apr 23.
Detection of 140 clinically relevant antibiotic-resistance genes in the plasmid metagenome of wastewater treatment plant bacteria showing reduced susceptibility to selected antibiotics
Rafael Szczepanowski 1, Burkhard Linke 2, Irene Krahn 1, Karl-Heinz Gartemann 3, Tim Gützkow 1, Wolfgang Eichler 4, Alfred Pühler 1, Andreas Schlüter 1

## 2008

https://pubmed.ncbi.nlm.nih.gov/18603322/
J Biotechnol
. 2008 Aug 31;136(1-2):65-76. doi: 10.1016/j.jbiotec.2008.03.017. Epub 2008 Apr 8.
Genetic diversity and composition of a plasmid metagenome from a wastewater treatment plant
Andreas Schlüter 1, Lutz Krause, Rafael Szczepanowski, Alexander Goesmann, Alfred Pühler

https://pubmed.ncbi.nlm.nih.gov/18586057/
J Biotechnol
. 2008 Aug 31;136(1-2):54-64. doi: 10.1016/j.jbiotec.2008.03.020. Epub 2008 Jun 10.
Insight into the plasmid metagenome of wastewater treatment plant bacteria showing reduced susceptibility to antimicrobial drugs analysed by the 454-pyrosequencing technology
Rafael Szczepanowski 1, Thomas Bekel, Alexander Goesmann, Lutz Krause, Holger Krömeke, Olaf Kaiser, Wolfgang Eichler, Alfred Pühler, Andreas Schlüter

----------
## kazumaxneo

http://kazumaxneo.hatenablog.com/archive/category/plasmid
plasmid カテゴリーの記事一覧 - macでインフォマティクス

http://kazumaxneo.hatenablog.com/entry/2019/10/09/234533
メタゲノムのアセンブリ配列からファージやプラスミドを予測する PPR-Meta - macでインフォマティクス

メタゲノムでは、しかしながら、コンティグ間のカバレッジ範囲の違いはコピー数よりもむしろ種の異なる存在量から生じる。ツールcBar [ref.24]は、主にメタゲノムにおけるプラスミド同定のために設計された最初のツールである。このツールは、k-merスペクトラムに基づく分類器として逐次最小最適化を適用する。 cBarと同様に、PlasFlow [ref.25]もプラスミドを同定するためのk-merベースのツールである。 cBarと比較して、PlasFlowはさらに異なるk-mer長の情報を組み合わせ、votingデバイスとして複数のニューラルネットワークを使用して配列がプラスミドに属するかどうかを判断し、cBarよりも優れたパフォーマンスを達成する。

本稿では、深層学習アルゴリズムに基づいてメタゲノム断片をファージ、プラスミド、またはクロモソームとして識別するための3クラス分類器であるPPR-Meta（Phage and Plasmid Recognizer for Metagenomes）を紹介する。

http://kazumaxneo.hatenablog.com/entry/2019/06/05/073000
metaplasmidSPAdes - macでインフォマティクス


http://kazumaxneo.hatenablog.com/entry/2019/05/11/073000
メタゲノムのアセンブリ配列からプラスミド配列を予測する PlasFlow - macでインフォマティクス


http://kazumaxneo.hatenablog.com/entry/2018/07/03/113439
抗生物質耐性遺伝子や病原性遺伝子を素早く検索できる ABRicate - macでインフォマティクス

----------


https://www.ncbi.nlm.nih.gov/pubmed/26657537
FEMS Microbiol Rev. 2016 Mar;40(2):258-72. doi: 10.1093/femsre/fuv048. Epub 2015 Dec 9.
Computational approaches to predict bacteriophage-host relationships.
Edwards RA1, McNair K2, Faust K3, Raes J3, Dutilh BE4.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5831537/
All the bioinformatics code used in this work is available online at http://edwards.sdsu.edu/PhageHosts/.

http://edwards.sdsu.edu/PhageHosts/



----------

