# plasmid IncP1

**Table of Contents**

- [](#)
- [unclassified](#unclassified)
- [mob](#mob)
- [Neisseria gonorrhoeae](#neisseria-gonorrhoeae)
  - [pEP5289](#pep5289)
- [basal](#basal)
- [MCR](#mcr)
  - [pMCR_1511](#pmcr_1511)
- [updates](#updates)
[2021](#2021)
[2020](#2020)
[2019](#2019)
[2018](#2018)
[2017](#2017)
[2016](#2016)
[2015](#2015)
[2013](#2013)
[2012](#2012)
[2011](#2011)
[2010](#2010)
[2007](#2007)
[2006](#2006)
[1998](#1998)
[1994](#1994)
- [](#)
- [2016](#2016)
- [pDTC28](#pdtc28) IncP-1beta1
- [2015](#2015)
- [pAcX50c](#pacx50c)
- [2013](#2013)
- [pDS1](#pds1) IncP-1eta
- [pMAB01](#pmab01) IncP-1beta
- [2012](#2012)
- [MEALZ_p](#mealz_p)
- [2011](#2011)
- [pMCBF1](#pmcbf1) IncP-1zeta
- [2010](#2010)
- [pAKD4](#pakd4) IncP-1delta
- [2007](#2007)
- [pKJK5](#pkjk5) IncP-1epsilon
- [2006](#2006)
- [pBP136](#pbp136) IncP-1beta1
- [pQKH54](#pqkh54) IncP-1gamma
- [pA1](#pa1) IncP-1beta2
- [2004](#2004)
- [pEST4011](#pest4011) IncP-1delta
- [pJP4](#pjp4) IncP-1beta
- [2003](#2003)
- [pB10](#pb10) IncP-1beta1
- [1998](#1998)
- [R751](#r751) IncP-1beta1
- [1994](#1994)
- [RK2](#rk2) IncP-1alpha
- [](#)
- [non-IncP1](#non-IncP1)
[IncA](#)
[IncC](#)
[IncF](#)
[IncK](#)
[IncR](#)
[IncU](#)
[IncW](#)
[PromA](#proma)
- [chromosome](#chromosome)
- [people](#people)
[Shintani](#shintani)
[Suzuki](#suzuki)
[Yano](#yano)
- [](#)
- [theta](#theta) IncP-1theta
- [p7ME01](#p7me01) IncP-1theta
- [database](#database)
  - [plasmidfinder](#plasmidfinder)

----------
## database

### plasmidfinder

https://github.com/genomicepidemiology/plasmidfinder
```
git clone https://bitbucket.org/genomicepidemiology/plasmidfinder_db.git
cd plasmidfinder_db
grep "^>" *.fsa | grep "IncP" | sort

enterobacteriales.fsa:>IncP1_1__BN000925
enterobacteriales.fsa:>IncP1_2__U67194
enterobacteriales.fsa:>IncP1_3__KX377410
enterobacteriales.fsa:>IncP1_4__CM007914
enterobacteriales.fsa:>IncP6_1__JF785550
```

https://www.ncbi.nlm.nih.gov/nuccore/CM007914.1/
```
LOCUS       CM007914               59459 bp    DNA     circular CON 11-MAY-2017
DEFINITION  Escherichia coli strain CH613_eco plasmid unnamed7, complete
            sequence, whole genome shotgun sequence.

                     /isolation_source="Urine, bladder (ub)"
                     /host="Homo sapiens"
                     /geo_loc_name="China: Guangzhou"
                     /collection_date="19-Jun-2015"
                     /collected_by="Peking University People's Hospital,
                     Department of Clinical Laboratory"

     CDS             32946..33824
                     /inference="EXISTENCE: similar to AA
                     sequence:RefSeq:WP_019842081.1"
                     /product="TrfA protein"
                     /protein_id="OTA08989.1"


     CDS             44716..47349
                     /inference="EXISTENCE: similar to AA
                     sequence:RefSeq:WP_004091435.1"
                     /product="conjugal transfer protein TraI"
                     /protein_id="OTA09004.1"

```

Run BLAST
https://blast.ncbi.nlm.nih.gov/Blast.cgi

CM007914
に類似のプラスミド配列は、例えば、以下の通り：

https://www.ncbi.nlm.nih.gov/nucleotide/CP057780.1
Escherichia coli strain RHB15-C18 plasmid pRHB15-C18_3, complete sequence
   PUBMED   34609275

https://www.ncbi.nlm.nih.gov/nucleotide/CP055103.1
Shigella flexneri strain SWHIN_107 plasmid unnamed4, complete sequence
   PUBMED   35078531

### 2021-10

https://pubmed.ncbi.nlm.nih.gov/34609275/
Microb Genom
. 2021 Oct;7(10):000630. doi: 10.1099/mgen.0.000630.
A genomic epidemiological study shows that prevalence of antimicrobial resistance in Enterobacterales is associated with the livestock host, as well as antimicrobial usage
Manal AbuOun 1, Hannah Jones 2, Emma Stubberfield 1, Daniel Gilson 2, Liam P Shaw 3, Alasdair T M Hubbard 3 4, Kevin K Chau 3, Robert Sebra 5, Tim E A Peto 3, Derrick W Crook 3 6, Daniel S Read 7, H Soon Gweon 7 8, A Sarah Walker 3 6, Nicole Stoesser 3 6, Richard P Smith 2, Muna F Anjum 1, On Behalf Of The Rehab Consortium
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000630#tab2
https://pmc.ncbi.nlm.nih.gov/articles/PMC8627209/

https://www.ncbi.nlm.nih.gov/nucleotide/CP057780.1
```
LOCUS       CP057780               41696 bp    DNA     circular BCT 13-MAY-2022
DEFINITION  Escherichia coli strain RHB15-C18 plasmid pRHB15-C18_3, complete
            sequence.

                     /strain="RHB15-C18"
                     /isolation_source="Pooled sheep faecal samples collected
                     from floor of farm"
                     /geo_loc_name="United Kingdom"
                     /collection_date="2017"


     CDS             11631..12509
                     /inference="COORDINATES: similar to AA
                     sequence:RefSeq:WP_020405883.1"
                     /product="replication initiator protein A"
                     /protein_id="QMM50915.1"

```

### 2022-01-25

https://pubmed.ncbi.nlm.nih.gov/35078531/
Microbiome
. 2022 Jan 25;10(1):16. doi: 10.1186/s40168-021-01192-w.
High-resolution genomic surveillance elucidates a multilayered hierarchical transfer of resistance between WWTP- and human/animal-associated bacteria
You Che 1 2, Xiaoqing Xu 1, Yu Yang 1, Karel Břinda 2 3, William Hanage 2, Chao Yang 4, Tong Zhang 5
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-021-01192-w
https://pmc.ncbi.nlm.nih.gov/articles/PMC8790882/

https://www.ncbi.nlm.nih.gov/nucleotide/CP055103.1
```
LOCUS       CP055103               43384 bp    DNA     circular BCT 22-FEB-2022
DEFINITION  Shigella flexneri strain SWHIN_107 plasmid unnamed4, complete
            sequence.

                     /strain="SWHIN_107"
                     /isolation_source="Shek Wu Hui WWTPs_107 WWTPs influent"
                     /geo_loc_name="Hong Kong"
                     /lat_lon="22.5 N 114.1 E"
                     /collection_date="2018-12"

     CDS             5974..6852
                     /inference="COORDINATES: similar to AA
                     sequence:RefSeq:WP_020405883.1"
                     /product="replication initiator protein A"
                     /protein_id="ULK73954.1"

```

----------

### Sen_2013-01

https://pubmed.ncbi.nlm.nih.gov/22936717/
Mol Biol Evol
. 2013 Jan;30(1):154-66. doi: 10.1093/molbev/mss210. Epub 2012 Aug 30.
Inferring the evolutionary history of IncP-1 plasmids despite incongruence among backbone gene trees
Diya Sen 1, Celeste J Brown, Eva M Top, Jack Sullivan
https://academic.oup.com/mbe/article/30/1/154/1020766
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3525142/
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3525142/table/mss210-T1/
Table 1.
General Features of the Plasmids Included in This Study Listed by Subgroup.
```
Plasmids	IncP-1 Subgroup	Origin/Isolation Method/Hosta	Accession Numberb

pA81	IncP-1β	PCB contaminated soil in Czech Republic, Achromobacter xylosoxidans A8	AJ515144
pADP-1	IncP-1β	Soil in the United States, Pseudomonas sp. ADP	NC_004956
```

Introduction

Fig. 1.
Genetic map of a typical IncP-1 plasmid showing the different functional modules:

Phylogenies inferred from single or concatenated backbone genes have shown the IncP-1 group to be composed of five diverse subgroups: IncP-1α, -β, -γ, -δ, and -ε (Pansegrau et al. 1994; Thorsted et al. 1998; Vedler et al. 2004; Haines et al. 2006; Bahl et al. 2007). Recently, three new potential IncP-1 subgroups have been identified: ζ (Norberg et al. 2011), η (Sen D, Yano H, Bauer M, Rogers LM, Van der Auwera GA, Brown C, and Top E, unpublished), and an unnamed subgroup (Pachulec and van der Does 2010), thus indicating the tremendous diversity that exists within this plasmid group. The biological significance of these subgroups is unknown, and more work is required to distinguish between them phenotypically.

Materials and Methods
Plasmids and Genes

The plasmids from the recently proposed ζ, η, and an unnamed subgroups (see Introduction) were not included because their sequences only became publically available in the middle of this study and because of their wide sequence divergence and lack of evidence that they are physically incompatible with the prototype IncP-1 plasmids.

Nucleotide Sequence Alignments and Model Selection
The amino acid sequences of each gene were aligned with ClustalX (Thompson et al. 2002). Tranalign (Rice et al. 2000) was used to align the nucleotide sequences of each gene guided by the aligned amino acid sequences. 

Detection of Recombinants
To detect recombination among the plasmids in the data set, alignment files of the backbone genes were concatenated in the order and orientation in which they appear on IncP-1 plasmids (fig. 1). The recombination detection programs RDP, GENECONV, BootScan, MaxChi, and Chimaera, which are implemented in RDP3 (Martin et al. 2010), were run with default parameters. Only recombinants that were identified by at least two programs were considered.

Fig. 2.
Cladograms showing four topologies produced by 21 gene trees. (A) Topology 1: supported by 46% of gene trees, namely, those of trfA2, trbA, trbC, trbG, traG, traH, traI, kfrA, kfrB, kfrC, korB, korA, and kleE. (B) Topology 2: supported by gene trees of trbD, trbK, and traJ. (C) Topology 3: supported by gene trees of trbF, trbI, and traE. (D) Topology 4: supported by gene trees of korC and klcA. Trees were rooted using IncP-1γ as outgroup.

For the concatenated tree (shown in fig. 3), individual genes were aligned and concatenated in the order in which they appear on IncP-1 plasmids: trfA2, trbA, trbB, trbC, trbD, trbF, trbG, trbI, trbJ, trbK, traD, traE, traF, traG, traH, traI, traJ, traK, traL, kfrC, kfrB, kfrA, korB, incC, korA, kleE, korC, and klcA (fig. 1). The concatenated tree had the same topology as topology 3 described (fig. 2C). It represents the null hypothesis that all genes have a single history (i.e., all gene trees are estimates of a single-gene tree) as would be the case in the absence of recombination, either within or among groups.

Fig. 3.
ML tree of concatenated data estimated from a partitioned analysis based on codon position. Nodal support is shown as nonparametric ML bootstrap values. The tree was rooted using IncP-1γ as outgroup.

----------

----------
## transposase

https://www.ncbi.nlm.nih.gov/protein/AEC19138
```
LOCUS       AEC19138                 687 aa            linear   BCT 31-JAN-2014
DEFINITION  transposase [Pusillimonas sp. T7-7].

     Protein         1..687
                     /product="transposase"
     Region          14..264
                     /region_name="Rep_3"
                     /note="Initiator Replication protein; cl19398"
                     /db_xref="CDD:450297"
     Region          326..675
                     /region_name="transpos_ISL3"
                     /note="ISL3 family transposase; NF033550"
                     /db_xref="CDD:411183"

```

https://pubmed.ncbi.nlm.nih.gov/21622753/
J Bacteriol
. 2011 Aug;193(15):4021-2. doi: 10.1128/JB.05242-11. Epub 2011 May 27.
Complete genome sequence of Pusillimonas sp. T7-7, a cold-tolerant diesel oil-degrading bacterium isolated from the Bohai Sea in China
Boyang Cao 1, Ting Ma, Yan Ren, Yi Ren, Guoqiang Li, Ping Li, Xi Guo, Peng Ding, Lu Feng
https://journals.asm.org/doi/10.1128/jb.05242-11
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3147516/
GENOME ANNOUNCEMENT

----------
## TrfA TraI

https://pubmed.ncbi.nlm.nih.gov/8954881/
Plasmid
. 1996 Sep;36(2):95-111. doi: 10.1006/plas.1996.0037.
Conservation of the genetic switch between replication and transfer genes of IncP plasmids but divergence of the replication functions which are major host-range determinants
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_ingenta_academicpress_ap_pl_1996_00000036_00000002_art00037


https://ncbi.nlm.nih.gov/protein/WP_241190711
```
LOCUS       WP_241190711             282 aa            linear   BCT 11-APR-2022
DEFINITION  plasmid replication initiator TrfA [Pseudomonas chlororaphis].

```

https://www.ncbi.nlm.nih.gov/protein/WP_239990634
```
LOCUS       WP_239990634             603 aa            linear   BCT 05-AUG-2022
DEFINITION  MULTISPECIES: plasmid replication initiator TrfA [unclassified
            Pseudomonas].

     gene            1..603
                     /gene="trfA"
     Protein         1..603
                     /product="plasmid replication initiator TrfA"
                     /calculated_mol_wt=67655
     Region          2..208
                     /region_name="Rep_3"
                     /note="Initiator Replication protein; cl19398"
                     /db_xref="CDD:450297"
     Region          <297..>603
                     /region_name="PRK13878"
                     /note="conjugal transfer relaxase TraI; Provisional"
                     /db_xref="CDD:237539"

```

----------
## theta
```
CP003381 Methylophaga frappieri strain JAM7 plasmid
CP001979 Marinobacter adhaerens HP15 plasmid pHP-42
CP006601 Cycloclasticus zancles 78-ME plasmid p7ME01
```

### 2012-08
IncP-1theta

https://pubmed.ncbi.nlm.nih.gov/22815445/
J Bacteriol
. 2012 Aug;194(15):4126-7. doi: 10.1128/JB.00726-12.
Complete genome sequences of Methylophaga sp. strain JAM1 and Methylophaga sp. strain JAM7
Céline Villeneuve 1, Christine Martineau, Florian Mauffrey, Richard Villemur
https://journals.asm.org/doi/10.1128/JB.00726-12
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3416558/
Nucleotide sequence accession numbers. The genome sequences and annotations of Methylophaga strains JAM1 and JAM7 have been deposited in GenBank under accession number CP003390 for the strain JAM1 chromosome, CP003380 for the strain JAM7 chromosome, and CP003381 for the strain JAM7 plasmid.

https://www.ncbi.nlm.nih.gov/nuccore/CP003381
```
LOCUS       CP003381               47825 bp    DNA     circular BCT 11-AUG-2015
DEFINITION  Methylophaga frappieri strain JAM7 plasmid, complete sequence.

  ORGANISM  Methylophaga frappieri
            Bacteria; Proteobacteria; Gammaproteobacteria; Thiotrichales;
            Piscirickettsiaceae; Methylophaga.

                     /plasmid="unnamed"
```

### 2010-09-28
### pHP-42
IncP-1theta

https://pubmed.ncbi.nlm.nih.gov/21304739/
Stand Genomic Sci
. 2010 Sep 28;3(2):97-107. doi: 10.4056/sigs.922139.
Complete genome sequence of Marinobacter adhaerens type strain (HP15), a diatom-interacting marine microorganism
Astrid Gärdes, Eva Kaeppel, Aamir Shehzad, Shalin Seebah, Hanno Teeling, Pablo Yarza, Frank Oliver Glöckner, Hans-Peter Grossart, Matthias S Ullrich
https://environmentalmicrobiome.biomedcentral.com/articles/10.4056/sigs.922139
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3035377/
Plasmid pHP-42*	  0.042	   circular	  CP001979

Plasmid
https://www.ncbi.nlm.nih.gov/nuccore/CP001979
```
LOCUS       CP001979               42349 bp    DNA     circular BCT 25-JUL-2016
DEFINITION  Marinobacter adhaerens HP15 plasmid pHP-42, complete sequence.

  ORGANISM  Marinobacter adhaerens HP15
            Bacteria; Proteobacteria; Gammaproteobacteria; Pseudomonadales;
            Marinobacteraceae; Marinobacter.

```

Chromosome
https://www.ncbi.nlm.nih.gov/nuccore/CP001978
```
LOCUS       CP001978             4421911 bp    DNA     circular BCT 12-OCT-2011
DEFINITION  Marinobacter adhaerens HP15, complete genome.
```

### 2016-08
### p7ME01
IncP-1theta

https://pubmed.ncbi.nlm.nih.gov/27345842/
Environ Microbiol Rep
. 2016 Aug;8(4):508-19. doi: 10.1111/1758-2229.12424. Epub 2016 Jun 27.
Analysis of defence systems and a conjugative IncP-1 plasmid in the marine polyaromatic hydrocarbons-degrading bacterium Cycloclasticus sp. 78-ME
Michail M Yakimov 1, Francesca Crisafi 1, Enzo Messina 1, Francesco Smedile 1, Anna Lopatina 2, Renata Denaro 1, Dietmar H Pieper 3, Peter N Golyshin 4, Laura Giuliano 1
https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/1758-2229.12424
Get access to the full version of this article.
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_crossref_primary_10_1111_1758_2229_12424
Wiley Online Library Open Access

Two ‘defence islands’ were identified in 78-ME genome: the first harbouring CRISPR-Cas with toxin-antitoxin system, while the second was composed by an array of genes for toxin-antitoxin and restriction-modification proteins. Among all identified spacers of CRISPR-Cas system only seven spacers match sequences of phages and plasmids. Furthermore, a conjugative plasmid p7ME01, which belongs to a new IncP-1θ ancestral archetype without any accessory mobile elements was found in 78-ME. 

Using approach of Norberg et al. (2011), we compared the concatenated backbone regions A, B and C of p7ME01 with corresponding regions of 23 IncP-1 plasmids retrieved from GenBank through BLAST and literature searches. As supported by phylogenetic analysis of the IncP-1 backbones, p7ME01 does belong to a novel clade, hereafter called θ (from the Greek word ‘θαλαθθα or thalassa’, meaning ‘sea’) (del Castillo et al., 2013). Additionally to p7ME01, this novel IncP-1θ clade currently consists of three other closely related plasmids (Fig. 4). Noteworthy, all of them were found in marine biofilm-forming gammaproteobacteria: Alcanivorax hongengensis A-11-3 (Lai and Shao, 2012b), Marinobacter adhaerens HP15 (Gärdes et al., 2010) and Methylophaga frappieri JAM7 (Auclair et al., 2010; Villeneuve et al., 2013). 

![](https://sfamjournals.onlinelibrary.wiley.com/cms/asset/641c8c24-a3b6-4b66-8aec-190bb934cf8f/emi412424-fig-0004-m.png)

Figure 4
Phylogenetic analysis showing the relationship of p7ME01 with other incP‐1 plasmids of the IncP‐1plasmid backbone. Maximum Likelihood tree was inferred from concatenated backbone regions A, B and C of 23 IncP‐1 plasmids belonging to all currently recognized clades (Norberg et al., 2011; del Castillo et al., 2013). Novel IncP‐1θ clade is shaded in grey. Five IncP‐1 plasmids without any accessory mobile elements known so far are highlighted in bold. Sequence of the uncharacterized plasmid MEALZ_p (FD082061) from Methylomicrobum alcaliphilum 20Z was used as an outgroup. The scale bar represents the probability of amino acid substitutions per site.

Two ‘defence islands’ were identified in its genome: one contained CRISPR-Cas and toxin-antitoxin system, while the second was composed of an array of genes for toxin-antitoxin and restriction-modification proteins. Among 116 spacers of CRISPR-Cas system only seven spacers matched phages and plasmids. Additionally, we showed that Cycloclasticus sp. 78-ME harbours a conjugative plasmid p7ME01 of a new IncP-1θ ancestral archetype, which likely suppresses the acquisition of exogenous DNA by this organism. Based on this finding, we suggested that the adaptive immunity of Cycloclasticus sp. 78-ME is linked with the acquisition of this plasmid.


MEALZ_p

https://www.ncbi.nlm.nih.gov/bioproject/PRJNA201013

https://www.ncbi.nlm.nih.gov/assembly/GCA_000442595.1

https://www.ncbi.nlm.nih.gov/genome/17665?genome_assembly_id=301223

https://www.ncbi.nlm.nih.gov/nuccore/CP005996
```
LOCUS       CP005996             2613078 bp    DNA     circular BCT 03-DEC-2015
DEFINITION  Cycloclasticus zancles 78-ME chromosome, complete genome.

            collection_date     :: 2011-06-22
```

https://www.ncbi.nlm.nih.gov/nuccore/CP006601
```
LOCUS       CP006601               42347 bp    DNA     circular BCT 03-DEC-2015
DEFINITION  Cycloclasticus zancles 78-ME plasmid p7ME01, complete sequence.

  ORGANISM  Cycloclasticus zancles 78-ME
            Bacteria; Proteobacteria; Gammaproteobacteria; Thiotrichales;
            Piscirickettsiaceae; Cycloclasticus.

            collection_date     :: 2011-06-22

                     /country="Italy: Arenzano"
                     /lat_lon="44.3667 N 8.7097 E"
                     /altitude="-78 m"

     CDS             complement(2091..3221)
                     /product="TrfA protein"
                     /protein_id="AGS40879.1"

     CDS             3714..4097
                     /product="Conjugal transfer protein TrbA"
                     /protein_id="AGS40881.1"

     CDS             complement(29494..31761)
                     /note="DNA relaxase"
                     /product="Conjugal transfer protein TraI"
                     /protein_id="AGS40912.1"

     CDS             complement(39208..39504)
                     /product="KorA protein"
                     /protein_id="AGS40923.1"

```

----------

### 2012-01
### MEALZ_p

https://www.ncbi.nlm.nih.gov/nuccore/FO082061
```
LOCUS       FO082061              128415 bp    DNA     circular BCT 25-JUL-2016
DEFINITION  Methylomicrobium alcaliphilum str. 20Z plasmid MEALZ_p, complete
            genome.

   PUBMED   22207753

     CDS             complement(29003..30022)
                     /product="putative plasmid replication initiator protein"
                     /protein_id="CCE25738.1"
https://www.ncbi.nlm.nih.gov/protein/CCE25738

     CDS             complement(21758..22615)
                     /product="putative Replication protein"
                     /protein_id="CCE25731.1"
https://www.ncbi.nlm.nih.gov/protein/CCE25731
                     /note="replication initiation protein RepM; NF038290"

     CDS             complement(111304..113505)
                     /gene="traI"
                     /product="Protein traI"
                     /protein_id="CCE25806.1"

```

GENOME ANNOUNCEMENT
https://pubmed.ncbi.nlm.nih.gov/22207753/
J Bacteriol
. 2012 Jan;194(2):551-2. doi: 10.1128/JB.06392-11.
Genome sequence of the haloalkaliphilic methanotrophic bacterium Methylomicrobium alcaliphilum 20Z
Stéphane Vuilleumier 1, Valentina N Khmelenina, Françoise Bringel, Alexandr S Reshetnikov, Aurélie Lajus, Sophie Mangenot, Zoé Rouy, Huub J M Op den Camp, Mike S M Jetten, Alan A Dispirito, Peter Dunfield, Martin G Klotz, Jeremy D Semrau, Lisa Y Stein, Valérie Barbe, Claudine Médigue, Yuri A Trotsenko, Marina G Kalyuzhnaya
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3256673/
Nucleotide sequence accession numbers. The M. alcaliphilum 20Z genome sequence was deposited in GENEMBL under accession numbers FO082060 and FO082061 (project ID 73721).

----------
## mob
relaxase

### Guglielmini_2013
### 2013-02

https://pubmed.ncbi.nlm.nih.gov/22977114/
Mol Biol Evol
. 2013 Feb;30(2):315-31. doi: 10.1093/molbev/mss221. Epub 2012 Sep 13.
Evolution of conjugation and type IV secretion systems
Julien Guglielmini 1, Fernando de la Cruz, Eduardo P C Rocha
https://academic.oup.com/mbe/article/30/2/315/1013981
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3548315/

### Garcillan-Barcia_2009
### 2009-05

https://pubmed.ncbi.nlm.nih.gov/19396961/
Review FEMS Microbiol Rev
. 2009 May;33(3):657-87. doi: 10.1111/j.1574-6976.2009.00168.x.
The diversity of conjugative relaxases and its application in plasmid classification
María Pilar Garcillán-Barcia 1, María Victoria Francia, Fernando de la Cruz
https://academic.oup.com/femsre/article/33/3/657/591359

The MOBQ family

From the first iteration, the family included a number of relaxases that also appeared in the BLAST search querying TraI_RP4 (the MOBP prototype), indicating that MOBQ and MOBP overlap. 

Figure 9

The members of this clade are related to MOBP, because they appeared in a BLAST using TraI_RP4 as a query. The intersection with MOBP only applied to the MOBQ1 clade. The rest of MOBQ relaxases have further evolved so that similarity with MOBP was not longer detected by PSI-BLAST when using the standard threshold. MOBQ1 relaxases are highly divergent; hence, some internal nodes of this subclade change in the ML topology.

The MOBP cluster
A PSI-BLAST search using TraI_RP4 as a query and a score threshold P<10−4 converged after 14 iterations and retrieved 259 relaxase sequences (Tables S4–S6). 

#<img alt="" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsre/33/3/10.1111_j.1574-6976.2009.00168.x/1/m_FMR_168_f12.jpeg?Expires=1620716729&Signature=nyh68fd1LQuuisY7DCnpvuUkiTH1ED2aQjYYpSj-16RJFbzL7fq3U1sKUW6p49lK4cbPVQgDlsJ4d8PIsPIcHu-CIA4wAkO3ORFixWNq9qiNlPyL5FmwD87PJzkRVmkfkvKJR6ChYDUFYnzXJLZ6bAFr5aYCGu6zDdEonzvXxrDF~qxuFqSMnDUXh1oF1yVLuwGo9zGPbZAeYo4~mFI3lff2UTweInCwUf6hTs8Htat0TDb3qU~nWFXZChdxcyJ3mgBnFZLxwAcTD~LIwD8t-U79j0ceXTcK5xYSSWQylauAK1wO6Clt5rG534QqCogxpsT8tZOBg10x68ZBTyhKXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" width=25%>

![](https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/femsre/33/3/10.1111_j.1574-6976.2009.00168.x/1/m_FMR_168_f12.jpeg?Expires=1620716729&Signature=nyh68fd1LQuuisY7DCnpvuUkiTH1ED2aQjYYpSj-16RJFbzL7fq3U1sKUW6p49lK4cbPVQgDlsJ4d8PIsPIcHu-CIA4wAkO3ORFixWNq9qiNlPyL5FmwD87PJzkRVmkfkvKJR6ChYDUFYnzXJLZ6bAFr5aYCGu6zDdEonzvXxrDF~qxuFqSMnDUXh1oF1yVLuwGo9zGPbZAeYo4~mFI3lff2UTweInCwUf6hTs8Htat0TDb3qU~nWFXZChdxcyJ3mgBnFZLxwAcTD~LIwD8t-U79j0ceXTcK5xYSSWQylauAK1wO6Clt5rG534QqCogxpsT8tZOBg10x68ZBTyhKXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA)

Figure 12

Phylogeny tree of MOBP1 relaxases. Codes for underlines, asterisks, boxes, gene synteny and vertical bars are the same as in Fig. 2. Azosp, Azoarcus sp. EbN1; Niteu, Nitrosomonas eutropha; Desps=Desulfotalea psychrophila; Nitmu=Nitrosospira multiformis.

Clade MOBP11 is composed exclusively of conjugative plasmids and has RP4 (IncP1α) and R751 (IncP1β) as prototypes. In fact, the IncP cluster (Heuer et al., 2004; Haines et al., 2006b; Bahl et al., 2007) is broad, and includes probably all plasmids listed in Fig. 12 from RP4 down to ‘plasmid 59 kb’. Thus, the IncP group is a deep, well-populated clade that contains a large number of conjugative plasmids. We propose to call it MOBP111. Because this is probably the most thoroughly analysed plasmid incompatibility group, it serves to underline the spread we can expect of an Inc group when it is analysed exhaustively. Although many other Inc groups are only represented by a single member in this work, each Inc group might be expected to encompass a clade that can reach the extension of IncP or IncF groups.


The conserved motifs in MOBP1 are shown in Fig. 13. As can be seen, MOBP1 relaxases share the classical motifs of MOBF and MOBQ relaxases. The most conspicuous is the 3H motif, which shows the signature H-x-(DE)-T-(DE)-x2-H-x-H-x3-N-x3-P. The argument about the acidic residue in motif II, which was invoked in section The MOBQ family, continues here because MOBP1 relaxases also contain a conserved D/E (E80 in TraI_RP4, signalled by a white arrowhead in Fig. 13) that could play the same role as D85 in TrwC_R388. Site-directed mutagenesis of TraI_RP4 (Pansegrau et al., 1994) defined the three sequence motifs in MOBP relaxases and identified one essential residue in each motif (Y22, S74 and H116). E80 was not tested and, surprisingly, H109 or H118 showed to be nonessential (Fig. 13).

Figure 13

Conserved sequence motifs in the MOBP1 subclades. Vertical black arrowheads and black circles point to residues configuring the catalytic centre. Mutations in TraI_RP4 affecting those residues either affected conjugation (black arrowheads) or did not (black circles). The vertical white arrowhead points to E79 in TraI_RP4, another potential key residue (see The MOBP cluster). *The N-terminal sequences of these relaxases were wrongly annotated (by comparison with homologous relaxases). Visual inspection allowed us to realize that if the annotated CDSs were extended upstream (pRAS3: −57 codons; pTF-FC2: −41 codons; pPNAP06: −13 codons), motif I was found. Azosp, Azoarcus sp. EbN1; Niteu, Nitrosomonas eutropha C91; Desps, Desulfotalea psychrophila LSv54.


Niteu, Nitrosomonas eutropha C91

Desps, Desulfotalea psychrophila LSv54

- Yersinia pseudotuberculosis IP 31758 plasmid p_59kb
- Nitrosomonas eutropha C91 plasmid p1
- Stenotrophomonas maltophilia plasmid pBI1063, partial sequence.
- Desulfotalea psychrophila LSv54, large plasmid.

### 2004-09
https://pubmed.ncbi.nlm.nih.gov/15305914/
Comparative Study Environ Microbiol
. 2004 Sep;6(9):887-902. doi: 10.1111/j.1462-2920.2004.00665.x.
The genome of Desulfotalea psychrophila, a sulfate-reducing bacterium from permanently cold Arctic sediments
R Rabus 1, A Ruepp, T Frickey, T Rattei, B Fartmann, M Stark, M Bauer, A Zibat, T Lombardot, I Becker, J Amann, K Gellner, H Teeling, W D Leuschner, F-O Glöckner, A N Lupas, R Amann, H-P Klenk
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_openaire_primary_doi_dedup_ad0d3bffe10f88898f2ae64d7a383e0f
Wiley Online Library

https://www.ncbi.nlm.nih.gov/nuccore/CR522871
```
LOCUS       CR522871              121587 bp    DNA     circular BCT 27-FEB-2015
DEFINITION  Desulfotalea psychrophila LSv54, large plasmid.


  ORGANISM  Desulfotalea psychrophila LSv54
            Bacteria; Proteobacteria; Deltaproteobacteria; Desulfobacterales;
            Desulfocapsaceae; Desulfotalea.

     CDS             complement(2020..3093)
                     /locus_tag="DPPB01"
                     /product="related to transcriptional repressor protein
                     TrfA"
                     /protein_id="CAG37865.1"

     CDS             complement(44056..45060)
                     /locus_tag="DPPB43"
                     /product="related to TraI protein (partial length)"
                     /protein_id="CAG37907.1"

     CDS             complement(72794..73684)
                     /locus_tag="DPPB57"
                     /product="related to TraI protein (partial length)"
                     /protein_id="CAG37921.1"

```

### pBI1063

https://www.ncbi.nlm.nih.gov/nuccore/AY299014
```
LOCUS       AY299014               23483 bp    DNA     linear   BCT 25-JUL-2016
DEFINITION  Stenotrophomonas maltophilia plasmid pBI1063, partial sequence.

  ORGANISM  Stenotrophomonas maltophilia
            Bacteria; Proteobacteria; Gammaproteobacteria; Xanthomonadales;
            Xanthomonadaceae; Stenotrophomonas; Stenotrophomonas maltophilia
            group.

     CDS             complement(2204..4675)
                     /gene="traI"
                     /codon_start=1
                     /transl_table=11
                     /product="putative transfer protein"
                     /protein_id="AAP82040.1"


     CDS             complement(11327..12295)
                     /gene="repA"
                     /codon_start=1
                     /transl_table=11
                     /product="putative replication protein"
                     /protein_id="AAP82047.1"

```

### 2007-12
### Stein_2007-12

https://pubmed.ncbi.nlm.nih.gov/17991028/
Environ Microbiol
. 2007 Dec;9(12):2993-3007. doi: 10.1111/j.1462-2920.2007.01409.x.
Whole-genome analysis of the ammonia-oxidizing bacterium, Nitrosomonas eutropha C91: implications for niche adaptation
Lisa Y Stein 1, Daniel J Arp, Paul M Berube, Patrick S G Chain, Loren Hauser, Mike S M Jetten, Martin G Klotz, Frank W Larimer, Jeanette M Norton, Huub J M Op den Camp, Maria Shin, Xueming Wei
https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/j.1462-2920.2007.01409.x
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_osti_scitechconnect_931233

Multiple gene‐shuffling events are apparent, including mobilized and replicated transposition, as well as plasmid or phage integration events into the 2.66 Mbp chromosome and two plasmids (65 and 56 kbp) of N. eutropha C91. A 117 kbp genomic island encodes multiple genes for heavy metal resistance, including clusters for copper and mercury transport, which are absent from the genomes of other ammonia‐oxidizing bacteria (AOB). 

The N. eutropha C91 genome is composed of a single circular chromosome (2 661 057 bp, 48.5% G + C content) and two large plasmids, pNeutP1 (65 132 bp, 49.8% G + C) and pNeutP2 (55 635 bp, 49.7% G + C) (Table 1). The two plasmids each contain an identical 11 414 bp (56.1% G + C) transposon (Tn_Neut) flanked by short inverted repeats (Table 2).

The chromosome and the 65 and 56 kbp plasmids of N. eutropha (pNeutP1 and pNeutP2, respectively) can be found in the GenBank/EMBL/DDBJ database under accession numbers CP000450, CP000451 and CP000452 respectively.

https://www.ncbi.nlm.nih.gov/nuccore/CP000451
```
LOCUS       CP000451               65132 bp    DNA     circular BCT 27-SEP-2019
DEFINITION  Nitrosomonas eutropha C91 plasmid p1, complete sequence.

  ORGANISM  Nitrosomonas eutropha C91
            Bacteria; Proteobacteria; Betaproteobacteria; Nitrosomonadales;
            Nitrosomonadaceae; Nitrosomonas.

     CDS             29038..30006
                     /locus_tag="Neut_2557"
                     /note="PFAM: TrfA family protein;
                     KEGG: reu:Reut_D6499 TrfA"
                     /product="TrfA family protein"
                     /protein_id="ABI60760.1"

     CDS             50171..52219
                     /locus_tag="Neut_2579"
                     /note="PFAM: Relaxase/mobilization nuclease family
                     protein;
                     KEGG: bms:BRA0373 TraI protein, putative"
                     /product="Relaxase/mobilization nuclease family protein"
                     /protein_id="ABI60782.1"

```

### Eppinger_2007-08
### pYpsIP31758.2
### p_59kb

https://pubmed.ncbi.nlm.nih.gov/17784789/
Comparative Study PLoS Genet
. 2007 Aug;3(8):e142. doi: 10.1371/journal.pgen.0030142. Epub 2007 Jul 10.
The complete genome sequence of Yersinia pseudotuberculosis IP31758, the causative agent of Far East scarlet-like fever
Mark Eppinger 1, M J Rosovitz, Wolfgang Florian Fricke, David A Rasko, Galina Kokorina, Corinne Fayolle, Luther E Lindler, Elisabeth Carniel, Jacques Ravel
https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.0030142
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC1959361/
The mobile genome pool includes two novel plasmids phylogenetically unrelated to all currently reported Yersinia plasmids.

General Genome Features
The genome of Y. pseudotuberculosis IP31758 consists of a circular chromosome of 4,723,306 bp (Figure 1A) and two novel plasmids called pYpsIP31758.1 (153,140 bp; Figure 1B) and pYpsIP31758.2 (58,679 bp; Figure 1C).

pYpsIP31758.1 appears to lack a complete plasmid transfer system; however, such a system is present on the smaller plasmid pYpsIP31758.2 (Figure 1C). The pYpsIP31758.2 transfer system is most similar to that of the Pseudomonas species IncP-1beta group pB3 plasmids [94,95], and may also provide the transfer function for the large plasmid. pYpsIP31758.2 is replicated and maintained through a kil/kor system. Such a mechanism has not previously been reported in Yersinia, nor has the incompatibility surface exclusion protein also found on pYpsIP31758.2 (YpsIP31758_A0016) [96,97].

https://www.ncbi.nlm.nih.gov/nuccore/CP000718
```
LOCUS       CP000718               58679 bp    DNA     circular BCT 27-SEP-2019
DEFINITION  Yersinia pseudotuberculosis IP 31758 plasmid p_59kb, complete

   PUBMED   17784789

     CDS             5634..6002
                     /locus_tag="YpsIP31758_A0046"
                     /note="identified by match to protein family HMM PF01381"
                     /codon_start=1
                     /transl_table=11
                     /product="conjugal transfer protein TrbA"
                     /protein_id="ABS45633.1"

     CDS             complement(47397..50018)
                     /locus_tag="YpsIP31758_A0026"
                     /note="identified by match to protein family HMM PF03432"
                     /codon_start=1
                     /transl_table=11
                     /product="conjugal transfer protein TraI"
                     /protein_id="ABS45618.1"

     CDS             complement(55522..56574)
                     /locus_tag="YpsIP31758_A0037"
                     /note="identified by similarity to SP:P07674; match to
                     protein family HMM PF02195; match to protein family HMM
                     PF06613; match to protein family HMM TIGR00180"
                     /codon_start=1
                     /transl_table=11
                     /product="transcription repressor protein KorB"
                     /protein_id="ABS45584.1"

```


----------

## Neisseria gonorrhoeae

### Baines_2020-12
### pAUSMDU00010541_01

https://pubmed.ncbi.nlm.nih.gov/33180013/
Microb Genom
. 2020 Dec;6(12). doi: 10.1099/mgen.0.000471. Epub 2020 Nov 12.
Complete microbial genomes for public health in Australia and the Southwest Pacific
Sarah L Baines 1, Anders Gonçalves da Silva 2, Glen P Carter 1, Amy Jennison 3, Irani Rathnayake 3, Rikki M Graham 3, Vitali Sintchenko 4 5, Qinning Wang 5, Rebecca J Rockett 4 5, Verlaine J Timms 4 5, Elena Martinez 5, Susan Ballard 2, Takehiro Tomita 2, Nicole Isles 2, Kristy A Horan 2, William Pitchers 2, Timothy P Stinear 1, Deborah A Williamson 2 1, Benjamin P Howden 2 1, Torsten Seemann 1 2, Communicable Diseases Genomics 
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000471
Table 1.

Neisseria gonorrhoeae	AUSMDU00010541_02 (p01)	41 998 bp

plasmid
https://www.ncbi.nlm.nih.gov/nuccore/CP045833
```
LOCUS       CP045833               41998 bp    DNA     circular BCT 12-NOV-2019
DEFINITION  Neisseria gonorrhoeae strain AUSMDU00010541 plasmid
            pAUSMDU00010541_01, complete sequence.

                     /host="Homo sapiens"
                     /country="Australia"
                     /collection_date="2017"

     CDS             5219..7612
                     /product="relaxase"
                     /protein_id="QGG73817.1"


     CDS             39820..40698
                     /product="TrfA family protein"
                     /protein_id="QGG73851.1"

```

chromosome
https://www.ncbi.nlm.nih.gov/nuccore/CP045832
```
LOCUS       CP045832             2174817 bp    DNA     circular BCT 12-NOV-2019
DEFINITION  Neisseria gonorrhoeae strain AUSMDU00010541 chromosome, complete
            genome.
```

https://www.ncbi.nlm.nih.gov/biosample/SAMN10920452
https://www.ncbi.nlm.nih.gov/assembly/GCA_009650075

### pConjugative
### PRJDB5915
https://www.ncbi.nlm.nih.gov/bioproject/PRJDB5915
Whole genome comparison of four ceftriaxone resistant Neisseria gonorrhoeae isolated in Japan during 2015-2016

Four ceftriaxone resistant Neisseria gonorrhoae were isolated from clinical gonorrhea patients. Two of them were isolated intermittently from the same patient. The whole genomes of the strains were compared to find accumulation of the sequence variants through transmission and long-term carriage. 

Submission	Registration date: 14-Dec-2018
Department of Bacteriology, National Institute of Infectious Diseases

https://www.ncbi.nlm.nih.gov/assembly/GCA_004001185
https://www.ncbi.nlm.nih.gov/nuccore/AP018386
```
LOCUS       AP018386               39058 bp    DNA     circular BCT 13-DEC-2018
DEFINITION  Neisseria gonorrhoeae plasmid pConjugative FC498 DNA, complete

                     /host="Homo sapiens"
                     /collection_date="2015"

                     /product="TrfA protein"
```

https://www.ncbi.nlm.nih.gov/assembly/GCA_004001165
https://www.ncbi.nlm.nih.gov/nuccore/AP018382
```
LOCUS       AP018382               39058 bp    DNA     circular BCT 13-DEC-2018
DEFINITION  Neisseria gonorrhoeae plasmid pConjugative FC460 DNA, complete
```

https://www.ncbi.nlm.nih.gov/assembly/GCA_004001105
https://www.ncbi.nlm.nih.gov/nuccore/AP018378
```

LOCUS       AP018378               39058 bp    DNA     circular BCT 13-DEC-2018
DEFINITION  Neisseria gonorrhoeae plasmid pConjugative FC428 DNA, complete
```

### Miyake_2020-09-24
### pConjugative
### PRJDB8572
https://www.ncbi.nlm.nih.gov/bioproject/PRJDB8572
https://www.ncbi.nlm.nih.gov/assembly/GCA_008065015.2
https://www.ncbi.nlm.nih.gov/nuccore/AP019854
```
LOCUS       AP019854               39057 bp    DNA     circular BCT 18-NOV-2020
DEFINITION  Neisseria gonorrhoeae SS3160 plasmid pConjugative DNA, complete
            genome.

                     /isolation_source="human urine"
                     /host="Homo sapines"
                     /db_xref="taxon:485"
                     /plasmid="pConjugative"
                     /country="Japan:Tokyo"
                     /collection_date="2017"


```

https://pubmed.ncbi.nlm.nih.gov/32972945/
Microbiol Resour Announc
. 2020 Sep 24;9(39):e00886-20. doi: 10.1128/MRA.00886-20.
Complete Genome Sequence of Ceftriaxone-Resistant Neisseria gonorrhoeae SS3160, Isolated in Tokyo, Japan
Hirofumi Miyake 1, Morika Mitobe 2, Hiroaki Kubota 2, Hiroki Takano 2, Keiko Yokoyama 2, Kenji Sadamasu 2
https://journals.asm.org/doi/10.1128/MRA.00886-20
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7516156/
It carries two plasmids, pConjugative (39,057 bp) and pCryptic (4,207 bp).

Three circular contigs, corresponding to the chromosome (2,214,955 bp) and two plasmids, identified as pConjugative (39,057 bp) and pCryptic (4,207 bp), were obtained by Unicycler without additional analysis, including overlap trimming.

### Cherdtrakulkiat_2020-05-07
### p1

https://pubmed.ncbi.nlm.nih.gov/32381609/
Microbiol Resour Announc
. 2020 May 7;9(19):e00198-20. doi: 10.1128/MRA.00198-20.
Complete Genome Sequences of Three Neisseria gonorrhoeae Isolates from Thailand with Multidrug Resistance and Multilocus Sequence Type 1903
Thitima Cherdtrakulkiat 1 2 3, Thidathip Wongsurawat 4, Piroon Jenjaroenpun 4, Sawannee Sutheeworapong 5, Wanna Leelawiwat 1 2, Andrew C Hickey 1 2, Eileen F Dunne 1 2, Boonyos Raengsakulrach 1 2, Chanwit Tribuddharat 6
https://mra.asm.org/content/9/19/e00198-20
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7206487/
This study emphasizes the usefulness of combining ONT and Illumina sequencing data for completing circular plasmids. All isolates contain three circular plasmids each. We detected plasmid 1 as a conjugative plasmid and plasmid 3 as a cryptic plasmid.

plasmid p1
https://www.ncbi.nlm.nih.gov/nuccore/CP043872
```
LOCUS       CP043872               41355 bp    DNA     circular BCT 11-MAY-2020
DEFINITION  Neisseria gonorrhoeae strain NG196 plasmid p1, complete sequence.

                     /isolation_source="Urethral discharge"
                     /host="Homo sapiens"
                     /country="Thailand: Bangkok"
                     /lat_lon="13.757925 N 100.485849 E"
                     /collection_date="07-May-2009"
                     /collected_by="Faculty of Medicine Siriraj Hospital,
                     Mahidol University"

```

chromosome
https://www.ncbi.nlm.nih.gov/nuccore/CP043871
```
LOCUS       CP043871             2218879 bp    DNA     circular BCT 11-MAY-2020
DEFINITION  Neisseria gonorrhoeae strain NG196 chromosome, complete genome.
```

### Chen_2011-04
### pNGTCDC08107

https://pubmed.ncbi.nlm.nih.gov/21257765/
J Bacteriol
. 2011 Apr;193(7):1788-9. doi: 10.1128/JB.00007-11. Epub 2011 Jan 21.
Draft genome sequence of a dominant, multidrug-resistant Neisseria gonorrhoeae strain, TCDC-NG08107, from a sexual group at high risk of acquiring human immunodeficiency virus infection and syphilis
Chun-Chen Chen 1, Ko-Chiang Hsia, Chung-Ter Huang, Wing-Wai Wong, Muh-Yong Yen, Lan-Hui Li, Kun-Yen Lin, Kuo-Wei Chen, Shu-Ying Li
https://jb.asm.org/content/193/7/1788.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3067668/
We also isolated a 39,054-bp circular plasmid, pNGTCDC08107, whose sequence was ∼3 kb shorter than but nearly identical to that of pEP5289 (GenBank accession no. GU479466), which has a Dutch-type backbone (7).

https://www.ncbi.nlm.nih.gov/nuccore/CP002441
```
LOCUS       CP002441               39054 bp    DNA     circular BCT 25-JUL-2016
DEFINITION  Neisseria gonorrhoeae TCDC-NG08107 plasmid pNGTCDC08107, complete

                     /isolation_source="17 year-old male"
                     /host="Homo sapiens"
                     /country="Taiwan: Taipei"
                     /collection_date="May-2008"

     CDS             2..2395
                     /locus_tag="NGTW08_p0001"
                     /codon_start=1
                     /transl_table=11
                     /product="TraI"
                     /protein_id="ADV09131.1"

     CDS             31659..32537
                     /locus_tag="NGTW08_p0035"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfA"
                     /protein_id="ADV09165.1"

```

### Pachulec_2010-04-01
### pEP5289

https://pubmed.ncbi.nlm.nih.gov/20376355/
PLoS One
. 2010 Apr 1;5(4):e9962. doi: 10.1371/journal.pone.0009962.
Conjugative plasmids of Neisseria gonorrhoeae
Emilia Pachulec 1, Chris van der Does
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0009962
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848598/
Many clinical isolates of the human pathogen Neisseria gonorrhoeae contain conjugative plasmids. The host range of these plasmids is limited to Neisseria species, 
Analysis of the backbone sequences showed that they belong to a novel IncP1 subfamily divergent from the IncP1α, β, γ, δ and ε subfamilies. 
The presence of the toxin and antitoxin on different plasmids might explain why the host range of this IncP1 plasmid is limited to Neisseria species. 

Results

Complete sequencing of the plasmid (pEP5289) resulted in a sequence of 42.004 bp. 

Figure 1
Circular map of the Neisserial conjugative plasmid (pEP5289) with the Dutch type tetM determinant and a Dutch type backbone.

The conjugative plasmids with the Dutch type backbone are phylogenetically divergent from the other IncP1 sub-families
The modular arrangement of the conjugative plasmid and derivates thereof, the absence of several genes often found in the IncP1-α subfamily, and an insertion of an additional module suggested that these plasmids are members of the IncP1-β sub-family. To verify placement of the plasmids in the IncP1-β sub-family, a phylogenetic analysis was performed on the TraI and TraG proteins encoded in the Tra region, the TrbE and TrbC proteins encoded in the Trb region, and the TrfA and KorB proteins encoded within the Rep and Ctl regions. Neighbor-joining tree containing representative members of the different IncP1 plasmid subfamilies (α, RP4; β, pADP-1; γ, QKH54; δ, pEST4011 and ε, pKJK5) and the neisserial conjugative plasmids with the Dutch type backbone were created (See figure 3). Remarkably this showed that these neisserial conjugative plasmids fall within the IncP1 family, but are phylogenetically divergent from the other IncP1 sub-families, and are members of a new subfamily.

Figure 3
Phylogenetic analysis of genes of different IncP1 plasmids.
The TraI (A), TrbE (B), TraG (C) and TrfA (D) proteins of the neisserial conjugative plasmids and of representative members of different IncP1 plasmid subfamilies (α, RP4; β, pADP-1; γ, QKH54; δ, pEST4011 and ε, pKJK5) representative for different regions of the conjugative plasmids were used to create phylogenetic trees of the IncP1 plasmid family. (For details see materials and methods).

Figure 4
Comparison of the oriT sequences of different members of the IncP1 subfamilies.

Figure 5
Comparison of the oriV iterons of different members of the IncP1 subfamilies.

The sequence of the conjugative plasmid with the Dutch type tetM determinant (pEP5289), and of the genetic load regions of the conjugative (pEP5233) and the plasmid with the American type tetM determinant (pEP5050) were deposited under genbank numbers GU479464, GU479465 and GU479466.


https://www.ncbi.nlm.nih.gov/nuccore/GU479464
```
LOCUS       GU479464                7825 bp    DNA     linear   BCT 25-JUL-2016
DEFINITION  Neisseria gonorrhoeae strain 5050 plasmid pEP5050 genetic load
            region genomic sequence.
```

https://www.ncbi.nlm.nih.gov/nuccore/GU479465
```
LOCUS       GU479465                3997 bp    DNA     linear   BCT 25-JUL-2016
DEFINITION  Neisseria gonorrhoeae strain 5233 plasmid pEP5233 genetic load
            region genomic sequence.
```

https://www.ncbi.nlm.nih.gov/nuccore/GU479466
```
LOCUS       GU479466               42004 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Neisseria gonorrhoeae strain 5289 plasmid pEP5289, complete

  ORGANISM  Neisseria gonorrhoeae
            Bacteria; Proteobacteria; Betaproteobacteria; Neisseriales;
            Neisseriaceae; Neisseria.

                     /note="Dutch type tetM conjugative plasmid; clinical
                     isolate"

     CDS             complement(6730..8664)
                     /note="tetracyclin resistance protein"
                     /product="TetM"
                     /protein_id="ADF36634.1"

     CDS             complement(25111..26109)
                     /note="VirB11-like conjugal transfer ATPase"
                     /product="TrbB"
                     /protein_id="ADF36655.1"

     CDS             complement(26238..26603)
                     /note="transcriptional repressor"
                     /product="TrbA"
                     /protein_id="ADF36656.1"

     CDS             26663..27046
                     /note="putative single standed DNA binding protein"
                     /codon_start=1
                     /transl_table=11
                     /product="Ssb"

     CDS             27050..27928
                     /note="oriV activator protein; plasmid replication"
                     /product="TrfA"
                     /protein_id="ADF36658.1"

     CDS             34447..36840
                     /note="DNA relaxase"
                     /product="TraI"
                     /protein_id="ADF36669.1"

```

----------
## basal

### Stormo_2020-03
### CP038019

https://www.ncbi.nlm.nih.gov/genome/84691?genome_assembly_id=701726

https://www.ncbi.nlm.nih.gov/nuccore/CP038018
```
LOCUS       CP038018             1953244 bp    DNA     circular BCT 04-MAY-2022
DEFINITION  Eikenella exigua strain PXX chromosome, complete genome.

```

https://www.ncbi.nlm.nih.gov/nuccore/CP038019
```
LOCUS       CP038019               39819 bp    DNA     circular BCT 19-AUG-2019
DEFINITION  Eikenella exigua strain PXX plasmid unnamed1, complete sequence.

  ORGANISM  Eikenella exigua
            Bacteria; Proteobacteria; Betaproteobacteria; Neisseriales;
            Neisseriaceae; Eikenella.

                     /isolation_source="brain abscess"
                     /host="Homo sapiens"

                     /country="Norway: Bergen"
                     /lat_lon="60.37 N 5.36 E"
                     /collection_date="Oct-2018"

     CDS             2300..4408
                     /product="relaxase"
                     /protein_id="QED92985.1"


     CDS             complement(32226..33224)
                     /gene="trbB"
                     /product="P-type conjugative transfer ATPase TrbB"
                     /protein_id="QED93013.1"

     CDS             complement(33368..33739)
                     /product="helix-turn-helix domain-containing protein"
                     /protein_id="QED93014.1"

     CDS             33798..34184
                     /product="single-stranded DNA-binding protein"
                     /protein_id="QED93015.1"



     CDS             34265..35137
                     /product="TrfA family protein"
                     /protein_id="QED93016.1"

```

https://pubmed.ncbi.nlm.nih.gov/31693476/
Int J Syst Evol Microbiol
. 2020 Mar;70(3):1478-1488. doi: 10.1099/ijsem.0.003827. Epub 2019 Nov 6.
Eikenella exigua sp. nov., isolated from brain abscess and blood
Kristin A Stormo 1 2, Randi M Nygaard 1, Torbjørn S Bruvold 1, Grete Dimmen 1, Paul Christoffer Lindemann 1, Stina Jordal 2, Øyvind Kommedal 1
https://www.microbiologyresearch.org/content/journal/ijsem/10.1099/ijsem.0.003827#tab2

Author Notes
The GenBank accession numbers for the complete whole-genome sequence are CP038018 (chromosome) and CP038019 (plasmid). The GenBank accession number for the for the full-length 16S rRNA gene sequence is MK541038.

Strain PXXT also contained a single 39 819 bp plasmid encoding 19 genes. Except for genes related to the conjugation system and a toxin–antitoxin system, only hypothetical proteins were found on the plasmid. The plasmid was not found in E. exigua strain EI-02, but a homologous gen-segment was detected in two GenBank whole-genome projects, one in the E. exigua cluster (NML01-A-08, 98.9 % homology) and the other probably representing an undescribed Eikenella species (NML130454, 94.8 % homology). The plasmid was not found in any of the study E. corrodens isolates nor in any E. corrodens GenBank whole-genome projects.


### pIPMB12
```
DEFINITION  Orbus sp. IPMB12 plasmid pIPMB12, complete sequence.
```
https://www.ncbi.nlm.nih.gov/nuccore/CP050254
```
LOCUS       CP050254               32446 bp    DNA     circular BCT 25-MAR-2020
DEFINITION  Zophobihabitans entericus strain IPMB12 plasmid pIPMB12, complete

  ORGANISM  Zophobihabitans entericus
            Bacteria; Proteobacteria; Gammaproteobacteria; Orbales; Orbaceae;
            Zophobihabitans.

                     /country="Taiwan"
                     /collection_date="2013-05-09"

     CDS             complement(22267..24456)
                     /product="relaxase/mobilization nuclease domain-containing
                     protein"
                     /protein_id="QIQ22530.1"

     CDS             complement(30709..31527)
                     /product="TrfA protein"
                     /protein_id="QIQ22540.1"

```

### pKAM329_2
https://www.ncbi.nlm.nih.gov/nuccore/AP024138
```
LOCUS       AP024138               49174 bp    DNA     circular BCT 31-OCT-2020
DEFINITION  Aeromonas caviae KAM329 plasmid pKAM329_2 DNA, complete sequence.

  ORGANISM  Aeromonas caviae
            Bacteria; Proteobacteria; Gammaproteobacteria; Aeromonadales;
            Aeromonadaceae; Aeromonas.

  JOURNAL   Submitted (29-OCT-2020) Contact:Masato Suzuki National Institute of
            Infectious Diseases; 4-2-1 Aobacho, Higashimurayama, Tokyo
            189-0002, Japan


                     /country="Japan:Kanagawa"
                     /collection_date="2018"

                     /product="TrfA family protein"
                     /protein_id="BCM78151.1"


                     /product="conjugal transfer relaxase TraI"
                     /protein_id="BCM78168.1"

```

----------
### pXOOYN24
https://www.ncbi.nlm.nih.gov/nuccore/CP018090
```
LOCUS       CP018090               40857 bp    DNA     circular BCT 10-DEC-2018
DEFINITION  Xanthomonas oryzae pv. oryzae strain YN24 plasmid pXOOYN24,
            complete sequence.

  ORGANISM  Xanthomonas oryzae pv. oryzae
            Bacteria; Proteobacteria; Gammaproteobacteria; Xanthomonadales;
            Xanthomonadaceae; Xanthomonas.

                     /country="China: Yunan"
                     /collection_date="2003"

     CDS             complement(15506..16555)
                     /product="TrfA protein"
                     /protein_id="AZK89864.1"

     CDS             complement(40646..40846)
                     /pseudo
```

----------
### pL21564-1
https://www.ncbi.nlm.nih.gov/nuccore/CP016850
```
LOCUS       CP016850               99718 bp    DNA     circular BCT 11-AUG-2016
DEFINITION  Pseudomonas sp. TMW 2.1634 plasmid pL21564-1, complete sequence.

                     /country="Germany"

     CDS             64078..64950
                     /product="TrfA protein"
                     /protein_id="AOA09234.1"

     CDS             87306..89822
                     /product="hypothetical protein"
                     /protein_id="AOA09225.1"

```



----------
### Andersen_2013-12-05
### pJJ1886_4

https://www.ncbi.nlm.nih.gov/nuccore/CP006788
```
LOCUS       CP006788               55956 bp    DNA     circular BCT 26-JUL-2016
DEFINITION  Escherichia coli JJ1886 plasmid pJJ1886_4, complete sequence.

     CDS             complement(767..3379)
                     /product="conjugal transfer protein TraI"
                     /protein_id="AGY87903.1"

     CDS             complement(21361..22254)
                     /product="hypothetical protein"
                     /protein_id="AGY87930.1"

```

https://pubmed.ncbi.nlm.nih.gov/24309736/
Genome Announc
. 2013 Dec 5;1(6):e00988-13. doi: 10.1128/genomeA.00988-13.
Complete Genome Sequence of the Epidemic and Highly Virulent CTX-M-15-Producing H30-Rx Subclone of Escherichia coli ST131
Paal S Andersen 1, Marc Stegger, Maliha Aziz, Tania Contente-Cuomo, Henry S Gibbons, Paul Keim, Evgeni V Sokurenko, James R Johnson, Lance B Price
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3853059/
Nucleotide sequence accession numbers. The complete sequences of the chromosome of E. coli JJ1886 and its five plasmids, pJJ1886-1 through pJJ1886-5, have been deposited in GenBank (accession numbers CP006784, CP006785, CP006786, CP006787, CP006788, and CP006789, respectively).



----------
### CP082959

https://www.ncbi.nlm.nih.gov/nuccore/CP082959
```
LOCUS       CP082959               52879 bp    DNA     circular BCT 14-SEP-2021
DEFINITION  Salmonella enterica subsp. arizonae strain S499 plasmid unnamed5,
            complete sequence.

                     /country="China: Wuhan"
                     /collection_date="2020-06-20"


     CDS             14598..17231
                     /locus_tag="K8I78_26780"
                     /inference="COORDINATES: similar to AA
                     sequence:RefSeq:WP_003019746.1"
                     /note="Derived by automated computational analysis using
                     gene prediction method: Protein Homology."
                     /codon_start=1
                     /transl_table=11
                     /product="relaxase/mobilization nuclease domain-containing
                     protein"
                     /protein_id="UAM88301.1"

     CDS             51812..52690
                     /locus_tag="K8I78_26980"
                     /inference="COORDINATES: similar to AA
                     sequence:RefSeq:WP_020405883.1"
                     /note="Derived by automated computational analysis using
                     gene prediction method: Protein Homology."
                     /codon_start=1
                     /transl_table=11
                     /product="replication initiator protein A"
                     /protein_id="UAM88285.1"

```

----------
## MCR
----------

----------
### 2023-11-02

https://pubmed.ncbi.nlm.nih.gov/37917755/
PLoS One
. 2023 Nov 2;18(11):e0293940. doi: 10.1371/journal.pone.0293940. eCollection 2023.
Comparative genome analysis of colistin-resistant Escherichia coli harboring mcr isolated from rural community residents in Ecuador and Vietnam
Hoa Thi Thanh Hoang 1, Mayumi Yamamoto 1 2, Manuel Calvopina 3, Carlos Bastidas-Caldes 3, Diep Thi Khong 4, Thang Nam Nguyen 4, Ryuji Kawahara 5, Takahiro Yamaguchi 5, Yoshimasa Yamamoto 1
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0293940
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10621974/
In this study, we obtained a total of 84 complete E. coli genomes (S1 and S2 Tables).
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10621974/#pone.0293940.s006

S1 Table
Genome of the colistin-resistant E. coli isolates from healthy residents in Ecuador.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10621974/bin/pone.0293940.s006.zip

S2 Table
Genome of the colistin-resistant E. coli isolates from healthy residents in Vietnam.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10621974/bin/pone.0293940.s007.zip

.tif画像ファイル。。

----------
### 2021-02-15

https://pubmed.ncbi.nlm.nih.gov/33671955/
Microorganisms
. 2021 Feb 15;9(2):398. doi: 10.3390/microorganisms9020398.
Mobile Colistin Resistance Gene mcr- 1 Detected on an IncI2 Plasmid in Salmonella Typhimurium Sequence Type 19 from a Healthy Pig in South Korea
Dong Chan Moon 1, Su-Jeong Kim 1, Abraham Fikru Mechesso 1, Hee Young Kang 1, Hyun-Ju Song 1, Ji-Hyun Choi 1, Soon-Seek Yoon 1, Suk-Kyung Lim 1
https://www.mdpi.com/2076-2607/9/2/398
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7919004/
The mcr-1 gene was so far associated with diverse plasmids belonging to the IncI2, IncHI1, IncHI2, IncP1, IncX4, IncFII, and IncY replicon types. 

----------
### 2021-01-03
### Ma_2021-01-03
### pHNEP124

https://pubmed.ncbi.nlm.nih.gov/33401636/
Pathogens
. 2021 Jan 3;10(1):33. doi: 10.3390/pathogens10010033.
Rapid Increase in the IS 26-Mediated cfr Gene in E. coli Isolates with IncP and IncX4 Plasmids and Co-Existing cfr and mcr-1 Genes in a Swine Farm
Zhenbao Ma 1 2 3, Jiao Liu 1 2 3, Lin Chen 1 2 3, Xiaoqin Liu 1 2 3, Wenguang Xiong 1 2 3, Jian-Hua Liu 1 2 3, Zhenling Zeng 1 2 3
https://www.mdpi.com/2076-0817/10/1/33/htm
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7823714/
Sequence analysis suggested that the plasmids pHNEP124 and pHNEP129 co-harboring the cfr and mcr-1 genes belonged to the plasmids IncP plasmid and IncX4 plasmid, respectively.

2.5. Plasmid Analysis of pHNEP124
The sequence analysis revealed that plasmid pHNEP124 is 60430 base pairs (bp) in size with an average GC content of 47.01% and belonged to plasmid pMCR_1511-like IncP type plasmid, which consisted of a typical plasmid backbone and two mosaic variant regions. The backbone of plasmid pHNEP124 is comprised of the trfA encoding plasmid replication initiation protein, two par modules for plasmid partitioning, a toxin-antitoxin higA-B system, and a “KlcA-kleE” region encoding a host-lethal protein for plasmid maintenance and stability, as well as the two conjugative regions tra (~17.3-kb) and trb (~12.7-kb) for plasmid horizontal transfer. BLASTn analysis indicated that the backbone of the plasmid pHNEP124 shared a high identity (>99%) with those mcr-1-carrying IncP type plasmids found in Enterobacteriaceae, such as E. coli plasmid pHNGDF36-1 (Genbank accession no. MF978389), Klebsiella pneumoniae pMCR_1511 (Genbank accession no. KX377410), Salmonella enterica serovar Typhimurium pMCR16_P053 (Genbank accession no. KY352406), and Citrobacter braakii pSCC4 (Genbank accession no. CP021078), which are obtained from diverse sources such as fish products, hospital sewage, chickens, and humans. Moreover, it also showed high homology (>99%) to IncP plasmid pHNFP671 (Genbank accession no. KP324830) which carries the cfr gene and was isolated from swine feces in Guangdong Province, China. A comparative analysis of the plasmid pHNEP124 and other IncP plasmids indicated that IncP type plasmids have a conserved backbone structure, except plasmid pSCC4, which is missing an ~8.4 kb conjugative region (Figure 3A). In addition, plasmid pHNFP671, with ~35.5-kb insertion, contained a conjugative region and a cfr-carrying module (Figure 3A).

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7823714/bin/pathogens-10-00033-g003.jpg)
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7823714/figure/pathogens-10-00033-f003/
Figure 3
Liner comparison of the cfr-carrying plasmids. Arrows indicate the positions of the genes and the direction. Regions with >99% homology are shaded in gray. ∆ indicates a truncated gene or mobile element. (A) Comparative analysis of plasmid pHNEP124 and other IncP-type plasmids.

4.7. Nucleotide Sequence Accession Number
The nucleotide sequences of plasmids pHNEP124 and pHNEP129 were deposited in the GenBank database under the accession numbers MT667260 and MT667261, respectively.

https://www.ncbi.nlm.nih.gov/nuccore/MT667260
```
LOCUS       MT667260               60430 bp    DNA     circular BCT 15-JUL-2020
DEFINITION  Escherichia coli strain GDE6P124 plasmid pHNEP124, complete
            sequence.


     CDS             14836..15714
                     /gene="trfA"
                     /product="Plasmid replication initiator protein TrfA"
                     /protein_id="QLG04493.1"

     CDS             22240..23289
                     /gene="cfr"
                     /product="23S ribosomal RNA methyltransferase"
                     /protein_id="QLG04502.1"

     CDS             complement(27192..28817)
                     /gene="mcr-1"
                     /product="Colistin resistance protein Mcr-1"
                     /protein_id="QLG04505.1"

     CDS             43395..46004
                     /gene="traI"
                     /product="DNA relaxase TraI"
                     /protein_id="QLG04528.1"

```

----------
### 2020-10-19
### Du_2020-10-19
### pQDFD216.2
IncP1 type

https://www.ncbi.nlm.nih.gov/nuccore/CP053213
```
LOCUS       CP053213               50638 bp    DNA     circular BCT 08-APR-2021
DEFINITION  Escherichia coli strain QDFD216 plasmid unnamed2, complete


     CDS             complement(16182..18812)
                     /product="relaxase/mobilization nuclease domain-containing
                     protein"
                     /protein_id="QTP53151.1"


     CDS             complement(32555..33433)
                     /product="TrfA protein"
                     /protein_id="QTP53167.1"


     CDS             47855..49480
                     /product="phosphoethanolamine--lipid A transferase
                     MCR-3.5"
                     /protein_id="QTP53182.1"

```

IncX4
https://www.ncbi.nlm.nih.gov/nuccore/CP053212
```
LOCUS       CP053212               33308 bp    DNA     circular BCT 08-APR-2021
DEFINITION  Escherichia coli strain QDFD216 plasmid unnamed1, complete

                     /product="replication initiation protein"
                     /protein_id="QTP53109.1"
```

chromosome
https://www.ncbi.nlm.nih.gov/nuccore/CP053211
```
LOCUS       CP053211             4577885 bp    DNA     circular BCT 08-APR-2021
DEFINITION  Escherichia coli strain QDFD216 chromosome, complete genome.
```

https://pubmed.ncbi.nlm.nih.gov/33116684/
Infect Drug Resist
. 2020 Oct 19;13:3649-3655. doi: 10.2147/IDR.S268787. eCollection 2020.
Co-Occurrence of the mcr-1.1 and mcr-3.7 Genes in a Multidrug-Resistant Escherichia coli Isolate from China
Chongtao Du # 1, Yuyang Feng # 1, Guizhen Wang 2, Zhiyuan Zhang 1, Huimin Hu 1, Yu Yu 1, Jiayang Liu 1, Lihao Qiu 1, Hongtao Liu 1, Zhimin Guo 3, Jing Huang 3, Jiazhang Qiu 1
https://dx.doi.org/10.2147/IDR.S268787
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7585518/

The Genetic Context of mcr-1.1 on pQDFD216.1
The mcr-1.1 was located on pQDFD216.1 which was 33,308 bp in length and had an average GC content of 41.83%. PlasmidFinder analysis indicated that pQDFD216.1 belonged to the IncX4 incompatibility group (Table S2). 

https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7585518/figure/f0001/
Figure 1
Comparison of mcr-carrying plasmids and other closely related plasmids. 
(B) Comparative analysis of pQDFD216.2 with five closely related mcr-3-carrying plasmids, pMCR3_WCHEC1943 (GenBank accession: MF678351.1), pGDZJ003-1-MCR-3 (GenBank accession: MH043625.1), pMCR3_WCHEC-LL123 (GenBank accession: MF489760.1), pZR12 (GenBank accession: MF455227.1) and pMCR_WCHEC1622 (GenBank accession: KY463452.1). The plasmid pQDFD216.2 was used as the reference plasmid for BRIG.

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7585518/bin/IDR-13-3649-g0002.jpg)
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC7585518/figure/f0002/
Figure 2
A linear depiction of the genetic configuration of mcr-carrying plasmids compared to other closely related plasmids.
(B) Structure comparisons of pQDFD216.2, pGDZJ003-1-MCR-3 (GenBank accession: MH043625.1) and pMCR3_WCHEC-LL123 (GenBank accession: MF489760.1). The map was generated with Easyfig.

The Genetic Contexts of mcr-3.7 on pQDFD216.2
The mcr-3.7-bearing plasmid pQDFD216.2 was 50,638 bp in size and had a GC content of 47.08%. PlasmidFinder analysis revealed that pQDFD216.2 belonged to the IncP1 type plasmid (Table S2). The IncP1 plasmids have a broad host range and are widespread in Gram-negative organisms, including E. coli, K. pneumonia and S. enteric.34–36 The association of mcr-3.7 with the IncP plasmid is worrisome because it may have the potential to promote the dissemination of mcr-3 in Gram-negative pathogens.

The plasmid pQDFD216.2 did not carry any other resistance gene besides mcr-3.7. The pQDFD216.2 plasmid has the conjugative transfer/type IV secretion system (tra and trb), replication initiator (trfA), replication and partition genes (parA and parB), toxin-antitoxin genes (higA-higB), host-lethal genes (klc) and their regulators (kor) (Figures 1B and ​and2B).2B). BLASTn analysis showed that plasmids pMCR3_WCHEC1943 (GenBank accession: MF678351.1), pGDZJ003-1-MCR-3 (GenBank accession: MH043625.1), pMCR3_WCHEC-LL123 (GenBank accession: MF489760.1), pZR12 (GenBank accession: MF455227.1) and pMCR_WCHEC1622 (GenBank accession: KY463452.1) were most similar to pQDFD216.2 (Figure 1B). These plasmids showed 99.96–99.98% nucleotide sequence identity with pQDFD216.2. The mcr-3.7 gene was located in pQDFD216.2 with the structure TnAs2-mcr-3.7-dgkA-IS26, which was also found in the mcr-3.7-carrying IncP1 plasmids pMCR3_WCHEC-LL123 and pGDZJ003-1-MCR-3 (Figure 2B). However, unlike pQDFD216.2, pGDZJ003-1-MCR-3 partially lacked the conjugal transfer gene TraC of the IncP-type plasmid (Figure 2B). Furthermore, when compared with pMCR3_WCHEC-LL123, the mobile genetic element IS1294 was absent in pQDFD216.2 (Figure 2B).

Plasmid Conjugation

Nucleotide Sequence Accession Numbers
The sequences of the chromosome and plasmids have been submitted in the NCBI database under the accession numbers: CP053211, CP053212 and CP053213.

----------
### 2020-03-04

https://pubmed.ncbi.nlm.nih.gov/32132160/
mSphere
. 2020 Mar 4;5(2):e00117-20. doi: 10.1128/mSphere.00117-20.
High Prevalence of Colistin-Resistant Escherichia coli with Chromosomally Carried mcr-1 in Healthy Residents in Vietnam
Takahiro Yamaguchi 1, Ryuji Kawahara 1, Kouta Hamamoto 2, Itaru Hirai 2, Diep Thi Khong 3, Thang Nam Nguyen 3, Hoa Thi Tran 3, Daisuke Motooka 4, Shota Nakamura 4, Yoshimasa Yamamoto 5 6
https://journals.asm.org/doi/full/10.1128/mSphere.00117-20
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7056805/
Other plasmids showed different mcr-1 transposons, including ISApl1-mcr-1-PAP2, which was carried by different Inc types, such as IncP1, IncI2, and IncY.

plasmid
https://www.ncbi.nlm.nih.gov/nuccore/AP021897
```
LOCUS       AP021897               47676 bp    DNA     circular BCT 19-JUN-2020
DEFINITION  Escherichia coli 2018.02.02CC plasmid p2018_02_02CC DNA, complete
            genome.

                     /country="Viet Nam"
                     /collection_date="2018-02"

```

chromosome
https://www.ncbi.nlm.nih.gov/nuccore/AP021896
```
LOCUS       AP021896             4574715 bp    DNA     circular BCT 19-JUN-2020
DEFINITION  Escherichia coli 2018.02.02CC DNA, complete genome.
```

----------
### 2019-11-26
### Hadjadj_2019-11-26
### pLH102-A
IncP1

https://pubmed.ncbi.nlm.nih.gov/31849875/
Front Microbiol
. 2019 Nov 26;10:2720. doi: 10.3389/fmicb.2019.02720. eCollection 2019.
Co-occurrence of Variants of mcr-3 and mcr- 8 Genes in a Klebsiella pneumoniae Isolate From Laos
Linda Hadjadj 1 2, Sophie Alexandra Baron 1 2 3, Abiola Olumuyiwa Olaitan 1 2, Serge Morand 4, Jean-Marc Rolain 1 2 3
https://www.frontiersin.org/articles/10.3389/fmicb.2019.02720/full
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6887894/

TABLE 1
Characteristics of Klebsiella spp. strains analyzed in this study.

The three mcr-3.21 genes were carried by an IncI1 plasmid type (LH94, TH114 and TH164) whereas the mcr-3.28 gene was harbored by an IncP1 (LH102-A) and the mcr3.26 by an IncFII (LH375). The mcr-8.3 gene was also found on an IncFII plasmid type (LH94) (Table 1).

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6887894/bin/fmicb-10-02720-g003.jpg)
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6887894/figure/F3/
FIGURE 3
(Not to scale) (A) Comparison of the genetic environment of the mcr-3 gene in plasmids: pWJ1 (KY924928), pLH102-A (CP035195), pECSC102 (MG552133), pLH375-3 (CP035200), pTH114-3 (CP035208), pLH94-3 (CP035205), and pTH164-3 (CP035213). (B) Comparison of the genetic environment of the mcr-8 gene in plasmid pK91 (MG736312) and in the plasmid of strain LH94 carrying mcr-8.3 gene: pLH94-8 (CP035204). HP, hypothetical protein, IS, insertion sequence, Tn, transposon.

https://www.ncbi.nlm.nih.gov/nuccore/CP035195
```
LOCUS       CP035195               60128 bp    DNA     circular BCT 29-OCT-2019
DEFINITION  Klebsiella pneumoniae strain LH102-A plasmid pLH102-A, complete
            sequence.

     CDS             complement(710..2335)
                     /gene="mcr-3"
                     /product="MCR-3 family phosphoethanolamine--lipid A
                     transferase"
                     /protein_id="QFU52753.1"


     CDS             16757..17635
                     /product="TrfA protein"
                     /protein_id="QFU52768.1"

     CDS             31364..33973
                     /product="conjugal transfer protein TraI"
                     /protein_id="QFU52785.1"

```

----------
### 2019-08-01
### pCFSA664-2

https://pubmed.ncbi.nlm.nih.gov/31371529/
Microbiol Resour Announc
. 2019 Aug 1;8(31):e00114-19. doi: 10.1128/MRA.00114-19.
Complete Genome and Plasmid Sequences of Seven Isolates of Salmonella enterica subsp. enterica Harboring the mcr-1 Gene Obtained from Food in China
Yujie Hu 1 2, Scott V Nguyen 2, Chang Liu 3, Wei Wang 1, Yinping Dong 1 2, Séamus Fanning 1 2, Fengqin Li 4
https://journals.asm.org/doi/full/10.1128/MRA.00114-19
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6675977/
TABLE 1
Chromosome and plasmid sequence accession numbers and additional information for seven Salmonella enterica subsp. enterica strains harboring the mcr-1 gene

pCFSA664-2 was not predicted to have an Inc group using PlasmidFinder; however, the annotation contained an IncP-1-like trfA replication gene from an also-untypeable plasmid, pYDC107_41 (GenBank accession no. CP025711), which is 97.63% identical with 95% coverage to pCFSA664-2 (8).

https://www.ncbi.nlm.nih.gov/nuccore/CP033354
```
LOCUS       CP033354               41696 bp    DNA     circular BCT 08-MAY-2019
DEFINITION  Salmonella enterica subsp. enterica strain CFSA664 plasmid
            pCFSA664-2, complete sequence.

                     /isolation_source="food"
                     /country="China: Jiangsu"
                     /collection_date="Jun-2015"

     CDS             833..1711
                     /product="TrfA protein"
                     /protein_id="QAX34190.1"

     CDS             12625..15237
                     /product="conjugal transfer protein TraI"
                     /protein_id="QCO44744.1"

```

----------
### Xu_2018-08
### pGDZJ002-1
### pGDZJ003–2

https://pubmed.ncbi.nlm.nih.gov/30061009/
EBioMedicine
. 2018 Aug;34:139-157. doi: 10.1016/j.ebiom.2018.07.027. Epub 2018 Jul 27.
Spread of MCR-3 Colistin Resistance in China: An Epidemiological, Genomic and Mechanistic Study
Yongchang Xu 1, Lan-Lan Zhong 2, Swaminath Srinivas 3, Jian Sun 4, Man Huang 1, David L Paterson 5, Sheng Lei 1, Jingxia Lin 1, Xin Li 6, Zichen Tang 7, Siyuan Feng 2, Cong Shen 2, Guo-Bao Tian 8, Youjun Feng 9
https://www.thelancet.com/journals/ebiom/article/PIIS2352-3964(18)30270-6/fulltext
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6116419/

3.2. Comparative Genomics of mcr-3-Harboring Plasmids
We succeeded in sequencing of MCR-3-producing plasmids pGDZJ002-1, pGDZJ003–2 and pGDZJ004 from GDZJ002, GDZJ003 and GDZJ004, respectively, and mcr-1-encoding plasmid pGDZJ003-1 from the GDZJ003. Of the two plasmids isolated from GDZJ003, whole genome sequencing shows that plasmid pGDZJ003–2 is a 50,537 kb plasmid that contains 73 ORFs including the mcr-3.7 gene. MCR-3.7 has 18 amino acid substitutions when compared to MCR-3. This IncP-type plasmid is also seen to possess the tra and trb conjugative regions, portioning modules par, replication initiator trfA, host-lethal protein encoding genes klc and their regulators kor and a toxin-antitoxin system higA-B (Fig. 2B). It shows very high sequence identity to another IncP-type mcr-3.7-bearing plasmid pMCR3_WCHEC-LL123 [54] (Accession no.: MF489760) isolated from human isolates of E. coli. However, unlike pMCR3_WCHEC-LL123, pGDZJ003–2 is missing the insertion sequence IS1294 (Fig. 2C) and does not have additional antibiotic resistance genes like blaTEM (conferring ampicillin resistance). When compared to another IncP type mcr-1-carrying plasmid, pMCR_1511 [93], pGDZJ003–2 also lacks the transposition unit of mcr-1 (ISApl1-mcr-1-hp) on pMCR_1511 (Fig. 2D). The other plasmid from GDZJ003 is pGDZJ003-1 which harbors the mcr-1 gene and is 33,522 kb in length with a GC content of 42%. This plasmid contains 43 predicted ORFs, with 17 associated with formation of the type IV pilus (Fig. 2A). Comparative analysis shows that its backbone is very closely related to two narrow-host-range IncX4-type plasmids [94] pSH146-32 (Accession no.: JX258655) and the mcr-1-harboring plasmid pCSZ4 [95] (Accession no.: KX711706) (Fig. 2C). Like most IncX4 plasmids, both pCSZ4 and pGDZJ003–1 are missing the ISApl1 insertion sequence in front of the mcr-1 gene.

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6116419/bin/gr2.jpg)
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6116419/figure/f0010/
Fig. 2
Comparative genomics of the plasmids carrying mcr-3 and/or mcr-1 genes.

B. Schematic representation for circular genome of the mcr-3.7-harboring plasmid pGDZJ003_mcr-3.7.

D. Linear genome alignment of the mcr-3-positive plasmid pGDZJ003_mcr-3.7 with the other two plasmids (pMCR3_WCHEC-LL123 and pMCR_1511).

Similarly, genome sequences for two other mcr-3-carrying plasmids, pGDZJ002-1 (50,818 kb; Fig. S2) and pGDZJ004 (50,519 kb; Fig. S3) were obtained. Both plasmids have a GC content of 47% and contain similar elements for replication, conjugation, and partitioning (Figs S2 and S3). Colinear genome analysis of pGDZJ003–2 and pMCR3_WCHEC-LL123 shows that they possess similar genetic arrangements (Fig. S4) and belong to the IncP family of plasmids. 

2.7. Sequence Data
GenBank accession numbers of new plasmids with full genome sequences are listed as follow: MH043623 for pGDZJ002–1; MH043624 for pGDZJ003–1; MH043625 for pGDZJ003–2; and MH043626 for pGDZJ004.

https://www.ncbi.nlm.nih.gov/nuccore/MH043623
```
LOCUS       MH043623               50818 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Escherichia coli strain GDZJ002 plasmid pGDZJ002-1, complete

     CDS             complement(15915..18524)
                     /product="IncP-type DNA relaxase TraI"
                     /protein_id="AWM63225.1"

     CDS             complement(32388..33266)
                     /product="TrfA"
                     /protein_id="AWM63250.1"

     CDS             47878..49503
                     /note="phosphoethanolamine--lipid A transferase"
                     /product="MCR-3.7"
                     /protein_id="AWM63270.1"

```

https://www.ncbi.nlm.nih.gov/nuccore/MH043625
```
LOCUS       MH043625               50537 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Escherichia coli strain GDZJ003 plasmid pGDZJ003-1-MCR-3, complete

     CDS             complement(15824..17548)
                     /product="IncP-type DNA relaxase TraI"
                     /protein_id="AWM63352.1"

     CDS             complement(32205..33083)
                     /product="TrfA"
                     /protein_id="AWM63380.1"

     CDS             47508..49133
                     /note="phosphoethanolamine--lipid A transferase"
                     /product="MCR-3.7"
                     /protein_id="AWM63398.1"

```

----------
### Li_2018-06-01
pMCR3_WCHEC-LL123

https://pubmed.ncbi.nlm.nih.gov/29462314/
J Antimicrob Chemother
. 2018 Jun 1;73(6):1721-1723. doi: 10.1093/jac/dky038.
Occurrence of the mobile colistin resistance gene mcr-3 in Escherichia coli from household pigs in rural areas
Jiyun Li 1, Anette Hulth 2, Lennart E Nilsson 3, Stefan Börjesson 4, Baoli Chen 5, Zhenwang Bi 5, Yang Wang 1, Stefan Schwarz 1 6, Congming Wu 1
https://academic.oup.com/jac/article/73/6/1721/4866120
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5961201/
The largest contig (49.9 kb) of plasmid pBZ5 showed 99% nucleotide sequence identity to that in plasmid pMCR3_WCHEC-LL123 (GenBank accession no. MF489760) of human E. coli from China.12 Plasmid pBZ5 contained toxin–antitoxin systems (higB and higA) as well as conjugative elements, which can maintain mcr-3.5 in its host and facilitate the horizontal transfer of this mcr-3 variant to other bacteria.14

https://www.ncbi.nlm.nih.gov/nuccore/MF489760
```
LOCUS       MF489760               52208 bp    DNA     circular BCT 27-NOV-2017
DEFINITION  Escherichia coli strain WCHEC-LL123 plasmid pMCR3_WCHEC-LL123,

     CDS             complement(33880..34758)
                     /gene="trfA"

                     /product="TrfA"


     CDS             49180..50805
                     /gene="mcr-3.5"

                     /product="phosphoethanolamine transferase MCR-3.5"

```

----------
### 2018-05-25
### Xiang_2018-05-25
### pECSC102

https://pubmed.ncbi.nlm.nih.gov/29760138/
Antimicrob Agents Chemother
. 2018 May 25;62(6):e00501-18. doi: 10.1128/AAC.00501-18. Print 2018 Jun.
Colocation of the Polymyxin Resistance Gene mcr-1 and a Variant of mcr-3 on a Plasmid in an Escherichia coli Isolate from a Chicken Farm
Rong Xiang 1, Bi-Hui Liu 1, An-Yun Zhang 1, Chang-Wei Lei 1, Xiao-Lan Ye 1, Yan-Xian Yang 1, Yan-Peng Chen 1, Hong-Ning Wang 2
https://journals.asm.org/doi/10.1128/AAC.00501-18
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5971617/
A colistin-resistant Escherichia coli isolate from a commercial poultry farm in China carried two colistin resistance genes, mcr-1 and variant of mcr-3, in an IncP plasmid.

The variant of mcr-3 and the mcr-1 gene were colocated in a conjugative 60.7-kb IncP plasmid, designated pECSC102. It had a length of 60,744 bp, with 47.2% G+C content (Fig. S2). Plasmid pECSC102 aligned very well to two mcr-carrying IncP plasmids, pMCR3_WCHEC-LL123 (GenBank accession no. MF489760), found in an E. coli isolate, and pMCR_1511 (GenBank accession no. KX377410), found in a K. pneumoniae isolate (Fig. 1). 

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5971617/bin/zac0061871950001.jpg)
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5971617/figure/F1/
FIG 1
Comparison of the genetic configuration of pECSC102 with other plasmids. 

Accession number(s). The complete nucleotide sequence of the plasmid pECSC102 characterized in this study was submitted to the GenBank database and assigned accession number MG552133.

pECSC102
MG552133
https://www.ncbi.nlm.nih.gov/nuccore/MG552133
```
LOCUS       MG552133               60744 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Escherichia coli strain ECSC102 plasmid pECSC102, complete
            sequence.

     CDS             complement(30485..31363)
                     /product="TrfA"
                     /protein_id="AUO29378.1"

     CDS             47002..48627
                     /gene="mcr-3.11"
                     /product="phosphoethanolamine transferase MCR-3.11"
                     /protein_id="AUO29399.1"

     CDS             58310..59935
                     /gene="mcr-1"
                     /product="Colistin resistance protein mcr-1"
                     /protein_id="AUO29419.1"

     CDS             complement(13851..16460)
                     /product="IncP-type DNA relaxase TraI"
                     /protein_id="AUO29352.1"

```


----------
### 2017-10-25
### Zhao_2017-10-25
### pMCR_WCHEC1622

https://pubmed.ncbi.nlm.nih.gov/29118748/
Front Microbiol
. 2017 Oct 25;8:2094. doi: 10.3389/fmicb.2017.02094. eCollection 2017.
Remarkable Diversity of Escherichia coli Carrying mcr-1 from Hospital Sewage with the Identification of Two New mcr-1 Variants
Feifei Zhao 1 2, Yu Feng 1 2, Xiaoju Lü 1 2, Alan McNally 3, Zhiyong Zong 1 2 4
https://www.frontiersin.org/articles/10.3389/fmicb.2017.02094/full
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5660977/

In WCHEC1622, mcr-1.1 was carried by a 49,897-bp IncP plasmid, designated pMCR_WCHEC1622, which was almost identical (100% coverage and 99% identity) to pMCR_1511 (Figure ​Figure44), an IncP plasmid recovered from K. pneumoniae in the same sewage sample (Zhao et al., 2017). Like pMCR_1511, pMCR_WCHEC1622 belongs to a new clade of IncP (Zhao et al., 2017). As IncP plasmids are broad-host-range, the carriage of mcr-1 on IncP plasmids has the potential to mediate the dissemination of mcr-1 from the Enterobacteriaceae to other bacterial species.

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5660977/bin/fmicb-08-02094-g004.jpg)
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5660977/figure/F4/
FIGURE 4
Alignment of pMCR_WCHEC1622 and closely related IncP plasmids. pMCR_1511 (Zhao et al., 2017) was used as a reference. The alignment is a pairwise BLASTn alignment performed using BRIG. pMCR_1511, pMCR_WCHEC1622, pHNFP671 (GenBank accession no. KP324830), pHS102707 (GenBank accession no. KF701335), pMCR16_P053 (GenBank accession no. KY352406), and pJJ1886_4 (GenBank accession no. CP006788) belong to a new unnamed clade of IncP plasmids (Zhao et al., 2017).

https://www.ncbi.nlm.nih.gov/nuccore/KY463452
```
LOCUS       KY463452               49897 bp    DNA     circular BCT 08-JUL-2017
DEFINITION  Escherichia coli strain WCHEC1622 plasmid pMCR_WCHEC1622, complete

     CDS             18165..19043
                     /gene="trfA"
                     /locus_tag="pMCR_WCHEC1622_00027"
                     /inference="ab initio prediction:Prodigal:2.6"
                     /inference="similar to AA sequence:UniProtKB:P07676"
                     /codon_start=1
                     /transl_table=11
                     /product="Plasmid replication initiator protein TrfA"


     CDS             complement(47015..48640)
                     /gene="mcr-1"
                     /locus_tag="pMCR_WCHEC1622_00067"
                     /EC_number="2.7.-.-"
                     /inference="ab initio prediction:Prodigal:2.6"
                     /inference="similar to AA sequence:UniProtKB:P30845"
                     /codon_start=1
                     /transl_table=11
                     /product="MCR-1"

```

----------
### 2017-01-24
### Zhao_2017-01-24
### pMCR_1511

https://pubmed.ncbi.nlm.nih.gov/27895009/
Antimicrob Agents Chemother
. 2017 Jan 24;61(2):e02229-16. doi: 10.1128/AAC.02229-16. Print 2017 Feb.
IncP Plasmid Carrying Colistin Resistance Gene mcr-1 in Klebsiella pneumoniae from Hospital Sewage
Feifei Zhao 1 2, Yu Feng 1 2, Xiaoju Lü 1 2, Alan McNally 3, Zhiyong Zong 4 2 5
https://aac.asm.org/content/61/2/e02229-16.long
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5278755/

The backbone of pMCR_1511 was highly similar (99% identity) to that of plasmid pHNFP671 (GenBank accession number KP324830), which was an IncP plasmid in E. coli isolate FP671 from Guangzhou, China but did not carry mcr-1. IncP-1 plasmid has six assigned clades, i.e., α, β, γ, δ, ε, and ζ, among which β clade has two subclades (β1 and β2) (24). To determine which clade pMCR_1511 belonged to, the sequences of 30 genes belonging to the IncP-1 backbone were retrieved from pMCR_1511, were concatenated and were then aligned to the counterparts of one representative plasmid of each clade, including the β1 and β2 subclades, as described previously (25). Phylogenetic analysis of the IncP-1 plasmid backbone revealed that pMCR_1511 and pHNFP671 belonged to a new IncP-1 clade (see Fig. S1 in the supplemental material).

FIG 1
Genetic structure of IncP plasmid pMCR_1511 carrying mcr-1 and comparison with IncP plasmid pHNFP671.

It is known that ISApl1 generates 2-bp direct target repeats (DR) upon insertion (26) (https://www-is.biotoul.fr/scripts/ficheIS.php?name=ISApl1). The 2-bp flanking sequences of the region bracketed by the two copies of ISApl1 were identical (AC) (Fig. 2). 


![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5278755/bin/zac0021758990002.jpg)
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC5278755/figure/F2/
FIG 2
Genetic context of mcr-1 on pMCR_1511. 

The ISApl1 downstream of mcr-1 on pMCR_1511 was interrupted by the insertion of Tn3 with the characteristic 5-bp DR (Fig. 2). The Tn3 was also disrupted by IS26 and most of the Tn3 was absent, which may result from the action of IS26. It is well known that the insertion of IS26 can lead to the deletion of the adjacent sequence of the insertion site (28). Alternatively, the insertion of the second IS26 and the recombination between the two copies of IS26 might eliminate the intervening region, resulting in the loss of most of Tn3. Although Tn3 interrupted the ISApl1 downstream of mcr-1, the right-end inverted repeat (IRR) of ISApl1 remained intact (Fig. 2). The transposase encoded by the ISApl1 upstream of mcr-1 had the potential to recognize the IRR of the downstream ISApl1 and thus mobilize the region bracketed by the two copies of ISApl1.

Sequencing reads and the whole-genome shotgun sequencing project of K. pneumoniae strain WCHKP1511 have been deposited into DDBJ/EMBL/GenBank under accession numbers SRR3170679 and LSMF00000000, respectively. The sequence of pMCR_1511 has been deposited into DDBJ/EMBL/GenBank under accession number KX377410.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5278755/bin/supp_61_2_e02229-16__index.html
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5278755/bin/AAC.02229-16_zac002175899s1.pdf
Supplementary file
Figure S1. Phylogenetic tree of IncP-1 plasmids. The plasmids included are RK2 (IncP1-α clade; GenBank accession no. BN000925), R751 (β1 subclade; GenBank accession no. NC_001735), pA1 (β2 subclade; GenBank accession no. NC_007353), pQKH54 (γ clade; GenBank accession no. AM157767), pAKD4 (δ clade; GenBank accession no. NC_025029), pKJK5 (ε clade; GenBank accession no. AM261282), pMCBF1 (ζ clade; GenBank accession no. AY950444), pHNFP671 (a new clade; GenBank accession no. KP324830) and pMCR_1511 (the same new clade as pHNFP671; this study). The tree was constructed using concatenated sequences of 30 genes belonging to the IncP-1 backbone as described previously (1).

Reference
1. Norberg P, Bergstrom M, Jethava V, Dubhashi D, Hermansson M. 2011. The IncP-1 plasmid backbone adapts to different host bacterial species and evolves through homologous recombination. Nat Commun 2:268.

https://www.ncbi.nlm.nih.gov/nuccore/KX377410
```
LOCUS       KX377410               57278 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Klebsiella pneumoniae strain WCHKP1511 plasmid pMCR_1511, complete


     CDS             complement(22557..25166)
                     /gene="traI"
                     /inference="ab initio prediction:Prodigal:2.6"
                     /note="pMCR_1511_00031"
                     /codon_start=1
                     /transl_table=11
                     /product="TraI"
                     /protein_id="APB02205.1"


     CDS             complement(39191..40069)
                     /gene="trfA"
                     /inference="ab initio prediction:Prodigal:2.6"
                     /inference="similar to AA sequence:UniProtKB:P07676"
                     /note="pMCR_1511_00057"
                     /codon_start=1
                     /transl_table=11
                     /product="Plasmid replication initiator protein TrfA"
                     /protein_id="APB02231.1"

```

----------

### pHNFP671

https://www.ncbi.nlm.nih.gov/nuccore/KP324830
```
LOCUS       KP324830               82807 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Escherichia coli isolate FP671 plasmid pHNFP671, complete sequence.


  AUTHORS   Liu,X.-Q., Liu,J.-H. and Zeng,Z.-l.
  TITLE     Detection a broad host range IncP plasmid carrying cfr gene from
            extended-spectrum cephalosporin-resistant Escherichia coli
  JOURNAL   Unpublished

                     /isolation_source="pig feces"
                     /country="China"


     CDS             36412..37290
                     /gene="trfA"
                     /note="TrfA replication protein"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfA"
                     /protein_id="AJQ17269.1"

     CDS             complement(70414..71406)
                     /note="relaxase"
                     /codon_start=1
                     /transl_table=11
                     /product="Type IV secretory pathway VirD2 component"
                     /protein_id="AJQ17300.1"

```

----------
### Li_2015-01
### pHS102707

https://pubmed.ncbi.nlm.nih.gov/25367902/
Antimicrob Agents Chemother
. 2015 Jan;59(1):338-43. doi: 10.1128/AAC.03061-14. Epub 2014 Nov 3.
First report of a clinical, multidrug-resistant Enterobacteriaceae isolate coharboring fosfomycin resistance gene fosA3 and carbapenemase gene blaKPC-2 on the same transposon, Tn1721
Gang Li 1, Ying Zhang 2, Dexi Bi 3, Pinghua Shen 2, Fuqi Ai 2, Hong Liu 2, Yueru Tian 2, Yiming Ma 2, Bei Wang 2, Kumar Rajakumar 4, Hong-Yu Ou 5, Xiaofei Jiang 6
https://aac.asm.org/content/59/1/338.long
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4291370/
The complete sequence of pHS102707 showed that this plasmid belongs to the P11 subfamily (IncP1) and has a replication gene, several plasmid-stable genes, an intact type IV secretion system gene cluster, and a composite transposon Tn1721-Tn3 that harbored blaKPC-2. Interestingly, a composite IS26 transposon carrying fosA3 was inserted in the Tn1721-tnpA gene in pHS102707 and pHS112625, leading to the disruption of Tn1721-tnpA and the deletion of Tn1721-tnpR. However, only IS26 with a truncated Tn21-tnpR was inserted in pHS092839 at the same position. To our knowledge, this is the first report of fosA3 and blaKPC-2 colocated in the same Tn1721-Tn3–like composite transposon on a novel IncP group plasmid.

![](https://aac.asm.org/content/aac/59/1/338/F2.large.jpg)

![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4291370/bin/zac0011535940002.jpg)

FIG 2
(a) Schematic map of pHS102707, an IncP plasmid found in the carbapenem-resistant E. coli isolate investigated in this study. 
(b) Inferred phylogenetic relationships of the plasmid-coding relaxase homologs. Seventeen protein sequences were aligned and the tree was generated with MEGA5 (21, 22) using the maximum-likelihood method. The relaxase sequences obtained in this study are indicated by blank arrows, while others were taken from Alvarado et al. (25). The sequences marked by an asterisk denote the ones taken from the prototype plasmid of each subfamily.

Within each family, subfamilies corresponding to phylogenetic clades contain more members (e.g., MOBF12, which harbors relaxases encoded by IncF plasmids, and MOBP11, which groups relaxases encoded by IncP plasmids) (23). Phylogenetic analysis shows that the three relaxases encoded by pHS102707, pHS092839, and pHS112625 were grouped as the P11 relaxase subfamily (Fig. 2b). Relaxase protein sequence comparison of pHS102707 revealed 30.5% identity with RP4 (Inc P1α), the prototype plasmid for the P11 subfamily. The relaxase most similar to that of pHS102707, pXFAS01, exhibited 47% identity; no identity was found with any other non-P11 subfamily relaxases by NCBI blast.

The blaKPC-2 and fosA3 genes are borne on a novel mosaic transposon.

pKP048 belongs to the F family (IncF), while pHS102707, pHS092839, and pHS112625 belong to the P11 subfamily (IncP1). This suggests that a block mobilization of the complex transposon may happen among IncP1 plasmids. Interestingly, EcoRI and HindIII digestion profiles (Fig. 1c) show that the three P11 subfamily plasmids are not identical, and some differences exist.

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4291370/bin/zac0011535940003.jpg)
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4291370/figure/F3/
FIG 3
Comparative analysis of the blaKPC-2 and fosA3 bearing mosaic Tn1721-Tn3-derived transposons present on pHS102707. The genes are depicted as arrows according to the direction of transcription. blaKPC-2 and fosA3 are shown in black and dark gray, respectively. The inverted repeats are indicated by the variable, vertical gray bars. Regions with similar sequences are indicated in gray between the different plasmids. The asterisk indicates that the plasmid type was determined by the degenerate primer MOB typing (DPMT) method (25).

Nucleotide sequence accession numbers. The complete sequences of the plasmid pHS102707, a 15,464-bp region of pHS112625, and a 15,499-bp region of pHS092839 were deposited in GenBank under the accession numbers KF701335, KF724506, and KJ210592, respectively. The sequences of the relaxase genes of pHS092839 and pHS112625 were deposited under accession numbers KJ210594 and KJ210593, respectively.

https://www.ncbi.nlm.nih.gov/nuccore/KF701335
```
LOCUS       KF701335               69453 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Escherichia coli strain HS102707 plasmid pHS102707, complete
            sequence.

     CDS             complement(60086..62695)
                     /gene="traI"
                     /note="conjugal transfer nickase/helicase TraI"
                     /codon_start=1
                     /transl_table=11
                     /product="TraI"
                     /protein_id="AHI61854.1"

```

部分配列
https://www.ncbi.nlm.nih.gov/nuccore/KF724506
```
LOCUS       KF724506               15499 bp    DNA     linear   BCT 26-JUL-2016
DEFINITION  Klebsiella pneumoniae subsp. pneumoniae strain HS092839 plasmid
            pHS092839, partial sequence.
```

部分配列
https://www.ncbi.nlm.nih.gov/nuccore/KJ210592
```
LOCUS       KJ210592               18184 bp    DNA     linear   BCT 26-JUL-2016
DEFINITION  Enterobacter aerogenes strain HS112625 plasmid pHS112625, partial
            sequence.
```


https://www.ncbi.nlm.nih.gov/nuccore/CP001012
```
LOCUS       CP001012               38297 bp    DNA     circular BCT 26-JUL-2016
DEFINITION  Xylella fastidiosa M23 plasmid pXFAS01, complete sequence.

     CDS             complement(23913..26405)
                     /locus_tag="XfasM23_2249"
                     /note="PFAM: Relaxase/mobilization nuclease family
                     protein;
                     KEGG: ypi:YpsIP31758_A0026 conjugal transfer protein TraI"
                     /codon_start=1
                     /transl_table=11
                     /product="Relaxase/mobilization nuclease family protein"
                     /protein_id="ACB93642.1"

```



----------
## updates

## 2021

### Wegrzyn_2021

https://pubmed.ncbi.nlm.nih.gov/33660784/
Nucleic Acids Res
. 2021 Apr 6;49(6):3394-3408. doi: 10.1093/nar/gkab113.
Defining a novel domain that provides an essential contribution to site-specific interaction of Rep protein with DNA
Katarzyna Wegrzyn 1, Elzbieta Zabrocka 1, Katarzyna Bury 1, Bartlomiej Tomiczek 1, Milosz Wieczor 2, Jacek Czub 2, Urszula Uciechowska 1, María Moreno-Del Alamo 3, Urszula Walkow 1, Igor Grochowina 1, Rafal Dutkiewicz 1, Janusz M Bujnicki 4 5, Rafael Giraldo 3, Igor Konieczny 1
https://academic.oup.com/nar/article/49/6/3394/6154473
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8034659/

Supplementary Materials
gkab113_Supplemental_Files.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8034659/bin/gkab113_supplemental_files.zip
Supplementary Data_3.02.21.pdf

Figure S1. Phylogenetic relations between Origin Binding Proteins (OBPs) based on their conserved WH domain. Maximum Likelihood phylogenetic tree calculated with the GAMMA+I+LG model rooted to WH domain from Eukaryotes (Orc1, Orc4, Orc5 and cdc6; orange). TrfA sequences are displayed in purple (TrfA33 in bold), whereas the proteins most closely related to RepC are marked in blue. Sequences with genomic annotations are labelled in grey (italics), plasmid sequences labelled black. Level of bootstrap support is indicated with dots, bootstrap >75 in black, bootstrap ≤75 and >25 in grey, bootstrap ≤25 in yellow.

Figure S2. Phylogenetic relations within the Rep superfamily, based on the full-length proteins. The tree was calculated with Maximum Likelihood method with the GAMMA+I+LG model. TrfA sequences are displayed in purple, other Rep superfamily members in green. Level of bootstrap support is indicated with dots, bootstrap >75 in black, bootstrap ≤75 and >25 in grey, bootstrap ≤25 in yellow.

ABSTRACT

The phylogenetic analysis revealed that the composition of this unique domain is typical within the described TrfA-like protein family.

MATERIALS AND METHODS

Phylogenetic analysis and determination of sequence divergence
Protein sequences of the Rep; TrfA; Orc1, 2, 3, 4, 5; and Cdc6 families were obtained from the Pfam database, version 31.0 (28). Sequences were aligned separately using Clustal Omega v1.2.2 with default parameters (29). The alignment was corrected manually. 


12 Orc, 16 Rep and 8 TrfA sequences were chosen for phylogenetic analysis based on sequence divergence and literature data. One thousand ML searches were performed using RAxML v.8.2.10 (32) with 100 rapid bootstrap replicates under the LG model of amino acid substitution and GAMMA model of rate heterogeneity with four discrete rate categories and the estimates proportion of invariable sites (LG + I + G) (33), which was determined to be the best-fit model for all families by ProtTest v3.2 following the Akaike criterion (34).

RESULTS

Interestingly, in some Rep proteins, we identified a short hydrophobic region located before the WH1 domain. This region shares a degree of similarity with a sequence within the N-terminal TrfA region (Figure 1). We thus asked how the TrfA-like proteins are related to other replication initiators. To answer this question, we performed phylogenetic analysis of the representative DNA replication initiators using the maximum likelihood method (ML) (Supplementary Figure S1). Our ML analysis of the WH domains revealed a close monophyletic relationship among all TrfA-like proteins containing extended N-termini. Moreover, the phylogenetic analysis indicated that the TrfA-like proteins are most closely related to RepC, an IncQ-type replication protein C (51,52), and belong to a common protein family together with other plasmid replication initiation factors (Supplementary Figures S1, S2). 



http://pfam.xfam.org/families
Pfam: All Pfam families

This is a list of all 19179 Pfam-A families in Pfam release 34.0 (release date 2021-03-04).

Accession	Id	Description

PF04084	ORC2	Origin recognition complex subunit 2

PF05460	ORC6	Origin recognition complex subunit 6 (ORC6)

PF06504	RepC	Replication protein C (RepC)

PF07034	ORC3_N	Origin recognition complex (ORC) subunit 3 N-terminus

PF07042	TrfA	TrfA protein

PF14629	ORC4_C	Origin recognition complex (ORC) subunit 4 C-terminus
PF14630	ORC5_C	Origin recognition complex (ORC) subunit 5 C-terminus

PF18137	ORC_WH_C	Origin recognition complex winged helix C-terminal

----------

## 2020

### 2020-03

### Xiang_2020

https://pubmed.ncbi.nlm.nih.gov/32133852/
J Agric Food Chem
. 2020 Mar 25;68(12):3786-3795. doi: 10.1021/acs.jafc.9b05061. Epub 2020 Mar 17.
Efficient Degradation of Phenoxyalkanoic Acid Herbicides by the Alkali-Tolerant Cupriavidus oxalaticus Strain X32
Sheng Xiang 1, Ronghua Lin 2, Hongyi Shang 1, Yong Xu 1, Zhenhua Zhang 1, Xuemin Wu 1, Fulin Zong 2
https://pubs.acs.org/doi/abs/10.1021/acs.jafc.9b05061
https://pubs.acs.org/doi/full/10.1021/acs.jafc.9b05061

IncP-1beta?
unnamed5
CP038640

https://www.ncbi.nlm.nih.gov/bioproject/PRJNA525272
https://www.ncbi.nlm.nih.gov/assembly/GCA_004768545.1

https://www.ncbi.nlm.nih.gov/nuccore/CP038640
```
LOCUS       CP038640               89459 bp    DNA     circular BCT 10-APR-2019
DEFINITION  Cupriavidus oxalaticus strain X32 plasmid unnamed5, complete
            sequence.
```

https://www.ncbi.nlm.nih.gov/nuccore/CP038635
```
LOCUS       CP038635             3701485 bp    DNA     circular BCT 10-APR-2019
DEFINITION  Cupriavidus oxalaticus strain X32 chromosome 2, complete sequence.
```

https://www.ncbi.nlm.nih.gov/nuccore/CP038634
```
LOCUS       CP038634             2410412 bp    DNA     circular BCT 10-APR-2019
DEFINITION  Cupriavidus oxalaticus strain X32 chromosome 1, complete sequence.
```

----------

## 2019

----------

### Dang_2016-05-06
### pDTC28
IncP-1beta1

https://www.ncbi.nlm.nih.gov/nuccore/KU238092
```
LOCUS       KU238092               61503 bp    DNA     circular ENV 17-MAY-2016
DEFINITION  Uncultured bacterium plasmid pDTC28, complete sequence.

                     /country="China"

```

https://pubmed.ncbi.nlm.nih.gov/27152950/
PLoS One
. 2016 May 6;11(5):e0154975. doi: 10.1371/journal.pone.0154975. eCollection 2016.
Complete Nucleotide Sequence of IncP-1β Plasmid pDTC28 Reveals a Non-Functional Variant of the blaGES-Type Gene
Bingjun Dang 1 2, Daqing Mao 1, Yi Luo 2
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0154975
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4859535/
A phylogenetic tree was constructed using the trfA proteins of 54 IncP-1 plasmids including plasmid pDTC28 (Fig 2). The tree shows that plasmid pDTC28 belongs to the IncP-1β group and clusters with the IncP-1β prototype R751. The tree also shows that plasmid pDTC28 is slightly different from a smaller IncP-1β clade formed by plasmids pB1, pB12, pNB8c, pLME1, pB4, pTB30, pBAM1 and pRSB223.


----------

## 2015
### Robson_2015-06-10
### pAcX50c

https://www.ncbi.nlm.nih.gov/genome/17633?genome_assembly_id=217211

Azotobacter chroococcum NCIMB 8003

https://www.ncbi.nlm.nih.gov/nuccore/CP010415
```
LOCUS       CP010415             4591803 bp    DNA     circular BCT 02-JUL-2015
DEFINITION  Azotobacter chroococcum NCIMB 8003, complete genome.
```

https://www.ncbi.nlm.nih.gov/nuccore/CP010418
```
LOCUS       CP010418               62783 bp    DNA     circular BCT 02-JUL-2015
DEFINITION  Azotobacter chroococcum NCIMB 8003 plasmid pAcX50c, complete

  ORGANISM  Azotobacter chroococcum NCIMB 8003
            Bacteria; Proteobacteria; Gammaproteobacteria; Pseudomonadales;
            Pseudomonadaceae; Azotobacter.

                     /isolation_source="soil"

                     /country="USA"
                     /collection_date="1950-02-01"


     CDS             724..1809
                     /gene="trfA"
                     /locus_tag="Achr_c20"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfA replication initiation protein"
                     /protein_id="AJE23484.1"

     CDS             13218..15617
                     /gene="traI"
                     /locus_tag="Achr_c170"
                     /codon_start=1
                     /transl_table=11
                     /product="Conjugal transfer relaxase TraI"
                     /protein_id="AJE23498.1"


```

https://pubmed.ncbi.nlm.nih.gov/26061173/
PLoS One
. 2015 Jun 10;10(6):e0127997. doi: 10.1371/journal.pone.0127997. eCollection 2015.
Azotobacter Genomes: The Genome of Azotobacter chroococcum NCIMB 8003 (ATCC 4412)
Robert L Robson 1, Robert Jones 2, R Moyra Robson 1, Ariel Schwartz 3, Toby H Richardson 3
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0127997
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4465626/

Plasmid pAcX50c

It appears that pACX50c most closely matches plasmids isolated from Pseudomonas species amongst the γ-proteobacteria including pQKH54 [108]. In our earlier work pACX50c was apparently cured when the IncP-1 plasmid RP4 was introduced and stably maintained in the strain by selecting for the antibiotic resistance it encodes [40]. This would appear to provide further evidence that pACX50c belongs to the IncP-1 incompatibility group. 

One of the plasmids in Ac-8003, pAcX50c, belongs to the IncP-1 incompatability group and the genomic analysis shows it contains tra and trb gene clusters found in the IncP1-γ family of plasmids from Pseudomonads.


The sequences of the chromosome and 6 plasmids have been deposited with Genbank under the accession numbers as follows: chromosome, CP010415; pAcX50a, CP010416; pAcX50b, CP010417; pAcX50c, CP010418; pAcX50d, CP010419; pAcX50e, CP010420; pAcX50f, CP010421. Supplementary material can be found at http://azotobacter.craic.com.

----------

## 2013

----------
### 2013-12
### pDS1
IncP-1eta

https://www.ncbi.nlm.nih.gov/nuccore/KC170283
```
LOCUS       KC170283               40596 bp    DNA     circular ENV 26-JUL-2016
DEFINITION  Uncultured bacterium plasmid pDS1, complete sequence.

  ORGANISM  uncultured bacterium
            Bacteria; environmental samples.

     CDS             complement(1..936)
                     /gene="trfA"
                     /product="TrfA replication initiation protein"
                     /protein_id="AGH89267.1"

     CDS             complement(961..1338)
                     /gene="ssb"
                     /codon_start=1
                     /transl_table=11
                     /product="Ssb single-strand DNA binding protein"
                     /protein_id="AGH89268.1"

     CDS             1443..1844
                     /gene="trbA"
                     /product="TrbA mating pair formation protein,
                     transcription regulator"
                     /protein_id="AGH89269.1"

     CDS             complement(16783..20595)
                     /gene="traC"
                     /product="TraC DNA processing protein, DNA primase"
                     /protein_id="AGH89288.1"

     CDS             complement(34044..34391)
                     /gene="korA"
                     /product="KorA transcriptional repressor protein"
                     /protein_id="AGH89302.1"

     CDS             complement(25862..28456)
                     /gene="traI"
                     /product="TraI DNA processing protein, DNA relaxase"
                     /protein_id="AGH89293.1"

     gene            complement(28505..28858)
                     /gene="traJ"

```

https://pubmed.ncbi.nlm.nih.gov/24096417/
Appl Environ Microbiol
. 2013 Dec;79(24):7684-95. doi: 10.1128/AEM.02252-13. Epub 2013 Oct 4.
Diverse broad-host-range plasmids from freshwater carry few accessory genes
Celeste J Brown 1, Diya Sen, Hirokazu Yano, Matthew L Bauer, Linda M Rogers, Geraldine A Van der Auwera, Eva M Top
https://aem.asm.org/content/79/24/7684.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3837812/

MATERIALS AND METHODS

Host range assays.
The host range of the marked plasmids was evaluated by conjugative transfer to three bacterial strains: Pseudomonas putida UCW1gfp, Cupriavidus pinatubonensis JMP228gfp, and Sinorhizobium meliloti RM1021, belonging to the Gamma-, Beta-, and Alphaproteobacteria, respectively (36). 

Phylogenetic analysis of IncP-1 plasmids.
The part of the trfA gene that is conserved among the IncP-1 plasmids was used to infer the phylogenetic relationship of pDS1 with the other IncP-1 subgroups. 

RESULTS

Capture of plasmids from freshwater sources.

Except for pDS1 and pDS2, all plasmids showed a broad host range as they transferred and replicated in alpha-, beta-, and gammaproteobacteria (Table 2).


Table 2
General features of completely sequenced plasmids
dPlasmids pDS1 and pDS2 were transferable into E. coli but could not be transferred to P. putida, C. pinatubonensis, or S. meliloti.


A phylogeny inferred from the trfA gene showed that pDS1 is distinct from other known subgroups including the new IncP-1ζ plasmids isolated from marine biofilms (57) (Fig. 4). Surprisingly, unlike most other IncP-1 plasmids, pDS1 could not be transferred to P. putida or C. pinatubonensis (Table 2). To verify the placement of pDS1 in the IncP-1 group, an incompatibility assay was performed. The marked plasmid pDS1T was introduced into E. coli DH10B(pQKH54) by conjugation, and loss of pQKH54 was monitored when we selected for the incoming plasmid only. Nearly 80% of the clones tested had lost pQKH54 by day 1, in contrast to only a 16% loss of pQKH54 in a separate stability assay. This indicates that pDS1T is incompatible with IncP-1 plasmids and can be classified as an IncP-1 plasmid. We therefore propose the new subgroup IncP-1η for pDS1.

Fig 4
Phylogenetic relationship of pDS1 to other IncP-1 plasmid subgroups. The conserved 3′ end of the trfA gene was used to infer the phylogeny of 45 IncP-1 plasmids using the maximum-likelihood method (base frequencies, gamma shape parameter, proportion of invariant sites, and rate matrix [a b c c b a] were estimated from the data). The incompatibility subgroup of each clade is indicated on the right.

DISCUSSION

The narrow host range of the IncP-1η plasmid pDS1 was unexpected, given that most IncP-1-like plasmids have a broad host range. A preliminary plasmid genomic signature analysis suggested a signature that is quite different from the other IncP-1 plasmids captured in this study (H. Suzuki, personal communication). As pDS1 transferred very well between E. coli strains, the plasmid's transfer system seems intact. Since the molecular basis of the apparent narrow host range of this IncP-1η plasmid is of great interest, a detailed host range investigation will be the subject of a future study, combining experimental host range testing with a plasmid genomic signature analysis to assess its potential host range (81).

----------

### Leao_2013-11-07
### pMAB01
IncP-1beta

https://www.ncbi.nlm.nih.gov/bioproject/85611
https://www.ncbi.nlm.nih.gov/assembly/GCA_000363915.2

https://www.ncbi.nlm.nih.gov/nuccore/CP003376
```
LOCUS       CP003376               56267 bp    DNA     linear   BCT 26-JUL-2016
DEFINITION  Mycobacterium abscessus subsp. bolletii INCQS 00594 plasmid pMAB01,
            complete sequence.
```

https://pubmed.ncbi.nlm.nih.gov/24201191/
Genome Announc
. 2013 Nov 7;1(6):e00896-13. doi: 10.1128/genomeA.00896-13.
Draft Genome Sequence of Mycobacterium abscessus subsp. bolletii INCQS 00594
Sylvia Cardoso Leão 1, Cristianne Kayoko Matsumoto, Cristina Viana-Niero, Rommel Thiago Jucá Ramos, Adriana Ribeiro Carneiro, Maria Silvanira Barbosa, Karla Valéria Batista Lima, Maria Luiza Lopes, Vasco Azevedo, Artur Silva
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3820772/
This isolate was shown to carry a circular plasmid of the IncP-1β group, named pMAB01, which was the subject of a previous publication by Leão et al. (13).
Eighteen contigs matched sequences in the plasmid pMAB01 (GenBank accession no. CP003376), and these were not considered here. 

https://pubmed.ncbi.nlm.nih.gov/23565273/
PLoS One
. 2013;8(4):e60746. doi: 10.1371/journal.pone.0060746. Epub 2013 Apr 2.
The detection and sequencing of a broad-host-range conjugative IncP-1β plasmid in an epidemic strain of Mycobacterium abscessus subsp. bolletii
Sylvia Cardoso Leão 1, Cristianne Kayoko Matsumoto, Adriana Carneiro, Rommel Thiago Ramos, Christiane Lourenço Nogueira, James Daltro Lima Jr, Karla Valéria Lima, Maria Luiza Lopes, Horacio Schneider, Vasco Ariston Azevedo, Artur da Costa da Silva
https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0060746
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3614916/

### Popowska_2013-07

https://pubmed.ncbi.nlm.nih.gov/23471189/
Front Microbiol
. 2013 Mar 7;4:44. doi: 10.3389/fmicb.2013.00044. eCollection 2013.
Broad-host-range IncP-1 plasmids and their resistance potential
Magdalena Popowska 1, Agata Krawczyk-Balska
https://www.frontiersin.org/articles/10.3389/fmicb.2013.00044/full
MINI REVIEW article
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3590792/

These plasmids demonstrate great diversity – a phylogenetic study has shown the presence of 45 backbone genes on IncP-1 plasmids, but only 33 are shared by all the plasmids that have been completely sequenced to date.

A detailed phylogenetic analysis and study of the amino acid sequence of protein TrfA, which initiates plasmid replication, have allowed the classification of group IncP-1 into six subgroups: -α, -β, -γ, -ε, -δ (Bahl, 2009), and -ζ (Norberg et al., 2011) and also into an unnamed subgroup (Pachulec and van der Does, 2010). Unpublished data also indicate the existence of a η subgroup (Sen et al., 2012). IncP-1 plasmids have been identified in clinical and environmental, phylogenetically very distant, host bacterial species from around the world. 

----------
## 2011
----------

### 2011-11

https://pubmed.ncbi.nlm.nih.gov/21948829/
Appl Environ Microbiol
. 2011 Nov;77(22):7975-83. doi: 10.1128/AEM.05439-11. Epub 2011 Sep 23.
Broad-host-range plasmids from agricultural soils have IncP-1 backbones with diverse accessory genes
Diya Sen 1, Geraldine A Van der Auwera, Linda M Rogers, Christopher M Thomas, Celeste J Brown, Eva M Top
https://journals.asm.org/doi/10.1128/AEM.05439-11
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3209000/
ABSTRACT
Complete nucleotide sequencing followed by phylogenetic analyses based on the relaxase gene traI showed that all the plasmids belong to one of two subgroups (β and ε) of the well-studied incompatibility group IncP-1. A diverse array of accessory genes was found to be involved in resistance to antimicrobials (streptomycin, spectinomycin, and sulfonamides), degradation of herbicides (2,4-dichlorophenoxyacetic acid and 2,4-dichlorophenoxypropionic acid), and a putative new catabolic pathway. Intramolecular transposition of insertion sequences followed by deletion was found to contribute to the diversity of some of these plasmids. The previous observation that the insertion sites of a Tn501-related element are identical in four IncP-1β plasmids (pJP4, pB10, R906, and R772) was further extended to three more IncP-1β plasmids (pAKD15, pAKD18, and pAKD29). 

Test for degradation of 2,4-D. The tfd gene-bearing plasmids pAKD25 and pAKD26 were tested for their ability to confer degradation of 2,4-dichlorophenoxyacetic acid (2,4-D) to their bacterial hosts. 

Catabolic pathways encoded on four plasmids. Four of the 12 pAKD plasmids contain multiple catabolic genes, even though none was isolated from soil based on the ability to confer degradation of an organic compound. In line with several previously published IncP-1 plasmids (26, 38), at least three plasmids encode gene products known to be involved in the degradation of chlorinated aromatic compounds: (i) 2,4-dichlorophenoxyacetic acid (2,4-D) degradation (tfd genes on pAKD25 and pAKD26), (ii) chlorocatechol degradation (mocp genes on pAKD26), (iii) dichlorophenoxypropionic acid (dichlorprop) degradation (sdpA and rdpA genes on pAKD34), and (iv) a possible novel pathway of unknown function on pAKD15.
The two plasmids that carry 2,4-D degradation (tfd) genes, pAKD25 and pAKD26, were isolated from the same agricultural soil and yet belong to different IncP-1 subgroups, IncP-1ε and IncP-1β, respectively (Fig. 1). 

Nucleotide sequence accession numbers. Complete nucleotide sequences of 12 pAKD plasmids have been submitted to GenBank, and their accession numbers are as follows: pAKD1 (JN106164), pAKD14 (JN106165), pAKD15 (JN106166), pAKD16 (JN106167), pAKD17 (JN106168), pAKD18 (JN106169), pAKD25 (JN106170), pAKD26 (JN106171), pAKD29 (JN106172), pAKD31 (JN106173), pAKD33 (JN106174), and pAKD34 (JN106175).


----------
### 2011-04-05
### Norberg_2011-04-05
### pMCBF1
IncP-1zeta

https://pubmed.ncbi.nlm.nih.gov/21468020/
Nat Commun
. 2011;2:268. doi: 10.1038/ncomms1267. Epub 2011 Apr 5.
The IncP-1 plasmid backbone adapts to different host bacterial species and evolves through homologous recombination
Peter Norberg 1, Maria Bergström, Vinay Jethava, Devdatt Dubhashi, Malte Hermansson
https://www.nature.com/articles/ncomms1267
Published: 05 April 2011
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3104523/

Phylogenetic analysis of the IncP-1 backbone
A splits network (Fig. 2a) was initially constructed for 1,000 bootstrap replicates of the concatenated segments A1, A2, B and C of 24 IncP-1 plasmids (plasmid pEST4011 was excluded from the analysis as it lacks the genes in A2). The network, which presents a combinatorial generalization of phylogenetic trees, presented a star-like topology with seven main clades. pMCBF1 formed a novel clade, hereafter called ζ. As visible in a previous study26, the β-clade16 could be divided into two subclades, β-1 and β-2. Parallel edges in the phylogenetic network indicated, however, conflicting phylogenetic signals, possibly resulting from homologous recombination. In particular, in addition to plasmid pIJB1, plasmid pAOVO02 was a putative recombinant, not clustering to any of the above-described clades. A second network, excluding these two plasmids, was therefore constructed for comparison (Fig. 2b).

Figure 2
Phylogenetic analysis of the IncP-1 plasmid backbone.
(a) Phylogenetic network based on the concatenated backbone regions A, B and C of 25 IncP-1 plasmids. The network displays seven main clades, including a novel clade containing the two newly sequenced plasmids pMCBF1 (in bold) and two sub-clades, β-1 and β-2, of the previously described β-clade. The putative recent recombinant plasmids pIJB1 and pAOVO02 are marked with red ellipses. (b) Phylogenetic network excluding the putative recent recombinant plasmids pIJB1 and pAOVO02.

#<img alt="" src="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3104523/bin/ncomms1267-f2.jpg" width=50%>
#![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3104523/bin/ncomms1267-f2.jpg)

Sequences of pMCBF1 and pMCBF6 were deposited in GenBank; Nucleotide Core #(accession AY950444 and EF107516).

https://www.ncbi.nlm.nih.gov/nuccore/AY950444
```
LOCUS       AY950444               62689 bp    DNA     circular BCT 03-APR-2020
DEFINITION  Plasmid pMCBF1, complete sequence.

  ORGANISM  Plasmid pMCBF1
            other sequences; plasmids.


     CDS             1..315
                     /gene="trbA"
                     /note="regulatory protein for mating pair formation"
                     /product="TrbA"
                     /protein_id="AAY97915.1"

     CDS             complement(35267..37480)
                     /gene="traI"
                     /note="involved in conjgative DNA transfer"
                     /product="DNA relaxase"
                     /protein_id="AAY97952.1"


     CDS             complement(53521..53811)
                     /gene="korA"
                     /note="DNA binding protein involved in transcriptional
                     regulation"
                     /product="KorA"
                     /protein_id="AAY97969.1"

     CDS             complement(60971..62410)
                     /gene="trfA"
                     /note="initiation of replication"
                     /product="replication initiation protein"
                     /protein_id="AAY97979.1"


```

https://www.ncbi.nlm.nih.gov/nuccore/EF107516
```
LOCUS       EF107516               66615 bp    DNA     circular ENV 14-JUL-2016
DEFINITION  Uncultured bacterium pMCBF6 plasmid pMCBF6, complete sequence.

  ORGANISM  uncultured bacterium pMCBF6
            Bacteria; environmental samples.


     CDS             complement(39193..41406)
                     /gene="traI"
                     /note="involved in conjugative DNA transfer"
                     /product="DNA relaxase"
                     /protein_id="ABO36589.1"


     CDS             complement(64897..66336)
                     /gene="trfA"
                     /note="initiation of replication"
                     /product="replication initiation protein"
                     /protein_id="ABO36616.1"

```





----------

## 2010

### Sen_2010-03
### pAKD4
IncP-1delta

https://pubmed.ncbi.nlm.nih.gov/20018208/
Comparative Study Plasmid
. 2010 Mar;63(2):98-107. doi: 10.1016/j.plasmid.2009.11.005. Epub 2009 Dec 16.
Comparative genomics of pAKD4, the prototype IncP-1delta plasmid with a complete backbone
Diya Sen 1, Hirokazu Yano, Haruo Suzuki, Jaroslaw E Król, Linda Rogers, Celeste J Brown, Eva M Top
https://www.sciencedirect.com/science/article/abs/pii/S0147619X09001267?via%3Dihub
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2819548/

Moreover, since both pEST4011 and pIJB1 are missing a portion of their transfer genes, pAKD4 represents the first completely sequenced self-transferable plasmid with a complete IncP-1δ backbone. We therefore propose it to be the prototype IncP-1δ plasmid.

3.2.1. The replication initiation protein TrfA
An interesting feature of the IncP-1δ plasmids is the replication initiation gene trfA. In most IncP-1 plasmids known so far this gene has two in-frame translational start codons, and thus encodes a long TrfA1 and a shorter TrfA2 protein (also called TrfA-44 and TrfA-33 in IncP-1α plasmid RK2) [31]. Interestingly, since all three IncP-1δ plasmids are missing the 5′end of trfA, which encodes the start codon for and the N-terminal region (350 bp) of TrfA1, TrfA2 is the only replication initiation protein encoded by these plasmids. The 3′-end of the ssb gene, which is usually found upstream of trfA in most IncP-1 plasmids, is also missing in these plasmids. This gene codes for a single strand DNA-binding protein, which helps in DNA replication, recombination and repair. TrfA1 and Ssb proteins play a crucial role in plasmid replication in some bacterial hosts such as P. aeruginosa and may not be needed in some others such as P. putida and E. coli [32]. Therefore we postulate that IncP-1δ plasmids co-evolved with hosts that do not require TrfA1 and Ssb proteins and thereby contracted their host range. Further research is needed to test this hypothesis.

3.2.4. Phylogenetic analysis of plasmid pAKD4
To confirm that pAKD4 is an IncP-1δ plasmid, a maximum likelihood tree was inferred from concatenated sequences of 20 backbone proteins shared by all IncP-1 plasmids and pAKD4 (Fig. 3). Note that none of the trbF-P genes missing in pEST4011 were included in this list. Also, of the two sets of trbABCD genes on pIJB1, only the ones having 99% identity to pAKD4 and pEST4011 were used. This tree shows that the three IncP-1δ plasmids share a common ancestor and form a monophyletic group.

Figure 3
Phylogenetic tree showing the relationship of pAKD4 with other IncP-1 plasmids. Maximum Likelihood tree was inferred from concatenated amino acid sequences of 20 backbone proteins shared by completely sequenced IncP-1 plasmids and pAKD4. The scale bar represents the probability of amino acid substitutions per site. References for the plasmids used to construct the phylogenetic tree are given in the Materials and Methods section.

3.3. Transposable elements in pAKD4
In spite of the high similarity in plasmid backbone gene content, sequence and synteny between plasmid pAKD4 and plasmids pEST4011 and pIJB1, their accessory genes and the transposons that carry them are very different. Plasmids pEST4011 and pIJB1 both encode 2,4-dichlorophenoxyacetic acid (2,4-D) on transposons and pIJB1 potentially also biphenyl degradation (Fig. 2). In contrast, plasmid pAKD4 does not encode 2,4-D or biphenyl degradation determinants. It is the smallest of the three IncP-1δ plasmids and has relatively few accessory genes (Fig. 2). These accessory genes are described below.

3.4. Evolutionary history of the IncP-1δ plasmids
Now that we have compared the backbones and accessory elements of the three IncP-1δ plasmids known so far, the putative evolutionary mechanisms that led to these three plasmids, and their effects on the plasmid phenotypes and ecology can be discussed. All three have been isolated from soils in Europe, and the only other indication of the presence of IncP-1δ plasmids is also from Europe (Denmark) [17]. Plasmid pAKD4 was isolated by a plasmid capture method from agricultural soil in Norway, and its host in that soil is not known [19], pEST4011 was found in a 2,4-D degrading strain, Achromobacter denitrificans EST4011, isolated from agricultural soil in Estonia [16], and pIJB1 was isolated from another 2,4-D degrading strain, Burkholderia cepacia 2a, from soil in Aberdeen, Scotland [21]. Future studies should determine the geographic distribution of these plasmids worldwide.

3.5. Plasmid host range
To confirm previous findings that plasmid pAKD4 can transfer and replicate in γ- and β-Proteobacteria, but not in α-Proteobacteria [19], the host range of pAKD4 was examined again. We showed transfer of pAKD4 from its laboratory host P. putida UWC1 (γ-Proteobacteria) to P. putida KT2442::gfp (γ-Proteobacteria), E. coli CV601::gfp (γ-Proteobacteria) and C. necator JMP228 (β-Proteobacteria), but transfer to Agrobacterium tumefaciens UBAPF2::gfp (α-Proteobacteria) could not be detected (data not shown). This is in agreement with previous results [19] and confirms that pAKD4 is a broad-host-range plasmid, albeit with a narrower host range than most other IncP-1 plasmids. Future rigorous host range studies will be needed to determine to what extent truncation of ssb and trfA in the IncP-1δ plasmids, or other missing backbone genes, caused a contraction of their host range, and if natural recombination events like the one that likely resulted in plasmid pIJB1, can reverse that host range contraction.

https://www.ncbi.nlm.nih.gov/nuccore/NC_025029.1?report=genbank
Record removed. This record was removed by RefSeq staff. Please contact info@ncbi.nlm.nih.gov for further details.

https://www.ncbi.nlm.nih.gov/nuccore/GQ983559
```
LOCUS       GQ983559               56803 bp    DNA     circular ENV 25-JUL-2016
DEFINITION  Uncultured bacterium pAKD4 plasmid pAKD4, complete sequence.

                     /note="IncP-1 delta plasmid"
     gene            complement(1..870)
                     /gene="trfA2"
     CDS             complement(1..870)
                     /gene="trfA2"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfA2 replication initiation protein"
                     /protein_id="ADD63272.1"

     gene            1191..1568
                     /gene="trbA"


     CDS             1191..1568
                     /gene="trbA"
                     /codon_start=1
                     /transl_table=11
                     /product="TrbA mating pair formation protein,
                     transcription regulator"
                     /protein_id="ADD63273.1"

     CDS             complement(35443..37641)
                     /gene="traI"
                     /codon_start=1
                     /transl_table=11
                     /product="TraI conjugative transfer protein, DNA relaxase"
                     /protein_id="ADD63310.1"


     CDS             complement(45051..45359)
                     /gene="korA"
                     /codon_start=1
                     /transl_table=11
                     /product="KorA transcriptional repressor protein"
                     /protein_id="ADD63322.1"

```


----------

## 2007

### Schluter_2007

https://pubmed.ncbi.nlm.nih.gov/17553065/
Review FEMS Microbiol Rev
. 2007 Jul;31(4):449-77. doi: 10.1111/j.1574-6976.2007.00074.x. Epub 2007 Jun 6.
Genomics of IncP-1 antibiotic resistance plasmids isolated from wastewater treatment plants provides evidence for a widely accessible drug resistance gene pool
Andreas Schlüter 1, Rafael Szczepanowski, Alfred Pühler, Eva M Top
https://onlinelibrary.wiley.com/doi/full/10.1111/j.1574-6976.2007.00074.x
https://academic.oup.com/femsre/article/31/4/449/2399130



----------

### 2016-01
### Li_2016-01
IncP-1epsilon

https://pubmed.ncbi.nlm.nih.gov/26635412/
FEMS Microbiol Ecol
. 2016 Jan;92(1):fiv151. doi: 10.1093/femsec/fiv151. Epub 2015 Dec 2.
Diversification of broad host range plasmids correlates with the presence of antibiotic resistance genes
Xiaobin Li 1, Yafei Wang 2, Celeste J Brown 3, Fei Yao 1, Yong Jiang 2, Eva M Top 3, Hui Li 4
https://academic.oup.com/femsec/article/92/1/fiv151/2467447
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4809973/
Abstract
The IncP-1ε plasmids can be clearly subdivided into two subclades, designated as ε-I and ε-II, based on phylogenetic analysis of backbone proteins TraI and TrfA. 


INTRODUCTION

Four IncP-1ε plasmids carried catabolic pathway genes responsible for herbicide degradation, with plasmids pAKD25, p712 and pEMT3 being associated with 2,4-dichlorophenoxyacetic acid degradation, and pAKD34 carrying putative 2,4-dichlorophenoxypropionic acid-degrading genes (Top, Holben and Forney 1995; Sen et al. 2011; Kim et al. 2013).

The partial genome sequences of these plasmids were determined by Illumina sequencing, and the backbone regions were completely assembled.

MATERIALS AND METHODS
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4809973/#sec2title

Host range test
To determine the host range of captured plasmids, the plasmids were transferred from E. coli EC100 (γ-Proteobacteria, SmR) to Agrobacterium tumefaciens C58 (α-Proteobacteria, RifR) (Wood et al. 2001) and Cupriavidus necator JMP228 (β-Proteobacteria, RifR) (Top, Holben and Forney 1995) by biparental matings as described previously (Binh et al. 2008).

Sequencing and annotation

The backbone genes of four plasmids (pSFS12, pSFS26, pSFS52 and pSFW53) were deposited in GenBank under accession numbers KM655308 to KM655311.
https://www.ncbi.nlm.nih.gov/nuccore/?term=KM655308:KM655311[pacc]

Bioinformatic analysis and software

RESULTS
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4809973/#sec3title

To determine the host range of the four plasmids, we tested their ability to transfer from the γ-Proteobacterium E. coli and replicate in A. tumefaciens C58 and C. necator JMP228, members of the α- and β-Proteobacteria, respectively. These host range tests indicated that the four plasmids were self-transferrable and could replicate in the tested strains of α-, β- and γ-Proteobacteria.


Table 2.
Structural organization of the four IncP-1ε plasmids isolated in this study.

Although the sequences of these four plasmids were not completely closed, the fully assembled backbone regions enabled us to infer the insertion sites of the partially sequenced accessory elements (Fig. 1 and Fig. S2, Supporting Information).


Phylogenetic analysis of selected IncP-1 plasmids based on TraI and TrfA


ε-I subclade
pSFS26 and pSFS52

pSFS26
https://www.ncbi.nlm.nih.gov/nuccore/KM655309
```
LOCUS       KM655309               48459 bp    DNA     linear   ENV 30-JUN-2015
DEFINITION  Uncultured bacterium plasmid pSFS26, partial sequence.

     CDS             complement(1241..2464)
                     /gene="trfA"
                     /product="TrfA"
                     /protein_id="AKI06452.1"


     CDS             complement(36128..38350)
                     /gene="traI"
                     /product="TraI"
                     /protein_id="AKI06490.1"


     CDS             complement(47994..48428)
                     /gene="klcA"
                     /product="KlcA"

ORIGIN      
```

pSFS52
"trfA"が最初に配置されていない。"putative DNA topoisomerase"が分割されている。
https://www.ncbi.nlm.nih.gov/nuccore/KM655310
```
LOCUS       KM655310               45002 bp    DNA     linear   ENV 30-JUN-2015
DEFINITION  Uncultured bacterium plasmid pSFS52, partial sequence.

     CDS             1838..2113
                     /product="hypothetical protein"

     CDS             complement(2233..3120)
                     /product="putative DNA topoisomerase"

     CDS             complement(3092..4297)
                     /product="putative DNA topoisomerase"

     CDS             complement(4312..4839)
                     /gene="traF"


     CDS             complement(6737..8959)
                     /gene="traI"
                     /product="TraI"
                     /protein_id="AKI06514.1"


     CDS             complement(20229..21452)
                     /gene="trfA"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfA"
                     /protein_id="AKI06533.1"


     CDS             complement(43130..44932)
                     /gene="traC"

ORIGIN      
```

ε-II subclade
pSFS12 and pSFW53

pSFS12
https://www.ncbi.nlm.nih.gov/nuccore/KM655308
```
LOCUS       KM655308               38744 bp    DNA     linear   ENV 30-JUN-2015
DEFINITION  Uncultured bacterium plasmid pSFS12, partial sequence.

     CDS             complement(17..388)
                     /product="hypothetical protein"

     CDS             complement(450..1685)
                     /gene="trfA"
                     /product="TrfA"
                     /protein_id="AKI06408.1"


     CDS             complement(26437..28659)
                     /gene="traI"
                     /product="TraI"
                     /protein_id="AKI06434.1"


     CDS             complement(38239..38718)
                     /gene="klcA"
                     /product="KlcA"

ORIGIN      
```

pSFW53
https://www.ncbi.nlm.nih.gov/nuccore/KM655311
```
LOCUS       KM655311               40089 bp    DNA     linear   ENV 30-JUN-2015
DEFINITION  Uncultured bacterium plasmid pSFW53, partial sequence.

     CDS             95..223
                     /product="hypothetical protein"

     CDS             complement(413..1648)
                     /gene="trfA1"
                     /product="TrfA1"
                     /protein_id="AKI06560.1"


     CDS             complement(27157..29379)
                     /gene="traI"
                     /product="TraI"
                     /protein_id="AKI06587.1"


     CDS             complement(38992..39471)
                     /gene="klcA"
                     /product="KlcA"

     CDS             39688..39906
                     /gene="dinJ"
                     /product="DinJ"

ORIGIN      
```

### Oliveira_2013-11

https://pubmed.ncbi.nlm.nih.gov/23831558/
Comparative Study Plasmid
. 2013 Nov;70(3):412-9. doi: 10.1016/j.plasmid.2013.06.002. Epub 2013 Jul 2.
Comparative genomics of IncP-1ε plasmids from water environments reveals diverse and unique accessory genetic elements
Cláudia S Oliveira 1, Alexandra Moura, Isabel Henriques, Celeste J Brown, Linda M Rogers, Eva M Top, António Correia
https://linkinghub.elsevier.com/retrieve/pii/S0147-619X(13)00076-0

### 2015-03-17

https://pubmed.ncbi.nlm.nih.gov/25333461/
ISME J
. 2015 Mar 17;9(4):934-45. doi: 10.1038/ismej.2014.191.
Broad host range plasmids can invade an unexpectedly diverse fraction of a soil bacterial community
Uli Klümper 1, Leise Riber 2, Arnaud Dechesne 1, Analia Sannazzarro 2, Lars H Hansen 3, Søren J Sørensen 2, Barth F Smets 1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4817699/

Construction of gfpmut3-tagged plasmid pKJK5
Plasmids RP4 and pIPO2tet have been constructed earlier (Musovic et al., 2010, 2014). The 54 kbp IncP-1 plasmid, pKJK5, originally isolated from a soil/manure environment, harbors a tetracycline and a trimethoprim resistance determinant, and a class 1 integron (Sengeløv et al., 2001). 

Figure 4
Phylogenetic tree showing all identified transconjugant OTUs for three different plasmids (pKJK5, RP4 and pIPO2tet) from the same donor (P. putida). 

The core super-permissive community consists mainly of diverse Proteobacteria such as Enterobacteriales (γ), Burkholderiales (β), Pseudomonadales (γ) and Rhizobiales (α) (Figure 4). 

These species are mainly found within Enterobacteriales (Gammaproteobacteria), Burkholderiales (Betaproteobacteria) and Staphylococci (Bacilli), groups that contain most of our super-permissive OTUs.


### pKJK5
IncP-1epsilon

### Bahl_2007-07

https://pubmed.ncbi.nlm.nih.gov/17306874/
Plasmid
. 2007 Jul;58(1):31-43. doi: 10.1016/j.plasmid.2006.11.007. Epub 2007 Feb 16.
The multiple antibiotic resistance IncP-1 plasmid pKJK5 isolated from a soil environment is phylogenetically divergent from members of the previously established alpha, beta and delta sub-groups
Martin Iain Bahl 1, Lars Hestbjerg Hansen, Alexander Goesmann, Søren J Sørensen
https://linkinghub.elsevier.com/retrieve/pii/S0147-619X(06)00116-8
https://www.sciencedirect.com/science/article/abs/pii/S0147619X06001168?via%3Dihub
https://www.sciencedirect.com/science/article/pii/S0147619X06001168?via%3Dihub
Abstract

The 54,383bp plasmid pKJK5 was recovered from a soil environment by exogenous plasmid isolation and conveys resistance towards tetracycline and trimethoprim. Sequencing and annotation revealed a high level of structural similarity of the backbone genes to other IncP-1 plasmids containing a Tra1 and Tra2 region, a central control module and a replication initiation module. 

Individual backbone genes in pKJK5 have a GC-content, which is consistently lower (average 6.3%) than the homologous genes from the archetype IncP-1β plasmid R751 indicating homogenous amelioration of IncP-1 plasmid backbone genes. 

Load 1 consists of an insertion sequence ISPa17 and load 2 is a Tn402-derivative containing a class 1 integron, IS1326 and a fragment identical to a region of plasmid pTB11 harboring a tetracycline resistance determinant and part of an IncP-1α oriV region.


2. Materials and methods

2.3. Phenotypic characterization of plasmid pKJK5 and plasmid DNA isolation

Furthermore the ability of pKJK5 to transfer within and between different sub-divisions of proteobacteria was tested using donor E. coli CSH26 (γ-proteobacteria) and recipients Pseudomonas putida (γ-proteobacteria) and Ochrobactrum sp. (α-proteobacteria), respectively, by a slightly modified standard 24 h filter mating procedure (Sengeløv et al., 2001). 

3. Results and discussion

3.1. Multiple antibiotic resistance and a broad host range is associated with plasmid pKJK5

The ability of pKJK5 to be transferred to and stably maintained in the same and other sub-divisions of proteobacteria was exemplified by P. putida and Ochrobactrum sp., which belong to the gamma and alpha subdivision respectively. Thus pKJK5 is defined as a broad-host-range plasmid (Szpirer et al., 1999). A more extensive analysis of the natural host-range of pKJK5 will be published elsewhere.

3.2.1. 

One such mechanism could be to gradually adjust the GC-content of the plasmid to better suit the host cell.

3.5. The GC-content of the individual backbone genes in pKJK5 is lower than that of the homologous genes in prototype IncP-1β plasmid R751, which indicates a homogeneous amelioration of IncP-1 plasmids across their entire backbone sequence

Fig. 4. Comparison between the mol% of GC in all backbone genes of plasmid R751 (black line) and pKJK5 (gray line). The percentage of GC-content increases from the centre (50% GC-content) towards the outer perimeter (75% GC-content) of the circular plot.

### Bahl_2007-01

https://pubmed.ncbi.nlm.nih.gov/17132149/
FEMS Microbiol Lett
. 2007 Jan;266(2):250-6. doi: 10.1111/j.1574-6968.2006.00536.x. Epub 2006 Nov 21.
Impact of conjugal transfer on the stability of IncP-1 plasmid pKJK5 in bacterial populations
Martin Iain Bahl 1, Lars Hestbjerg Hansen, Søren J Sørensen
https://academic.oup.com/femsle/article/266/2/250/565715

https://www.ncbi.nlm.nih.gov/nuccore/AM261282
```
LOCUS       AM261282               54383 bp    DNA     circular ENV 14-JUL-2016
DEFINITION  Uncultured bacterium IncP-1 plasmid pKJK5 complete sequence.


     CDS             complement(1315..2538)
                     /gene="trfA"
                     /locus_tag="pKJK5_1"
                     /function="initiation of replication"
                     /product="replication initiation protein"
                     /protein_id="CAK02642.1"
                     /db_xref="InterPro:IPR010751"
                     /db_xref="UniProtKB/TrEMBL:Q0VUX3"


     CDS             complement(29679..30329)
                     /gene="tetR"
                     /locus_tag="pKJK5_36"
                     /function="transcriptional regulation of tetA"
                     /product="tetracycline resistance repressor protein"

     CDS             30435..31634
                     /gene="tetA"
                     /locus_tag="pKJK5_37"
                     /function="confers tetracycline resistance"
                     /product="tetracycline efflux protein"



     CDS             complement(42140..44362)
                     /gene="traI"
                     /locus_tag="pKJK5_46"
                     /function="DNA relaxase"
                     /product="TraI protein"
                     /protein_id="CAK02686.1"
                     /db_xref="InterPro:IPR005094"
                     /db_xref="UniProtKB/TrEMBL:Q0VUS9"


     CDS             complement(50794..51096)
                     /gene="korA"
                     /locus_tag="pKJK5_56"
                     /function="putative transcriptional repressor protein"
                     /product="KorA protein"
                     /protein_id="CAK02698.1"

     CDS             3043..3417
                     /gene="trbA"
                     /locus_tag="pKJK5_3"
                     /function="transcriptional regulation"
                     /product="transcriptional regulator protein"
                     /protein_id="CAK02644.1"
                     /db_xref="GOA:Q0VUX1"
                     /db_xref="InterPro:IPR001387"
                     /db_xref="InterPro:IPR010982"
                     /db_xref="UniProtKB/TrEMBL:Q0VUX1"


```

----------

## 2006

### Kamachi_2006-12
### pBP136
IncP-1beta1

https://www.ncbi.nlm.nih.gov/nuccore/NC_008459
https://www.ncbi.nlm.nih.gov/nuccore/AB237782
```
LOCUS       AB237782               41268 bp    DNA     circular BCT 14-JUL-2016
DEFINITION  Bordetella pertussis plasmid pBP136 DNA, complete sequence.

     CDS             complement(9526..10749)
                     /gene="trfA"
                     /function="DNA-binding protein for plasmid replication"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfA"
                     /protein_id="BAF33454.1"

```

https://pubmed.ncbi.nlm.nih.gov/17159199/
Microbiology (Reading)
. 2006 Dec;152(Pt 12):3477-3484. doi: 10.1099/mic.0.29056-0.
Plasmid pBP136 from Bordetella pertussis represents an ancestral form of IncP-1beta plasmids without accessory mobile elements
Kazunari Kamachi 1, Masahiro Sota 2, Yuji Tamai 3, Noriyo Nagata 4, Toshifumi Konda 1, Toshiro Inoue 3, Eva M Top 2, Yoshichika Arakawa 1
https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.29056-0#tab2

### pQKH54
IncP-1gamma
### Haines_2006-09

https://pubmed.ncbi.nlm.nih.gov/16946264/
Microbiology (Reading)
. 2006 Sep;152(Pt 9):2689-2701. doi: 10.1099/mic.0.28941-0.
Plasmids from freshwater environments capable of IncQ retrotransfer are diverse and include pQKH54, a new IncP-1 subgroup archetype
Anthony S Haines 1, Parveen Akhtar 1, Elton R Stephens 1, Karen Jones 1, Christopher M Thomas 1, Caroline D Perkins 2, Jacqueline R Williams 2, Martin J Day 2, John C Fry 2
https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.28941-0#tab2

RESULTS

Plasmid host range

Host range was tested using P. putida PaW340 as donor against the various representative species listed in Table 1T1 as recipients: Escherichia coli, Alcaligenes eutrophus, Agrobacterium tumefaciens and Cytophaga heparina. Transfer to the C. heparina strain could not be detected for any of the plasmids tested (Table 3T3). However, the RP4 derivative plasmid pQM899 and the IncP-1β plasmids showed a broad host range, being transferred and maintained in all subgroups of the Proteobacteria. Epilithic plasmids pQKH6 and pQM719 showed a pattern of transfer similar to that of the IncP-1β plasmids pJP4 and pSS50, with higher transfer frequencies to A. tumefaciens. The transfer frequency of pQKH54 was somewhat lower to this alphaproteobacterial species but high transfer frequencies, as shown by pQKH6 and pQM719, were detected to the gamma- and betaproteobacteria subgroups. The remaining plasmid tested, pQKH33, only showed a higher transfer frequency to P. putida, with transfer to the other strains at or below the limit of detection.


DNA sequencing and the IncP-1 backbone of pQKH54

The G+C content of the backbone is generally lower than for the other subgroups. If the trb region is excluded beyond the end of trbL (as the majority of these genes are unique to pQKH54), then the mean G+C content of the backbone is 56.5 mol%, while it is 63.1 mol% for RK2 (Pansegrau et al., 1994), 66.6 mol% for R751 (Thorsted et al., 1998) and 62.0 mol% for pEST4011 (Vedler et al., 2004). Thus it appears that pQKH54 has spent a considerable proportion of its recent evolutionary history in species with a lower G+C content than the other branches of the IncP-1 family. 
バックボーンのG+C含量は他のサブグループと比較して一般に低い。trbL末端からtrb領域を除いた場合（これらの遺伝子の大部分はpQKH54に固有のものであるため）、バックボーンの平均G+C含量は56.5 mol%であるが、RK2 (Pansegrau et al., 1994) では 63.1 mol%、R751 (Thorsted et al., 1998) では 66.6 mol%、pEST4011 (Vedler et al., 2004) では 62.0 mol% であった。したがって、pQKH54は最近の進化の歴史のかなりの部分を、IncP-1ファミリーの他の枝よりも低いG+C含量の種で過ごしてきたようである。


DISCUSSION

While pQKH54 clearly belongs to the IncP-1 family it is distinct from IncP-1β plasmids and appears to be somewhat less broad host range than pQKH6 and pQM719. This would also be consistent with the lower G+C content of the genome since it suggests adaptation to a different group of hosts. 
pQKH54は明らかにIncP-1ファミリーに属するが、IncP-1βプラスミドとは異なり、pQKH6やpQM719よりも宿主範囲がやや狭いようである。このことは、ゲノムのG+C含量が低いこととも一致し、異なる宿主群に適応していることを示唆していると思われる。

Fig. 2.
Sequence logos of iterons for representatives of the four known IncP-1 subgroups. In a DNA sequence logo (Schneider & Stephens, 1990; Crooks et al., 2004), the height of each base represents its conservation, with the information content of each position measured in bits. There are some minor variations between the groups, which may to some extent be due to neutral mutations. However, the consensus for pQKH54 shows a very clear difference of the last two bases. This suggests that this replicon is the most distant member of the IncP-1 group yet found, confirming the phylogenetic tree (Fig. 3F3), and possibly explaining the only partial incompatibility phenotype against RP4.

![](https://www.microbiologyresearch.org/docserver/ahah/fulltext/micro/152/9/2689fig2_thmb.gif)

Fig. 3.
Phylogenetic tree for IncP-1 plasmids currently available in public databases, using several proteins. Primary sequences of KorA, IncC2, KorB, KorC and KfrC (Upf54.4) were joined together and then aligned for IncP-1 plasmids plus a plasmid of the IncU group, which carries versions of these genes (Rhodes et al., 2004). The alignment was then used to build the phylogeny. The plasmids pB10 and pJP4 are not included since they appear to be chimeric products of recombination between pB4-like and R751 subgroup plasmids.

![](https://www.microbiologyresearch.org/docserver/ahah/fulltext/micro/152/9/2689fig3_thmb.gif)

Table 2.

Plasmid	Size (kb)	Known characteristics	Known host range*
pQKH54	80	Hgr Tra+	ABG

*16S rRNA classification subgroups of Proteobacteria to which plasmids can transfer. BHR, broad host range, can be transferred to a wide range of Gram-negative bacteria (A=α, B=β, and G=γ 16S rRNA subgroups of the Proteobacteria listed in Table 1 T1 ). nd, Not determined.


Table 3.
Summary of characteristics of plasmids studied in this work

Plasmid	Hybridization* R751 probes	Hybridization† pQKH6 probes	IncP-1 by replicon probing‡	Excluded by/excludes RP4§	PCR trfA/korA||	IncP-1β by restriction profile¶	Transfer/Retrotransfer/Mobilization#	Host range††

pQKH54	bde	ABCDE	−	+/+	−/−	−	100/10−5/10−2**	aBG

††Host range is indicated in the same way as in Table 2 T2 . Upper-case letters are used when tested species/strains of that group could receive the plasmid; lower-case letters indicate that transfer frequency into the test strain for that group is significantly reduced. The host range of RP4 was not tested in this study.

https://www.ncbi.nlm.nih.gov/nuccore/AM157767
```
LOCUS       AM157767               69966 bp    DNA     circular ENV 26-JUL-2016
DEFINITION  Uncultured bacterium IncP-1gamma plasmid QKH54 complete genome.

     protein_bind    8892..8906
                     /note="iteron"
                     /bound_moiety="trfA"
     protein_bind    8941..8955
                     /note="iteron"
                     /bound_moiety="trfA"

     CDS             complement(23120..24241)
                     /gene="trfA1"
                     /operon="ssb-trfA"
                     /codon_start=1
                     /transl_table=11
                     /product="trans-acting replication factor A"
                     /protein_id="CAJ43307.1"
                     /db_xref="InterPro:IPR010751"
                     /db_xref="UniProtKB/TrEMBL:Q1H9T9"

     CDS             complement(24282..24641)
                     /gene="ssb"
                     /operon="ssb-trfA"



     CDS             complement(4959..5261)
                     /gene="korA"
                     /operon="korAB-incC1/2"
                     /codon_start=1
                     /transl_table=11
                     /product="KorA repressor protein"
                     /protein_id="CAJ43288.1"
                     /db_xref="UniProtKB/TrEMBL:Q1H9Z5"
                     /translation="MKKRLTEAEFQAALSGLDVGHQTVEIARGVLVQGRAQAEFAEAL
                     NLSKGAVSQAVNRVWAAHEAQVPPGCERLTVLLPKHQAFIVKKWASEFARKREAKA"
     regulatory      complement(5335..5340)
                     /regulatory_class="minus_10_signal"
                     /operon="korAB-incC1/2"
     protein_bind    5336..5347
                     /standard_name="OA1"
                     /bound_moiety="KorA"


     CDS             24803..25120
                     /gene="trbA"
                     /operon="trbA"
                     /codon_start=1
                     /transl_table=11
                     /product="TrbA repressor protein"
                     /protein_id="CAJ43309.1"
                     /db_xref="GOA:Q1H9V4"
                     /db_xref="InterPro:IPR001387"
                     /db_xref="InterPro:IPR010982"
                     /db_xref="UniProtKB/TrEMBL:Q1H9V4"
                     /translation="MTKSELAAKADMSVSFLSDLTTGKANPSLKIMESIGTALETPLP
                     MLLESSDLDREALDAIAGGKMPQSLPKGFVRICAVLPEAKAYVVRKWEEEAKKKMQEK
                     GRP"



     CDS             complement(65323..67731)
                     /gene="traI"
                     /operon="traJXHI"
                     /codon_start=1
                     /transl_table=11
                     /product="TraI DNA relaxase"
                     /protein_id="CAJ43356.1"

```

### Harada_2006-07
### pA1
IncP-1beta2

https://pubmed.ncbi.nlm.nih.gov/16445980/
Plasmid
. 2006 Jul;56(1):11-23. doi: 10.1016/j.plasmid.2005.12.002. Epub 2006 Jan 30.
Sequence and analysis of the 46.6-kb plasmid pA1 from Sphingomonas sp. A1 that corresponds to the typical IncP-1beta plasmid backbone without any accessory gene
Karen Mine Harada 1, Yuji Aso, Wataru Hashimoto, Bunzo Mikami, Kousaku Murata
https://www.sciencedirect.com/science/article/abs/pii/S0147619X05001289?via%3Dihub

https://www.ncbi.nlm.nih.gov/nuccore/NC_007353

https://www.ncbi.nlm.nih.gov/nuccore/AB231906
```
LOCUS       AB231906               46557 bp    DNA     circular BCT 26-JUL-2016
DEFINITION  Sphingomonas sp. A1 plasmid pA1 DNA, complete sequence.

            Bacteria; Proteobacteria; Alphaproteobacteria; Sphingomonadales;
            Sphingomonadaceae; Sphingomonas.


     CDS             complement(13771..16038)
                     /gene="traI"
                     /function="DNA relaxase"
                     /product="TraI"
                     /protein_id="BAE19709.1"


     protein_bind    29970..29984
                     /bound_moiety="TrfA binding"

                     /bound_moiety="TrfA binding"
     repeat_region   complement(31061..31080)
                     /note="palindromic sequence"


     CDS             complement(23035..23343)
                     /gene="korA"
                     /function="Transcriptional repressor"
                     /product="KorA"
                     /protein_id="BAE19720.1"


     CDS             complement(31316..32542)
                     /gene="trfA"
                     /function="Replication initiation protein"
                     /product="TrfA"
                     /protein_id="BAE19726.2"


     CDS             complement(32590..32928)
                     /gene="ssb"
                     /function="Single-strand binding protein"
                     /product="Ssb"
                     /protein_id="BAE19727.1"

     CDS             33091..33396
                     /gene="trbA"
                     /function="Transcriptional regulator"
                     /product="TrbA"
                     /protein_id="BAE19728.1"

```

## 2011

### 2011-08-20

### pB5
IncP-1alpha

https://pubmed.ncbi.nlm.nih.gov/21115076/
J Biotechnol
. 2011 Aug 20;155(1):95-103. doi: 10.1016/j.jbiotec.2010.11.018. Epub 2010 Nov 27.
Sequencing and comparative analysis of IncP-1α antibiotic resistance plasmids reveal a highly conserved backbone and differences within accessory regions
Rafael Szczepanowski 1, Felix Eikmeyer, Jannis Harfmann, Jochen Blom, Linda M Rogers, Eva M Top, Andreas Schlüter
https://www.sciencedirect.com/science/article/abs/pii/S0168165610020511?via%3Dihub

The three IncP-1α plasmids pB5, pB11 and pSP21 were isolated from bacteria of two different sewage treatment plants and sequenced by a combination of next-generation and capillary sequencing technologies. A comparative analysis including the previously analysed IncP-1α plasmids RK2, pTB11 and pBS228 revealed a highly conserved plasmid backbone (at least 99.9% DNA sequence identity) comprising 54 core genes. The accessory elements of the plasmid pB5 constitute a class 1 integron interrupting the parC gene and an IS6100 copy inserted into the integron. In addition, the tetracycline resistance genes tetAR and the ISTB11-like element are located between the klc operon and the trfA–ssb operon. 


https://www.ncbi.nlm.nih.gov/nuccore/CP002151
```
LOCUS       CP002151               64696 bp    DNA     circular ENV 25-JUL-2016
DEFINITION  Uncultured bacterium plasmid PB5, complete sequence.



     CDS             complement(1..1149)
                     /gene="trfA1"
                     /locus_tag="PB5_0001"
                     /note="alternative; initiation of plasmid replication,
                     essential for replication in Pseudomonas aeruginosa; long
                     form TrfA1, alternative start codon"
                     /product="TrfA transcriptional repressor protein, long
                     form"
                     /protein_id="ADU90710.1"

     CDS             complement(1..858)
                     /gene="trfA2"
                     /locus_tag="PB5_0002"
                     /note="alternative; initiation of plasmid replication,
                     essential for replication in Pseudomonas aeruginosa; long
                     form TrfA1, alternative start codon"
                     /product="TrfA transcriptional repressor protein, short
                     form"
                     /protein_id="ADU90711.1"


     CDS             complement(35798..37996)
                     /gene="traI"
                     /locus_tag="PB5_0053"
                     /note="Endonuclease relaxase, MobA/VirD2,
                     Relaxase/Mobilisation nuclease domain, involved in
                     conjugative DNA transfer; mediates cleavage at the nick
                     site of oriT"
                     /product="conjugal transfer relaxase TraI"
                     /protein_id="ADU90756.1"


     CDS             complement(55740..56417)
                     /gene="tetR"
                     /locus_tag="PB5_0081"
                     /note="Tetracycline transcriptional regulator, TetR-like,
                     C-terminal all-alpha domain; Transcriptional regulator"
                     /product="tetracycline resistance repressor protein"
                     /protein_id="ADU90784.1"

     CDS             56496..57695
                     /gene="tetA"
                     /locus_tag="PB5_0082"
                     /note="Tetracycline resistance protein, class B (TetA(B))
                     (Metal-tetracycline/H(+) antiporter), Major facilitator
                     superfamily, 2_A_01_02: Multidrug resistance protein;
                     Arabinose efflux permease"
                     /product="tetracycline efflux protein"

```

## 2005

### 2005-05
### pTB11
IncP-1alpha

https://pubmed.ncbi.nlm.nih.gov/15848226/
Comparative Study Plasmid
. 2005 May;53(3):218-38. doi: 10.1016/j.plasmid.2004.09.004. Epub 2004 Dec 9.
Sequence of the 68,869 bp IncP-1alpha plasmid pTB11 from a waste-water treatment plant reveals a highly conserved backbone, a Tn402-like integron and other transposable elements
Thomas Tennstedt 1, Rafael Szczepanowski, Irene Krahn, Alfred Pühler, Andreas Schlüter
https://www.sciencedirect.com/science/article/abs/pii/S0147619X04001027?via%3Dihub

In contrast to the ‘Birmingham’ plasmids the pTB11 tetracycline resistance module carries a pecM- and a pncA-like gene downstream of the tetracycline resistance gene tetA and contains an insertion of the new insertion sequence element ISTB11. 

https://www.ncbi.nlm.nih.gov/nuccore/AJ744860
```
LOCUS       AJ744860               68869 bp    DNA     circular ENV 26-JUL-2016
DEFINITION  uncultured bacterium pTB11 plasmid complete genome.

     CDS             complement(3826..4476)
                     /gene="tetR"
                     /note="transcriptional regulation of tetA expression"
                     /product="tetracycline repressor protein"

     CDS             4582..5781
                     /gene="tetA"
                     /note="confers tetracycline resistance"
                     /product="tetracyclin efflux protein"


     CDS             complement(12783..13931)
                     /gene="trfA1"
                     /note="alternative;
                     initiation of plasmid replication"
                     /product="transcriptional repressor protein"
                     /protein_id="CAG30857.1"


     CDS             complement(52751..54949)
                     /gene="traI"
                     /note="involved in conjugative DNA transfer; mediates
                     cleavage at the nick site of oriT"
                     /product="DNA relaxase"

```

### 2005-01

https://pubmed.ncbi.nlm.nih.gov/15551059/
Arch Microbiol
. 2005 Jan;183(1):27-36. doi: 10.1007/s00203-004-0742-9. Epub 2004 Nov 13.
The genome sequence of an anaerobic aromatic-degrading denitrifying bacterium, strain EbN1
Ralf Rabus 1, Michael Kube, Johann Heider, Alfred Beck, Katja Heitmann, Friedrich Widdel, Richard Reinhardt
https://link.springer.com/article/10.1007/s00203-004-0742-9
Published: 13 November 2004
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_proquest_miscellaneous_67341160

https://www.ncbi.nlm.nih.gov/nuccore/CR555308
```
LOCUS       CR555308              223670 bp    DNA     circular BCT 26-JUL-2016
DEFINITION  Azoarcus sp. EbN1 plasmid 2.


     CDS             9849..10451
                     /locus_tag="p2A17"
                     /product="hypothetical protein"
                     /protein_id="CAI10535.1"


     CDS             11073..11573
                     /gene="tnp2PF1"
                     /product="putative transposase subunit, fragment"
                     /protein_id="CAI10536.1"


     CDS             12049..12411
                     /gene="trbA"
                     /product="TrbA protein of DNA transfer system"
                     /protein_id="CAI10537.1"


     CDS             complement(55761..57983)
                     /gene="traI"
                     /product="TraI protein of DNA transfer system"
                     /protein_id="CAI10575.1"

```


## 2004

### Vedler_2004-11
### pEST4011
IncP-1delta

https://www.ncbi.nlm.nih.gov/nuccore/AY540995
```
LOCUS       AY540995               76958 bp    DNA     circular BCT 03-APR-2020
DEFINITION  Achromobacter xylosoxidans subsp. denitrificans strain EST4002
            plasmid pEST4011, complete sequence.

  ORGANISM  Achromobacter denitrificans
            Bacteria; Pseudomonadota; Betaproteobacteria; Burkholderiales;
            Alcaligenaceae; Achromobacter.

     misc_feature    8790..9592
                     /note="vegetative replication origin, oriV"
     protein_bind    8790..8810
                     /note="iteron 11"
                     /bound_moiety="TrfA"

     protein_bind    complement(60176..60196)
                     /note="iteron 1"
                     /bound_moiety="TrfA"

     CDS             complement(60241..61110)
                     /gene="trfA"
                     /note="initiation of vegetative replication; DNA binding
                     at oriV; similar to Enterobacter aerogenes plasmid R751
                     TrfA2 of NC_001735"
                     /product="TrfA"
                     /protein_id="AAS49458.1"

     CDS             complement(71647..73839)
                     /gene="traI"
                     /note="DNA relaxase; nicks the DNA strand to be
                     transferred"
                     /product="TraI"
                     /protein_id="AAS49470.1"



```

https://pubmed.ncbi.nlm.nih.gov/15489427/
J Bacteriol
. 2004 Nov;186(21):7161-74. doi: 10.1128/JB.186.21.7161-7174.2004.
The completely sequenced plasmid pEST4011 contains a novel IncP1 backbone and a catabolic transposon harboring tfd genes for 2,4-dichlorophenoxyacetic acid degradation
Eve Vedler 1, Merle Vahter, Ain Heinaru
https://jb.asm.org/content/186/21/7161.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC523222/
We show that despite the overall similarity in plasmid organization, the pEST4011 backbone is sufficiently different (51 to 86% amino acid sequence identity between individual backbone genes) from the backbones of members of the three IncP1 subgroups (theα , β, and γ subgroups) that it belongs to a new IncP1subgroup, the δ subgroup. This conclusion was also supported by a phylogenetic analysis of the trfA2, korA, and traG gene products of different IncP1 plasmids.

The bootstrapped neighbor-joining trees derived from these alignments (obtained by using the program CLUSTALX) (Fig. ​(Fig.7)7) showed that pEST4011 TrfA, KorA, and TraG are distinct from all other corresponding sequences and that they occupy individual branches of each tree. Similar results were obtained with the different PHYLIP programs used (data not shown). As C. M. Thomas and coworkers have assigned plasmid pQKH54 (14) to the γ subgroup (personal communication) and the amino acid sequence of TrfA of pQKH54 appeared to be different from the pEST4011 TrfA sequence (unpublished data), we placed pEST4011 in a new IncP1 subgroup, the δ subgroup. The other four IncP1 catabolic plasmids sequenced, pJP4, pADP-1, pUO1, and pTSA, together with pIJB1, possess the well-conserved IncP1 subgroup β backbone. The TrfA2 phylogenetic tree (Fig. ​(Fig.7A)7A) shows that the 2,4-D-degradative plasmid pEMT3 is also different from all other IncP1 plasmids and could also form a separate subgroup. Unfortunately, the incompatibility group of the 2,4-D-degradative plasmid pTV1 is unknown (T. Vallaeys, personal communication).

FIG. 7.
Bootstrapped neighbor-joining trees derived from multiple alignments of trfA2 (A), korA (B), and traG (C) gene products of different IncP1 plasmids, constructed with the CLUSTALX program. The GenBank accession numbers of the proteins are indicated in parentheses. The numbers at the nodes of the trees represent the bootstrap values (1,000 replicates) for each node.

### Trefault_2004-07
### pJP4
IncP-1beta

https://www.ncbi.nlm.nih.gov/nuccore/NZ_AY365053
```
LOCUS       NZ_AY365053            87688 bp    DNA     circular CON 05-NOV-2020
DEFINITION  Cupriavidus pinatubonensis JMP134 plasmid pJP4, complete sequence.

     CDS             complement(55727..56947)
                     /gene="trfA"
                     /locus_tag="HS370_RS00300"
                     /function="initiation of replication"
                     /product="plasmid replication initiator TrfA"
                     /protein_id="WP_011114072.1"

     CDS             complement(83064..85304)
                     /locus_tag="HS370_RS00430"
                     /function="involved in conjugative DNA transfer"
                     /product="relaxase/mobilization nuclease domain-containing
                     protein"
                     /protein_id="WP_011114057.1"

```

https://www.ncbi.nlm.nih.gov/nuccore/AY365053
```
LOCUS       AY365053               87688 bp    DNA     circular BCT 03-APR-2020
DEFINITION  Ralstonia eutropha JMP134 plasmid pJP4, complete sequence.

     mobile_element  55610..55647
                     /rpt_type=inverted
                     /mobile_element_type="transposon:Tn501"

     CDS             complement(55727..56947)
                     /gene="trfA1"
                     /note="ORF52"
                     /product="transreplication factor"
                     /protein_id="AAR31070.1"

     CDS             complement(55727..56581)
                     /gene="trfA2"
                     /note="ORF52.1"
                     /product="transreplication factor"
                     /protein_id="AAR31071.1"

     CDS             complement(83064..85304)
                     /gene="traI"
                     /note="ORF78"
                     /product="relaxosome stabilization protein"
                     /protein_id="AAR31100.1"

     CDS             complement(83366..83758)
                     /gene="traH"
                     /note="ORF79"
                     /product="DNA relaxase"
                     /protein_id="AAR31099.1"

     CDS             complement(85339..85656)
                     /gene="traJ"
                     /note="ORF80"
                     /product="oriT binding protein"
                     /protein_id="AAR31101.1"

     misc_feature    85790..85881
                     /note="relaxation region oriT"

```

https://pubmed.ncbi.nlm.nih.gov/15186344/
Environ Microbiol
. 2004 Jul;6(7):655-68. doi: 10.1111/j.1462-2920.2004.00596.x.
Genetic organization of the catabolic plasmid pJP4 from Ralstonia eutropha JMP134 (pJP4) reveals mechanisms of adaptation to chloroaromatic pollutants and evolution of specialized chloroaromatic degradation pathways
N Trefault 1, R De la Iglesia, A M Molina, M Manzano, T Ledger, D Pérez-Pantoja, M A Sánchez, M Stuardo, B González
https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/j.1462-2920.2004.00596.x?sid=nlm%3Apubmed
Get access to the full version of this article.
View access options below.


## 2003
### Schluter_2003-11
### pB10
IncP-1beta1

https://www.ncbi.nlm.nih.gov/nuccore/AJ564903
```
LOCUS       AJ564903               64508 bp    DNA     circular ENV 25-JUL-2016
DEFINITION  Uncultured bacterium IncP-1beta multiresistance plasmid pB10.
```

https://pubmed.ncbi.nlm.nih.gov/16151119/
Appl Environ Microbiol
. 2005 Sep;71(9):5309-17. doi: 10.1128/AEM.71.9.5309-5317.2005.
Plasmid donor affects host range of promiscuous IncP-1beta plasmid pB10 in an activated-sludge microbial community
Leen De Gelder 1, Frederik P J Vandecasteele, Celeste J Brown, Larry J Forney, Eva M Top
https://journals.asm.org/doi/10.1128/AEM.71.9.5309-5317.2005
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1214629/

https://pubmed.ncbi.nlm.nih.gov/14600226/
Microbiology (Reading)
. 2003 Nov;149(Pt 11):3139-3153. doi: 10.1099/mic.0.26570-0.
The 64 508 bp IncP-1beta antibiotic multiresistance plasmid pB10 isolated from a waste-water treatment plant provides evidence for recombination between members of different branches of the IncP-1beta group
A Schlüter 1, H Heuer 2, R Szczepanowski 1, L J Forney 2, C M Thomas 3, A Pühler 1, E M Top 2
https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.26570-0#tab2

## 1998

### pIJB1
IncP-1delta

https://pubmed.ncbi.nlm.nih.gov/12206751/
Plasmid
. 2002 Jul;48(1):1-12. doi: 10.1016/s0147-619x(02)00018-5.
Complete characterisation of Tn5530 from Burkholderia cepacia strain 2a (pIJB1) and studies of 2,4-dichlorophenoxyacetate uptake by the organism
R P-C Poh 1, A R W Smith, I J Bruce

https://pubmed.ncbi.nlm.nih.gov/9514710/
Plasmid
. 1998;39(2):154-9. doi: 10.1006/plas.1997.1332.
A novel plasmid pIJB1 possessing a putative 2,4-dichlorophenoxyacetate degradative transposon Tn5530 in Burkholderia cepacia strain 2a
X S Xia 1, S Aathithan, K Oswiecimska, A R Smith, I J Bruce

https://www.ncbi.nlm.nih.gov/nuccore/JX847411
```
LOCUS       JX847411               99001 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Burkholderia cepacia strain 2a plasmid pIJB1, complete sequence.
```

### Thorsted_1998
### R751
IncP-1beta1

https://pubmed.ncbi.nlm.nih.gov/9753548/
J Mol Biol
. 1998 Oct 9;282(5):969-90. doi: 10.1006/jmbi.1998.2060.
Complete sequence of the IncPbeta plasmid R751: implications for evolution and organisation of the IncP backbone
P B Thorsted 1, D P Macartney, P Akhtar, A S Haines, N Ali, P Davidson, T Stafford, M J Pocklington, W Pansegrau, B M Wilkins, E Lanka, C M Thomas
https://www.sciencedirect.com/science/article/abs/pii/S0022283698920605?via%3Dihub

https://www.ncbi.nlm.nih.gov/nuccore/U67194
```
LOCUS       EAU67194               53423 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Enterobacter aerogenes plasmid R751, complete sequence.

  ORGANISM  Klebsiella aerogenes
            Bacteria; Pseudomonadota; Gammaproteobacteria; Enterobacterales;
            Enterobacteriaceae; Klebsiella/Raoultella group; Klebsiella.

  TITLE     The ancestral IncP replication system consisted of contiguous oriV
            and trfA segments as deduced from a comparison of the nucleotide
            sequences of diverse IncP plasmids
  JOURNAL   J. Gen. Microbiol. 139 (Pt 8), 1761-1766 (1993)
   PUBMED   8409919


     CDS             complement(4381..4683)
                     /gene="korA"
                     /function="regulation of gene expression"
                     /codon_start=1
                     /transl_table=11
                     /product="KorA"
                     /protein_id="AAC64422.1"

     repeat_region   14716..14726
                     /note="probable transposase binding site"
                     /rpt_type=inverted
     misc_feature    14730..14741
                     /note="an inverted repetition of this sequence occurs at
                     nucleotides 10026..10047"
     protein_bind    14947..14961
                     /bound_moiety="TrfA"
     repeat_region   complement(14975..14993)
                     /rpt_type=direct
     repeat_region   complement(15015..15032)
                     /rpt_type=direct
     repeat_region   complement(15046..15065)
                     /rpt_type=direct
     repeat_region   complement(15092..15111)
                     /rpt_type=direct
     repeat_region   complement(15149..15168)
                     /rpt_type=direct
     regulatory      15175..15203
                     /regulatory_class="terminator"
     gene            complement(15226..16460)
                     /gene="trfA1"
     CDS             complement(15226..16449)
                     /gene="trfA1"
                     /function="initiation of vegetative plasmid replication;
                     DNA binding"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfA1"
                     /protein_id="AAC64439.1"

     CDS             complement(15226..16083)
                     /gene="trfA2"
                     /note="initiation of vegetative plasmid replication; DNA
                     binding"
                     /codon_start=1
                     /transl_table=11
                     /product="S-TrfA"
                     /protein_id="AAC64440.1"

     regulatory      complement(16091..16096)
                     /regulatory_class="ribosome_binding_site"
                     /gene="trfA2"
     regulatory      complement(16453..16460)
                     /regulatory_class="ribosome_binding_site"
                     /gene="trfA1"
     gene            complement(16496..16852)
                     /gene="ssb"
     CDS             complement(16496..16837)
                     /gene="ssb"
                     /function="single strand DNA binding protein"
                     /codon_start=1
                     /transl_table=11
                     /product="SSB"
                     /protein_id="AAC64441.1"


     CDS             16951..17313
                     /gene="trbA"
                     /function="gene regulation"
                     /product="TrbA"
                     /protein_id="AAC64442.1"


     CDS             complement(48797..51040)
                     /gene="traI"
                     /function="DNA relaxase"
                     /product="TraI"
                     /protein_id="AAC64475.1"

```

## 1994
### RK2
IncP-1alpha
### Pansegrau_1994

https://pubmed.ncbi.nlm.nih.gov/10430917/
Comparative Study Proc Natl Acad Sci U S A
. 1999 Aug 3;96(16):9184-9. doi: 10.1073/pnas.96.16.9184.
Genome signature comparisons among prokaryote, plasmid, and mitochondrial DNA
A Campbell 1, J Mrázek, S Karlin
https://www.pnas.org/doi/10.1073/pnas.96.16.9184
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC17754/

https://pubmed.ncbi.nlm.nih.gov/8014987/
Comparative Study J Mol Biol
. 1994 Jun 24;239(5):623-63. doi: 10.1006/jmbi.1994.1404.
Complete nucleotide sequence of Birmingham IncP alpha plasmids. Compilation and comparative analysis
W Pansegrau 1, E Lanka, P T Barth, D H Figurski, D G Guiney, D Haas, D R Helinski, H Schwab, V A Stanisich, C M Thomas
https://www.sciencedirect.com/science/article/abs/pii/S0022283684714045?via%3Dihub
The IncP alpha promiscuous plasmid (R18, R68, RK2, RP1 and RP4) comprises 60,099 bp of nucleotide sequence, encoding at least 74 genes. 
The most frequently used initiation codon is AUG but translation for a few open reading frames begins at GUG or UUG. The most common stop-codon is UGA followed by UAA and then UAG. 

https://www.ncbi.nlm.nih.gov/nuccore/L27758
```
LOCUS       BIACOMGEN              60099 bp    DNA     linear   BCT 31-JAN-2013
DEFINITION  Pseudomonas aeruginosa plasmid Birmingham IncP-alpha, complete
            sequence.

```

https://www.ncbi.nlm.nih.gov/nuccore/BN000925
```
LOCUS       BN000925               60096 bp    DNA     circular BCT 23-OCT-2008
DEFINITION  TPA_exp: Birmingham IncP-alpha plasmid, complete sequence, isolate
            RK2.

  ORGANISM  Birmingham IncP-alpha plasmid
            other sequences; plasmids.

                     /country="United Kingdom:Birmingham"

     CDS             complement(join(4813..5487,10442..11152))
                     /gene="klcB"
                     /note="putative klcAB operon"
                     /pseudo
                     /codon_start=1
                     /transl_table=11
                     /db_xref="PSEUDO:CAJ85676.1"



     protein_bind    11864..11878
                     /note="orphan iteron"
                     /bound_moiety="TrfA"
     regulatory      12089..12120
                     /regulatory_class="terminator"
     misc_feature    12256..12301
                     /note="AT rich"
     protein_bind    complement(12279..12287)
                     /bound_moiety="DnaA"
     protein_bind    complement(12368..12382)
                     /note="iteron 9"
                     /bound_moiety="TrfA"


     CDS             complement(16521..17669)
                     /gene="trfA"
                     /operon="ssb,trfA,upf16.5"
                     /function="replication"
                     /note="essential for replication in P. aeruginosa;
                     long form TrfA1, alternative start codon"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfA replication protein"
                     /protein_id="CAJ85684.1"
                     /db_xref="InterPro:IPR010751"
                     /db_xref="UniProtKB/TrEMBL:A6H936"
                     /translation="MNRTFDRKAYRQELIDAGFSAEDAETIASRTVMRAPRETFQSVG
                     SMVQQATAKIERDSVQLAPPALPAPSAAVERSRRLEQEAAGLAKSMTIDTRGTMTTKK
                     RKTAGEDLAKQVSEAKQAALLKHTKQQIKEMQLSLFDIAPWPDTMRAMPNDTARSALF
                     TTRNKKIPREALQNKVIFHVNKDVKITYTGVELRADDDELVWQQVLEYAKRTPIGEPI
                     TFTFYELCQDLGWSINGRYYTKAEECLSRLQATAMGFTSDRVGHLESVSLLHRFRVLD
                     RGKKTSRCQVLIDEEIVVLFAGDHYTKFIWEKYRKLSPTARRMFDYFSSHREPYPLKL
                     ETFRLMCGSDSTRVKKWREQVGEACEELRGSGLVEHAWVNDDLVHCKR"
     CDS             complement(16521..17378)
                     /gene="trfA"
                     /operon="ssb,trfA,upf16.5"
                     /function="replication"
                     /note="replication protein - sufficient (with oriV) for
                     replication in all hosts tested except P. aeruginosa;
                     short form TrfA2, alternative start codon"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfA replication protein"
                     /protein_id="CAJ85685.1"
                     /db_xref="InterPro:IPR010751"
                     /db_xref="UniProtKB/TrEMBL:A6H937"
                     /translation="MTTKKRKTAGEDLAKQVSEAKQAALLKHTKQQIKEMQLSLFDIA
                     PWPDTMRAMPNDTARSALFTTRNKKIPREALQNKVIFHVNKDVKITYTGVELRADDDE
                     LVWQQVLEYAKRTPIGEPITFTFYELCQDLGWSINGRYYTKAEECLSRLQATAMGFTS
                     DRVGHLESVSLLHRFRVLDRGKKTSRCQVLIDEEIVVLFAGDHYTKFIWEKYRKLSPT
                     ARRMFDYFSSHREPYPLKLETFRLMCGSDSTRVKKWREQVGEACEELRGSGLVEHAWV
                     NDDLVHCKR"
     gene            complement(17718..18068)
                     /gene="ssb"
                     /operon="ssb,trfA,upf16.5"
     CDS             complement(17718..18068)
                     /gene="ssb"
                     /operon="ssb,trfA,upf16.5"
                     /codon_start=1
                     /transl_table=11
                     /product="single stranded DNA binding protein"
                     /protein_id="CAJ85686.1"
                     /db_xref="GOA:A6H938"
                     /db_xref="InterPro:IPR000424"
                     /db_xref="InterPro:IPR011344"
                     /db_xref="InterPro:IPR012340"
                     /db_xref="InterPro:IPR016027"
                     /db_xref="UniProtKB/TrEMBL:A6H938"
                     /translation="MSHNQFQFIGNLTRDTEVRHGNSNKPQAIFDIAVNEEWRNDAGD
                     KQERTDFFRIKCFGSQAEAHGKYLGKGSLVFVQGKIRNTKYEKDGQTVYGTDFIADKV
                     DYLDTKAPGGSNQE"


     CDS             18189..18554
                     /gene="trbA"
                     /codon_start=1
                     /transl_table=11
                     /product="TrbA repressor protein"
                     /protein_id="CAJ85687.1"
                     /db_xref="GOA:A6H939"
                     /db_xref="InterPro:IPR001387"
                     /db_xref="InterPro:IPR010982"
                     /db_xref="UniProtKB/TrEMBL:A6H939"
                     /translation="MYNQIFFTNILRLLDERGMTKHELSERAGVSISFLSDLTNGKAN
                     PSLKVMEAIADALETPLPLLLESTDLDREALAEIAGHPFKSSVPPGYERISVVLPSHK
                     AFIVKKWGDDTRKKLRGRL"


     CDS             complement(48492..50690)
                     /gene="traI"
                     /operon="traJXIH"
                     /codon_start=1
                     /transl_table=11
                     /product="TraI DNA relaxase"
                     /protein_id="CAJ85725.1"



     CDS             complement(59033..59338)
                     /gene="korA"
                     /operon="incC, korABFG"
                     /function="global repressor"
                     /codon_start=1
                     /transl_table=11
                     /product="KorA repressor protein"
                     /protein_id="CAJ85740.1"
                     /db_xref="UniProtKB/TrEMBL:A6H992"
                     /translation="MKKRLTESQFQEAIQGLEVGQQTIEIARGVLVDGKPQATFATSL
                     GLTRGAVSQAVHRVWAAFEDKNLPEGYARVTAVLPEHQAYIVRKWEADAKKKQETKR"


```


----------
## chromosome

### 2020-11-05

https://pubmed.ncbi.nlm.nih.gov/33154010/
Microbiol Resour Announc
. 2020 Nov 5;9(45):e01014-20. doi: 10.1128/MRA.01014-20.
Complete Genome Sequences of the Escherichia coli Donor Strains ST18 and MFD pir
Simon A Jackson 1 2, Bridget J Fellows 3 2, Peter C Fineran 3 2 4
https://mra.asm.org/content/9/45/e01014-20
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7645665/
Escherichia coli ST18 and MFDpir are donors commonly used to transfer oriTRP4-containing plasmids to diverse bacteria via conjugation. 

ANNOUNCEMENT
A common laboratory method for transferring plasmids to diverse bacteria is via conjugation from an Escherichia coli donor strain. Several E. coli K-12 derivative strains have been engineered as donors, typically based on mobilization machinery derived from the broad-host-range plasmid RP4 (1,–4). To facilitate efficient postmating donor counterselection, several auxotrophic E. coli donors, such as ST18, have been constructed (5). ST18 is a 5-aminolevulinic acid auxotroph derivative of S17-1 λpir that was derived by deletion of hemA (5,–8). 

The complete genome sequences have been deposited in NCBI GenBank with accession numbers CP060709 (ST18) and CP060708 (MFDpir) 


https://www.ncbi.nlm.nih.gov/nuccore/CP060709
```
LOCUS       CP060709             4825686 bp    DNA     circular BCT 31-AUG-2020
DEFINITION  Escherichia coli strain ST18 chromosome, complete genome.

     CDS             416431..416802
                     /gene="traJ"
                     /product="conjugal transfer transcriptional regulator
                     TraJ"

     CDS             416837..419035
                     /product="relaxase/mobilization nuclease domain-containing
                     protein"

     CDS             complement(461363..461800)
                     /gene="trbC"
                     /product="IncP-type conjugal transfer pilin TrbC"
                     /protein_id="QNM56946.1"

     CDS             complement(461813..462772)
                     /gene="trbB"
                     /product="P-type conjugative transfer ATPase TrbB"

     CDS             463928..465076
                     /product="plasmid replication initiator protein TrfA"

```

https://www.ncbi.nlm.nih.gov/nuccore/CP060708
```
LOCUS       CP060708             4686434 bp    DNA     circular BCT 31-AUG-2020
DEFINITION  Escherichia coli strain MFDpir chromosome, complete genome.
```

### S17-1

http://www.agr.kyushu-u.ac.jp/sanshi-kyushu/entomotech_38/p17_19.pdf
Escherichia coli S17-1
のゲノム上には，プラスミドの伝播に関与する tra
遺伝子が存在する。この遺伝子が mob 領域を認識
し，受容菌へとプラスミドを伝播するという特性
がある（SIMON et al., 1983）。 

https://www.ncbi.nlm.nih.gov/nuccore/CP040667
```
LOCUS       CP040667             4772290 bp    DNA     circular BCT 04-JUN-2019
DEFINITION  Escherichia coli strain S17-1 chromosome, complete genome.

     CDS             complement(3069276..3070424)
                     /product="plasmid replication initiator protein TrfA"
                     /protein_id="QCW47545.1"

     CDS             complement(3117550..3117921)
                     /gene="traJ"
                     /product="conjugal transfer transcriptional regulator
                     TraJ"

     CDS             3118289..3118693
                     /product="conjugal transfer protein TraK"

     CDS             3118693..3119418
                     /product="protein TraL"

     CDS             3119415..3119852
                     /product="conjugal transfer protein TraM"


     CDS             3139017..3139259
                     /product="relaxase"

```

### Stenger_2011

https://www.ncbi.nlm.nih.gov/pubmed/21296948
Appl Environ Microbiol. 2011 Apr;77(7):2522-6. 
Phylogeny of replication initiator protein TrfA reveals a highly divergent clade of incompatibility group P1 plasmids.
Stenger DC, Lee MW.
https://aem.asm.org/content/77/7/2522.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3067460/
Genes for four TrfA homologues are integrated into chromosomes of their respective hosts
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3067460/table/t1/
cNA, not applicable; integrated in host chromosome.

FIG. 1.
Phylogeny of IncP-1 TrfA homologues. Presented is a neighbor joining tree (1,000 bootstrap iterations) based on alignment of TrfA amino acid sequences. 

![](https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3067460/bin/zam9991019330001.jpg)

----------
## people

### Shintani
新谷政己

### 2023-02-06
https://pubmed.ncbi.nlm.nih.gov/36744408/
Environ Microbiol
. 2023 Feb 6. doi: 10.1111/1462-2920.16345. Online ahead of print.
Reconsideration of the previously classified incompatibility groups of plasmids, IncP-1 and IncP-11
Masaki Shintani 1 2 3, Haruo Suzuki 4, Hideaki Nojiri 5, Masato Suzuki 6
https://ami-journals.onlinelibrary.wiley.com/doi/10.1111/1462-2920.16345

### 2022-01-25
https://pubmed.ncbi.nlm.nih.gov/35084026/
J Antimicrob Chemother
. 2022 Jan 25;dkac006. doi: 10.1093/jac/dkac006. Online ahead of print.
Precise classification of antimicrobial resistance-associated IncP-2 megaplasmids for molecular epidemiological studies on Pseudomonas species
Masaki Shintani 1 2 3, Haruo Suzuki 4, Hideaki Nojiri 5, Masato Suzuki 6
https://academic.oup.com/jac/advance-article/doi/10.1093/jac/dkac006/6515317
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8969547/

### 2022-09-27
https://pubmed.ncbi.nlm.nih.gov/36069618/
Appl Environ Microbiol
. 2022 Sep 27;88(18):e0111422. doi: 10.1128/aem.01114-22. Epub 2022 Sep 7.
Hitherto-Unnoticed Self-Transmissible Plasmids Widely Distributed among Different Environments in Japan
Masaya Hayakawa # 1, Maho Tokuda # 2, Kensei Kaneko # 1, Koichiro Nakamichi # 1, Yukie Yamamoto # 1, Tatsuya Kamijo # 1, Honoka Umeki # 1, Reimi Chiba 1, Ryo Yamada 1, Mitsuya Mori 1, Kosuke Yanagiya 1, Ryota Moriuchi 3, Masahiro Yuki 4, Hideo Dohra 3 5, Hiroyuki Futamata 1 2 6, Moriya Ohkuma 4, Kazuhide Kimbara 1 2, Masaki Shintani 1 2 4 6
https://journals.asm.org/doi/full/10.1128/aem.01114-22
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9499019/

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9499019/table/T3/?report=objectonly
TABLE 3
IncP/P-1 plasmids obtained in this study
Name	Inc_group	Size (bp)	Source	Antibiotic resistance genes and the resistances of the hosts (R, resistance, S, sensitive, ND, not determined)a
Accessory genes	Tc (12.5)b	Gm (30)	Ap (50)	Km (50)	Km (25)	Km (12.5)	Cm (30)	Sm (50)	Sm (25)	Em (25)

pMNCG080-1	IncP/P-1ρ	69,454	Activated sludge	No accessory	ND	ND	ND	ND	ND	ND	ND	ND	ND	ND
pMNCG082-1	IncP/P-1μ	74,115	Activated sludge	bcs	ND	ND	ND	ND	ND	ND	ND	ND	ND	ND

Clade II contained the newly identified plasmids pMNCG080-1 (IncP/P-1μ) and pMNCG082-1 (IncP/P-1ρ), as well as pDS1 (IncP/P-1η) (4), and other plasmids in unidentified subgroups (Fig. 2).

Data availability. The accession numbers deposited in DDBJ of the plasmid sequences (70 plasmids) were LC623882 to LC623932 and LC663721 to LC663740. The 16S rRNA genes of the transconjugants (transconjugants of PromA plasmids) were LC655384 to LC655683.

https://www.ncbi.nlm.nih.gov/nuccore/LC623894
```
LOCUS       LC623894               69454 bp    DNA     circular ENV 25-MAY-2022
DEFINITION  Uncultured bacterium plasmid pMNCG080 DNA, complete sequence.
```

https://www.ncbi.nlm.nih.gov/nuccore/LC663734
```
LOCUS       LC663734                3992 bp    DNA     circular ENV 25-MAY-2022
DEFINITION  Uncultured bacterium plasmid pMNCG080-2 DNA, complete sequence.
```

https://www.ncbi.nlm.nih.gov/nuccore/LC623895
```
LOCUS       LC623895               74115 bp    DNA     circular ENV 25-MAY-2022
DEFINITION  Uncultured bacterium plasmid pMNCG082 DNA, complete sequence.
```

https://www.ncbi.nlm.nih.gov/nuccore/LC663735
```
LOCUS       LC663735                9531 bp    DNA     circular ENV 25-MAY-2022
DEFINITION  Uncultured bacterium plasmid pMNCG082-2 DNA, complete sequence.
```

### 2020-11-27
https://pubmed.ncbi.nlm.nih.gov/33329469/
Front Microbiol
. 2020 Nov 27;11:590776. doi: 10.3389/fmicb.2020.590776. eCollection 2020.
Plant Species-Dependent Increased Abundance and Diversity of IncP-1 Plasmids in the Rhizosphere: New Insights Into Their Role and Ecology
Masaki Shintani 1 2 3, Eman Nour 4, Tarek Elsayed 4, Khald Blau 4, Inessa Wall 4, Sven Jechalke 4, Cathrin Spröer 5, Boyke Bunk 5, Jörg Overmann 5, Kornelia Smalla 4
https://www.frontiersin.org/articles/10.3389/fmicb.2020.590776/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7728920/

### 2020-06-09
https://pubmed.ncbi.nlm.nih.gov/32582111/
Front Microbiol
. 2020 Jun 9;11:1187. doi: 10.3389/fmicb.2020.01187. eCollection 2020.
Determination of Plasmid pSN1216-29 Host Range and the Similarity in Oligonucleotide Composition Between Plasmid and Host Chromosomes
Maho Tokuda 1, Haruo Suzuki 2 3, Kosuke Yanagiya 1, Masahiro Yuki 4, Kengo Inoue 5, Moriya Ohkuma 4, Kazuhide Kimbara 1, Masaki Shintani 1 4 6
https://www.frontiersin.org/articles/10.3389/fmicb.2020.01187/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7296055/

Shintani M., Suzuki H. (2019) Plasmids and Their Hosts. In: Nishida H., Oshima T. (eds) DNA Traffic in the Environment. Springer, Singapore. DOI: https://doi.org/10.1007/978-981-13-3411-5_6

### 2015-03-31
https://pubmed.ncbi.nlm.nih.gov/25873913/
Review Front Microbiol
. 2015 Mar 31;6:242. doi: 10.3389/fmicb.2015.00242. eCollection 2015.
Genomics of microbial plasmids: classification and identification based on replication and transfer systems and host taxonomy
Masaki Shintani 1, Zoe K Sanchez 2, Kazuhide Kimbara 2
https://www.frontiersin.org/articles/10.3389/fmicb.2015.00242/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4379921/

### 2015-03-22
http://ci.nii.ac.jp/naid/40020727466
新谷政己，金原和秀．環境バイオテクノロジー学会誌 14(2), 81-86.
"プラスミドゲノミクス~全塩基配列解読済のプラスミドデータベースの整備"
Genomics of Plasmids
https://www.jseb.jp/wordpress/wp-content/uploads/14-02-081.pdf

### 2013-11-22
http://ci.nii.ac.jp/naid/40020320833
新谷 政己, 松井 一泰, 金原 和秀, 野尻 秀昭. 環境バイオテクノロジー学会誌 13(2), 125-134.
"環境中におけるプラスミドの挙動解析"
Comparisons of Conjugation Frequency in Different Environmental Conditions
https://www.jseb.jp/wordpress/wp-content/uploads/13-02-125.pdf
表 2．Pseudomonas 属細菌におけるプラスミドの不和合性群とその接合伝達機構における分類

https://pubmed.ncbi.nlm.nih.gov/24141122/
Appl Environ Microbiol
. 2014 Jan;80(1):138-45. doi: 10.1128/AEM.02571-13. Epub 2013 Oct 18.
Single-cell analyses revealed transfer ranges of IncP-1, IncP-7, and IncP-9 plasmids in a soil bacterial community
Masaki Shintani 1, Kazuhiro Matsui, Jun-Ichi Inoue, Akira Hosoyama, Shoko Ohji, Atsushi Yamazoe, Hideaki Nojiri, Kazuhide Kimbara, Moriya Ohkuma
https://aem.asm.org/content/80/1/138.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911017/
The host ranges of the three plasmids have been previously documented as being broad (pBP136), narrow (pCAR1), and intermediate (NAH7) (13). A broad-host-range plasmid is proposed to be able to replicate itself and be maintained stably in at least two different bacterial classes without selection, whereas a narrow-host-range plasmid is in the same class or genus (13).

https://www.ncbi.nlm.nih.gov/pubmed/24141122
Appl Environ Microbiol. 2014 Jan;80(1):138-45. doi: 10.1128/AEM.02571-13. Epub 2013 Oct 18.
Single-cell analyses revealed transfer ranges of IncP-1, IncP-7, and IncP-9 plasmids in a soil bacterial community.
Shintani M1, Matsui K, Inoue J, Hosoyama A, Ohji S, Yamazoe A, Nojiri H, Kimbara K, Ohkuma M.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911017/
Nucleotide sequence accession numbers.
The partial sequences of 519 transconjugants were deposited in the DDBJ, EMBL, and GenBank databases (accession numbers AB772697 to AB773215). The pyrosequencing run data of the 16S rRNA genes in soil samples were deposited in the DDBJ Sequence Read Archive (DRA) (http://trace.ddbj.nig.ac.jp/dra/index_e.html; accession number DRA001209).
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911017/bin/supp_80_1_138__index.html
Table S3. Top hit type strains with transconjugants identified in this study in BLAST searches.

### Suzuki

https://link.springer.com/referenceworkentry/10.1007%2F978-1-4614-6436-5_574-1
Suzuki, H., Brown, C.J., Top, E.M. (2014). Genomic Signature Analysis to Predict Plasmid Host Range. In: Bell, E. (eds) Molecular Life Sciences. Springer, New York, NY. https://doi.org/10.1007/978-1-4614-6436-5_574-1

https://pubmed.ncbi.nlm.nih.gov/20851899/
J Bacteriol
. 2010 Nov;192(22):6045-55. doi: 10.1128/JB.00277-10. Epub 2010 Sep 17.
Predicting plasmid promiscuity based on genomic signature
Haruo Suzuki 1, Hirokazu Yano, Celeste J Brown, Eva M Top
https://jb.asm.org/content/192/22/6045.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2976448/


http://www.ncbi.nlm.nih.gov/pubmed/18953039
Nucleic Acids Res. 2008 Dec;36(22):e147. doi: 10.1093/nar/gkn753. Epub 2008 Oct 25.
Using Mahalanobis distance to compare genomic signatures between bacterial plasmids and chromosomes.
Suzuki H1, Sota M, Brown CJ, Top EM.
https://academic.oup.com/nar/article/36/22/e147/1196271
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2602791/

### Yano
### 2018-12-15

総説
https://www.ncbi.nlm.nih.gov/pubmed/30619542
Comput Struct Biotechnol J. 2018 Dec 15;17:70-81. doi: 10.1016/j.csbj.2018.12.001. eCollection 2019.
Reconsidering plasmid maintenance factors for computational plasmid design.
Yano H, Shintani M, Tomita M, Suzuki H, Oshima T.
https://www.sciencedirect.com/science/article/pii/S2001037018301685
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6312765/

Fig. 1
Modular structure of the plasmid genome. An example of the IncP-1 plasmid genome is shown. The replication module is the only element essential for plasmid maintenance. The other modules assist in plasmid maintenance. IncP-1β and IncP-1α are lineages of IncP-1 plasmids. Horizontal arrows indicate the direction of transcription. Each functional module is regulated by a transcriptional regulator encoded in the module block or global regulators encoded in the central control region [188,189].

![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6312765/bin/gr1.jpg)

Fig. 2
Replication module. (A) Replication module of IncP-1 plasmids. Ssb encodes a single-strand DNA binding protein. trfA encodes the Rep protein. (B) Replication initiation of IncP-1 plasmid. Monomeric TrfA bound to iterons opens base pairs in the DUE, and then host DnaA or TrfA itself recruits DnaB onto the unwound DUE. TrfA molecules bind to single-stranded DNA to assist replisome assembly on one strand via direct interaction with β-clamp, starting unidirectional replication. Lagging strand synthesis is not shown for simplicity. Illustration follows [8], with minor modifications. (C) Replication inhibition by Rep protein dimer. Paired oriV can dissociate via proteolysis or an increased Rep protein monomer to dimer ratio [32].

https://sites.google.com/view/hirokazuyano/koremadeno-yan-jiu
Hirokazu Yano - これまでの研究 (in Japanese)

薬剤耐性プラスミドの宿主適応

本現象の背景にある分子機構を解明するため、ある宿主集団内で進化した進化型プラスミドの配列を決定したのち、進化型プラスミドを先祖型宿主に戻す実験を行ったところ、プラスミド存続性の向上がプラスミド の複製タンパク質（TrfA1）のN末の変異に依存していることがわかりました。このTrfAのN末は宿主のDnaBヘリカーゼと結合する部位であることが知られていました。生化学、分子遺伝学実験の結果から、プラスミ ドが宿主の中で複製できる場合、「宿主のDNAヘリカーゼとプラスミドとの相互作用の度合いが、プラスミドの細菌集団内での存続性を決める要因の一つ、フィットネス・コストの正体である」ことがわかりました（Yano et al., Mol. Microbiol. 2016)。


プラスミドの種分化と宿主域

自然宿主の培養に依存しないプラスミド単離法の普及によって、伝達性プラスミドの集団内には遺伝子のレパートリーや並びが似ていても、ゲノムシグニチャーが顕著に異なる系統群が存在していることが明らかに なって来ました (Sen et al., Mol. Biol.Evol. 2013; Norberg et al., Nat. Comm. 2011)。そこで、「隔離による組換えの機会の減少によって種分化が進む」という生命進化の原則にもとづき、「ゲノムシグニチャーの異 なるプラスミドグループは、同じ不和合性群に属しても、宿主域が異なる」いう仮説を立て、その仮説が妥当であることを、実験によって示しました（Yano et al., Microbiol. 2013)。


タンパク質のN末の長さ多型の意義

上記の進化実験では、複製開始タンパク質に集中的に変異が入りましたが、類似の変異がデータベースに登録されている他のプラスミドにも見つかることに研究の過程で気づきました。そこで、得られた変異体プラ スミドの表現型を様々な角度から解析することで、 プラスミドに見られる複製開始タンパク質遺伝子の5’-endの長さの多様性（in/del) が、genetic driftではなくトレード・オフの関係にある形質（この場合は水平伝播 と垂直伝播）に対するnatural selectionの結果として説明できるというアイデアを提案しました（Yano et al., J. Bacteriol. 2012)。TrfAのN末の多型のような長さ多型は他のタンパク質（遺伝子座）にも見かけられま す。他のタンパク質に関しても、N末の長さ多型にには何か重要な意味があるのではないでしょうか。


### Yano_2013-11
https://pubmed.ncbi.nlm.nih.gov/24002747/
Microbiology (Reading)
. 2013 Nov;159(Pt 11):2303-2315. doi: 10.1099/mic.0.068387-0. Epub 2013 Sep 3.
Host range diversification within the IncP-1 plasmid group
Hirokazu Yano 1 2, Linda M Rogers 1 2, Molly G Knox 1, Holger Heuer 3, Kornelia Smalla 3, Celeste J Brown 1 2, Eva M Top 1 2
https://www.microbiologyresearch.org/content/journal/micro/10.1099/mic.0.068387-0#tab2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3836486/
Of the two IncP-1γ plasmids, both were unable to replicate in alphaproteobacterial host Sphingobium japonicum, and one established itself in Agrobacterium tumefaciens but was very unstable.

Introduction
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3836486/#sec1-1title

Since Yakobson & Guiney (1983) proposed to separate plasmids RP4 and R751 as different subgroups (α, β) based on the sequence similarity in the transfer origin (Yakobson & Guiney, 1983), six IncP-1 subgroups (α, β, γ, δ, ϵ and ξ, see Fig. 1a) have been proposed based on the results of phylogenetic analysis for plasmid backbone genes (Jobanputra & Datta, 1974; Haines et al., 2006; Sen et al., 2010, 2013; Bahl et al., 2007; Norberg et al., 2011). To date, more than 55 IncP-1 plasmid sequences have been deposited in the NCBI database. 

https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC3836486/figure/f1/
Fig. 1.
Diversity of IncP-1 plasmids. (a) Phylogenetic tree 
(b) Similarities of replication initiation proteins and relaxases (TraI) between representative plasmids of each IncP-1 subgroup (
Amino-acid sequence identities were determined using blastp, and are represented in the heat map.

Methods
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3836486/#sec1-2title

Results
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3836486/#sec1-3title

General features of the pKS208 and pMBUI1 genomes
Plasmids pKS208 and pMBUI1 consist of 50 604 and 44 303 bp, respectively. 

Replication host range of the intact IncP-1 plasmids

Table 2.
Plasmid transfer frequencies

When the alphaproteobacteria were used as recipients, no transfer was detected for either of the IncP-1γ plasmids from P. putida as donor, whereas pBP136Km was transferred successfully. When E. coli was the donor, transfer of the IncP-1γ plasmids to A. tumefaciens was detected, but at low to extremely low frequencies, and no transfer to S. japonicum was observed.

https://www.ncbi.nlm.nih.gov/nuccore/JQ432564
```
LOCUS       JQ432564               50604 bp    DNA     circular ENV 26-JUL-2016
DEFINITION  Uncultured bacterium plasmid pKS208, complete sequence.

     CDS             complement(1..1089)
                     /gene="trfA1"
                     /note="trans-acting factor A; Homologous to TrfA1 of R751"
                     /product="TrfA replication initiation protein"
                     /protein_id="AFS51515.1"

     CDS             complement(36446..38857)
                     /gene="traI"
                     /note="DNA transfer protein"
                     /product="TraI relaxase"
                     /protein_id="AFS51557.1"

```

https://www.ncbi.nlm.nih.gov/nuccore/JQ432563
```
LOCUS       JQ432563               44304 bp    DNA     circular ENV 26-JUL-2016
DEFINITION  Uncultured bacterium plasmid pMBUI1, complete sequence.

     CDS             complement(1..1122)
                     /gene="trfA1"
                     /note="trans-acting factor A, homologous to TrfA1 of R751"
                     /product="TrfA1 replication initiation protein"
                     /protein_id="AFS51460.1"

     CDS             complement(29447..31855)
                     /gene="traI"
                     /note="DNA transfer protein"
                     /product="TraI relaxase"
                     /protein_id="AFS51493.1"

```

### Yano_2012

https://pubmed.ncbi.nlm.nih.gov/22228734/
J Bacteriol
. 2012 Mar;194(6):1533-43. doi: 10.1128/JB.06395-11. Epub 2012 Jan 6.
Roles of long and short replication initiation proteins in the fate of IncP-1 plasmids
Hirokazu Yano 1, Gail E Deckert, Linda M Rogers, Eva M Top
https://jb.asm.org/content/194/6/1533.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294859/

Fig 1
(A) Schematic representation of the diversity in TrfA structure. M98 and M124 indicate the start methionines of TrfA-33 in RK2 and TrfA2 in pBP136, respectively. QLSLF indicates the conserved amino acid sequence motif responsible for interaction with DNA polymerase III (33). DBD1 and DBD2 indicate DNA-binding (winged-helix) domains deduced from secondary structure comparisons (22, 49). (B) The trfA gene locus of pBP136 and its mini-pBP136 derivative pMS0506. A+T and G+C indicate the A/T- and G/C-rich regions, respectively. Triangles and pentagons indicate iterons (TrfA-binding motif) and DnaA-boxes (DnaA-binding motif), respectively. Flags indicate promoters. 

![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294859/bin/zjb9990912760001.jpg)

https://www.ncbi.nlm.nih.gov/nuccore/X58896
```
LOCUS       X58896                  1823 bp    DNA     linear   BCT 26-JUL-2016
DEFINITION  P.syringae pPS10 plasmid DNA for basic replicon.

     CDS             575..1270
                     /gene="repA"
                     /codon_start=1
                     /transl_table=11
                     /product="replication protein"
                     /protein_id="CAA41700.1"

```

----------
## non-IncP1

Reconsidering plasmid maintenance factors for computational plasmid design - ScienceDirect
https://www.sciencedirect.com/science/article/pii/S2001037018301685#t0005
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6312765/table/t0005/?report=objectonly
Table 1. Lists of plasmids in different incompatibility groups.

### IncA
### IncC

https://pubmed.ncbi.nlm.nih.gov/29486211/
Plasmid
. Mar-May 2018;96-97:7-12. doi: 10.1016/j.plasmid.2018.02.002. Epub 2018 Feb 24.
Compatibility and entry exclusion of IncA and IncC plasmids revisited: IncA and IncC plasmids are compatible
Stephanie J Ambrose 1, Christopher J Harmer 2, Ruth M Hall 1
https://www.sciencedirect.com/science/article/abs/pii/S0147619X1730152X?via%3Dihub

### IncF
### F

IncF
F
AP001918

https://www.ncbi.nlm.nih.gov/nuccore/AP001918
LOCUS       AP001918               99159 bp    DNA     circular BCT 29-SEP-2018
DEFINITION  Escherichia coli K-12 plasmid F DNA, complete sequence.

### IncK

https://pubmed.ncbi.nlm.nih.gov/28052854/
Antimicrob Agents Chemother
. 2017 Feb 23;61(3):e01954-16. doi: 10.1128/AAC.01954-16. Print 2017 Mar.
Plasmids of Distinct IncK Lineages Show Compatible Phenotypes
Marta Rozwandowicz 1, Michael S M Brouwer 2, Aldert L Zomer 1, Alex Bossers 2, Frank Harders 2, Dik J Mevius 1 2, Jaap A Wagenaar 1 2, Joost Hordijk 3
https://journals.asm.org/doi/full/10.1128/AAC.01954-16
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5328535/

### IncR
### pKP1780

IncR
pKP1780
JX424614

https://www.ncbi.nlm.nih.gov/nuccore/JX424614
```
LOCUS       JX424614               49770 bp    DNA     linear   BCT 26-JUL-2016
DEFINITION  Klebsiella pneumoniae plasmid pKP1780, complete sequence.
```

https://pubmed.ncbi.nlm.nih.gov/23711894/
J Antimicrob Chemother
. 2013 Oct;68(10):2259-62. doi: 10.1093/jac/dkt196. Epub 2013 May 27.
Characterization of pKP1780, a novel IncR plasmid from the emerging Klebsiella pneumoniae ST147, encoding the VIM-1 metallo-β-lactamase
C C Papagiannitsis 1, V Miriagou, P Giakkoupi, L S Tzouvelekis, A C Vatopoulos

### IncU
### pFBAOT6

https://pubmed.ncbi.nlm.nih.gov/15574953/
Appl Environ Microbiol
. 2004 Dec;70(12):7497-510. doi: 10.1128/AEM.70.12.7497-7510.2004.
Complete nucleotide sequence of the conjugative tetracycline resistance plasmid pFBAOT6, a member of a group of IncU plasmids with global ubiquity
Glenn Rhodes 1, Julian Parkhill, Christine Bird, Kerrie Ambrose, Matthew C Jones, Geert Huys, Jean Swings, Roger W Pickup
https://journals.asm.org/doi/full/10.1128/AEM.70.12.7497-7510.2004
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC535204/

https://www.ncbi.nlm.nih.gov/nuccore/CR376602
```
LOCUS       CR376602               84749 bp    DNA     circular BCT 26-JUL-2016
DEFINITION  plasmid pFBAOT6 from Aeromonas punctata (Aeromonas caviae) HGB5,
            complete sequence.

     CDS             6010..6348
                     /gene="trfB"
                     /locus_tag="pFBAOT6.08"
                     /gene_synonym="korA"
                     /note="Similar to Escherichia coli trfb transcriptional
                     repressor protein trfb or korA SWALL:KOA1_ECOLI
                     (SWALL:Q57423) (100 aa) fasta scores: E(): 0.00011, 36% id
                     in 100 aa"
                     /codon_start=1
                     /transl_table=11
                     /product="TrfB transcriptional repressor protein"
                     /protein_id="CAG15055.1"
                     /db_xref="UniProtKB/TrEMBL:Q68E88"
                     /translation="MAACDRVKVMALTNEEFESALKHLPRRLAAKNVEIARAILVQGR
                     RQVDLVKESGLSRSAVAALVRKVRQAHEKHGTPPAGWVRVSVCVPVDMAPIVKAIEDE
                     AYKQANKPKG"

```

----------

### IncW
### R388

IncW
R388
BR000038
https://www.ncbi.nlm.nih.gov/nuccore/NC_028464
https://www.ncbi.nlm.nih.gov/nuccore/BR000038
```
LOCUS       BR000038               33913 bp    DNA     circular BCT 09-MAY-2006
DEFINITION  TPA_inf: Escherichia coli plasmid R388 DNA, complete sequence.
```

https://pubmed.ncbi.nlm.nih.gov/17026718/
Review FEMS Microbiol Rev
. 2006 Nov;30(6):942-66. doi: 10.1111/j.1574-6976.2006.00042.x. Epub 2006 Oct 6.
Dynamics of the IncW genetic backbone imply general trends in conjugative plasmid evolution
Raúl Fernández-López 1, M Pilar Garcillán-Barcia, Carlos Revilla, Miguel Lázaro, Luis Vielva, Fernando de la Cruz
https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1574-6976.2006.00042.x
https://academic.oup.com/femsre/article/30/6/942/494544
As a consequence, and in order to avoid long-branch attraction phenomena, the phylogeny of TrwC was reconstructed from protein sequence alignments using only the relaxase domain (300 N-terminal amino acids).

----------

### PromA

https://pubmed.ncbi.nlm.nih.gov/37341708/
Microb Genom
. 2023 Jun;9(6):mgen001043. doi: 10.1099/mgen.0.001043.
Transconjugant range of PromA plasmids in microbial communities is predicted by sequence similarity with the bacterial host chromosome
Maho Tokuda 1, Masahiro Yuki 2, Moriya Ohkuma 2, Kazuhide Kimbara 1, Haruo Suzuki 3, Masaki Shintani 1 2 4
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.001043#tab2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10327500/

https://pubmed.ncbi.nlm.nih.gov/33967971/
Front Microbiol
. 2021 Apr 21;12:606409. doi: 10.3389/fmicb.2021.606409. eCollection 2021.
Biosolids as a Source of Antibiotic Resistance Plasmids for Commensal and Pathogenic Bacteria
Aaron Law 1, Olubunmi Solano 1 2, Celeste J Brown 1 3, Samuel S Hunter 3 4, Matt Fagnan 3, Eva M Top 1 3, Thibault Stalder 1 3
https://www.frontiersin.org/articles/10.3389/fmicb.2021.606409/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8098119/


https://pubmed.ncbi.nlm.nih.gov/32132980/
Front Microbiol
. 2020 Feb 18;11:149. doi: 10.3389/fmicb.2020.00149. eCollection 2020.
PromA Plasmids Are Instrumental in the Dissemination of Linuron Catabolic Genes Between Different Genera
Johannes Werner 1, Eman Nour 2 3, Boyke Bunk 4, Cathrin Spröer 5, Kornelia Smalla 2, Dirk Springael 6, Başak Öztürk 6 7
https://www.frontiersin.org/articles/10.3389/fmicb.2020.00149/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7039861/


https://pubmed.ncbi.nlm.nih.gov/30637959/
Environ Microbiol
. 2019 Mar;21(3):1113-1123. doi: 10.1111/1462-2920.14531. Epub 2019 Feb 22.
Real time monitoring of Aeromonas salmonicida evolution in response to successive antibiotic therapies in a commercial fish farm
Xiaochen Du 1, Sion C Bayliss 2, Edward J Feil 2, Ying Liu 3, Chao Wang 1, Gang Zhang 1, Dongsheng Zhou 4, Dawei Wei 1, Na Tang 1, Sébastien O Leclercq 5, Jie Feng 1
https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.14531
Plasmid pS28‐3 present in isolate S28 (floR, oxyS, enrS) is 48.922 kb, (Supporting Information Fig. S3A) and shares 99% identity to the environmental plasmid pSFA231 isolated from petroleum‐contaminated sediment in China (Li et al., 2015). The major difference between pS28‐3 and pSFA231 is that pS28‐3 contains an additional 7.3 kb transposon, Tn6351, which harbours the floR gene known to confer resistance to florfenicol (Table 1). 

https://pubmed.ncbi.nlm.nih.gov/25628616/
Front Microbiol
. 2015 Jan 12;5:777. doi: 10.3389/fmicb.2014.00777. eCollection 2014.
The broad-host-range plasmid pSFA231 isolated from petroleum-contaminated sediment represents a new member of the PromA plasmid family
Xiaobin Li 1, Eva M Top 2, Yafei Wang 3, Celeste J Brown 2, Fei Yao 1, Shan Yang 3, Yong Jiang 3, Hui Li 3

----------

### pP9014
### 2013-07

https://pubmed.ncbi.nlm.nih.gov/23602560/
Int J Antimicrob Agents
. 2013 Jul;42(1):10-8. doi: 10.1016/j.ijantimicag.2013.02.027. Epub 2013 Apr 18.
Comparative analysis and distribution of pP9014, a novel drug resistance IncP-1 plasmid from Photobacterium damselae subsp. piscicida
Carmelo S del Castillo 1, Ho Bin Jang, Jun-Ichi Hikima, Tae Sung Jung, Hideaki Morii, Ikuo Hirono, Hidehiro Kondo, Chihiro Kurosaka, Takashi Aoki
https://www.sciencedirect.com/science/article/pii/S0924857913001015?via%3Dihub
https://search.lib.keio.ac.jp/permalink/81SOKEI_KEIO/uccs31/cdi_proquest_miscellaneous_1370639312
Elsevier ScienceDirect Journals

https://www.ncbi.nlm.nih.gov/nuccore/AB453229
```
LOCUS       AB453229               55851 bp    DNA     circular BCT 24-JUL-2016
DEFINITION  Photobacterium damselae subsp. piscicida plasmid pP9014 DNA,
            complete sequence.


     CDS             complement(12694..15069)
                     /gene="TraI"
                     /codon_start=1
                     /transl_table=11
                     /product="TraI DNA relaxase"
                     /protein_id="BAH83586.1"

     CDS             complement(23694..24128)
                     /gene="KlcA"
                     /product="KlcA protein"
                     /protein_id="BAH83599.1"

     CDS             24851..25615
                     /gene="RepB"
                     /codon_start=1
                     /transl_table=11
                     /product="RepB initiator protein"
                     /protein_id="BAH83600.1"

     CDS             complement(25972..26304)
                     /product="hypothetical protein"
                     /protein_id="BAH83601.1"

     CDS             complement(26291..26707)
                     /product="hypothetical protein"
                     /protein_id="BAH83602.1"

     CDS             complement(27107..27250)
                     /product="truncated Ssb protein"
                     /protein_id="BAH83603.1"
                     /translation="MSFNKLQFIEYLTHDMNFIPNKNNSARAVFDPAVTSIYGKIEIE
                     RKM"

     CDS             27351..27728
                     /gene="TrbA"
                     /product="TrbA DNA binding protein"
                     /protein_id="BAH83604.1"

```

### Top_1995

https://pubmed.ncbi.nlm.nih.gov/7646006/
Comparative Study Appl Environ Microbiol
. 1995 May;61(5):1691-8. doi: 10.1128/AEM.61.5.1691-1698.1995.
Characterization of diverse 2,4-dichlorophenoxyacetic acid-degradative plasmids isolated from soil by complementation
E M Top 1, W E Holben, L J Forney
https://aem.asm.org/content/61/5/1691.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC167431/
https://aem.asm.org/content/aem/61/5/1691.full.pdf

Plasmid pEMT1 obtained in this study is a non-IncP1 plasmid that has catabolic genes that show high degrees of homology to the tfdABCDEFR/S pathways of the IncP1 plasmid pJP4.

Both IncP1 and non-IncP1 plasmids encoding 2,4-D degradation were present in the soil samples. The incompatibility group designation of several 2,4-D-degradative plasmids described in the literature is still unknown (5, 35). 




----------



----------
## unclassified


----------
### 


----------
### 

----------
### 2021-09-23
### pBAM1
IncP1-β

https://pubmed.ncbi.nlm.nih.gov/34556718/
Sci Rep
. 2021 Sep 23;11(1):18943. doi: 10.1038/s41598-021-98184-5.
The complete genome of 2,6-dichlorobenzamide (BAM) degrader Aminobacter sp. MSH1 suggests a polyploid chromosome, phylogenetic reassignment, and functions of plasmids
Tue Kjærgaard Nielsen # 1, Benjamin Horemans # 2 3, Cédric Lood 4 5, Jeroen T'Syen 2, Vera van Noort 4, Rob Lavigne 5, Lea Ellegaard-Jensen 6, Ole Hylling 6, Jens Aamand 7, Dirk Springael 8, Lars Hestbjerg Hansen 9
https://www.nature.com/articles/s41598-021-98184-5
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8460812/
The first step of BAM-mineralization involves the hydrolysis of BAM to 2,6-dichlorobenzoic acid (2,6-DCBA) by the amidase BbdA encoded on the 41 kb IncP1-β plasmid pBAM17.

Besides the previously described IncP1-β and repABC plasmids, pBAM1 and pBAM27,8,

Data availability
The genome sequences of strain MSH1 substrains MK1/DK1 are available under the following GenBank accession numbers CP026265/CP028968 (chromosome), CP026268/CP028967 (pBAM1), CP026267/CP028966 (pBAM2), CP026266/CP028969 (pUSP1) and CP028970 (pUSP2), CP028971 (pUSP3), CP028972 (pUSP4) and CP028973 (pUSP5).

https://www.ncbi.nlm.nih.gov/nuccore/CP026268
```
LOCUS       CP026268               40564 bp    DNA     circular BCT 25-APR-2018
DEFINITION  Aminobacter sp. MSH1 plasmid pBAM1, complete sequence.

  ORGANISM  Aminobacter sp. MSH1
            Bacteria; Proteobacteria; Alphaproteobacteria; Hyphomicrobiales;
            Phyllobacteriaceae; Aminobacter; unclassified Aminobacter.

COMMENT     On Apr 25, 2018 this sequence version replaced KP792998.1.
            Bacteria and source DNA available from Benjamin Horemans KULeuven.

                     /country="Denmark: near Hvidovre"
                     /collection_date="2007"
                     /collected_by="Maria S.Holtze"
                     /identified_by="Sebastian R. Sorensen"

     CDS             complement(10179..12437)
                     /product="conjugal transfer relaxase TraI"
                     /protein_id="AKD43467.1"

     CDS             complement(25481..26704)
                     /gene="trfA"
                     /product="Plasmid replication initiator protein TrfA"
                     /protein_id="AKD43447.1"

```

https://www.ncbi.nlm.nih.gov/nuccore/CP028967
```
LOCUS       CP028967               40559 bp    DNA     circular BCT 23-APR-2018
DEFINITION  Aminobacter sp. MSH1 plasmid pBAM1, complete sequence.

                     /country="Denmark: Hvidovre"
                     /collection_date="2007"

```

----------
### pSR8

https://github.com/SionBayliss/PIRATE/issues/65
length is longer in CP062120 (197271 bp) than in the other sequences (

https://www.ncbi.nlm.nih.gov/bioproject/PRJNA664737
https://www.ncbi.nlm.nih.gov/assembly/GCA_014843375.1

https://www.ncbi.nlm.nih.gov/nuccore/CP062120
```
LOCUS       CP062120              197271 bp    DNA     linear   BCT 05-OCT-2020
DEFINITION  Aminobacter sp. SR38 plasmid pSR8.

  ORGANISM  Aminobacter sp. SR38
            Bacteria; Proteobacteria; Alphaproteobacteria; Hyphomicrobiales;
            Phyllobacteriaceae; Aminobacter; unclassified Aminobacter.

                     /isolation_source="soil"
                     /country="France"
                     /collection_date="2000"

     CDS             90629..91849
                     /product="replication initiator protein A"
                     /protein_id="QOF75817.1"

     CDS             188635..189851
                     /note="frameshifted; Derived by automated computational
                     analysis using gene prediction method: Protein Homology."
                     /pseudo
                     /product="transcriptional regulator"


     CDS             148572..150812
                     /product="relaxase/mobilization nuclease domain-containing
                     protein"
                     /protein_id="QOF75865.1"

     CDS             17842..20068
                     /note="frameshifted; Derived by automated computational
                     analysis using gene prediction method: Protein Homology."
                     /pseudo
                     /product="relaxase/mobilization nuclease domain-containing
                     protein"

```

----------
### 2015-04-30
### pBII_1

https://pubmed.ncbi.nlm.nih.gov/25931592/
Genome Announc
. 2015 Apr 30;3(2):e00159-15. doi: 10.1128/genomeA.00159-15.
Complete genome sequences for 59 burkholderia isolates, both pathogenic and near neighbor
Shannon L Johnson 1, Kimberly A Bishop-Lilly, Jason T Ladner 2, Hajnalka E Daligault 3, Karen W Davenport 3, James Jaissle 4, Kenneth G Frey, Galina I Koroleva 2, David C Bruce 5, Susan R Coyne 4, Stacey M Broomall 6, Po-E Li 3, Hazuki Teshima 3, Henry S Gibbons 6, Gustavo F Palacios 2, C Nicole Rosenzweig 6, Cassie L Redden, Yan Xu 3, Timothy D Minogue 4, Patrick S Chain 3
https://journals.asm.org/doi/10.1128/genomeA.00159-15
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4417688/

https://www.ncbi.nlm.nih.gov/nuccore/CP009797
```
LOCUS       CP009797               43581 bp    DNA     circular BCT 26-JUL-2016
DEFINITION  Burkholderia ambifaria AMMD plasmid pBII_1, complete sequence.

  ORGANISM  Burkholderia ambifaria AMMD
            Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales;
            Burkholderiaceae; Burkholderia; Burkholderia cepacia complex.

                     /country="USA: Wisconsin"
                     /collection_date="1985"

     CDS             complement(505..1725)
                     /product="trfA family protein"
                     /protein_id="AJY20331.1"

     CDS             complement(28956..31196)
                     /gene="traI"
                     /product="protein traI"
                     /protein_id="AJY20360.1"

```

----------
### 2011-09
### pALIDE02

https://pubmed.ncbi.nlm.nih.gov/21742888/
J Bacteriol
. 2011 Sep;193(18):5028-9. doi: 10.1128/JB.00365-11. Epub 2011 Jul 8.
Genome sequences of Alicycliphilus denitrificans strains BC and K601T
Margreet J Oosterkamp 1, Teun Veuskens, Caroline M Plugge, Alette A M Langenhoff, Jan Gerritse, Willem J H van Berkel, Dietmar H Pieper, Howard Junca, Lynne A Goodwin, Hajnalka E Daligault, David C Bruce, John C Detter, Roxanne Tapia, Cliff S Han, Miriam L Land, Loren J Hauser, Hauke Smidt, Alfons J M Stams
https://journals.asm.org/doi/10.1128/JB.00365-11
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3165661/
For strain BC, the accession numbers are CP002449 (chromosome), CP002450 (megaplasmid), and CP002451 (plasmid).

https://www.ncbi.nlm.nih.gov/nuccore/CP002451
```
LOCUS       CP002451               78982 bp    DNA     circular BCT 25-JUL-2016
DEFINITION  Alicycliphilus denitrificans BC plasmid pALIDE02, complete
            sequence.

  ORGANISM  Alicycliphilus denitrificans BC
            Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales;
            Comamonadaceae; Alicycliphilus.

     CDS             complement(9784..12024)
                     /note="PFAM: Relaxase/mobilization nuclease family
                     protein;
                     KEGG: reu:Reut_D6526 conjugal transfer relaxase TraI"
                     /product="Relaxase/mobilization nuclease family protein"
                     /protein_id="ADV02250.1"

     CDS             complement(45090..46310)
                     /product="TrfA family protein"
                     /protein_id="ADV02287.1"

```

----------

### pA81
CP002288
AJ515144

### 2011-02

https://pubmed.ncbi.nlm.nih.gov/21097610/
J Bacteriol
. 2011 Feb;193(3):791-2. doi: 10.1128/JB.01299-10. Epub 2010 Nov 19.
Complete genome sequence of the haloaromatic acid-degrading bacterium Achromobacter xylosoxidans A8
Hynek Strnad 1, Jakub Ridl, Jan Paces, Michal Kolar, Cestmir Vlcek, Vaclav Paces
https://journals.asm.org/doi/10.1128/JB.01299-10
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3021230/
Its genome consists of a single chromosome comprising 7,013,095 bp, one circular conjugative plasmid, pA81, of 98,156 bp, and one circular plasmid, pA82, of 247,895 bp.

The nucleotide sequences were deposited in GenBank with accession numbers CP002287 (chromosome), CP002288 (plasmid pA81), and CP002289 (plasmid pA82).

chromosome
https://www.ncbi.nlm.nih.gov/nuccore/CP002287

plasmid
https://www.ncbi.nlm.nih.gov/nuccore/CP002288
```
LOCUS       CP002288               98156 bp    DNA     circular BCT 25-JUL-2016
DEFINITION  Achromobacter xylosoxidans A8 plasmid pA81, complete sequence.

  ORGANISM  Achromobacter xylosoxidans A8
            Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales;
            Alcaligenaceae; Achromobacter.

   PUBMED   18249097

   PUBMED   21097610

     CDS             complement(60..1283)
                     /product="TrfA family protein"
                     /protein_id="ADP19822.1"

     CDS             complement(65692..67950)
                     /gene="traI"
                     /product="conjugal transfer relaxase TraI"
                     /protein_id="ADP19887.1"

```

### 2008-03

https://pubmed.ncbi.nlm.nih.gov/18249097/
Res Microbiol
. 2008 Mar;159(2):118-27. doi: 10.1016/j.resmic.2007.11.018. Epub 2007 Dec 8.
Nucleotide sequence, organization and characterization of the (halo)aromatic acid catabolic plasmid pA81 from Achromobacter xylosoxidans A8
Vera Jencova 1, Hynek Strnad, Zdenek Chodora, Pavel Ulbrich, Cestmir Vlcek, W J Hickey, Vaclav Paces
https://www.sciencedirect.com/science/article/abs/pii/S0923250807002410?via%3Dihub
The backbone genes controlled conjugative transfer, replication and plasmid stability, and were well conserved with other IncP1-beta plasmids. 


https://www.ncbi.nlm.nih.gov/nuccore/AJ515144
```
LOCUS       AJ515144               98192 bp    DNA     circular BCT 14-NOV-2006
DEFINITION  Achromobacter xylosoxidans plasmid pA81, strain A8.

                     /country="Czech Republic"

     CDS             complement(60..1286)
                     /gene="trfA"
                     /product="replication initiation protein"
                     /protein_id="CAI47811.1"


     CDS             complement(65695..67953)
                     /gene="traI"
                     /product="DNA relaxase"
                     /protein_id="CAI47869.1"

```

----------
### pADP-1

https://www.ncbi.nlm.nih.gov/nuccore/U66917
```
LOCUS       U66917                108845 bp    DNA     circular BCT 06-APR-2020
DEFINITION  Pseudomonas sp. ADP atrazine catabolic plasmid pADP-1, complete

  ORGANISM  Pseudomonas sp. ADP
            Bacteria; Pseudomonadota; Gammaproteobacteria; Pseudomonadales;
            Pseudomonadaceae; Pseudomonas.
REFERENCE   1  (bases 34964 to 36388)
  AUTHORS   de Souza,M.L., Wackett,L.P., Boundy-Mills,K.L., Mandelbaum,R.T. and
            Sadowsky,M.J.
  TITLE     Cloning, characterization, and expression of a gene region from
            Pseudomonas sp. strain ADP involved in the dechlorination of
            atrazine
  JOURNAL   Appl. Environ. Microbiol. 61 (9), 3373-3378 (1995)
   PUBMED   7574646
REFERENCE   2  (bases 43552 to 46212)
  AUTHORS   Boundy-Mills,K.L., de Souza,M.L., Mandelbaum,R.T., Wackett,L.P. and
            Sadowsky,M.J.
  TITLE     The atzB gene of Pseudomonas sp. strain ADP encodes the second
            enzyme of a novel atrazine degradation pathway
  JOURNAL   Appl. Environ. Microbiol. 63 (3), 916-923 (1997)
   PUBMED   9055410
REFERENCE   3  (bases 69775 to 71867)
  AUTHORS   Sadowsky,M.J., Tong,Z., de Souza,M. and Wackett,L.P.
  TITLE     AtzC is a new member of the amidohydrolase protein superfamily and
            is homologous to other atrazine-metabolizing enzymes
  JOURNAL   J. Bacteriol. 180 (1), 152-158 (1998)
   PUBMED   9422605


     CDS             12543..14783
                     /gene="traI"
                     /note="DNA relaxase"
                     /product="TraI protein"
                     /protein_id="AAK50254.1"

     CDS             89488..90399
                     /gene="trfA"
                     /product="TrfA protein"
                     /protein_id="AAK50322.1"

```



----------
### Arsene-Ploetze_2010-02-26
### pTHI

https://pubmed.ncbi.nlm.nih.gov/20195515/
PLoS Genet
. 2010 Feb 26;6(2):e1000859. doi: 10.1371/journal.pgen.1000859.
Structure, function, and evolution of the Thiomonas spp. genome
Florence Arsène-Ploetze 1, Sandrine Koechler, Marie Marchal, Jean-Yves Coppée, Michael Chandler, Violaine Bonnefoy, Céline Brochier-Armanet, Mohamed Barakat, Valérie Barbe, Fabienne Battaglia-Brunet, Odile Bruneel, Christopher G Bryan, Jessica Cleiss-Arnold, Stéphane Cruveiller, Mathieu Erhardt, Audrey Heinrich-Salmeron, Florence Hommais, Catherine Joulian, Evelyne Krin, Aurélie Lieutaud, Didier Lièvremont, Caroline Michel, Daniel Muller, Philippe Ortet, Caroline Proux, Patricia Siguier, David Roche, Zoé Rouy, Grégory Salvignol, Djamila Slyemi, Emmanuel Talla, Stéphanie Weiss, Jean Weissenbach, Claudine Médigue, Philippe N Bertin
https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000859
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2829063/

The plasmid, pTHI, contains 68 predicted CDS. 21 genes were found in synteny with genes carried by the R. eutropha JMP134 plasmid pJP4, and among them, par/trf/pem genes necessary for plasmid partitioning, stability and replication (Figure S1B). These observations suggest that pTHI, as JMP134, belongs to the IncP-1β group [28]. pTHI contains 13 of the 14 genes involved in conjugation (vir and tra genes) and genes that could fulfill the function of the missing components were found on the chromosome. Therefore, Thiomonas sp. 3As may be able to express a complete Type IV secretory system (T4SS) of the Vir/Tra type required for pTHI conjugal transfer. IncP-1β members are known to carry multi-resistance determinants and degradative cassettes [28], and plasmid pTHI indeed carries a Tn3-related transposon. This transposon contains part of a mercury resistance operon found in many other Gram negative bacterial transposon such as Tn21, Tn501 and Tn5053 [29].

The EMBL (http://www.ebi.ac.uk/embl) accession numbers for the genome of Thiomonas sp. 3As are FP475956 (chromosome) and FP475957 (plasmid).

https://www.ncbi.nlm.nih.gov/nuccore/FP475957
```
LOCUS       FP475957               46756 bp    DNA     circular BCT 25-JUL-2016
DEFINITION  Thiomonas sp. str. 3As plasmid pTHI, complete genome.

  ORGANISM  Thiomonas arsenitoxydans
            Bacteria; Proteobacteria; Betaproteobacteria; Burkholderiales;
            Thiomonas.


     CDS             2554..3459
                     /locus_tag="THI_p0005"
                     /function="8 : DNA metabolism"
                     /inference="ab initio prediction:AMIGene:2.0"
                     /note="Evidence 3 : Function proposed based on presence of
                     conserved amino acid motif, structural feature or limited
                     homology; Product type pf : putative factor"
                     /product="putative TrfA protein"
                     /protein_id="CAZ90403.1"


     CDS             23390..24397
                     /gene="virD2"
                     /locus_tag="THI_p0040"
                     /function="7 : Transport and binding proteins"
                     /function="17 : Mobile and extrachromosomal element
                     functions"
                     /inference="ab initio prediction:AMIGene:2.0"
                     /note="Evidence 2b : Function of strongly homologous gene;
                     PubMedId : 3021341; Product type e : enzyme"
                     /codon_start=1
                     /transl_table=11
                     /product="Type IV secretory pathway, VirD2 components
                     (relaxase)"
                     /protein_id="CAZ90436.1"


```



----------

https://archaea.i2bc.paris-saclay.fr/wasps/

https://en.wikipedia.org/wiki/Plasmid-mediated_resistance


<img alt="" src="" width=50%>
<img alt="" src="" width=25%>
![]()

```

```


https://www.ncbi.nlm.nih.gov/protein/WP_011114072
```
LOCUS       WP_011114072             406 aa            linear   BCT 27-APR-2020
DEFINITION  MULTISPECIES: replication initiator protein A [Bacteria].

                     /product="replication initiator protein A"
                     /function="initiation of replication"

                     /region_name="TrfA"
                     /note="TrfA protein; pfam07042"

```





----------









