# plasmids.papers

**Table of Contents**
- [wolbachia](#wolbachia)
- [updates](#updates)
  - [2021](#2021)
  - [2019](#2019)
  - [2018](#2018)
  - [2017](#2017)
- [mutation rate](#mutation-rate)
- [review](#review)
- [rRNA](#rrna)
- [compensatory](#compensatory)
- [IncP7](#incp7)
- [ICE](#ice)
- [rep](#rep)
- [annotation](#annotation)
- [unclassified](#unclassified)
- [compensatory](#compensatory)
- [MOB](#mob)
- [people](#people)
[Rocha](#rocha)
[Shintani](#shintani)
[Suzuki](#suzuki)
- [](#)
- [](#)
- [chromid](#chromid)
- [Hi-C](#hi-c)
- [long read](#long-read)

----------


----------
## long read

### 2024-03-27
https://pubmed.ncbi.nlm.nih.gov/38538803/
Commun Biol
. 2024 Mar 27;7(1):357. doi: 10.1038/s42003-024-05998-w.
Uncovering microbiomes of the rice phyllosphere using long-read metagenomic sequencing
Sachiko Masuda 1, Pamela Gan 1, Yuya Kiguchi 2 3 4, Mizue Anda 2, Kazuhiro Sasaki 5 6, Arisa Shibata 1, Wataru Iwasaki 2, Wataru Suda 4, Ken Shirasu 7 8
https://www.nature.com/articles/s42003-024-05998-w
https://pmc.ncbi.nlm.nih.gov/articles/PMC10973392/
Pacbio Sequel II sequencer

2025/01/04
https://www.youtube.com/watch?v=ukgkBQxqjYQ
Uncovering microbiomes of the rice phyllosphere using long-read metagenomic sequencing
PacBio
Sachiko Masuda, PhD, Research Scientist, Plant Immunity Research Group at RIKEN Center for Sustainable Resource Science speaks at PacBio Prism in Bangkok.

### 2019-08-27
https://pubmed.ncbi.nlm.nih.gov/31455406/
Microbiome
. 2019 Aug 27;7(1):119. doi: 10.1186/s40168-019-0737-z.
Long-read metagenomic exploration of extrachromosomal mobile genetic elements in the human gut
Yoshihiko Suzuki 1, Suguru Nishijima 1 2 3, Yoshikazu Furuta 4, Jun Yoshimura 1, Wataru Suda 1 5, Kenshiro Oshima 1, Masahira Hattori 6 7 8, Shinichi Morishita 9
https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-019-0737-z
https://pmc.ncbi.nlm.nih.gov/articles/PMC6712665/
PacBio long-read metagenomic data. De novo assembly of PacBio long reads from 12 faecal samples generated 82 eMGE contigs (2.5~666.7-kb), which were classified as 71 plasmids and 11 bacteriophages, including 58 novel plasmids and six bacteriophages, and complete genomes of five diverse crAssphages with terminal direct repeats.

### 2020-06
https://pubmed.ncbi.nlm.nih.gov/32514171/
Nat Med
. 2020 Jun;26(6):941-951. doi: 10.1038/s41591-020-0894-4. Epub 2020 Jun 8.
Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment
Kern Rei Chng # 1, Chenhao Li # 1, Denis Bertrand # 1, Amanda Hui Qi Ng 1, Junmei Samantha Kwah 1, Hwee Meng Low 1, Chengxuan Tong 1, Maanasa Natrajan 1, Michael Hongjie Zhang 1, Licheng Xu 2, Karrie Kwan Ki Ko 3 4 5, Eliza Xin Pei Ho 1, Tamar V Av-Shalom 1, Jeanette Woon Pei Teo 6, Chiea Chuen Khor 1; MetaSUB Consortium; Swaine L Chen 1, Christopher E Mason 7, Oon Tek Ng 8 9 10, Kalisvar Marimuthu 8 9 11, Brenda Ang 8 9, Niranjan Nagarajan 12 13
Collaborators, Affiliations Collapse
Collaborators
MetaSUB Consortium: 
Haruo Suzuki, 
https://www.nature.com/articles/s41591-020-0894-4
https://pmc.ncbi.nlm.nih.gov/articles/PMC7303012/
Abstract
Quasi-metagenomics with nanopore sequencing provided thousands of high-contiguity genomes, phage and plasmid sequences (>60% novel), enabling characterization of resistome and mobilome diversity and dynamic architectures in hospital environments.

Fig. 4. Species distribution and genomic proximity of drug resistance genes in the hospital-environment microbiome.

Genomic proximity network and clustering of ARGs based on 2,347 microbial genomes and 5,910 closed plasmid sequences obtained from the hospital environment. a, Multigraph of genomic proximity between ARGs.

Performing a similar analysis for closed plasmids, we first clustered them into shared backbones and annotated them for known hosts (identity â‰¥â€‰95%; Methods). By analyzing ARGs in this context, we found that many ARGs were variably present in backbones (93 of 143). For ARGs stably found in one backbone, many were variably present in another backbone (19 of 31), highlighting the dynamic nature of ARG combinations from plasmids in the hospital environment (Fig. 4b).

Genome assembly and species assignment
Nanopore reads for each plate were assembled using Canu64 (v1.3 and v1.7; genomeSizeâ€‰=â€‰8â€‰m). For samples where both Illumina and nanopore reads were available, a higher-quality hybrid assembly was obtained using OPERA-MS19 (v0.8.3; --polish --no-gap-filling --short-read-assembler spades). Assembled contigs were mapped to the NCBI nt database with BLAST (v2.2.28), to identify microbial species or plasmid assignments according to the best BLAST hit (highest total reference coverage).

Similarly, novel circular plasmids were identified by comparing to the PLSDB36 database with Mash distance and identifying clusters at 99% identity (single linkage) with no known sequence.

Analysis of ARG combinations
ARGs were annotated to contigs by mapping them to the ARG-ANNOT70 database provided in SRST2 (v3) with BLAST (best hit with >90% identity and >90% reference coverage). ARG combinations present in chromosomes and plasmid sequences were considered novel when they were not found in the reference databases (nt or PLSDB36). Assembled circular plasmids were clustered and annotated based on their best BLAST hit with identity >95% and >60% query coverage. A bipartite graph was constructed by connecting each plasmid cluster to ARGs found in it, with edge weights representing the frequency of occurrence (clusters with <5 representatives were excluded).

Data availability
Sequencing reads and assemblies are available from the European Nucleotide Archive under project PRJEB31632. Source code and data for reproducing figures are available under MIT license at https://github.com/csb5/hospital_microbiome. Assemblies and annotations for genomes, plasmids and phages are available at https://t.co/bdZxADGM7z.


----------
## Hi-C

https://pubmed.ncbi.nlm.nih.gov/40980884/
mBio
. 2025 Sep 22:e0224325. doi: 10.1128/mbio.02243-25. Online ahead of print.
Hi-C-resolved metagenomics reveals host range variation among mobile genetic elements within the European honey bee
https://journals.asm.org/doi/10.1128/mbio.02243-25

https://pubmed.ncbi.nlm.nih.gov/40161467/
ISME Commun
. 2025 Mar 9;5(1):ycae161. doi: 10.1093/ismeco/ycae161. eCollection 2025 Jan.
Detection of rare plasmid hosts using a targeted Hi-C approach
Salvador CastaÃ±eda-Barba 1 2 3, Benjamin J Ridenhour 2 3 4 5, Eva M Top 1 2 3 4, Thibault Stalder 1 3 4 6
https://academic.oup.com/ismecommun/article/5/1/ycae161/8064735?login=false
https://pmc.ncbi.nlm.nih.gov/articles/PMC11950669/


https://pubmed.ncbi.nlm.nih.gov/38228585/
Nat Commun
. 2024 Jan 16;15(1):555. doi: 10.1038/s41467-024-44827-w.
Host- plasmid network structure in wastewater is linked to antimicrobial resistance genes
Alice Risely # 1, Arthur Newbury # 2 3, Thibault Stalder 4 5, Benno I Simmons 2, Eva M Top 4 5, Angus Buckling 2 3, Dirk Sanders 6 7
https://www.nature.com/articles/s41467-024-44827-w
https://pmc.ncbi.nlm.nih.gov/articles/PMC10791616/


https://pubmed.ncbi.nlm.nih.gov/31147603/
ISME J
. 2019 Oct;13(10):2437-2446. doi: 10.1038/s41396-019-0446-4. Epub 2019 May 30.
Linking the resistome and plasmidome to the microbiome
Thibault Stalder 1 2, Maximilian O Press 3, Shawn Sullivan 3, Ivan Liachko 3, Eva M Top 4 5
https://academic.oup.com/ismej/article/13/10/2437/7475059?login=false
https://pmc.ncbi.nlm.nih.gov/articles/PMC6776055/


https://phasegenomics.com/hi-c-solves-the-problem-of-linking-plasmids-to-host-cells-in-a-mixed-sample/
Hi-C solves the problem of linking plasmids to host cells in a mixed sample - Phase Genomics

https://www.ncbi.nlm.nih.gov/pubmed/26104560
Microbiol Spectr. 2015 Feb;3(1):PLAS-0038-2014. doi: 10.1128/microbiolspec.PLAS-0038-2014.
Plasmid Detection, Characterization, and Ecology.
Smalla K1, Jechalke S1, Top EM2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4480600/
Figure 1
Overview of applying Hi-C technology to a mixed bacterial population to reliably associate plasmids with the chromosomes of their hosts (modified from Burton et al. [41]).


----------
## chromid


https://www.ncbi.nlm.nih.gov/pubmed/31138741
MBio. 2019 May 28;10(3). pii: e00505-19. doi: 10.1128/mBio.00505-19.
Codon Usage Heterogeneity in the Multipartite Prokaryote Genome: Selection-Based Coding Bias Associated with Gene Location, Expression Level, and Ancestry.
LÃ³pez JL1, Lozano MJ1, Lagares A Jr2,3, Fabre ML1, Draghi WO1, Del Papa MF1, Pistorio M1, Becker A3, Wibberg D4, SchlÃ¼ter A4, PÃ¼hler A4, Blom J5, Goesmann A5, Lagares A6.
- Sinorhizobium meliloti, which harbors a chromosome, a chromid (pSymB), a megaplasmid (pSymA), and, in many strains, one or more accessory plasmids.
- modal codon usages


9:24 PM Â· Feb 18, 2019
https://twitter.com/WaldminghausLab/status/1097471902711513088
Torsten Waldminghaus on Twitter: "Rule: Bacteria have one chromosome. Exception: Vibrios have two chromosomes. Exception to exception: Natural Single Chromosome Vibrio (NSCV) More about it in our recent paper: https://t.co/d6ezhPePGF @PMOrwin @biotweeps #microbefacts" / Twitter


12:18 AM Â· Feb 19, 2019
https://x.com/vscooper/status/1097515615298994176
Vaughn Cooper (see: vscooper@bsky) on X: "@WaldminghausLab @surt_lab @PMOrwin @biotweeps Really cool, congrats! Weâ€™ve been studying evolution of multi-chromosome bacteria for awhile bc of their broad implications e.g. https://t.co/3vMONxuHIY" / X

https://pubmed.ncbi.nlm.nih.gov/20369015/
PLoS Comput Biol
. 2010 Apr 1;6(4):e1000732. doi: 10.1371/journal.pcbi.1000732.
Why genes evolve faster on secondary chromosomes in bacteria
Vaughn S Cooper 1, Samuel H Vohr, Sarah C Wrocklage, Philip J Hatcher
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000732
https://pmc.ncbi.nlm.nih.gov/articles/PMC2848543/


https://pubmed.ncbi.nlm.nih.gov/30608550/
Comparative Study Mol Biol Evol
. 2019 Mar 1;36(3):562-574. doi: 10.1093/molbev/msy248.
Chromids Aid Genome Expansion and Functional Diversification in the Family Burkholderiaceae
George C diCenzo 1, Alessio Mengoni 1, Elena Perrin 1
https://academic.oup.com/mbe/article/36/3/562/5273485
The genomes of about 10% of bacterial species display a multipartite structure, consisting of a chromosome plus one or more large additional replicons of at least 350â€‰kb (Harrison et al. 2010; diCenzo and Finan 2017). The presence of a plasmid (nonessential replicon less than 350â€‰kb in size) is not sufficient for a genome to be designated as multipartite. The large secondary replicons (i.e., the nonchromosomal replicons) can be essential (termed â€œchromidâ€) or nonessential (termed â€œmegaplasmidâ€), and the primary features of these replicons and multipartite genomes have been reviewed recently (diCenzo and Finan 2017). 

Posted August 27, 2018.
https://www.biorxiv.org/content/10.1101/264945v3
Neo-formation of chromosomes in bacteria | bioRxiv

https://pubmed.ncbi.nlm.nih.gov/28794225/
Review Microbiol Mol Biol Rev
. 2017 Aug 9;81(3):e00019-17. doi: 10.1128/MMBR.00019-17. Print 2017 Sep.
The Divided Bacterial Genome: Structure, Function, and Evolution
George C diCenzo 1, Turlough M Finan 2
https://journals.asm.org/doi/10.1128/MMBR.00019-17
https://pmc.ncbi.nlm.nih.gov/articles/PMC5584315/

Replicon and Secondary Replicon
We suggest that each replicon be classified into one of the following five groups, as described below and in Fig. 1: chromosome, second chromosome, chromid, megaplasmid, and plasmid.

While any size limitation is essentially arbitrary, we suggest a lower cutoff of 350 kb for megaplasmid status, as this is equal to roughly 10% of the median bacterial genome size. Any nonessential replicon of <350 kb would therefore be a plasmid.

https://www.ncbi.nlm.nih.gov/pubmed/23123907
J Bacteriol. 2013 Jan;195(2):202-12. doi: 10.1128/JB.01758-12. Epub 2012 Nov 2.
The tRNAarg gene and engA are essential genes on the 1.7-Mb pSymB megaplasmid of Sinorhizobium meliloti and were translocated together from the chromosome in an ancestral strain.
diCenzo G1, Milunovic B, Cheng J, Finan TM.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3553834/
The challenges in defining autonomously replicating replicons as second chromosomes and megaplasmids have been discussed previously (21), and the term â€œchromidâ€ was proposed and defined (22).

https://www.ncbi.nlm.nih.gov/pubmed/20080407
Trends Microbiol. 2010 Apr;18(4):141-8. doi: 10.1016/j.tim.2009.12.010. Epub 2010 Jan 18.
Introducing the bacterial 'chromid': not a chromosome, not a plasmid.
Harrison PW1, Lower RP, Kim NK, Young JP.

----------

### Rocha

### 2024-02-20

https://pubmed.ncbi.nlm.nih.gov/38378896/
Nat Commun
. 2024 Feb 20;15(1):1545. doi: 10.1038/s41467-024-45757-3.
Phage-plasmids promote recombination and emergence of phages and plasmids
Eugen Pfeifer 1, Eduardo P C Rocha 2
https://www.nature.com/articles/s41467-024-45757-3
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10879196/
8:28 PM Â· Feb 21, 2024
https://twitter.com/epcrocha/status/1760265128082084272
Eduardo Rocha on X: "Our latest on phage-plasmids(PP) is out! See @UGene2020 thread. 2 main findings. PP recombine more with (other) plasmids and phages than the latter, thereby connecting them. PP can become just phages or plasmids, which may transition to be conjugative. All the worldâ€™s an MGE!" / X
7:32 PM Â· Feb 21, 2024
https://twitter.com/UGene2020/status/1760251163637670177
Eugen Pfeifer on X: "ðŸ§¬ Our latest study, done together with the inspiring @epcrocha, is out in @NatureComms !ðŸŽ‰ It delves into the fascinating world of phage-plasmids (P-Ps). Discover how P-Ps bridge the gap between phages and plasmids, shaping gene flow and evolution ðŸ‘‡ðŸ‘‡ðŸ‘‡ https://t.co/QLzMCqSShA" / X

### 2023-10-04

https://pubmed.ncbi.nlm.nih.gov/37788575/
Mol Biol Evol
. 2023 Oct 4;40(10):msad217. doi: 10.1093/molbev/msad217.
Chance Favors the Prepared Genomes: Horizontal Transfer Shapes the Emergence of Antibiotic Resistance Mutations in Core Genes
Charles Coluzzi 1, Martin Guillemet 1, Fanny Mazzamurro 1 2, Marie Touchon 1, Maxime Godfroid 3, Guillaume Achaz 3, Philippe Glaser 4, Eduardo P C Rocha 1
https://academic.oup.com/mbe/article/40/10/msad217/7288467
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10575684/

We then identified those positively or negatively associated with the trait (quinolone resistance) while accounting for the phylogenetic structure of the data using a linear mixed model (LMM) (see Methods section). 

We used all E. coli sequence type (ST) 131 (ST131) genomes from RefSeq (n = 85), and we retrieved all available E. coli ST 410 and 167 (ST410 and ST167) genomes from Enterobase (n = 1,000 and n = 801) (last accessed in December 2021) (Zhou et al. 2020). 

Genome-Wide Association Study
We performed a GWAS for the presence of fluoroquinolone resistance using 1,268 E. coli genomes with pyseer v.1.3.9 (Lees et al. 2018). The association between the gene presence/absence and the resistance phenotype (defined by the presence of a known resistance mutation) was assessed with an LMM in which the STs were considered as covariates. We used the recombination-free phylogenetic tree produced by Gubbins. This tree allowed us to generate a distance and a kinship matrix with the scripts coming with pyseer. The LMM used the multidimensional scaling (MDS) of these matrices to control for population structure. 

### 

https://pubmed.ncbi.nlm.nih.gov/37254807/
Nucleic Acids Res
. 2023 Jul 21;51(13):6806-6818. doi: 10.1093/nar/gkad452.
Restriction-modification systems have shaped the evolution and distribution of plasmids across bacteria
Liam P Shaw 1 2, Eduardo P C Rocha 3, R Craig MacLean 1
https://academic.oup.com/nar/article/51/13/6806/7186992
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10359461/


### 


https://pubmed.ncbi.nlm.nih.gov/36154183/
mBio
. 2022 Oct 26;13(5):e0185122. doi: 10.1128/mbio.01851-22. Epub 2022 Sep 26.
Phage-Plasmids Spread Antibiotic Resistance Genes through Infection and Lysogenic Conversion
Eugen Pfeifer 1, RÃ©my A Bonnin 2, Eduardo P C Rocha 1
https://journals.asm.org/doi/10.1128/mbio.01851-22
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9600943/

### 


https://pubmed.ncbi.nlm.nih.gov/33590101/
Nucleic Acids Res
. 2021 Mar 18;49(5):2655-2673. doi: 10.1093/nar/gkab064.
Bacteria have numerous distinctive groups of phage-plasmids with conserved phage and variable plasmid gene repertoires
Eugen Pfeifer 1, Jorge A Moura de Sousa 1, Marie Touchon 1, Eduardo P C Rocha 1
https://academic.oup.com/nar/article/49/5/2655/6137301?login=false
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7969092/


### 


https://pubmed.ncbi.nlm.nih.gov/29905872/
Mol Biol Evol
. 2018 Sep 1;35(9):2230-2239. doi: 10.1093/molbev/msy123.
Host Range and Genetic Plasticity Explain the Coexistence of Integrative and Extrachromosomal Mobile Genetic Elements
Jean Cury 1 2, Pedro H Oliveira 1 2, Fernando de la Cruz 3, Eduardo P C Rocha 1 2
https://academic.oup.com/mbe/article/35/9/2230/5037826
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6107060/
Conjugative systems were detected using the CONJscan module of MacSyFinder (Abby et al. 2014), with protein profiles and definitions of the MPF type T, published previously (Guglielmini et al. 2014).


### 

https://www.ncbi.nlm.nih.gov/pubmed/28911112
Nucleic Acids Res. 2017 Sep 6;45(15):8943-8956. doi: 10.1093/nar/gkx607.
Integrative and conjugative elements and their hosts: composition, distribution and organization.
Cury J1,2, Touchon M1,2, Rocha EPC1,2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5587801/
Detection of conjugative systems
Conjugative systems were found with the CONJscan module of MacSyFinder (29), using protein profiles and definitions following a previous work (16) 

### Guglielmini_2013

https://pubmed.ncbi.nlm.nih.gov/22977114/
Mol Biol Evol
. 2013 Feb;30(2):315-31. doi: 10.1093/molbev/mss221. Epub 2012 Sep 13.
Evolution of conjugation and type IV secretion systems
Julien Guglielmini 1, Fernando de la Cruz, Eduardo P C Rocha
https://academic.oup.com/mbe/article/30/2/315/1013981
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3548315/

Furthermore, this classification system can be applied to partial data, e.g., from metagenomics, because it requires the identification of a single gene.
 evolution of conjugation and T4SS


Conjugation of dsDNA takes place in mycelia-producing Actinobacteria (Grohmann et al. 2003; Ghinet et al. 2011). It relies on a single protein: TraBpSG5 that translocates dsDNA between neighboring cells in mycelia (Possoz et al. 2001). This protein resembles, in sequence and function, the essential protein FtsK that segregates sister chromosomes in the last stages of chromosomal replication (Bigot et al. 2007; Vogelmann et al. 2011).






----------
## people
### Rocha


### Shintani
æ–°è°· æ”¿å·±

https://pubmed.ncbi.nlm.nih.gov/32582111/
Front Microbiol
. 2020 Jun 9;11:1187. doi: 10.3389/fmicb.2020.01187. eCollection 2020.
Determination of Plasmid pSN1216-29 Host Range and the Similarity in Oligonucleotide Composition Between Plasmid and Host Chromosomes
Maho Tokuda 1, Haruo Suzuki 2 3, Kosuke Yanagiya 1, Masahiro Yuki 4, Kengo Inoue 5, Moriya Ohkuma 4, Kazuhide Kimbara 1, Masaki Shintani 1 4 6
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7296055/
https://www.frontiersin.org/articles/10.3389/fmicb.2020.01187/full
(Supplementary Table S1-1)
https://www.frontiersin.org/articles/10.3389/fmicb.2020.01187/full#supplementary-material

https://www.frontiersin.org/research-topics/11162/horizontal-gene-transfer-in-microbes


https://link.springer.com/chapter/10.1007/978-981-13-3411-5_6
Shintani M., Suzuki H. (2019) Plasmids and Their Hosts. In: Nishida H., Oshima T. (eds) DNA Traffic in the Environment. Springer, Singapore


https://tdb.shizuoka.ac.jp/RDB/public/Default2.aspx?id=11068&l=0&t=21
é™å²¡å¤§å­¦ï¼šæ•™å“¡ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ - æ–°è°· æ”¿å·± ï¼ˆSHINTANI Masakiï¼‰
[37]. æ–°è¦ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã‚°ãƒ«ãƒ¼ãƒ—PromAç¾¤ã«å±žã™ã‚‹GCå«é‡ã®ç•°ãªã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»åŸŸæ¯”è¼ƒ
æ—¥æœ¬è¾²èŠ¸åŒ–å­¦ä¼šï¼’ï¼ï¼‘ï¼™å¹´åº¦å¤§ä¼š ï¼ˆ2019å¹´3æœˆ24æ—¥ï¼‰ æ‹›å¾…è¬›æ¼”ä»¥å¤–

http://www.lc.shizuoka.ac.jp/pdf/baio36shintani.pdf
é›£åŸ¹é¤Šæ€§å¾®ç”Ÿç‰©ã‚’ã‚‚å¯¾è±¡ã«ã—ãŸãƒ•ã‚šãƒ©ã‚¹ãƒŸãƒˆã‚™ã®å®¿ä¸»åŸŸã®è§£æ˜Ž

https://kaken.nii.ac.jp/ja/grant/KAKENHI-PROJECT-15KK0278/
KAKEN â€” ç ”ç©¶èª²é¡Œã‚’ã•ãŒã™ | è¤‡åˆå¾®ç”Ÿç‰©ç³»ã«ãŠã‘ã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®ã€ŒçœŸã®ã€å®¿ä¸»åŸŸã®è§£æ˜Žï¼ˆå›½éš›å…±åŒç ”ç©¶å¼·åŒ–ï¼‰ (KAKENHI-PROJECT-15KK0278)
ã¾ãŸï¼Œå¾—ã‚‰ã‚ŒãŸãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã«ã¤ã„ã¦ï¼Œç¨®ã€…ã®æ¡ä»¶ã«ãŠã‘ã‚‹å®¿ä¸»åŸŸã‚’æ¯”è¼ƒã—ãŸã¨ã“ã‚ï¼Œé…¸ç´ æ¿ƒåº¦ã®é•ã„ã‚„ï¼Œãƒ—ãƒ©ã‚¹ãƒŸãƒ‰è‡ªä½“ã®å¡©åŸºçµ„æˆã®é•ã„ã«ã‚ˆã£ã¦ï¼Œãã®è¦‹ã‹ã‘ã®å®¿ä¸»åŸŸãŒå¤‰åŒ–ã™ã‚‹ã“ã¨ã‚‚åˆ¤æ˜Žã—ãŸï¼Ž

2019/09/10
https://www2.aeplan.co.jp/jsme2019/pdf/koen.pdf
æ—¥æœ¬å¾®ç”Ÿç‰©ç”Ÿæ…‹å­¦ä¼š ç¬¬33å›žå¤§ä¼š è¬›æ¼”è¦æ—¨é›†
- P2-17
è¤‡åˆå¾®ç”Ÿç‰©ç³»ã§å®Ÿéš›ã«ä¼æ’­ã™ã‚‹PromAç¾¤ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å–å¾—ã¨è§£æž
- P2-18
åŒä¸€ä¸å’Œåˆæ€§ç¾¤ã«å±žã™ã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã§ã‚ã£ã¦ã‚‚ãã®å®¿ä¸»åŸŸã¯ç•°ãªã‚‹
ã€èƒŒæ™¯ãƒ»ç›®çš„ã€‘ç’°å¢ƒä¸­ã«åºƒãåˆ†å¸ƒã™ã‚‹ã“ã¨ãŒäºˆæ¸¬ã•ã‚Œã¦ã„ã‚‹ï¼Œä¸å’Œåˆæ€§ç¾¤PromAã«å±žã™ã‚‹2ã¤ã®ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰pSN1104-11ã¨
pSN0729-62ã¯åŒä¸€ç´°èƒžå†…ã§å…±å­˜ã§ããªã„ï¼ˆä¸å’Œåˆæ€§ï¼‰ï¼Žã—ã‹ã—åŒæ–¹ã®GCå«é‡ãŒ10ï¼…ç•°ãªã‚‹ï¼ˆ1ï¼‰ï¼ŽåŒæ§˜ã«IncP-1ç¾¤ã«å±žã™ã‚‹
ã¨ã•ã‚Œã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰pBP136ã¨pDS1ã«ãŠã„ã¦ã‚‚GCå«é‡ãŒ18ï¼…ç•°ãªã‚‹ï¼Žãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®GCå«é‡ã¯å®¿ä¸»æŸ“è‰²ä½“ã®GCå«é‡ã‚ˆã‚Šã‚‚ä½Ž
ãï¼Œãã®é•ã„ã¯10ï¼…ä»¥å†…ã§ã‚ã‚‹ã“ã¨ãŒå¤šã„ï¼ˆ2ï¼‰ï¼Ž
ä»¥ä¸Šã®çµæžœã‹ã‚‰ï¼ŒåŒã˜ä¸å’Œ
åˆæ€§ç¾¤ã«å±žã™ã‚‹ã«ã‚‚é–¢ã‚ã‚‰ãšï¼Œå¡©åŸºçµ„æˆã®é•ã„ã«ã‚ˆã£ã¦å®¿ä¸»åŸŸãŒç•°ãªã‚Šï¼Œãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¨å®¿ä¸»æŸ“è‰²ä½“ã®GCå«é‡ã«ç›¸é–¢ãŒã‚ã‚‹ã“
ã¨ãŒç¤ºå”†ã•ã‚ŒãŸï¼Œç¾åœ¨ã¯ï¼ŒpBP136ã¨pDS1ã®å®¿ä¸»åŸŸã®æ¯”è¼ƒã‚’è¡Œã£ã¦ã„ã‚‹ï¼ŽåŒæ™‚ã«ï¼Œå®¿ä¸»åŸŸã®é•ã„ãŒå¡©åŸºçµ„æˆã«ã‚ˆã‚‹ã‚‚ã®ã§ã‚
ã‚‹ã‹ã‚’æ¤œè¨¼ã™ã‚‹ãŸã‚ï¼ŒåŒã˜PromAç¾¤ã«å±žã—ï¼Œ61ï¼…ã®GCå«é‡ã‚’ã‚‚ã¤pMH0613-68ã®å®¿ä¸»åŸŸã®èª¿æŸ»ã‚’è¡Œã£ã¦ã„ã‚‹ï¼Ž

https://www.ncbi.nlm.nih.gov/pubmed/30459733
Front Microbiol. 2018 Nov 6;9:2602. doi: 10.3389/fmicb.2018.02602. eCollection 2018.
Novel Self-Transmissible and Broad-Host-Range Plasmids Exogenously Captured From Anaerobic Granules or Cow Manure.
Yanagiya K1, Maejima Y1, Nakata H1, Tokuda M2, Moriuchi R3, Dohra H3, Inoue K4, Ohkuma M5, Kimbara K1,2, Shintani M1,2,5,6
https://www.frontiersin.org/articles/10.3389/fmicb.2018.02602/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6232296/

2018.06.25 
https://www.sbj.or.jp/sbj/sbj_vol96_no01.html
ç”Ÿç‰©å·¥å­¦ä¼šèªŒ â€“ 96å·»1å· | å…¬ç›Šç¤¾å›£æ³•äºº æ—¥æœ¬ç”Ÿç‰©å·¥å­¦ä¼š
ã€Œãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ãƒ‘ãƒ©ãƒ‰ãƒƒã‚¯ã‚¹ã€ï½žãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¯ãªãœã€Œç”Ÿãæ®‹ã£ã¦ã€ã„ã‚‹ï¼Ÿï½ž
æ–°è°· æ”¿å·±
https://www.sbj.or.jp/wp-content/uploads/file/sbj/9601/9601_biomedia_1.pdf
ã€Œãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ãƒ‘ãƒ©ãƒ‰ãƒƒã‚¯ã‚¹ã€
compensated mutationï¼ˆä»£å„Ÿçš„å¤‰ç•°ï¼‰
ä»¥ä¸Šã®ã“ã¨ã¯ï¼Œå°‘ãªãã¨ã‚‚è¤‡æ•°ã®ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»ã¯ï¼ŒãŸã¨ãˆé¸æŠžåœ§ãŒãªãã¨ã‚‚ï¼ŒæŸ“è‰²ä½“ãƒ»ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®éºä¼å­ã«ä»£å„Ÿçš„å¤‰ç•°ã‚’ç”Ÿã˜ã•ã›ï¼Œãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã‚’ã€Œç”Ÿãæ®‹ã‚‰ã›ã‚‹ã€ã‚ˆã†é€²åŒ–ã™ã‚‹ã“ã¨ã‚’ç¤ºå”†ã—ã¦ã„ã‚‹ï¼Ž

å‹•ç”»ï¼ˆ14:48ï¼‰
Nov 3, 2016
https://www.youtube.com/watch?v=W2UR1i6ihXE
ç¬¬10å›žè¶…é ˜åŸŸç ”ç©¶ä¼šã€Œè¤‡åˆå¾®ç”Ÿç‰©ç³»ã«ãŠã‘ã‚‹å¯å‹•æ€§éºä¼å› å­ã®æŒ™å‹•è§£æ˜Žã«å‘ã‘ã¦ã€ æ–°è°· æ”¿å·± å‡†æ•™æŽˆ-é™å²¡å¤§å­¦ ã€ç ”ç©¶è€…ç´¹ä»‹ã€‘

http://www.nougaku.jp/award/2016/3Masaki%20Shintani.pdf
ç’°å¢ƒå¾®ç”Ÿç‰©é–“ã«ãŠã‘ã‚‹å¯å‹•æ€§éºä¼å› å­ã®å‹•æ…‹ã«é–¢ã™ã‚‹ç ”ç©¶æ–°è°· æ”¿å·±(é™å²¡å¤§å­¦å­¦è¡“é™¢å·¥å­¦é ˜åŸŸ)

https://www.ncbi.nlm.nih.gov/pubmed/25873913
Front Microbiol. 2015 Mar 31;6:242. doi: 10.3389/fmicb.2015.00242. eCollection 2015.
Genomics of microbial plasmids: classification and identification based on replication and transfer systems and host taxonomy.
Shintani M1, Sanchez ZK2, Kimbara K2.
https://www.frontiersin.org/articles/10.3389/fmicb.2015.00242/full
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4379921/

2015-03
http://ci.nii.ac.jp/naid/40020727466
æ–°è°·æ”¿å·±ï¼Œé‡‘åŽŸå’Œç§€ï¼Žç’°å¢ƒãƒã‚¤ã‚ªãƒ†ã‚¯ãƒŽãƒ­ã‚¸ãƒ¼å­¦ä¼šèªŒ 14(2), 81-86.
"ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã‚²ãƒŽãƒŸã‚¯ã‚¹~å…¨å¡©åŸºé…åˆ—è§£èª­æ¸ˆã®ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã®æ•´å‚™"
Genomics of Plasmids
https://www.jseb.jp/wordpress/wp-content/uploads/14-02-081.pdf

https://www.ncbi.nlm.nih.gov/pubmed/24141122
Appl Environ Microbiol. 2014 Jan;80(1):138-45. doi: 10.1128/AEM.02571-13. Epub 2013 Oct 18.
Single-cell analyses revealed transfer ranges of IncP-1, IncP-7, and IncP-9 plasmids in a soil bacterial community.
Shintani M1, Matsui K, Inoue J, Hosoyama A, Ohji S, Yamazoe A, Nojiri H, Kimbara K, Ohkuma M.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911017/
Nucleotide sequence accession numbers.
The partial sequences of 519 transconjugants were deposited in the DDBJ, EMBL, and GenBank databases (accession numbers AB772697 to AB773215). The pyrosequencing run data of the 16S rRNA genes in soil samples were deposited in the DDBJ Sequence Read Archive (DRA) (http://trace.ddbj.nig.ac.jp/dra/index_e.html; accession number DRA001209).
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911017/bin/supp_80_1_138__index.html
Table S3. Top hit type strains with transconjugants identified in this study in BLAST searches.

2013-12
http://ci.nii.ac.jp/naid/40020320833
æ–°è°· æ”¿å·±, æ¾äº• ä¸€æ³°, é‡‘åŽŸ å’Œç§€, é‡Žå°» ç§€æ˜­. ç’°å¢ƒãƒã‚™ã‚¤ã‚ªãƒ†ã‚¯ãƒŽãƒ­ã‚·ã‚™ãƒ¼å­¦ä¼šèªŒ 13(2), 125-134.
"ç’°å¢ƒä¸­ã«ãŠã‘ã‚‹ãƒ•ã‚šãƒ©ã‚¹ãƒŸãƒˆã‚™ã®æŒ™å‹•è§£æž"
Comparisons of Conjugation Frequency in Different Environmental Conditions
https://www.jseb.jp/wordpress/wp-content/uploads/13-02-125.pdf
è¡¨ 2ï¼ŽPseudomonas å±žç´°èŒã«ãŠã‘ã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®ä¸å’Œåˆæ€§ç¾¤ã¨ãã®æŽ¥åˆä¼é”æ©Ÿæ§‹ã«ãŠã‘ã‚‹åˆ†é¡ž

æ—¥æœ¬è¾²èŠ¸åŒ–å­¦ä¼š 2012 å¹´åº¦å¤§ä¼šãƒˆãƒ”ãƒƒã‚¯ã‚¹è³ž ç™ºè¡¨ç•ªå·ï¼š2C22a03
http://www.jsbba.or.jp/wp-content/uploads/file/award/2012/topics/17_2C22a03.pdf
ä¸€ç´°èƒžãƒ¬ãƒ™ãƒ«ã®è§£æžã«ã‚ˆã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®åœŸå£Œç´°èŒé›†å›£ä¸­ã«ãŠã‘ã‚‹çœŸã®å®¿ä¸»åŸŸ


### Suzuki




----------
## MOB

https://pubmed.ncbi.nlm.nih.gov/35639760/
Mol Biol Evol
. 2022 Jun 2;39(6):msac115. doi: 10.1093/molbev/msac115.
Evolution of Plasmid Mobility: Origin and Fate of Conjugative and Nonconjugative Plasmids
Charles Coluzzi 1, Maria Pilar GarcillÃ¡n-Barcia 2, Fernando de la Cruz 2, Eduardo P C Rocha 1
https://academic.oup.com/mbe/article/39/6/msac115/6593704
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9185392/

5:12 AM Â· Dec 14, 2021
https://twitter.com/epcrocha/status/1470486804751126540
Eduardo Rocha on Twitter: "A decade ago we found that ca. 1/4 of plasmids are conjugative, 1/4 mobilisable &amp; 1/2 we donâ€™t know. How do these plasmids come about is the focus of our new preprint: â€œEvolution of plasmid mobility: origin and fate of non-conjugative plasmidsâ€ https://t.co/xxP4nXyHCx 1/8" / Twitter



### 2019

https://pubmed.ncbi.nlm.nih.gov/30718882/
Nat Biotechnol
. 2019 Feb;37(2):152-159. doi: 10.1038/s41587-018-0010-1. Epub 2019 Feb 4.
Ultrafast search of all deposited bacterial and viral genomic data
Phelim Bradley 1 2, Henk C den Bakker 3, Eduardo P C Rocha 4 5, Gil McVean 6, Zamin Iqbal 7 8
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6420049/
- Host-range of plasmids in the all-microbial index
The distribution of different versions of the machinery for conjugation of DNA between bacteria has previously been analysed in 1,124 genomes39 using sensitive, but slow, protein profile searches for relaxase (MOB) and type 4 secretion system (T4SS) genes. 
- Plasmid search and exclusion of contaminated datasets
2,826 plasmid sequences were taken from the ENA plasmid pages (www.ebi.ac.uk/genomes/plasmid.html; December 2016) (See Supplementary Data 2) and downloaded from the ENA. 
- Conjugative system search
MOB (MOB_B, MOB_C, MOB_CQ, MOB_F, MOB_H, MOB_P,MOB_T, MOB_V) and T4SS sequences (VirB4_TRaU, VirD4_TcpA) as defined in Guglielmini et al 39 and Supplementary Data 8 in the all-microbial-index with T=100%. Full search results are available in Supplementary Data 9. Results were filtered for bacteria and contamination following the same method as described in â€œPlasmid searchâ€. Accessions with at least one MOB and T4SS were said to contain a putative conjugative system. BIGSI does not return copy number, or location on chromosome or plasmid, so it was not possible to determine if the genes were co-located on a chromosome or on a plasmid.
- MCR-1,2,3
We searched for MCR-1, MCR-2, MCR-3 in the all-microbial-index using k-mer percent threshold T=100%. See Supplementary Data 1 for sequences and results.


https://pubmed.ncbi.nlm.nih.gov/21876676/
PLoS Genet
. 2011 Aug;7(8):e1002222. doi: 10.1371/journal.pgen.1002222. Epub 2011 Aug 18.
The repertoire of ICE in prokaryotes underscores the unity, diversity, and ubiquity of conjugation
Julien Guglielmini 1, Leonor Quintais, Maria Pilar GarcillÃ¡n-Barcia, Fernando de la Cruz, Eduardo P C Rocha
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3158045/
Relaxases (see [67] for a description of each family, except MOBB, and MOBT): MOBT (corresponding to protein Q47728 of Enterococcus faecalis conjugative transposon Tn916 Orf20 [70]), MOBB (corresponding to mobilization protein B of Bacteroides thetaiotaomicron VPI-5482 [71]), MOBV, MOBQ, MOBP, MOBH, MOBF, MOBC. Major ATPases: VirB4 and TraU. T4CP: VirD4. MPFF: TraLEKVCWUcNHD. MPFT: VirB3689. MPFI: TraIKLMNPQRWY. MPFG: p31, p35, p41, p44, p51, p52.



### Smillie_2010
### 2010-09

https://pubmed.ncbi.nlm.nih.gov/20805406/
Review Microbiol Mol Biol Rev
. 2010 Sep;74(3):434-52. doi: 10.1128/MMBR.00020-10.
Mobility of plasmids
Chris Smillie 1, M Pilar GarcillÃ¡n-Barcia, M Victoria Francia, Eduardo P C Rocha, Fernando de la Cruz
https://journals.asm.org/doi/10.1128/MMBR.00020-10
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2937521/


### Garcillan-Barcia_2009

https://pubmed.ncbi.nlm.nih.gov/19396961/
Review FEMS Microbiol Rev
. 2009 May;33(3):657-87. doi: 10.1111/j.1574-6976.2009.00168.x.
The diversity of conjugative relaxases and its application in plasmid classification
MarÃ­a Pilar GarcillÃ¡n-Barcia 1, MarÃ­a Victoria Francia, Fernando de la Cruz
https://academic.oup.com/femsre/article/33/3/657/591359

### Francia_2004

https://pubmed.ncbi.nlm.nih.gov/14975531/
Review FEMS Microbiol Rev
. 2004 Feb;28(1):79-100. doi: 10.1016/j.femsre.2003.09.001.
A classification scheme for mobilization regions of bacterial plasmids
M Victoria Francia 1, Athanasia Varsaki, M Pilar GarcillÃ¡n-Barcia, Amparo Latorre, Constantin Drainas, Fernando de la Cruz
https://onlinelibrary.wiley.com/doi/abs/10.1016/j.femsre.2003.09.001?sid=nlm%3Apubmed
https://academic.oup.com/femsre/article/28/1/79/635761






----------
## compensatory
ä»£å„Ÿå¤‰ç•°

https://pubmed.ncbi.nlm.nih.gov/32251388/
Nat Ecol Evol
. 2020 Jun;4(6):863-869. doi: 10.1038/s41559-020-1170-1. Epub 2020 Apr 6.
Coevolution of host-plasmid pairs facilitates the emergence of novel multidrug resistance
Hannah Jordt 1 2, Thibault Stalder 2 3, Olivia Kosterlitz 1 2, JosÃ© M Ponciano 4, Eva M Top 2 3, Benjamin Kerr 5 6
https://www.nature.com/articles/s41559-020-1170-1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8049106/


https://pubmed.ncbi.nlm.nih.gov/29046540/
Nat Ecol Evol
. 2017 Sep;1(9):1354-1363. doi: 10.1038/s41559-017-0243-2. Epub 2017 Aug 7.
Compensatory mutations improve general permissiveness to antibiotic resistance plasmids
Wesley Loftie-Eaton 1 2, Kelsie Bashford 1, Hannah Quinn 3, Kieran Dong 1, Jack Millstein 1 2, Samuel Hunter 2, Maureen K Thomason 4, Houra Merrikh 4, Jose M Ponciano 5, Eva M Top 6
https://www.nature.com/articles/s41559-017-0243-2
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5649373/


Posted September 11, 2017.
https://www.biorxiv.org/content/10.1101/187070v1
Compensatory evolution facilitates the acquisition of multiple plasmids in bacteria | bioRxiv
Alfonso Santos-Lopez, Cristina Bernabe-Balas, Alvaro San Millan, Rafael Ortega-Huedo, Andreas Hoefer, Manuel Ares-Arroyo, Bruno Gonzalez-Zorn

----------

### unclassified

https://www.frontiersin.org/research-topics/10952/plasmid-transfer-mechanisms-ecology-evolution-and-applications#articles
Plasmid Transfer: Mechanisms, Ecology, Evolution, and Applications | Frontiers Research Topic
Eva M Top
University of Idaho


https://www.ncbi.nlm.nih.gov/pubmed/27385827
Proc Natl Acad Sci U S A. 2016 Jul 19;113(29):8260-5. doi: 10.1073/pnas.1600974113. Epub 2016 Jul 6.
Source-sink plasmid transfer dynamics maintain gene mobility in soil bacterial communities.
Hall JP1, Wood AJ2, Harrison E3, Brockhurst MA3.


https://www.ncbi.nlm.nih.gov/pubmed/24587126
PLoS One. 2014 Feb 24;9(2):e89922. doi: 10.1371/journal.pone.0089922. eCollection 2014.
Cultivation-independent screening revealed hot spots of IncP-1, IncP-7 and IncP-9 plasmid occurrence in different environmental habitats.

https://www.ncbi.nlm.nih.gov/pubmed/17553065
FEMS Microbiol Rev. 2007 Jul;31(4):449-77.
Genomics of IncP-1 antibiotic resistance plasmids isolated from wastewater treatment plants provides evidence for a widely accessible drug resistance gene pool.
SchlÃ¼ter A1, Szczepanowski R, PÃ¼hler A, Top EM.

http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6976.2007.00074.x/full
All plasmids contain genes for replication initiation (Rep), conjugative DNA-transfer (Tra) and mating-pair-formation (Trb), stable plasmid inheritance and plasmid control (Ctl) in similar arrangements (see Figs 1 and 2). 

Plasmid backbone modules for replication initiation (Repâ€“grey), mating-pair-formation (Trbâ€“green), conjugative DNA-transfer (Traâ€“red) and plasmid control and stable maintenance and inheritance (Ctlâ€“yellow) are indicated by coloured bars.

- [Chan, Jane 2013 "Molecular epidemiology of fosfomycin-resistant Escherichia coli from humans and animals"](http://hub.hku.hk/handle/10722/197077) | [FullText.pdf](http://hub.hku.hk/bitstream/10722/197077/2/FullText.pdf) Table 1. 4 List of known incompatibility groups of Enterobacteriaceae
- [CiNii ArticlesÂ -Â  pUCãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã«ã¾ã¤ã‚ã‚‹ã‚¨ãƒˆã‚»ãƒˆãƒ©(ç”Ÿç‰©å·¥å­¦åŸºç¤Žè¬›åº§-ãƒã‚¤ã‚ªã‚ˆã‚‚ã‚„ã¾è©±-)](http://ci.nii.ac.jp/naid/110008750634) | [PDF](https://www.sbj.or.jp/wp-content/uploads/file/sbj/8910/8910_yomoyama-2.pdf)
- [Virtual Genome Project Blog](http://virtualgenomeproject.blogspot.jp)
 - 2009/03 [Genome's barcodes](http://virtualgenomeproject.blogspot.jp/2009/03/genomes-barcodes.html)
 - 2008/12 [Functions of horizontally transferred genes](http://virtualgenomeproject.blogspot.jp/2008/12/functions-of-horizontally-transferred.html)
 - 2008/09 [Contribution of horizontal gene transfer to microbial evolution](http://virtualgenomeproject.blogspot.jp/2008/09/contribution-of-horizontal-gene.html)
 - 2008/07 [Reticulate classification of mobile genetic elements](http://virtualgenomeproject.blogspot.jp/2008/07/reticulate-classification-of-mobile.html)
- Plasmids as genetic tools for microbial engineering
 - 2001-06 [Overcoming the codon bias of E. coli for enhanced protein expression](http://wolfson.huji.ac.il/expression/rosetta.pdf)

----------

## updates

## 2022

https://pubmed.ncbi.nlm.nih.gov/35382558/
mSystems
. 2022 Apr 26;7(2):e0118021. doi: 10.1128/msystems.01180-21. Epub 2022 Apr 6.
PlasmidHostFinder: Prediction of Plasmid Hosts Using Random Forest
Derya Aytan-Aktug 1, Philip T L C Clausen 1, Judit Szarvas 1, Patrick Munk 1, Saria Otani 1, Marcus Nguyen 2 3, James J Davis 2 3 4, Ole Lund 1, Frank M Aarestrup 1
https://journals.asm.org/doi/full/10.1128/msystems.01180-21
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9040769/


## 2021

https://www.science.org/doi/10.1126/sciadv.abj5056
Functions predict horizontal gene transfer and the emergence of antibiotic resistance

INTRODUCTION

Genome content, such as the presence of specific plasmid replicon and mobilization machinery or shared phage structural proteins, also defines gene flow across microbial species (4â€“6), whereas restriction-modification genes, the presence of CRISPR-Cas9 adaptive immune systems, and toxin-antitoxin systems serve as barriers to gene flow.

RESULTS

Transfer involving antibiotic resistance determinants is predictable by orthogonal functions

Fig. 3. The transfer of ARGs genes is predictable.

Despite the concern over multidrug-resistant plasmids, only 8729 edges in the network involved more than one ARG class (fig. S14A).

In addition, genes involved in the machinery and process of HGT were also identified: plasmid-segregation systems (e.g., parM), components of type IV coupling systems demarcating plasmid lineages (virD4), phage proteins (xtmB and rstA1), and proteins involved in recombination (xerC, rmuC, ruvC, and recU). We suspect that these barriers to HGT delimit subnetworks where genes are closely associated with specific transposable elements, phage, or plasmid lineages (37â€“39), as is the case of more recently mobilized colistin-resistance genes (40).

DISCUSSION

This framework may be leveraged to improve the design of mobile genetic elements intended for engineering the microbiome (44) or inform strategies to reduce ARG burden by curing or eliminating plasmids or inhibiting conjugation.


----------



### Rodriguez-Beltran_2021

https://pubmed.ncbi.nlm.nih.gov/33469168/
Review Nat Rev Microbiol
. 2021 Jan 19. doi: 10.1038/s41579-020-00497-1. Online ahead of print.
Beyond horizontal gene transfer: the role of plasmids in bacterial evolution
JerÃ³nimo RodrÃ­guez-BeltrÃ¡n 1, Javier DelaFuente 2, Ricardo LeÃ³n-Sampedro 2, R Craig MacLean 3, Ãlvaro San MillÃ¡n 4 5
https://www.nature.com/articles/s41579-020-00497-1
Data were retrieved from the PLSDB plasmid database119.

### 2021-05

https://pubmed.ncbi.nlm.nih.gov/33782584/
Nat Microbiol
. 2021 May;6(5):606-616. doi: 10.1038/s41564-021-00879-y. Epub 2021 Mar 29.
Pervasive transmission of a carbapenem resistance plasmid in the gut microbiota of hospitalized patients
Ricardo LeÃ³n-Sampedro # 1 2, Javier DelaFuente # 1, Cristina DÃ­az-Agero # 3, Thomas Crellen 4 5, Patrick Musicha 4 5, JerÃ³nimo RodrÃ­guez-BeltrÃ¡n 1 2, Carmen de la Vega 1, Marta HernÃ¡ndez-GarcÃ­a 1 6, R-GNOSIS WP5 Study Group; Nieves LÃ³pez-FresneÃ±a 3, Patricia Ruiz-Garbajosa 1 6, Rafael CantÃ³n 1 6, Ben S Cooper 4 5, Ãlvaro San MillÃ¡n 7 8 9
https://www.nature.com/articles/s41564-021-00879-y


### Modeling

https://pubmed.ncbi.nlm.nih.gov/33833414/
ISME J
. 2021 Apr 8. doi: 10.1038/s41396-021-00954-6. Online ahead of print.
Modeling the ecology of parasitic plasmids
Jaime G Lopez 1, Mohamed S Donia 2, Ned S Wingreen 3 4
https://www.nature.com/articles/s41396-021-00954-6


### 2020

### MetaSUB Consortium

https://pubmed.ncbi.nlm.nih.gov/32514171/
Nat Med
. 2020 Jun;26(6):941-951. doi: 10.1038/s41591-020-0894-4. Epub 2020 Jun 8.
Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment
Kern Rei Chng # 1, Chenhao Li # 1, Denis Bertrand # 1, Amanda Hui Qi Ng 1, Junmei Samantha Kwah 1, Hwee Meng Low 1, Chengxuan Tong 1, Maanasa Natrajan 1, Michael Hongjie Zhang 1, Licheng Xu 2, Karrie Kwan Ki Ko 3 4 5, Eliza Xin Pei Ho 1, Tamar V Av-Shalom 1, Jeanette Woon Pei Teo 6, Chiea Chuen Khor 1, MetaSUB Consortium; Swaine L Chen 1, Christopher E Mason 7, Oon Tek Ng 8 9 10, Kalisvar Marimuthu 8 9 11, Brenda Ang 8 9, Niranjan Nagarajan 12 13
https://www.nature.com/articles/s41591-020-0894-4
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7303012/


### 2020-05

https://pubmed.ncbi.nlm.nih.gov/32123369/
Comment Nat Rev Microbiol
. 2020 May;18(5):264-265. doi: 10.1038/s41579-020-0349-4.
Stop that plasmid
Ursula Hofer 1
https://www.nature.com/articles/s41579-020-0349-4



#### 2020-03

ãƒ¡ã‚¿ã‚¢ãƒŠãƒªã‚·ã‚¹
https://pubmed.ncbi.nlm.nih.gov/31926878/
Plasmid
. 2020 Mar;108:102489. doi: 10.1016/j.plasmid.2020.102489. Epub 2020 Jan 10.
The Role of Hosts, Plasmids and Environment in Determining Plasmid Transfer Rates: A Meta-Analysis
Richard J Sheppard 1, Alice E Beddis 2, Timothy G Barraclough 2
https://www.sciencedirect.com/science/article/abs/pii/S0147619X20300019?via%3Dihub


### 2019

https://www.ncbi.nlm.nih.gov/pubmed/31515374
Science. 2019 Sep 13;365(6458):1082-1083. doi: 10.1126/science.aax3879.
The evolution of antibiotic resistance.
MacLean RC1, San Millan A2.

https://twitter.com/BejaLab/status/1195926101509496832
Oded BÃ©jÃ  on Twitter: "while preparing my microbial ecology lecture for today i came upon this one (not for the first time
). Bacterial clade with the ribosomal RNA operon on a small plasmid rather than the chromosome https://t.co/PsC007lQoY" / Twitter
1:46 PM Â· Nov 17, 2019Â·Twitter Web App


https://www.ncbi.nlm.nih.gov/pubmed/31187446
Genes Genomics. 2019 Sep;41(9):1077-1083. doi: 10.1007/s13258-019-00839-1. Epub 2019 Jun 11.
Comparison of de-novo assembly tools for plasmid metagenome analysis.
Gupta SK1, Raza S1, Unno T2,3.

https://www.nature.com/articles/s41467-019-13709-x
Published: 20 December 2019
Evolution of satellite plasmids can prolong the maintenance of newly acquired accessory genes in bacteria




https://twitter.com/EvolvedBiofilm/status/1184427547234373632
Akos T. Kovacs on Twitter: "Extremely fast amelioration of plasmid fitness costs by multiple functionally diverse pathways -in Microbiology by @jpjhall et al w/ @ellieevolves @BrockhurstLab https://t.co/5Fis1aiaEE"
8:15 PM Â· Oct 16, 2019


https://www.ncbi.nlm.nih.gov/pubmed/30625112
Microb Genom. 2019 Jan;5(1). doi: 10.1099/mgen.0.000244.
Where the plasmids roam: large-scale sequence analysis reveals plasmids with large host ranges.
Brooks LE1,2, Kaze M2, Sistrom M2.
https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000244#tab2
The host range of this plasmid was expanded to include two members from the phylum Actinobacteria as well as additional representatives from the classes Alphaproteobacteria, Betaproteobacteria and Gammaproteobacteria of the Proteobacteria (Fig. 3) when incorporating additional sequences from the nucleotide collection sequences. 



https://www.ncbi.nlm.nih.gov/pubmed/31455406
Microbiome. 2019 Aug 27;7(1):119. doi: 10.1186/s40168-019-0737-z.
Long-read metagenomic exploration of extrachromosomal mobile genetic elements in the human gut.
Suzuki Y1, Nishijima S1,2,3, Furuta Y4, Yoshimura J1, Suda W1,5, Oshima K1, Hattori M6,7,8, Morishita S9.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6712665/
Circular contig generation from PacBio read assembly
circular contigs (CCs)
PlasFlow [30], checking the presence or absence of known plasmid-enriched genes, such as mobilisation- and conjugation-related genes, and a similarity search of the public database. 
PlasFlow (v1.1) was used with the default options for classification of CCs as plasmids [30].


https://twitter.com/WvSchaik/status/1148526049300496384
Willem van Schaik on Twitter: "Next up: Eduardo Rocha on the diversity of mobile genetic elements. Sequenced 1305 natural E. coli isolates: up to a third of the pan-genome can be in plasmids and phages #FEMS2019"
5:35 AM - 9 Jul 2019





GCå«é‡
https://twitter.com/Symbionticism/status/1123203760564637697
Seth Bordenstein on Twitter: "This is a game changer. Many intracellular microbes have a genome wide bias in Aâ€™s and Tâ€™s. The major assumption has been a mutational bias / drift impacts this outcome. Well, not so fast. Evidence below that selection can contribute or cause it.â€¦ https://t.co/yfDqN3b3Lj"
8:33 AM - 30 Apr 2019
https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1007778
Selective advantages favour high genomic AT-contents in intracellular elements
Accordingly, introducing AT-rich and GC-rich plasmids into other bacterial species with different genomic GC-contents revealed that the costs of G+C-rich plasmids decreased with an increasing GC-content of their hostâ€™s genomic DNA. 

https://twitter.com/LauraPiddock/status/1123498515664199680
Laura Piddock on Twitter: "Our latest paper 'Antibiotic resistance genes are abundant and diverse in raw sewage used for urban agriculture in Africa and associated with urban population density' is now online :) @BlaiseBougnom @alanmcn1 https://t.co/LREwajwm5Q"
4:04 AM - 1 May 2019
https://www.sciencedirect.com/science/article/pii/S0269749119303124
Antibiotic resistance genes are abundant and diverse in raw sewage used for urban agriculture in Africa and associated with urban population density - ScienceDirect
- The size of the city impacts on the abundance of ARGs in sewage water.
- ESBL gene abundance is impacted by Enterobacteriaceae and plasmid abundance.

https://twitter.com/CarinaBrehony/status/1122128633395650560
carina brehony on Twitter: "Our paper out now in @BSACandJAC on OXA-48 plasmid molecular epidemiology in Ireland using a gene by gene approach @ARME_Galway @MaidenLab @sylvainbrisse MLST approach to support tracking of plasmids carrying OXA-48-like carbapenemase https://t.co/EjaaJeHBj4"
9:21 AM - 27 Apr 2019


https://www.ncbi.nlm.nih.gov/pubmed/30670543
J Bacteriol. 2019 Mar 13;201(7). pii: e00430-18. doi: 10.1128/JB.00430-18. Print 2019 Apr 1.
Plasmid Characteristics Modulate the Propensity of Gene Exchange in Bacterial Vesicles.
Tran F1, Boedicker JQ2,3.
Here, a set of different plasmids was constructed to systematically test the impact of plasmid properties, specifically, plasmid copy number, size, and origin of replication, on gene transfer in vesicles. 

https://www.ncbi.nlm.nih.gov/pubmed/30835726
PLoS Comput Biol. 2019 Mar 5;15(3):e1006724. doi: 10.1371/journal.pcbi.1006724. eCollection 2019 Mar.
Evolutionary model for the unequal segregation of high copy plasmids.
MÃ¼nch K1,2, MÃ¼nch R1,2, Biedendieck R1,2, Jahn D1,2, MÃ¼ller J3,4.

### 2018

### 2018-12

https://pubmed.ncbi.nlm.nih.gov/30049587/
Review Trends Microbiol
. 2018 Dec;26(12):978-985. doi: 10.1016/j.tim.2018.06.007. Epub 2018 Jul 23.
Evolution of Plasmid-Mediated Antibiotic Resistance in the Clinical Context
Alvaro San Millan 1
https://www.cell.com/trends/microbiology/fulltext/S0966-842X(18)30142-2
In vitro experimental work in the field is being complemented by bioinformatics and systems biology approaches, and a recent computational analysis provided the first population-level evidence of adaptation to the carriage of AR plasmids in a pathogenic E. coli lineage through mutations in gene-regulatory regions [33]. Given the increasing number of bacterial genome sequences available, a combination of this type of detailed phylogenetic analysis with genome-wide association studies [34] could identify specific loci or mutations associated with plasmid carriage. These powerful computational tools will help to define the genetic basis of plasmidâ€“bacterium adaptation on a broader scale.
By combining the experimental and computational approaches described here, it should be possible to define molecular signatures and use them to forecast (i) the â€˜fitness compatibilityâ€™ between plasmid and bacterium, and (ii) the susceptibility of these associations to compensatory evolution. These signatures could then be used to predict which plasmidâ€“bacterium associations are likely to arise in the future from the plasmids and bacteria currently present in a given environment.
Concluding Remarks





### 2018-04


https://www.ncbi.nlm.nih.gov/pubmed/29633935
Microb Genom. 2018 Apr;4(4). doi: 10.1099/mgen.0.000168. Epub 2018 Apr 10.
Correlation between bacterial G+C content, genome size and the G+C content of associated plasmids and bacteriophages.
Almpanis A1,2, Swain M1, Gatherer D3, McEwan N1,4.


https://www.ncbi.nlm.nih.gov/pubmed/29650391
Trends Microbiol. 2018 Apr 10. pii: S0966-842X(18)30067-2. doi: 10.1016/j.tim.2018.03.004. [Epub ahead of print]
Convergent Evolution in Intracellular Elements: Plasmids as Model Endosymbionts.
Dietel AK1, Kaltenpoth M2, Kost C3.
REVIEW

https://www.biorxiv.org/content/early/2018/06/06/262956
KrakenHLL: Confident and fast metagenomics classification using unique k-mer counts | bioRxiv
A taxonomic node may also be added for each sequence â€“ e.g. specific bacterial chromosomes or plasmids.
https://www.biorxiv.org/content/biorxiv/early/2018/06/06/262956.full.pdf
KrakenHLL allows extending the taxonomy with virtual nodes for genomes,
311 chromosomes and plasmids, and thus enabling identifications at the most specific levels
312 (see Suppl. Methods Section 3)


https://twitter.com/WvSchaik/status/562549766790594560
Willem van Schaik on Twitter: "@pathogenomenick @koadman @daanspeth I guess could vary in broad-host range plasmid (different copy number in different bugs)"
4:54 AM - 3 Feb 2015

### 2017


### 2015

https://www.ncbi.nlm.nih.gov/pubmed/25540453
Mol Biol Evol. 2015 Dec;32(12):3079-88. doi: 10.1093/molbev/msu293. Epub 2014 Dec 23.
Why There Are No Essential Genes on Plasmids.
Tazzyman SJ1, Bonhoeffer S2.



https://www.ncbi.nlm.nih.gov/pubmed/25608745
BMC Genomics. 2015 Jan 22;16:6. doi: 10.1186/s12864-014-1206-5.
Plasmids are vectors for redundant chromosomal genes in the Bacillus cereus group.
Zheng J1, Guan Z2, Cao S3, Peng D4, Ruan L5, Jiang D6, Sun M7.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4326196/


### 2012

https://www.ncbi.nlm.nih.gov/pubmed/22617140
MBio. 2012 May 22;3(3). pii: e00112-12. doi: 10.1128/mBio.00112-12. Print 2012.
Comparative genomics of vancomycin-resistant Staphylococcus aureus strains and their positions within the clade most commonly associated with Methicillin-resistant S. aureus hospital-acquired infection in the United States.
Kos VN1, Desjardins CA, Griggs A, Cerqueira G, Van Tonder A, Holden MT, Godfrey P, Palmer KL, Bodi K, Mongodin EF, Wortman J, Feldgarden M, Lawley T, Gill SR, Haas BJ, Birren B, Gilmore MS.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372964/
FIG 3
Plasmid sequences in VRSA. Heat map showing the extent of occurrence of known enterococcal (graded green shades) and staphylococcal plasmids (graded yellow to red) (based on detail in Table S5 in the supplemental material). 
![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3372964/bin/mbo0031212770003.jpg)








----------

## annotation

## 2018-01-01
https://link.springer.com/referenceworkentry/10.1007/978-1-4614-1531-2_568
Plasmids, Naming and Annotation of
Laura S. Frost & Christopher M. Thomas 
Reference work entry
First Online: 01 January 2018

### Thomas_2017
## 2017-05

http://www.plasmidbiologysociety.org/plasmid-annotation/

https://pubmed.ncbi.nlm.nih.gov/28365184/
Review Plasmid
. 2017 May;91:61-67. doi: 10.1016/j.plasmid.2017.03.006. Epub 2017 Mar 30.
Annotation of plasmid genes
Christopher M Thomas 1, Nicholas R Thomson 2, Ana M CerdeÃ±o-TÃ¡rraga 2, Celeste J Brown 3, Eva M Top 3, Laura S Frost 4
https://linkinghub.elsevier.com/retrieve/pii/S0147-619X(16)30113-5




----------

## rep

----------

https://ja.wikipedia.org/wiki/ãƒ¢ãƒã‚¤ãƒ­ãƒ¼ãƒ 
ãƒ¢ãƒã‚¤ãƒ­ãƒ¼ãƒ (Mobilome)ã¯ã€ã‚²ãƒŽãƒ ä¸­ã®å¯å‹•éºä¼å› å­ã®ç·ä½“ã§ã‚ã‚‹ã€‚ã‚²ãƒŽãƒ å†…ã‚’ç§»å‹•ã™ã‚‹éºä¼å› å­ï¼ˆãƒˆãƒ©ãƒ³ã‚¹ãƒã‚¾ãƒ³ï¼‰ã¯ã€çœŸæ ¸ç”Ÿç‰©ã®ãƒ¢ãƒã‚¤ãƒ­ãƒ¼ãƒ ã®ä¸­å¿ƒã§ã‚ã‚‹ã€‚ã—ã‹ã—åŽŸæ ¸ç”Ÿç‰©ã§ã¯ã€å¯å‹•éºä¼å› å­ã¯ã€ãƒ—ãƒ­ãƒ•ã‚¡ãƒ¼ã‚¸ã‚„ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®ã‚ˆã†ã«ã‚²ãƒŽãƒ é–“ã‚’ç§»å‹•ã§ãã€ãƒ¢ãƒã‚¤ãƒ­ãƒ¼ãƒ ã®é‡è¦ãªè¦ç´ ã§ã‚ã‚‹ã€‚

----------

https://kotobank.jp/word/ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰-161388
ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰
ã·ã‚‰ã™ã¿ã©
plasmid

å¤§è…¸èŒã®ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã«ã¯ã€Få› å­ã€Rå› å­ã€ã‚³ãƒªã‚·ãƒ³å› å­ãªã©æ•°ç¨®ãŒçŸ¥ã‚‰ã‚Œè©³ã—ãç ”ç©¶ã•ã‚Œã¦ã„ã‚‹ã€‚

ã‚³ãƒªã‚·ãƒ³å› å­ã¯ä»–ã®ç´°èŒã‚’æ®ºã™ä½œç”¨ã‚’ã‚‚ã¤ã‚³ãƒªã‚·ãƒ³ã¨ã„ã‚ã‚Œã‚‹ç‰©è³ªã®ç”Ÿç”£ã‚’æ”¯é…ã—ã€æ™®é€šã€ä¸€ç´°èƒžã«10å€‹ã‚ã‚‹ã„ã¯ãã‚Œä»¥ä¸Šå«ã¾ã‚Œã¦ã„ã‚‹ã€‚é…µæ¯ã§ã¯2ãƒŸã‚¯ãƒ­ãƒ³DNAã¨ã‚ˆã°ã‚Œã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ãŒã¿ã„ã ã•ã‚Œã¦ã„ã‚‹ã€‚ã“ã‚Œã¯é•·ã•ãŒ2ãƒŸã‚¯ãƒ­ãƒ³ã®DNAåˆ†å­ã§ã€ä¸€ç´°èƒžå½“ã‚Š50å€‹ä½™ã‚Šå«ã¾ã‚Œã‚‹ãŒã€ãã®åƒãã¯ä¸æ˜Žã§ã‚ã‚‹ã€‚

å¤å…¸çš„æ€§å› å­ (Fãƒ—ãƒ©ã‚¹ãƒŸãƒ‰) ï¼Œãƒã‚¯ãƒ†ãƒªã‚ªã‚·ãƒ³ç”£ç”Ÿãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ (Col E1ãªã©) ï¼Œè–¬å‰¤è€æ€§å› å­ (Rãƒ—ãƒ©ã‚¹ãƒŸãƒ‰) ï¼Œç—…åŽŸæ€§æ±ºå®šãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ï¼ŒæŠ—ç”Ÿç‰©è³ªåˆæˆç³»ã«é–¢ä¸Žã™ã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ï¼Œç‰©è³ªä»£è¬ç³»ã«é–¢ä¸Žã™ã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ãªã©ãŒã‚ã‚‹ã€‚


https://ja.wikipedia.org/wiki/ColE1
ColE1ã¯ãƒã‚¯ãƒ†ãƒªã‚¢ã«ç”¨ã„ã‚‰ã‚Œã‚‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã€‚ãã®åå‰ã¯ã€ ã‚³ãƒªã‚·ãƒ³ E1éºä¼å­ï¼ˆ ceaéºä¼å­ï¼‰ã‚’æŒã£ã¦ã„ã‚‹ã“ã¨ã«ç”±æ¥ã—ã¦ã„ã‚‹ã€‚ã¾ãŸã€ imméºä¼å­ã‚’ä½¿ç”¨ã—ã¦ã€ã“ã®ç”£ç”Ÿç‰©ã‹ã‚‰ã®å…ç–«ã‚’ã‚³ãƒ¼ãƒ‰ã™ã‚‹ã€‚ã•ã‚‰ã«ã€ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã«ã¯ä¸€é€£ã®ç§»å‹•æ€§ï¼ˆ mob ï¼‰éºä¼å­ãŒã‚ã‚‹ã€‚ãã®ã‚µã‚¤ã‚ºã¨å˜ä¸€ã®EcoRIèªè­˜éƒ¨ä½ã®å­˜åœ¨ã«ã‚ˆã‚Šã€ãƒ™ã‚¯ã‚¿ãƒ¼å€™è£œã¨è¦‹ãªã•ã‚ŒãŸ [1]ã€‚


----------


https://github.com/haruosuz/mgsa/blob/master/references/mgsa.rRNA.md

### Yano

ç·èª¬
https://www.ncbi.nlm.nih.gov/pubmed/30619542
Comput Struct Biotechnol J. 2018 Dec 15;17:70-81. doi: 10.1016/j.csbj.2018.12.001. eCollection 2019.
Reconsidering plasmid maintenance factors for computational plasmid design.
Yano H, Shintani M, Tomita M, Suzuki H, Oshima T.
https://www.sciencedirect.com/science/article/pii/S2001037018301685
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6312765/

https://sites.google.com/view/hirokazuyano/koremadeno-yan-jiu
Hirokazu Yano - ã“ã‚Œã¾ã§ã®ç ”ç©¶ (in Japanese)

è–¬å‰¤è€æ€§ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®å®¿ä¸»é©å¿œ

æœ¬ç¾è±¡ã®èƒŒæ™¯ã«ã‚ã‚‹åˆ†å­æ©Ÿæ§‹ã‚’è§£æ˜Žã™ã‚‹ãŸã‚ã€ã‚ã‚‹å®¿ä¸»é›†å›£å†…ã§é€²åŒ–ã—ãŸé€²åŒ–åž‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®é…åˆ—ã‚’æ±ºå®šã—ãŸã®ã¡ã€é€²åŒ–åž‹ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã‚’å…ˆç¥–åž‹å®¿ä¸»ã«æˆ»ã™å®Ÿé¨“ã‚’è¡Œã£ãŸã¨ã“ã‚ã€ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰å­˜ç¶šæ€§ã®å‘ä¸ŠãŒãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ ã®è¤‡è£½ã‚¿ãƒ³ãƒ‘ã‚¯è³ªï¼ˆTrfA1ï¼‰ã®Næœ«ã®å¤‰ç•°ã«ä¾å­˜ã—ã¦ã„ã‚‹ã“ã¨ãŒã‚ã‹ã‚Šã¾ã—ãŸã€‚ã“ã®TrfAã®Næœ«ã¯å®¿ä¸»ã®DnaBãƒ˜ãƒªã‚«ãƒ¼ã‚¼ã¨çµåˆã™ã‚‹éƒ¨ä½ã§ã‚ã‚‹ã“ã¨ãŒçŸ¥ã‚‰ã‚Œã¦ã„ã¾ã—ãŸã€‚ç”ŸåŒ–å­¦ã€åˆ†å­éºä¼å­¦å®Ÿé¨“ã®çµæžœã‹ã‚‰ã€ãƒ—ãƒ©ã‚¹ãƒŸ ãƒ‰ãŒå®¿ä¸»ã®ä¸­ã§è¤‡è£½ã§ãã‚‹å ´åˆã€ã€Œå®¿ä¸»ã®DNAãƒ˜ãƒªã‚«ãƒ¼ã‚¼ã¨ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¨ã®ç›¸äº’ä½œç”¨ã®åº¦åˆã„ãŒã€ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã®ç´°èŒé›†å›£å†…ã§ã®å­˜ç¶šæ€§ã‚’æ±ºã‚ã‚‹è¦å› ã®ä¸€ã¤ã€ãƒ•ã‚£ãƒƒãƒˆãƒã‚¹ãƒ»ã‚³ã‚¹ãƒˆã®æ­£ä½“ã§ã‚ã‚‹ã€ã“ã¨ãŒã‚ã‹ã‚Šã¾ã—ãŸï¼ˆYano et al., Mol. Microbiol. 2016)ã€‚

ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã®Næœ«ã®é•·ã•å¤šåž‹ã®æ„ç¾©

ä¸Šè¨˜ã®é€²åŒ–å®Ÿé¨“ã§ã¯ã€è¤‡è£½é–‹å§‹ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã«é›†ä¸­çš„ã«å¤‰ç•°ãŒå…¥ã‚Šã¾ã—ãŸãŒã€é¡žä¼¼ã®å¤‰ç•°ãŒãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã«ç™»éŒ²ã•ã‚Œã¦ã„ã‚‹ä»–ã®ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã«ã‚‚è¦‹ã¤ã‹ã‚‹ã“ã¨ã«ç ”ç©¶ã®éŽç¨‹ã§æ°—ã¥ãã¾ã—ãŸã€‚ãã“ã§ã€å¾—ã‚‰ã‚ŒãŸå¤‰ç•°ä½“ãƒ—ãƒ© ã‚¹ãƒŸãƒ‰ã®è¡¨ç¾åž‹ã‚’æ§˜ã€…ãªè§’åº¦ã‹ã‚‰è§£æžã™ã‚‹ã“ã¨ã§ã€ ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã«è¦‹ã‚‰ã‚Œã‚‹è¤‡è£½é–‹å§‹ã‚¿ãƒ³ãƒ‘ã‚¯è³ªéºä¼å­ã®5â€™-endã®é•·ã•ã®å¤šæ§˜æ€§ï¼ˆin/del) ãŒã€genetic driftã§ã¯ãªããƒˆãƒ¬ãƒ¼ãƒ‰ãƒ»ã‚ªãƒ•ã®é–¢ä¿‚ã«ã‚ã‚‹å½¢è³ªï¼ˆã“ã®å ´åˆã¯æ°´å¹³ä¼æ’­ ã¨åž‚ç›´ä¼æ’­ï¼‰ã«å¯¾ã™ã‚‹natural selectionã®çµæžœã¨ã—ã¦èª¬æ˜Žã§ãã‚‹ã¨ã„ã†ã‚¢ã‚¤ãƒ‡ã‚¢ã‚’ææ¡ˆã—ã¾ã—ãŸï¼ˆYano et al., J. Bacteriol. 2012)ã€‚TrfAã®Næœ«ã®å¤šåž‹ã®ã‚ˆã†ãªé•·ã•å¤šåž‹ã¯ä»–ã®ã‚¿ãƒ³ãƒ‘ã‚¯è³ªï¼ˆéºä¼å­åº§ï¼‰ã«ã‚‚è¦‹ã‹ã‘ã‚‰ã‚Œã¾ ã™ã€‚ä»–ã®ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã«é–¢ã—ã¦ã‚‚ã€Næœ«ã®é•·ã•å¤šåž‹ã«ã«ã¯ä½•ã‹é‡è¦ãªæ„å‘³ãŒã‚ã‚‹ã®ã§ã¯ãªã„ã§ã—ã‚‡ã†ã‹ã€‚

https://pubmed.ncbi.nlm.nih.gov/22228734/
J Bacteriol
. 2012 Mar;194(6):1533-43. doi: 10.1128/JB.06395-11. Epub 2012 Jan 6.
Roles of long and short replication initiation proteins in the fate of IncP-1 plasmids
Hirokazu Yano 1, Gail E Deckert, Linda M Rogers, Eva M Top
https://jb.asm.org/content/194/6/1533.long
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294859/

https://pubmed.ncbi.nlm.nih.gov/29879198/
PLoS One
. 2018 Jun 7;13(6):e0198613. doi: 10.1371/journal.pone.0198613. eCollection 2018.
Interplay of a non-conjugative integrative element and a conjugative plasmid in the spread of antibiotic resistance via suicidal plasmid transfer from an aquaculture Vibrio isolate
Lisa Nonaka 1, Tatsuya Yamamoto 2, Fumito Maruyama 3, Yuu Hirose 4, Yuki Onishi 5, Takeshi Kobayashi 6, Satoru Suzuki 5, Nobuhiko Nomura 2, Michiaki Masuda 1, Hirokazu Yano 2


----------

## ICE
Integrative and Conjugative Elements

### ICEberg
https://integbio.jp/dbcatalog/record/nbdc02055
Integbio ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã‚«ã‚¿ãƒ­ã‚°
Integrative and Conjugative Elements in Bacteria:  ICEberg
ç´°èŒå†…ã§è¦‹ã‚‰ã‚Œã‚‹ integrative and conjugative element (ICE) ã«é–¢ã™ã‚‹ã‚¦ã‚§ãƒ–ãƒ™ãƒ¼ã‚¹ãƒªã‚½ãƒ¼ã‚¹ã§ã™ã€‚PostgreSQLãƒ™ãƒ¼ã‚¹ã®ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã¨ã—ã¦ã€æ—¢çŸ¥ã¾ãŸã¯æŽ¨å®šã•ã‚ŒãŸç´°èŒå†…ã® ICE ã«é–¢ã™ã‚‹å®Ÿé¨“çš„ï½¥ãƒã‚¤ã‚ªã‚¤ãƒ³ãƒ•ã‚©ãƒžãƒ†ã‚£ã‚¯ã‚¹çš„è§£æžã€ãŠã‚ˆã³æ–‡çŒ®ã‹ã‚‰å¾—ã‚‰ã‚ŒãŸãƒ‡ãƒ¼ã‚¿ã‚’åŽé›†ã—ã¦ã„ã¾ã™ã€‚ãƒ‡ãƒ¼ã‚¿ã«ã¯ã€å„çµ„ç¹”ã«ã‚ˆã‚Šã‚¢ãƒ¼ã‚«ã‚¤ãƒ–åŒ–ã•ã‚ŒãŸå…¨ã¦ã® ICE ãŠã‚ˆã³éºä¼å­ã«é–¢ã™ã‚‹è©³ç´°ãªæƒ…å ±ãŒå«ã¾ã‚Œã¾ã™ï¼ˆä¸€æ„ã®è­˜åˆ¥å­ã€ç¨®ã®è©³ç´°ã®ä»–ã€NCBIUniprotKBã€KEGG ãªã©ã®ä»–ã®å…¬å…±ãƒ‡ãƒ¼ã‚¿ãƒ™ãƒ¼ã‚¹ã¸ã®ãƒã‚¤ãƒ‘ãƒ¼ãƒªãƒ³ã‚¯ã‚’å«ã¿ã¾ã™ï¼‰ã€‚

http://db-mml.sjtu.edu.cn/ICEberg/
ICEberg

#### Liu_2019_ICEberg2

https://www.ncbi.nlm.nih.gov/pubmed/30407568
Nucleic Acids Res. 2019 Jan 8;47(D1):D660-D665. doi: 10.1093/nar/gky1123.
ICEberg 2.0: an updated database of bacterial integrative and conjugative elements.
Liu M1,2, Li X1, Xie Y1, Bi D3, Sun J4, Li J5, Tai C1, Deng Z1, Ou HY1,2.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6323972/

https://pubmed.ncbi.nlm.nih.gov/30407568/
Nucleic Acids Res
. 2019 Jan 8;47(D1):D660-D665. doi: 10.1093/nar/gky1123.
ICEberg 2.0: an updated database of bacterial integrative and conjugative elements
Meng Liu 1 2, Xiaobin Li 1, Yingzhou Xie 1, Dexi Bi 3, Jingyong Sun 4, Jun Li 5, Cui Tai 1, Zixin Deng 1, Hong-Yu Ou 1 2
https://academic.oup.com/nar/article/47/D1/D660/5165266
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC6323972/
added. At last, ICEberg 2.0 provides an online tool ICEfinder to predict ICEs or IMEs in bacterial genome sequences. It combines a similarity search for the integrase, relaxase and/or type IV secretion system and the co-localization of these corresponding homologous genes. With the



### Delavat_2017_Review

https://pubmed.ncbi.nlm.nih.gov/28369623/
Review FEMS Microbiol Rev
. 2017 Jul 1;41(4):512-537. doi: 10.1093/femsre/fux008.
The hidden life of integrative and conjugative elements
FranÃ§ois Delavat 1, Ryo Miyazaki 2, Nicolas Carraro 1, Nicolas Pradervand 1, Jan Roelof van der Meer 1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5812530/

### 

https://www.ncbi.nlm.nih.gov/pubmed/26473380
Annu Rev Genet. 2015;49:577-601. doi: 10.1146/annurev-genet-112414-055018. Epub 2015 Oct 14.
Integrative and Conjugative Elements (ICEs): What They Do and How They Work.
Johnson CM1, Grossman AD.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5180612/

----------

https://ja.wikipedia.org/wiki/åŽŸæ ¸ç”Ÿç‰©ã®ç´°èƒžéª¨æ ¼
FtsZ ã¯æœ€åˆã«åŒå®šã•ã‚ŒãŸåŽŸæ ¸ç”Ÿç‰©ã®ç´°èƒžéª¨æ ¼ã®æ§‹æˆè¦ç´ ã§ã‚ã‚‹ã€‚

https://ja.wikipedia.org/wiki/FtsZ
FtsZã¯ã€çœŸæ­£ç´°èŒã‚„ãƒ¦ãƒ¼ãƒªå¤ç´°èŒãªã©ã«å­˜åœ¨ã™ã‚‹ã€ftsZ éºä¼å­ã‹ã‚‰ç¿»è¨³ã•ã‚Œã‚‹ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã€‚ç´°èƒžè†œä¸‹ã«é›†åˆã—ã¦ç’°æ§‹é€ ã‚’å½¢æˆã—ã€ãã®ç®‡æ‰€ãŒåˆ†è£‚æ™‚ã«éš”å£ã¨ãªã‚‹ã€‚ã“ã‚Œã¯åŽŸæ ¸ç”Ÿç‰©ã«ãŠã„ã¦ã€çœŸæ ¸ç”Ÿç‰©ã®ãƒãƒ¥ãƒ¼ãƒ–ãƒªãƒ³ã¨ãƒ›ãƒ¢ãƒ­ã‚°ï¼ˆç›¸åŒï¼‰ã§ã‚ã‚‹ã€‚FtsZã®åç§°ã¯ã€"Filamenting temperature-sensitive mutant Z" ï¼ˆãƒ•ã‚£ãƒ©ãƒ¡ãƒ³ãƒˆçŠ¶æ¸©åº¦æ„Ÿå—æ€§å¤‰ç•°æ ªZï¼‰ã«ã‚ˆã‚‹ã€‚E. coliã®åˆ†è£‚ç•°å¸¸æ ªã§ãƒ•ã‚£ãƒ©ãƒ¡ãƒ³ãƒˆçŠ¶ã«æˆé•·ã™ã‚‹ã‚‚ã®ã®å ´åˆã€å¨˜ç´°èƒžã‚’åˆ†é›¢ã™ã‚‹èƒ½åŠ›ã‚’æ¬ ããŸã‚ã«ç´°é•·ãæˆé•·ã™ã‚‹ã¨è€ƒãˆã‚‰ã‚Œã¦ã„ã‚‹ã€‚

ZipAã‚„ã€ã‚¢ã‚¯ãƒãƒ³ã®ãƒ›ãƒ¢ãƒ­ã‚°ã§ã‚ã‚‹FtsAã¯ã€FtsZãŒæœ€åˆã«ç´°èƒžè†œã¸å±€æ‰€åŒ–ã™ã‚‹ã“ã¨ã‚’å¯èƒ½ã«ã™ã‚‹[9]ã€‚ãã‚Œä»¥é™ã®ç´°èƒžè†œã¸ã®å±€æ‰€åŒ–ã¯ã€Ftsãƒ•ã‚¡ãƒŸãƒªãƒ¼ã®åˆ†è£‚ã‚¿ãƒ³ãƒ‘ã‚¯è³ªãŒã€åŽç¸®ç’°ã‚’æ§‹æˆã™ã‚‹ãŸã‚ã«é›†ã‚ã‚‰ã‚Œã‚‹[10]ã€‚FtsWã€FtsKã€FtsQãªã©ã€ã“ã‚Œã‚‰ã®ã‚¿ãƒ³ãƒ‘ã‚¯è³ªã®å¤šãã¯ã€Zãƒªãƒ³ã‚°ã®å®‰å®šåŒ–ã•ã›ã‚‹å½¹ç›®ã‚’æžœãŸã—ã¦ã„ã‚‹ã€‚ã¾ãŸåˆ†è£‚æ™‚ã«å‹•çš„ãªé–¢ä¸Žã‚’ã—ã¦ã„ã‚‹å¯èƒ½æ€§ã‚‚ã‚ã‚‹ã€‚



----------

## IncP7

https://pubmed.ncbi.nlm.nih.gov/30863370/
Front Microbiol
. 2019 Feb 26;10:213. doi: 10.3389/fmicb.2019.00213. eCollection 2019.
Genetic Characterization of a bla VIM-24-Carrying IncP-7Î² Plasmid p1160-VIM and a bla VIM-4-Harboring Integrative and Conjugative Element Tn 6413 From Clinical Pseudomonas aeruginosa
Lijun Zeng 1 2, Zhe Zhan 1 3, Lingfei Hu 1, Xiaoyuan Jiang 1, Yanjun Zhang 3, Jiao Feng 1, Bo Gao 1, Yuee Zhao 1, Wenhui Yang 1, Huiying Yang 1, Zhe Yin 1, Dongsheng Zhou 1
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6399125/
FIGURE 1
Neighbor-joining phylogenetic trees of sequenced IncP-7 plasmids.


----------
## host range

å­¦ä½è«–æ–‡
https://backend.orbit.dtu.dk/ws/portalfiles/portal/110306544/Uli_Kl_mper_PhD_thesis_WWW_Version.pdf
U KlÃ¼mper è‘— - â€Ž2015 

4.2 Plasmid host ranges 

Three different host ranges of plasmids are defined through the duration and
intimacy of the considered plasmid-host relationship (Suzuki et al., 2010). 

The evolutionary host range describes the variety of organisms in which a given plasmid was maintained long enough to undergo adaptation of its backbone to the genetic code of its host organism
(Suzuki et al., 2010) (Figure 4). 


Barth F. Smetsã•ã‚“ã‚³ãƒ¬ã‚¹ãƒ
https://www.ncbi.nlm.nih.gov/pubmed/32020051
ISME J. 2020 May;14(5):1170-1181. doi: 10.1038/s41396-020-0596-4. Epub 2020 Feb 4.
Plasmids persist in a microbial community by providing fitness benefit to multiple phylotypes.
Li L1, Dechesne A1, Madsen JS2, Nesme J2, SÃ¸rensen SJ2, Smets BF3.
https://www.nature.com/articles/s41396-020-0596-4
Plasmids persist in a microbial community by providing fitness benefit to multiple phylotypes | The ISME Journal



----------
## amr


https://www.ncbi.nlm.nih.gov/pubmed/29370371
J Antimicrob Chemother. 2018 May 1;73(5):1121-1137. doi: 10.1093/jac/dkx488.
Plasmids carrying antimicrobial resistance genes in Enterobacteriaceae.
Rozwandowicz M1, Brouwer MSM2, Fischer J3, Wagenaar JA1,2, Gonzalez-Zorn B4, Guerra B3, Mevius DJ1,2, Hordijk J1.
https://academic.oup.com/jac/article/73/5/1121/4822282
PCR-based replicon typing (PBRT) or relaxase typing. Currently, there are 28 known plasmid types in Enterobacteriaceae distinguished by PBRT. Frequently reported plasmids [IncF, IncI, IncA/C, IncL (previously designated IncL/M), IncN and IncH] are the ones that bear the greatest variety of resistance genes. The purpose of this review is to provide an overview of all known AMR-related plasmid families in Enterobacteriaceae, the resistance genes they carry and their geographical distribution.



----------

## Fernandez-Lopez_2017_taxonomy_plasmids

https://www.ncbi.nlm.nih.gov/pubmed/28586714
Curr Opin Microbiol. 2017 Aug;38:106-113. doi: 10.1016/j.mib.2017.05.005. Epub 2017 Jun 3.
Towards a taxonomy of conjugative plasmids.
Fernandez-Lopez R1, Redondo S1, Garcillan-Barcia MP1, de la Cruz F2.
https://www.sciencedirect.com/science/article/pii/S1369527417300036?via%3Dihub
Natural transformation, phage transduction, and bacterial conjugation are three main routes of HGT in bacteria.

https://pfam.xfam.org/family/PF04796
Pfam: Family: RepA_C (PF04796)



----------

## mutation rate

https://mbio.asm.org/content/9/4/e01371-18
Periodic Variation of Mutation Rates in Bacterial Genomes Associated with Replication Timing | mBio

https://www.ncbi.nlm.nih.gov/pubmed/20369015
PLoS Comput Biol. 2010 Apr 1;6(4):e1000732. doi: 10.1371/journal.pcbi.1000732.
Why genes evolve faster on secondary chromosomes in bacteria.
Cooper VS1, Vohr SH, Wrocklage SC, Hatcher PJ.
Further, in every bacterial genome with multiple chromosomes that we studied, genes on secondary chromosomes exhibited significantly weaker codon usage bias than those on primary chromosomes. Faster evolution and reduced codon bias could in turn result from global effects of chromosome position, as genes on secondary chromosomes experience reduced dosage and expression due to their delayed replication, or selection on specific gene attributes.

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848543/
Why many bacteria have multiple chromosomes is largely unknown, but a leading hypothesis is that secondary chromosomes evolved from plasmids and now serve as accessory genomes. 
Chromosomes may originate by three different mechanisms: by the split of a single chromosome, by chromosome duplication, or by acquisition of a large plasmid with essential genes, which ensures its prolonged maintenance. Of these processes, the last has the greatest support because some secondary chromosomes have plasmid-like origins of replication [2]. 

A fourth possible mechanism is that secondary chromosomes could experience inherently higher mutation rates. Although mutation rates are known to vary among genome locations, such a widespread and systematic difference would be exceptional. The delayed replication of secondary chromosomes could potentially produce such an effect if nucleotide pools vary or become limiting as a function of the cell cycle [37] or if the replication apparatus tends to require reassembly in later replication stages, which is mutagenic [38]. The probable origin of secondary chromosomes as plasmids could also lead to increased mutation rates as a consequence of their greater supercoiling, which has been associated with greater rates of mutation [39]. Of the four potential explanations that we suggest for why secondary chromosomes evolve more quickly, this one (a systematically greater mutation rate) is the most speculative but also the most experimentally tractable.

https://www.ncbi.nlm.nih.gov/pubmed/16689800
Mol Microbiol. 2006 Jun;60(5):1251-61.
The effect of promoter strength, supercoiling and secondary structure on mutation rates in Escherichia coli.
Schmidt KH1, Reimers JM, Wright BE.


----------
## nucleotide composition
- [Suzuki H et al. Molecular Life Sciences. 2014 May "Genomic Signature Analysis to Predict Plasmid Host Range."](https://link.springer.com/referenceworkentry/10.1007/978-1-4614-6436-5_574-1) | [pdf](https://link.springer.com/content/pdf/10.1007/978-1-4614-6436-5_574-1.pdf)
- [Suzuki H et al. J Bacteriol. 2010 Nov "Predicting plasmid promiscuity based on genomic signature."](http://www.ncbi.nlm.nih.gov/pubmed/20851899) | [Supplemental material](http://jb.asm.org/content/192/22/6045/suppl/DC1)
  - [Predicting Plasmid Promiscuity Could Help Fight Antibiotic Resistance | BEACON](http://beacon-center.org/blog/2011/03/28/predicting-plasmid-promiscuity-could-help-fight-antibiotic-resistance/)
- [Suzuki H et al. Nucleic Acids Res. 2008 Dec "Using Mahalanobis distance to compare genomic signatures between bacterial plasmids and chromosomes."](http://www.ncbi.nlm.nih.gov/pubmed/18953039) | [Supplementary Data](http://nar.oxfordjournals.org/content/36/22/e147/suppl/DC1)
- [Norberg P et al. Nat Commun. 2011;2:268. "The IncP-1 plasmid backbone adapts to different host bacterial species and evolves through homologous recombination."](http://www.ncbi.nlm.nih.gov/pubmed/21468020)
- [van Passel MW et al. BMC Genomics. 2006 Feb "Compositional discordance between prokaryotic plasmids and host chromosomes."](http://www.ncbi.nlm.nih.gov/pubmed/16480495)
- [Nishida H. Int J Evol Biol. 2012 "Comparative analyses of base compositions, DNA sizes, and dinucleotide frequency profiles in archaeal and bacterial chromosomes and plasmids."](http://www.ncbi.nlm.nih.gov/pubmed/22536540)
- [Rocha EP, Danchin A. Trends Genet. 2002 Jun "Base composition bias might result from competition for metabolic resources."](http://www.ncbi.nlm.nih.gov/pubmed/12044357)

https://www.ncbi.nlm.nih.gov/pubmed/22325062
BMC Genomics. 2012 Feb 10;13:66. doi: 10.1186/1471-2164-13-66.
Relative entropy differences in bacterial chromosomes, plasmids, phages and genomic islands.
Bohlin J1, van Passel MW, Snipen L, Kristoffersen AB, Ussery D, Hardy SP.

https://www.ncbi.nlm.nih.gov/pubmed/19233962
Mol Biol Evol. 2009 May;26(5):1163-9. 
Phylogenetic signals in DNA composition: limitations and prospects.
MrÃ¡zek J


## codon usage
- [Davis JJ, Olsen GJ. Mol Biol Evol. 2010 Apr "Modal codon usage: assessing the typical codon usage of a genome."](https://www.ncbi.nlm.nih.gov/pubmed/20018979)
- [Guo FB, Yuan JB. DNA Res. 2009 Apr "Codon usages of genes on chromosome, and surprisingly, genes in plasmid are primarily affected by strand-specific mutational biases in Lawsonia intracellularis."](https://www.ncbi.nlm.nih.gov/pubmed/19221094)
- [Peixoto L et al. Gene. 2003 Nov 27 "The strength of translational selection for codon usage varies in the three replicons of Sinorhizobium meliloti."](https://www.ncbi.nlm.nih.gov/pubmed/14597394)


## host range
å®¿ä¸»åŸŸ

https://twitter.com/search?q=plasmid%20host%20range

https://www.ncbi.nlm.nih.gov/pubmed/23980652
FEMS Microbiol Lett. 2013 Nov;348(2):87-96. doi: 10.1111/1574-6968.12241. Epub 2013 Sep 10.
Broad host range plasmids.
Jain A1, Srivastava P.

The present review article describes the features that confer broad host range to the plasmids, 

https://twitter.com/NatureJapan/status/957538744894189569
Nature Japan on Twitter: "Nature Ecology & Evolutionå‰µåˆŠ1å‘¨å¹´è¨˜å¿µ Collectionã€Œä»£å„Ÿæ€§å¤‰ç•°ã¯æŠ—ç”Ÿç‰©è³ªè€æ€§ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã«å¯¾ã™ã‚‹å…¨èˆ¬çš„å¯›å®¹æ€§ã‚’å‘ä¸Šã•ã›ã‚‹ã€https://t.co/NII6UeGiDUâ€¦ "
28 Jan 2018

http://onlinelibrary.wiley.com/doi/10.1111/1574-6968.12417/full
Insertion sequence ISPst4 activates pUC plasmid replication in Pseudomonas stutzeri - Coleman - 2014 - FEMS Microbiology Letters - Wiley Online Library

IS transposition directly leading to an expansion of the effective host range of a plasmid

----------


## review

review
https://www.ncbi.nlm.nih.gov/pubmed/28586714
Curr Opin Microbiol. 2017 Jun 3;38:106-113. doi: 10.1016/j.mib.2017.05.005. [Epub ahead of print]
Towards a taxonomy of conjugative plasmids.
Fernandez-Lopez R1, Redondo S1, Garcillan-Barcia MP1, de la Cruz F2.
Natural transformation, phage transduction, and bacterial conjugation are three main routes of HGT in bacteria.


### 2017

Review Article
February 10, 2017
https://www.omicsonline.org/open-access/role-of-plasmids-in-microbiology-2155-9546-1000466.php?aid=85578
(2017) Role of Plasmids in Microbiology. J Aquac Res Development 8: 466.

https://www.ncbi.nlm.nih.gov/pubmed/28232822
Front Microbiol. 2017 Feb 9;8:182. doi: 10.3389/fmicb.2017.00182. eCollection 2017.
Plasmid Classification in an Era of Whole-Genome Sequencing: Application in Studies of Antibiotic Resistance Epidemiology.
Orlek A1, Stoesser N2, Anjum MF3, Doumith M4, Ellington MJ5, Peto T1, Crook D1, Woodford N5, Walker AS1, Phan H1, Sheppard AE1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5299020/
Table 2
Summary of common in silico tools used for plasmid analysis.

- [Loftie-Eaton W, Rawlings DE. Plasmid. 2012 Jan "Diversity, biology and evolution of IncQ-family plasmids."](http://www.ncbi.nlm.nih.gov/pubmed/22037393)

- [èƒ½ç™»  å„ª 1,é«™ç€¬ è­˜ä¹‹ 1,é«™æ©‹è£•é‡Œé¦™ 1,æ¾æœ¬ è²´å—£ 2,å‰å· åšæ–‡ 2,åœŸé‡‘ æµå­ 3, ç´°å±±  å“² 3,è—¤ç”° ä¿¡ä¹‹ 3,å²¡ç”° æ†²å…¸ 1,å±±æ ¹ ä¹…å’Œ 1,é‡Žå°» ç§€æ˜­ 1* ç’°å¢ƒãƒã‚¤ã‚ªãƒ†ã‚¯ãƒŽãƒ­ã‚¸ãƒ¼å­¦ä¼š 12(2), 135-139, 2012-12 "ãƒ•ã‚šãƒ©ã‚¹ãƒŸãƒˆã‚™ä¿æŒã«ä¼´ã†è² è·ã‚’è»½æ¸›åŒ–ã™ã‚‹å®¿ä¸»æŸ“è‰²ä½“å› å­" Host Chromosomal Factor Reducing Plasmid Fitness Cost](https://ci.nii.ac.jp/naid/40019972025) | [PDF](http://www.jseb.jp/jeb/12-02/12-02-135.pdf)

https://ci.nii.ac.jp/naid/40019971986/
http://www.jseb.jp/jeb/11-0102/11-12-069.pdf
Journal of Environmental Biotechnology(ç’°å¢ƒãƒã‚™ã‚¤ã‚ªãƒ†ã‚¯ãƒŽãƒ­ã‚·ã‚™ãƒ¼å­¦ä¼šèªŒ)Vol. 11, No. 1 Â· 2, 69â€“75, 2011 ç·  èª¬(ä¸€èˆ¬)       ãƒ•ã‚šãƒ©ã‚¹ãƒŸãƒˆã‚™ã®æŽ¥åˆä¼é”ã«é–¢ä¸Žã™ã‚‹éºä¼å› å­ãƒ»ç’°å¢ƒå› å­Genetic Elements and Environmental Factors Involved in the Plasmid Conjugationæ¾äº• ä¸€æ³° 1,æ–°è°· æ”¿å·± 2,å±±æ ¹ ä¹…å’Œ 1,é‡Žå°» ç§€æ˜­ 1*

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3383451/
Mob Genet Elements. 2012 Jan 1; 2(1): 55â€“58.
IncA/C plasmids
An emerging threat to human and animal health?
Timothy J. Johnson* and Kevin S. Lang

https://www.ncbi.nlm.nih.gov/pubmed/23471189
Front Microbiol. 2013 Mar 7;4:44. doi: 10.3389/fmicb.2013.00044. eCollection 2013.
Broad-host-range IncP-1 plasmids and their resistance potential.
Popowska M1, Krawczyk-Balska A.

### plasmidome
ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ãƒ»ãƒ¡ã‚¿ã‚²ãƒŽãƒ è§£æž

https://www.ncbi.nlm.nih.gov/pubmed/30341451
Curr Microbiol. 2018 Oct 19. doi: 10.1007/s00284-018-1581-2. [Epub ahead of print]
Comparative Genomics of Lactobacillus brevis Reveals a Significant Plasmidome Overlap of Brewery and Insect Isolates.
Fraunhofer ME1, GeiÃŸler AJ1, Behr J1,2, Vogel RF3.

https://www.ncbi.nlm.nih.gov/pubmed/27861983
Ann N Y Acad Sci. 2017 Jan;1388(1):26-41. doi: 10.1111/nyas.13282. Epub 2016 Nov 10.
Genomic and metagenomic technologies to explore the antibiotic resistance mobilome.
MartÃ­nez JL1, Coque TM2,3,4, Lanza VF2,3,4, de la Cruz F5, Baquero F2,3,4.
https://nyaspubs.onlinelibrary.wiley.com/doi/abs/10.1111/nyas.13282
analysis of plasmid metagenomes is still in its infancy. 

- [Li AD et al. Front Microbiol. 2015 Sep 24. "Exploring antibiotic resistance genes and metal resistance genes in plasmid metagenomes from wastewater treatment plants."](https://www.ncbi.nlm.nih.gov/pubmed/26441947)
- [Welcome to the plasmidome. Nat Rev Microbiol. 2012 May](http://www.ncbi.nlm.nih.gov/pubmed/22580363)
- [Insights into the bovine rumen plasmidome. Proc Natl Acad Sci U S A. 2012 Apr](http://www.ncbi.nlm.nih.gov/pubmed/22431592)
- [Evolutionary, ecological and biotechnological perspectives on plasmids resident in the human gut mobile metagenome. Bioeng Bugs. 2012 Jan](http://www.ncbi.nlm.nih.gov/pubmed/22126801)
- [Jones BV, Marchesi JR. Nat Methods. 2007 Jan. "Transposon-aided capture (TRACA) of plasmids resident in the human gut mobile metagenome."](https://www.ncbi.nlm.nih.gov/pubmed/17128268)

## Plasmid Copy Number
- [Plasmid copy number - Wikipedia](https://en.wikipedia.org/wiki/Plasmid_copy_number)
- 14th January, 2015 [How to Manipulate Plasmid Copy Number - Bitesize Bio](http://bitesizebio.com/22824/how-to-manipulate-plasmid-copy-number/)
- Feb 6, 2014 [Plasmids 101: Origin of Replication](http://blog.addgene.org/plasmid-101-origin-of-replication) Copy Number+
- []()
- [Anindyajati et al. Sci Pharm. 2016 Feb 14;84(1):89-101. "Plasmid Copy Number Determination by Quantitative Polymerase Chain Reaction."](https://www.ncbi.nlm.nih.gov/pubmed/27110501) | [Tab. 2. Plasmid Copy Number Determination by qPCR](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4839616/table/T2/)
- [San Millan A et al. Antimicrob Agents Chemother. 2015;59(6):3335-41. "Small-plasmid-mediated antibiotic resistance is enhanced by increases in plasmid copy number and bacterial fitness."](https://www.ncbi.nlm.nih.gov/pubmed/25824216) | [FIG 5. Plasmid copy number and bacterial fitness enhance antibiotic resistance.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4432117/figure/F5/)

## resistance

https://holtlab.net/tag/antibiotic-resistance/
antibiotic resistance | Holt Lab

https://www.ncbi.nlm.nih.gov/protein/AMP50525
TrfA protein [uncultured bacterium]
https://www.ncbi.nlm.nih.gov/pubmed/27172044
Nature. 2016 May 12;533(7602):212-6. 
Interconnected microbiomes and resistomes in low-income human habitats.

https://www.ncbi.nlm.nih.gov/pubmed/25232178
Sci Transl Med. 2014 Sep 17;6(254):254ra126. doi: 10.1126/scitranslmed.3009845.
Single-molecule sequencing to track plasmid diversity of hospital-associated carbapenemase-producing Enterobacteriaceae.
Conlan S1, Thomas PJ2, Deming C1, Park M2, Lau AF3, Dekker JP3, Snitkin ES1, Clark TA4, Luong K4, Song Y4, Tsai YC4, Boitano M4, Dayal J2, Brooks SY2, Schmidt B2, Young AC2, Thomas JW2, Bouffard GG2, Blakesley RW2; NISC Comparative Sequencing Program, Mullikin JC2, Korlach J4, Henderson DK3, Frank KM5, Palmore TN5, Segre JA6.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4203314/#SD1
NIHMS631653-supplement-SuppFiles.doc

In addition, an estimated copy number for each plasmid relative to the chromosome was determined from average depths of PacBio and MiSeq reads (Table S1& S4). Interestingly, most plasmids were present in relatively low copy number (1-3).    

Supplemental Tables

Table S1: Assembly metrics. SMRT sequencing and HGAP de novo genome assembly metrics of 20 isolates.

Table S4: Plasmid copy number. Estimated Copy Number of Plasmids from Relative Depth of MiSeq Reads

### NDM-1
New Delhi metallo-Î²-lactamase-1 (NDM-1)

- [Conlan S et al. Genome Announc. 2016 Jul 14;4(4). "Complete Genome Sequence of a Klebsiella pneumoniae Strain Carrying blaNDM-1 on a Multidrug Resistance Plasmid."](https://www.ncbi.nlm.nih.gov/pubmed/27417839)
- [Khong WX et al. BMC Genomics. 2016 Jun 13;17:452. "Local transmission and global dissemination of New Delhi Metallo-Beta-Lactamase (NDM): a whole genome analysis."](https://www.ncbi.nlm.nih.gov/pubmed/27297071)
- [Kumarasamy KK et al. Lancet Infect Dis. 2010 Sep "Emergence of a new antibiotic resistance mechanism in India, Pakistan, and the UK: a molecular, biological, and epidemiological study."](http://www.ncbi.nlm.nih.gov/pubmed/20705517)

### mcr-1
Colistin resistance (mcr-1 gene)


https://pubmed.ncbi.nlm.nih.gov/29563494/
Nat Commun
. 2018 Mar 21;9(1):1179. doi: 10.1038/s41467-018-03205-z.
The global distribution and spread of the mobilized colistin resistance gene mcr-1
Ruobing Wang 1, Lucy van Dorp 2, Liam P Shaw 2, Phelim Bradley 3, Qi Wang 1, Xiaojuan Wang 1, Longyang Jin 1, Qing Zhang 4, Yuqing Liu 4, Adrien Rieux 5, Thamarai Dorai-Schneiders 6, Lucy Anne Weinert 7, Zamin Iqbal 3 8, Xavier Didelot 9, Hui Wang 10, Francois Balloux 11
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5862964/
https://www.nature.com/articles/s41467-018-03205-z





https://www.ncbi.nlm.nih.gov/pubmed/28507118
Antimicrob Agents Chemother. 2017 May 15. pii: AAC.00404-17.
Colistin resistance in carbapenem-resistant Klebsiella pneumoniae mediated by chromosomal integration of plasmid DNA.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5527652/
https://aac.asm.org/content/61/8/e00404-17.long


https://www.ncbi.nlm.nih.gov/pubmed/28487432
MBio. 2017 May 9;8(3). pii: e00625-17. 
Deciphering MCR-2 Colistin Resistance.

https://www.ncbi.nlm.nih.gov/pubmed/28336940
Sci Rep. 2017 Mar 24;7(1):424. 
Genetic Analysis of the IncX4 Plasmids: Implications for a Unique Pattern in the mcr-1 Acquisition.

https://www.ncbi.nlm.nih.gov/pubmed/27893854
PLoS Pathog. 2016 Nov 28;12(11):e1005957. 
Dissemination and Mechanism for the MCR-1 Colistin Resistance.

- [McGann P et al. Antimicrob Agents Chemother. 2016 Jun "Escherichia coli Harboring mcr-1 and blaCTX-M on a Novel IncF Plasmid: First Report of mcr-1 in the United States."](https://www.ncbi.nlm.nih.gov/pubmed/27230792)
- [Fernandes MR et al. Euro Surveill. 2016 Apr 28;21(17). "Silent dissemination of colistin-resistant Escherichia coli in South America could contribute to the global spread of the mcr-1 gene."](https://www.ncbi.nlm.nih.gov/pubmed/27168587) | [Eurosurveillance - View Article](http://www.eurosurveillance.org/ViewArticle.aspx?ArticleId=22458)
- [Suzuki S et al. Lancet Infect Dis. 2016 Mar "Investigation of a plasmid genome database for colistin-resistance gene mcr-1."](http://www.ncbi.nlm.nih.gov/pubmed/26774245)
- [Skov RL, Monnet DL. Euro Surveill. 2016;21(9). "Plasmid-mediated colistin resistance (mcr-1 gene): three months later, the story unfolds."](http://www.ncbi.nlm.nih.gov/pubmed/26967914) | [Eurosurveillance - View Article](http://www.eurosurveillance.org/ViewArticle.aspx?ArticleId=21403)
- [Liu YY et al. Lancet Infect Dis. 2016 Feb "Emergence of plasmid-mediated colistin resistance mechanism MCR-1 in animals and human beings in China: a microbiological and molecular biological study."](http://www.ncbi.nlm.nih.gov/pubmed/26603172)

### IncP Colistin

### Zhao_2017_IncP_mcr-1

https://www.ncbi.nlm.nih.gov/pubmed/27895009
Antimicrob Agents Chemother. 2017 Jan 24;61(2). pii: e02229-16. doi: 10.1128/AAC.02229-16. Print 2017 Feb.
IncP Plasmid Carrying Colistin Resistance Gene mcr-1 in Klebsiella pneumoniae from Hospital Sewage.
Zhao F1,2, Feng Y1,2, LÃ¼ X1,2, McNally A3, Zong Z4,2,5.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5278755/
, suggesting that mcr-1 was carried by a self-transmissible plasmid, which was assigned pMCR_1511. 

Phylogenetic analysis of the IncP-1 plasmid backbone revealed that pMCR_1511 and pHNFP671 belonged to a new IncP-1 clade (see Fig. S1 in the supplemental material).
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5278755/bin/supp_61_2_e02229-16__index.html

https://www.ncbi.nlm.nih.gov/nuccore/KX377410
Klebsiella pneumoniae strain WCHKP1511 plasmid pMCR_1511, complete sequence

https://www.ncbi.nlm.nih.gov/nuccore/KP324830
Escherichia coli isolate FP671 plasmid pHNFP671, complete sequence

## IncP1

https://www.ncbi.nlm.nih.gov/pubmed/28264851
Antimicrob Agents Chemother. 2017 Apr 24;61(5).
MCR-1.6, a New MCR Variant Carried by an IncP Plasmid in a Colistin-Resistant Salmonella enterica Serovar Typhimurium Isolate from a Healthy Individual.
Lu X1, Hu Y2, Luo M3, Zhou H1, Wang X1,4, Du Y1, Li Z1, Xu J4, Zhu B5, Xu X6, Kan B7.

https://www.ncbi.nlm.nih.gov/nuccore/KY352406
Salmonella enterica subsp. enterica serovar Typhimurium strain YL14P053 plasmid pMCR16_P053, complete sequence
LOCUS       KY352406               47824 bp    DNA     linear   BCT 14-FEB-2017


https://pubmed.ncbi.nlm.nih.gov/21468020/
Nat Commun
. 2011;2:268. doi: 10.1038/ncomms1267. Epub 2011 Apr 5.
The IncP-1 plasmid backbone adapts to different host bacterial species and evolves through homologous recombination
Peter Norberg 1, Maria BergstrÃ¶m, Vinay Jethava, Devdatt Dubhashi, Malte Hermansson

### Pachulec_2010

https://pubmed.ncbi.nlm.nih.gov/20376355/
PLoS One
. 2010 Apr 1;5(4):e9962. doi: 10.1371/journal.pone.0009962.
Conjugative plasmids of Neisseria gonorrhoeae
Emilia Pachulec 1, Chris van der Does
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848598/
Figure 3
Phylogenetic analysis of genes of different IncP1 plasmids.



### IncQ
https://www.ncbi.nlm.nih.gov/pubmed/11729261
Microbiol Mol Biol Rev. 2001 Dec;65(4):481-96, table of contents.
Comparative biology of IncQ and IncQ-like plasmids.
Rawlings DE1, Tietze E.


### Top EM

https://www.ncbi.nlm.nih.gov/pubmed/16151119
Appl Environ Microbiol. 2005 Sep;71(9):5309-17.
Plasmid donor affects host range of promiscuous IncP-1beta plasmid pB10 in an activated-sludge microbial community.
De Gelder L1, Vandecasteele FP, Brown CJ, Forney LJ, Top EM.


https://www.ncbi.nlm.nih.gov/pubmed/29046540
Nat Ecol Evol. 2017 Sep;1(9):1354-1363. doi: 10.1038/s41559-017-0243-2. Epub 2017 Aug 7.
Compensatory mutations improve general permissiveness to antibiotic resistance plasmids.
Loftie-Eaton W1,2, Bashford K1, Quinn H3, Dong K1, Millstein J1,2, Hunter S2, Thomason MK4, Merrikh H4, Ponciano JM5, Top EM6,7.

https://www.ncbi.nlm.nih.gov/pubmed/26668183
Mol Biol Evol. 2016 Apr;33(4):885-97. doi: 10.1093/molbev/msv339. Epub 2015 Dec 14.
Evolutionary Paths That Expand Plasmid Host-Range: Implications for Spread of Antibiotic Resistance.
Loftie-Eaton W1, Yano H1, Burleigh S2, Simmons RS2, Hughes JM1, Rogers LM1, Hunter SS3, Settles ML1, Forney LJ1, Ponciano JM4, Top EM5.


https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1574-6976.2007.00074.x
FEMS Microbiol Rev. 2007 Jul;31(4):449-77. Epub 2007 Jun 6.
Genomics of IncP-1 antibiotic resistance plasmids isolated from wastewater treatment plants provides evidence for a widely accessible drug resistance gene pool.
SchlÃ¼ter A1, Szczepanowski R, PÃ¼hler A, Top EM.
https://academic.oup.com/femsre/article/31/4/449/2399130
Figure 2
Modular structure of completely sequenced IncP-1 plasmids.


### HGT
horizontal gene transfer (HGT) 

https://www.ncbi.nlm.nih.gov/pubmed/22564249
Trends Microbiol. 2012 Jun;20(6):262-7. doi: 10.1016/j.tim.2012.04.003. Epub 2012 May 5.
Plasmid-mediated horizontal gene transfer is a coevolutionary process.
Harrison E1, Brockhurst MA.

https://www.ncbi.nlm.nih.gov/pubmed/25370194
Life (Basel). 2014 May 16;4(2):217-24. doi: 10.3390/life4020217.
Horizontal Gene Transfer among Bacteria and Its Role in Biological Evolution.
Arber W1.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4187160/
- Fâ€™-like conjugative plasmids have soon become known to act sometimes in the horizontal transfer of antibiotic resistance determinants.
- Viral DNA molecules and F conjugative plasmids were envisaged to serve as autonomously replicating gene vectors, into which a relatively short DNA fragment could be spliced.

### gene sharing
https://www.ncbi.nlm.nih.gov/pubmed/22130968
Mol Biol Evol. 2012 Apr;29(4):1225-40. doi: 10.1093/molbev/msr292. Epub 2011 Nov 29.
Large-scale analysis of plasmid relationships through gene-sharing networks.
Tamminen M1, Virta M, Fani R, Fondi M.
https://www.ncbi.nlm.nih.gov/pubmed/20636373
Environ Microbiol. 2010 Dec;12(12):3228-42. doi: 10.1111/j.1462-2920.2010.02295.x.
The horizontal flow of the plasmid resistome: clues from inter-generic similarity networks.
Fondi M1, Fani R.

https://www.ncbi.nlm.nih.gov/pubmed/20181243
BMC Evol Biol. 2010 Feb 24;10:59. doi: 10.1186/1471-2148-10-59.
Exploring the evolutionary dynamics of plasmids: the Acinetobacter pan-plasmidome.
Fondi M1, Bacci G, Brilli M, Papaleo MC, Mengoni A, Vaneechoutte M, Dijkshoorn L, Fani R.

https://www.ncbi.nlm.nih.gov/pubmed/20007769
Proc Natl Acad Sci U S A. 2010 Jan 5;107(1):127-32. doi: 10.1073/pnas.0908978107. Epub 2009 Dec 10.
Network analyses structure genetic diversity in independent genetic worlds.
Halary S1, Leigh JW, Cheaib B, Lopez P, Bapteste E.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2806761/
- Mathematical studies of the centralities of these worldsâ€™ subnetworks demonstrate that plasmids, not viruses, were key vectors of genetic exchange between bacterial chromosomes, both recently and in the past.
- Most interestingly, these bridges were almost always plasmids in our networks (Fig. 2), strongly suggesting that these vehicles, and not viruses, are key vectors in the spreading of DNA in nature. Interestingly, many of these plasmids with a remarkable betweennessâ€”although diverse in size, G+C content, and percentage of coding genesâ€”were already known for their phylogenetic mosaicism, the presence of mobile elements such as transposons and integrons in their genomes (resulting from recombination events), and their resistance to drugs and/or heavy metals, which likely contributed to their diffusion in various hosts and environments (Table S5). For instance, pB10 [degree (d) = 63, betweenness (b) = 2,358, in the 100% identity network] is a promiscuous IncP-1 plasmid, isolated from a waste water treatment plant. Its recombined mosaic backbone structure encompasses 5 distinct mobile genetic elements. pB10 is able to self-transfer among diverse bacterial species and confers resistance to multiple antimicrobial agents and to inorganic mercury ions, and would play a major role in rapid adaptation of bacterial communities to changing environments (21). 

----------

## News
- [News about plasmid resistance on Twitter](https://twitter.com/search?vertical=news&q=plasmid%20resistance)

- December 16, 2015 [A New Superbug Attacks Our Last Line Of Antibiotic Defense | On Point](http://www.wbur.org/onpoint/2015/12/16/superbug-antibiotics-global-risk) 
- March 27, 2015 [FACT SHEET: Obama Administration Releases National Action Plan to Combat Antibiotic-Resistant Bacteria | whitehouse.gov](https://www.whitehouse.gov/the-press-office/2015/03/27/fact-sheet-obama-administration-releases-national-action-plan-combat-ant)

- Fri May 27, 2016 [U.S. sees first case of bacteria resistant to last-resort antibiotic | Reuters](http://www.reuters.com/article/us-health-superbug-idUSKCN0YH2KT)

![http://www.reuters.com/article/us-health-superbug-idUSKCN0YH2KT](http://s2.reutersmedia.net/resources/r/?m=02&d=20160527&t=2&i=1139027343&w=&fh=&fw=&ll=644&pl=429&sq=&r=LYNXNPEC4P1O4)

- June 12, 2016 [Bacteria gaining upper hand in war against humans- Nikkei Asian Review](http://asia.nikkei.com/Tech-Science/Science/Bacteria-gaining-upper-hand-in-war-against-humans)

- April 13, 2011 [Antibiotic resistance spreads rapidly between bacteria -- ScienceDaily](https://www.sciencedaily.com/releases/2011/04/110411163918.htm)

![https://www.sciencedaily.com/releases/2011/04/110411163918.htm](https://images.sciencedaily.com/2011/04/110411163918_1_540x360.jpg)

- May 2, 2011 [BEACON Researchers at Work: Plasmid evolution is the key to fighting antibiotic resistance | BEACON](http://beacon-center.org/blog/2011/05/02/beacon-researchers-at-work-plasmid-evolution-is-the-key/)

![http://elizabethely.com/2010/07/11/antibiotic-chloramphenicol-in-chinese-imported-honey/](http://elizabethely.com/wp-content/uploads/2010/07/nz149-superbug-cartoon.jpg)

----------



http://people.ibest.uidaho.edu/~etop/vgp/tnsBasics.html


https://www.ncbi.nlm.nih.gov/pubmed/10430917
Proc Natl Acad Sci U S A. 1999 Aug 3;96(16):9184-9.
Genome signature comparisons among prokaryote, plasmid, and mitochondrial DNA.
Campbell A1, MrÃ¡zek J, Karlin S.

![](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC17754/bin/pq1692140001.jpg)


----------
### HGT


å¡©åŸºçµ„æˆã€€Nucleotide composition profilingã€€
https://www.ncbi.nlm.nih.gov/pubmed/15653627
Nucleic Acids Res. 2005 Jan 13;33(1):e6.
Detection and characterization of horizontal transfers in prokaryotes using genomic signature.
Dufraigne C1, Fertil B, Lespinats S, Giron A, Deschavanne P.
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC546175/
RESULTS
Many plasmids and viruses are also found in agreement with the known molecular mechanisms of horizontal transfer (Table 1 and Supplementary Material 1).

DISCUSSION
Potential donors
Numerous viruses and plasmids qualify as potential donors (Tables 1 and â€‹and5,5, Supplementary Materials 1, 2, 4 and 6). It is not really surprising since they are known as HT vectors. They are often totally or partially inserted together with transferred genes in the host genome (14).

https://www.ncbi.nlm.nih.gov/pubmed/15208628
Nat Genet. 2004 Jul;36(7):760-6. Epub 2004 Jun 20.
Biased biological functions of horizontally transferred genes in prokaryotic genomes.
Nakamura Y1, Itoh T, Matsuda H, Gojobori T.
http://www.nature.com/articles/ng1381
For most species, the HT model predicted the plasmid or phage genes more effectively than the non-HT model. These observations imply that in many cases, the horizontally transferred genes were initially inserted into plasmids or phages and then introgressed into the recipient species. For Borrelia burgdorferi plasmids, however, all indices were higher with the non-HT model than the HT model, implying that the genes have been settled in B. burgdorferi for a long time and that their nucleotide compositions became similar to those of the host chromosomes by amelioration14.

https://www.ncbi.nlm.nih.gov/pubmed/16520338
Mol Biol Evol. 2006 Jun;23(6):1129-35. Epub 2006 Mar 6.
Horizontal transfer of a virulence operon to the ancestor of Mycobacterium tuberculosis.
Rosas-Magallanes V1, Deschavanne P, Quintana-Murci L, Brosch R, Gicquel B, Neyrolles O.
https://academic.oup.com/mbe/article/23/6/1129/1055299
Genomic signature profiling further suggested that the transfer was plasmid mediated and that the operon originated from a Î³-proteobacterium donor species.

Interestingly, âˆ¼30% of the most closely related sequences are plasmid borne, suggesting a possible plasmid-mediated transfer. 

----------

### Plasmid ecology

https://ci.nii.ac.jp/naid/40006735954
http://www.jseb.jp/jeb/04-02/04-02-131.PDF
è‡ªç„¶ç”Ÿæ…‹ç³»ã«ãŠã‘ã‚‹ç´°èƒžå¤–DNAã®å‹•æ…‹ã¨éºä¼å­ä¼æ’­
Dynamics and Contributions to Horizontal Gene Transfer of Extracellular DNA in Natural Ecosystems
Journal of environmental biotechnology 4(2), 131-137, 2005-02
ä¸¸å±± å²äºº,è°· ä½³æ´¥æ²»,é‚£é ˆ æ­£å¤«*

ã¾ãŸæµ·æ°´ã‹ã‚‰å˜é›¢ã—ãŸç´° èŒã®10%ã‹ã‚™,ãƒ•ã‚šãƒ©ã‚¹ãƒŸãƒˆã‚™ã‚’å–è¾¼ã‚€èƒ½åŠ›ã‹ã‚™ã‚ã‚Š,14%ã‹ã‚™æŸ“ è‰²ä½“ã‚’å–è¾¼ã‚€èƒ½åŠ›ã‚’ä¿æŒã—ã¦ã„ãŸ20)ã€‚

20) Frischer, M.E., G.J. Stewart, and J.H. Paul. 1994. Plasmid transfer to indigenous marine bacterial populations by natural transformation. FEMS Microbiol. Ecol. 15: 127â€“136.

https://www.jstage.jst.go.jp/article/jsb1944/46/2/46_2_529/_pdf
Ecological behavior of plasmids and resistance to lead of Enterobacter agglomerans isolated from soil
åœŸå£Œä¸­ã‹ã‚‰åˆ†é›¢ã•ã‚Œã‚‹Enterobacter agglomeransã®ãƒ•ã‚šãƒ©ã‚¹ãƒŸãƒˆã‚™ã®ç”Ÿæ…‹å­¦çš„æŒ™å‹•ã¨é‰›è€æ€§ã«ã¤ã„ã¦
æ—¥æœ¬ç´°èŒå­¦é›‘èªŒ 46 (2), 1991
åº¦ä¼š é›…ä¹…1)æ­¦ ç”° æµ©äºŒ1)ä¸Š èŠ ç§€åš2)
https://www.jstage.jst.go.jp/article/jssm/36/0/36_KJ00008103115/_pdf/-char/ja
ãƒ•ã‚šãƒ©ã‚¹ãƒŸãƒˆã‚™ã®ç”Ÿæ…‹å­¦
è¾²æ¥­ç’°å¢ƒæŠ€è¡“ç ”ç©¶æ‰€
ä½è—¤ å®ˆ

----------


http://www.seibutsushi.net/blog/2009/08/860.html
2009-08-19
ãƒ—ãƒ©ã‚¹ãƒŸãƒ‰ã¯ã©ã“ã‹ã‚‰ããŸã®ã‹ï¼Ÿï¼’-ä¸å®Œå…¨ãªç”Ÿå‘½åŒå£«ã®éºä¼å­äº¤æ›


----------

## Wolbachia

https://twitter.com/search?q=Wolbachia%20plasmid

https://naturemicrobiologycommunity.nature.com/users/208603-julie-reveillaud/posts/44531-an-interdisciplinary-wolbachia-story-through-connections-across-the-atlantic-metagenomics-long-read-sequencing-and-other-things
An interdisciplinary Wolbachia story through connections across the Atlantic: Metagenomics, long-read sequencing, and other things | Nature Research Microbiology Community

https://www.nature.com/articles/s41467-019-08973-w
The Wolbachia mobilome in Culex pipiens includes a putative plasmid | Nature Communications

https://twitter.com/vectorgen/status/1103032982191906817
Jason Rasgon on Twitter: "That's very interesting. Wouldn't be the first time I heard someone talk about a wolbachia plasmid; never panned out before. Sequencing methods are better now. Were they able to isolate it?â€¦ https://t.co/tTJcKK9lBd"
3:42 PM - 5 Mar 2019

https://twitter.com/merenbey/status/1102959988098633729
A. Murat Eren (Meren) on Twitter: "A Wolbachia plasmid?! A study by @JulieReve & @srbordenstein et al: https://t.co/J61RISirT6 This study also includes a reproducible bioinformatics workflow, https://t.co/RrZePXmc9p and all the data you can ask for ðŸ˜‡ https://t.co/E54vVXyjGR #openscience #openaccessâ€¦ https://t.co/znKIoiwaRw"
10:52 AM - 5 Mar 2019 from Chicago, IL

https://twitter.com/JulieReve/status/1102893058784481281
Julie Reveillaud on Twitter: "Very happy to announce the discovery of a plasmid in the mobilome of Wolbachia in Culex pipiens mosquitoes with fantastic team @merenbey @srbordenstein @Symbionticism @alon_shaiber @c4vansite @ozcanesen et al! https://t.co/Qrm9uGu788 just out in @NatureComms"
6:26 AM - 5 Mar 2019



https://twitter.com/Symbionticism/status/211387158425116672
Seth Bordenstein on Twitter: "Rickettsial plasmid also exchanged genes with Wolbachia phage WO http://t.co/ibAig0aE #Wolbachia2012"
5:20 AM - 9 Jun 2012

----------





